{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "4b57d9e8",
   "metadata": {
    "colab_type": "text",
    "id": "view-in-github"
   },
   "source": [
    "<a href=\"https://colab.research.google.com/github/jlee2843/Peer-Review-22-23/blob/Albert-2022/API_Biorvix.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "28ba5ea5-17a2-4e94-bd7a-286f4be7dc1c",
   "metadata": {
    "id": "7Q304L8t4_jg"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "version: 3.9.7 (default, Sep 16 2021, 13:09:58) \n",
      "[GCC 7.5.0]\n"
     ]
    }
   ],
   "source": [
    "import sys\n",
    "print(\"version:\", sys.version)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "1e7b372d-5570-410f-9247-e3447dff2d5e",
   "metadata": {
    "id": "Q3I4WKRkvcQr"
   },
   "outputs": [],
   "source": [
    "#installing of models\n",
    "!pip install python-doi\n",
    "!pip install pyarrow\n",
    "!pip install multipledispatch\n",
    "!pip install pixiedust"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1a07e81b-7c10-457b-9618-c515c8440fa2",
   "metadata": {
    "id": "Q3I4WKRkvcQr"
   },
   "outputs": [],
   "source": [
    "from google.colab import drive\n",
    "\n",
    "drive.mount('/content/gdrive')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "63d0a567",
   "metadata": {
    "id": "63d0a567"
   },
   "source": [
    "**Biorxiv API** information is found [here](https://api.biorxiv.org/details/medrxiv/help).<br>\n",
    "**Final edit:** December 02, 2022.<br>\n",
    "**NB:** Using Jenny's notebook  as a template\n",
    "<br>\n",
    "tqdm is used as visualization for the processing fo data. (Its manual can be found https://github.com/tqdm/tqdm#manual)\n",
    "<br>\n",
    "<br>\n",
    "**NB: Some issues when using BioRxiv API**\n",
    "1. there are empty entry e.g. no subject area declaired\n",
    "    - sometimes these entries can be fixed and sometimes it cannot. For example, one can manual enter the corresponding other but there is no way to find the intended subject area when it is not given.\n",
    "2. sometimes the publication date is before the preprint date\n",
    "3. sometimes the date return is not in the format of 'yyyy-mm-dd' when querying for published article detail\n",
    "4. sometimes the information returned have miscellanous information for example '- Genetic Institute...' in the author field which requires manual reentry of information\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "P58Efn060Fbe",
   "metadata": {
    "id": "P58Efn060Fbe"
   },
   "source": [
    "# Common functions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "Ep_xUfgYeUur",
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "Ep_xUfgYeUur",
    "outputId": "0ae1566f-5a5b-4a72-d617-6e3268e9783d"
   },
   "outputs": [],
   "source": [
    "from pandas.core.internals.ops import Iterator\n",
    "#from tqdm.notebook import tqdm, trange\n",
    "from typing import Union, List\n",
    "from concurrent.futures import ThreadPoolExecutor\n",
    "from datetime import datetime\n",
    "from threading import current_thread\n",
    "from pathlib import Path\n",
    "\n",
    "import math\n",
    "import pandas as pd\n",
    "import time\n",
    "import numpy as np\n",
    "import requests\n",
    "import json\n",
    "import urllib, urllib.request\n",
    "import pathlib\n",
    "import tqdm.contrib.concurrent as tq\n",
    "import doi\n",
    "\n",
    "# common helper function\n",
    "def get_total (url) -> int:\n",
    "    json_info = get_json_data(url)\n",
    "    return json_info[\"messages\"][0][\"total\"]\n",
    "\n",
    "def get_json_data (url):\n",
    "    try:\n",
    "        request_API = requests.get(url)\n",
    "        #print(f\"{url} request_API: {request_API}\")\n",
    "        return json.loads(request_API.text)\n",
    "    except Exception as e:\n",
    "        time.sleep(120)\n",
    "        request_API = requests.get(url)\n",
    "        return json.loads(request_API.text)\n",
    "\n",
    "def get_data(url:str, loop_range:range, disable:bool):\n",
    "    #print(f\"values: {list(loop_range)}\")\n",
    "    results = []\n",
    "    #print(args)\n",
    "    results = tq.thread_map(get_json_data, (f'{url}/{cursor}' for cursor in loop_range), \n",
    "                            desc=f'get_json_data {current_thread().name}', total=len(loop_range),\n",
    "                            disable=disable, leave=False)\n",
    "    \n",
    "    return results\n",
    "\n",
    "#def get_article_detail(url, articles):\n",
    "#    result = get_data(url,articles)\n",
    "#\n",
    "#    return result\n",
    "\n",
    "def process_data(json_info, keys:List[str], cursor:int, disable:bool) -> List:\n",
    "    journal_list = [[entry + cursor] + [getValue(journal, key) for key in keys] for entry, journal in enumerate(json_info[\"collection\"])]\n",
    "    if disable is False:\n",
    "        time.sleep(0.001 * len(journal_list))\n",
    "    return journal_list\n",
    "\n",
    "def query_to_df(results, keys:List[str], col_names:List[str], loop_list:range, disable:bool) -> pd.DataFrame:\n",
    "    args = zip(results, \n",
    "               [keys for _ in loop_list],\n",
    "               loop_list,\n",
    "               [disable for _ in loop_list])\n",
    "    args = list(args)\n",
    "    total = len(args)\n",
    "    result_list = []\n",
    "    result_list = tq.thread_map(lambda p: process_data(*p), args, desc=f'processing data {current_thread().name}', total=total, leave=False, disable=disable)\n",
    "    \n",
    "    data = np.array(flatten(result_list))\n",
    "    #print(data)\n",
    "    #print(col_names)0\n",
    "    #print(f'loop_list: {loop_list}')\n",
    "    return create_df(data, col_names)\n",
    "\n",
    "# prepublish helper function\n",
    "def create_prepublish_df(df:pd.DataFrame) -> pd.DataFrame:\n",
    "    try:\n",
    "        df['Num_of_Authors'] = df.Authors.apply(lambda x: len(x.split(';')))\n",
    "        df.DOI = df.DOI.astype('str')\n",
    "        df.Title = df.Title.astype('str').map(lambda x: x.strip())\n",
    "        df.Authors = df.Authors.astype('str').map(lambda x: x.strip())\n",
    "        df.Corresponding_Authors = df.Corresponding_Authors.astype('str').map(lambda x: x.strip())\n",
    "        df.Institution = df.Institution.map(lambda x: x.strip().upper()).astype('category')\n",
    "        df.Date = df.Date.map(lambda x: convert_date(x)).astype('datetime64')\n",
    "        df.Type = df.Type.map(lambda x: x.strip().lower()).astype('category')\n",
    "        df.Category = df.Category.map(lambda x: x.strip().title()).astype('category')\n",
    "#        df.Xml = df.Xml.astype('str')\n",
    "        df.Published = df.Published.astype('str')\n",
    "    except Exception as e:\n",
    "        print(f'Error in data format:{e.args}\\n')\n",
    "        print(e.with_traceback)\n",
    "\n",
    "    return df\n",
    "\n",
    "# helper functions for published articles\n",
    "# pub = [\"preprint_doi\", \"published_doi\", \"preprint_title\", \"preprint_authors\", \"preprint_author_corresponding\", \"preprint_author_corresponding_institution\", \"preprint_category\", \"published_journal\", \"preprint_date\", \"published_date\"]\n",
    "def create_published_df(df:pd.DataFrame) ->pd.DataFrame:\n",
    "    try:\n",
    "        df['Num_of_Authors'] = df.Authors.apply(lambda x: len(x.split(';')))\n",
    "        df.DOI = df.DOI.astype('str')\n",
    "        df.pub_DOI = df.pub_DOI.astype('str')\n",
    "        df.Title = df.Title.astype('str').map(lambda x: x.strip())\n",
    "        df.Authors = df.Authors.astype('str').map(lambda x: x.replace('\\'','').strip())\n",
    "        df.Corresponding_Authors = df.Corresponding_Authors.astype('str').str.strip()\n",
    "        df.Institution = df.Institution.map(lambda x: x.strip().upper()).astype('category')\n",
    "        df.Category = df.Category.map(lambda x: x.strip().title()).astype('category')\n",
    "        df.Journal = df.Journal.astype('str').map(lambda x: x.strip().title()).astype('category')\n",
    "        #tqdm.pandas(desc=\"Preprint_Date conversion!\")\n",
    "        df.Preprint_Date = df.Preprint_Date.map(lambda x: convert_date(x)).astype('datetime64')\n",
    "        #tqdm.pandas(desc=\"Published_Date conversion!\")\n",
    "        df.Published_Date = df.Published_Date.map(lambda x: convert_date(x)).astype('datetime64')\n",
    "    except Exception as e:\n",
    "        print(f'Error in data format:{e.args}\\n')\n",
    "        print(e.with_traceback)\n",
    "    #    print(f'Preprint_Date: {df.Preprint_Date} Published_Date: {df.Published_Date}')\n",
    "#\n",
    "    return df\n",
    "\n",
    "def convert_date(value:str) -> datetime:\n",
    "    try:\n",
    "        return datetime.strptime(value.strip().split(':')[0], '%Y-%m-%d')\n",
    "    except Exception as e:\n",
    "        print(e)\n",
    "        return pd.NaT\n",
    "\n",
    "# Now you can use `progress_apply` instead of `apply`\n",
    "# and `progress_map` instead of `map`\n",
    "        #df.Published_Date = datetime.strptime(df.Published_Date.str.strip().str.split(':')[0], '%Y-%m-%d')\n",
    "\n",
    "def getValue(journal, key):\n",
    "    result = np.NaN\n",
    "    try:\n",
    "        result = journal[key]\n",
    "    except Exception as e:\n",
    "        print (f'key: {key} journal: {journal}\\n{e}')\n",
    "        raise e\n",
    "\n",
    "    finally:\n",
    "        return result\n",
    "  \n",
    "def get_big_data(path:str, url:str, cursor:int, json_keys:List[str], col_names:List[str], step:int, disable:bool):\n",
    "    result_list = [get_json_data(f'{url}/{cursor}')]\n",
    "    df = query_to_df(result_list, json_keys, col_names, range(cursor, cursor + step, step), disable)\n",
    "    df.to_parquet(pathlib.Path(f'{path}/{datetime.utcnow().timestamp()}.parquet'))\n",
    "    #time.sleep(0.001)  # to visualize the progress\n",
    "\n",
    "def multithread_processor(path:str, url:str, json_keys:List[str], col_names:List[str], step:int, loop_range:range, disable:bool):\n",
    "    #print(f\"values: {list(loop_range)}\")\n",
    "    results = []\n",
    "    args = [(path, url, cursor, json_keys, col_names, step, disable) for cursor in loop_range]\n",
    "    #print(f'args: {len(args)}\\n{args}')\n",
    "    tq.thread_map(lambda p: get_big_data(*p), args, desc='get_big_data', total=len(args))\n",
    "\n",
    "def process_doi_data(path:str, url:str, doi:pd.Series, meta: List, col_names: List, item:int, disable:bool = False):\n",
    "    results = get_data(url, doi[item:item+step], disable)\n",
    "    tmp = list(results)\n",
    "    df = query_to_df([r for r in tmp], meta, col_names, \n",
    "                     range(item, item + (len(tmp) * step), step), disable)\n",
    "    df.to_parquet(pathlib.Path(f'{path}/{datetime.utcnow().timestamp()}.parquet'))\n",
    "\n",
    "def checkDOI(x:str):\n",
    "  if doi.validate_doi(x.strip()) is None:\n",
    "    raise Exception(f'invalid doi: {x.strip()}')\n",
    "  else:\n",
    "    return x.strip()\n",
    "\n",
    "def getSlurmID():\n",
    "    import subprocess\n",
    "    id = None\n",
    "    try:\n",
    "        id = subprocess.Popen(\"squeue -u $USER | tail -1 | awk {'print $1'}\", shell=True, stdout=subprocess.PIPE).stdout\n",
    "    except Exception:\n",
    "        id = subprocess.Popen(\"ssh -f arc squeue -u $USER | tail -1 | awk {'print $1'}\", shell=True, stdout=subprocess.PIPE).stdout\n",
    "    return str(id.read().rstrip(), 'utf-8')\n",
    "    \n",
    "def getSlurmDir() -> Path:\n",
    "    return Path('/scratch', getSlurmID())\n",
    "\n",
    "# returns (SparkContext, SqlContext) objects.\n",
    "def getSparkSqlContext():\n",
    "    import os\n",
    "    import atexit\n",
    "    import sys\n",
    "    import re\n",
    "    import pyspark\n",
    "    from pyspark.conf import SparkConf\n",
    "    from pyspark.context import SparkContext\n",
    "    from pyspark.sql import SQLContext\n",
    "\n",
    "import pixiedust\n",
    "pixiedust.enableJobMonitor()\n",
    "conflines=[tuple(a.rstrip().split(\" \")) for a in open(os.environ['SPARK_CONFIG_FILE']).readlines()]\n",
    "    conf=SparkConf()\n",
    "    conf.setAll(conflines)\n",
    "    conf.setMaster(\"spark://%s:%s\"% (os.environ['SPARK_MASTER_HOST'],os.environ['SPARK_MASTER_PORT']))\n",
    "    sc=pyspark.SparkContext(conf=conf)\n",
    "\n",
    "    sqlCtx=SQLContext(sc)\n",
    "    \n",
    "    return sc, sqlCtx\n",
    "\n",
    "freq_count = lambda x,y: x[y].value_counts()\n",
    "\n",
    "flatten = lambda y: sorted([sublist for inner in y for sublist in inner],\n",
    "                           key=lambda x:x[0])\n",
    "\n",
    "create_df = lambda x, y: pd.DataFrame(data=x[:, 1:], index=x[:, 0], columns=y)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "d9a35acf-89be-4b25-bdb0-b654100c8718",
   "metadata": {
    "id": "3E24xcwG1kXQ",
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Slurm id: 17482823\n",
      "Does /scratch/17482823 exist: True\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: An illegal reflective access operation has occurred\n",
      "WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/global/software/spark/spark-3.2.0-bin-hadoop3.2/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)\n",
      "WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform\n",
      "WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations\n",
      "WARNING: All illegal access operations will be denied in a future release\n",
      "Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties\n",
      "Setting default log level to \"WARN\".\n",
      "To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).\n",
      "23/01/18 19:35:17 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable\n",
      "/global/software/spark/spark-3.2.0-bin-hadoop3.2/python/pyspark/sql/context.py:77: FutureWarning: Deprecated in 3.0.0. Use SparkSession.builder.getOrCreate() instead.\n",
      "  warnings.warn(\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(<SparkContext master=spark://mc50:8460 appName=pyspark-shell>,\n",
       " <pyspark.sql.context.SQLContext at 0x7f90653c8220>)"
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(f'Slurm id: {getSlurmID()}')\n",
    "print(f'Does {getSlurmDir()} exist: {getSlurmDir().exists()}')\n",
    "getSparkSqlContext()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "614f69be-a804-4624-92e8-44281d94aefc",
   "metadata": {
    "id": "3E24xcwG1kXQ",
    "tags": []
   },
   "source": [
    "## Case \\#1: Pre-publish Data query for given timeframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "1WDh9gs9XfMp",
   "metadata": {
    "id": "1WDh9gs9XfMp"
   },
   "outputs": [],
   "source": [
    "# \"global\" variables\n",
    "base_url: str = 'https://api.biorxiv.org'\n",
    "query_type: str = 'details'\n",
    "server: str = 'biorxiv'\n",
    "start_interval: str = '2011-01-01'\n",
    "end_interval: str = '2020-12-31'\n",
    "step = 100\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "TLRswUpf1zlx",
   "metadata": {
    "id": "TLRswUpf1zlx",
    "tags": []
   },
   "source": [
    "### BioRxiv Server"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "cSYP2rLX05cQ",
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 121,
     "referenced_widgets": [
      "4edf0b643db94dfeae6db7b8a0519cdb",
      "7c45fe41b4dd4db7a0a1c964f8c68e7c",
      "af6fac0308a34252af8aa82673b98406",
      "5993c8519cc3465eae2f87b2318414c4",
      "86847fee2c014f76a008a321f5209a4a",
      "01b6eddd2404428bb81a4547d60444b3",
      "aa058be5dc914ec28a81bff85de7168e",
      "543c5ba3a78946f58b91fe5bd0e7f1d1",
      "49032de80ac44169b9a8337bed34ead2",
      "1bbfab880f2149e685d407db49da94a1",
      "e4a56b5756c94cc68e6b4f9ba88f66e8"
     ]
    },
    "id": "cSYP2rLX05cQ",
    "outputId": "24c52150-3538-4da7-cd60-1bcda53db0a3"
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "774eaadbcfa04da9b0ac031f8902e0d3",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "get_big_data:   0%|          | 0/1481 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# \"local\" variables\n",
    "url: str = f'{base_url}/{query_type}/{server}/{start_interval}/{end_interval}'\n",
    "path: str = f'data/prepub-{query_type}-{server}-{start_interval}!{end_interval}-{datetime.now()}'\n",
    "step = 100\n",
    "journal_df = None\n",
    "\n",
    "#finding the number of records for a given time period\n",
    "#tmp = f\"{base_url}/{start_interval}/{end_interval}\"\n",
    "#tmp = 'https://api.biorxiv.org/details/biorxiv/2020-08-21/2020-08-28'\n",
    "\n",
    "#create directory\n",
    "pathlib.Path(path).mkdir(parents=True, exist_ok=True)\n",
    "\n",
    "multithread_processor(path, url, \n",
    "                      [\"doi\", \"title\", \"authors\", \"author_corresponding\", \"author_corresponding_institution\", \"date\", \"version\", \"type\", \"category\", \"jatsxml\", \"published\"],\n",
    "                      [\"DOI\", \"Title\", \"Authors\", \"Corresponding_Authors\", \"Institution\", \"Date\", \"Version\", \"Type\", \"Category\", \"Xml\", \"Published\"],\n",
    "                      step, range(0, get_total(url), step), True)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "je_tNnPyKoNG",
   "metadata": {
    "id": "je_tNnPyKoNG"
   },
   "outputs": [],
   "source": [
    "journal_df = pd.read_parquet(pathlib.Path(path))\n",
    "journal_df = create_prepublish_df(journal_df)\n",
    "\n",
    "#display(journal_df)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1M7lIyNEEx42",
   "metadata": {
    "id": "1M7lIyNEEx42"
   },
   "source": [
    "#### Cleaning and fixing missing data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "V2Ix_MjB2W-e",
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 1000
    },
    "id": "V2Ix_MjB2W-e",
    "outputId": "f1e02c53-fef5-4a3b-8245-d6bb10b55a3b"
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding_Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Date</th>\n",
       "      <th>Version</th>\n",
       "      <th>Type</th>\n",
       "      <th>Category</th>\n",
       "      <th>Xml</th>\n",
       "      <th>Published</th>\n",
       "      <th>Num_of_Authors</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>12673</th>\n",
       "      <td>10.1101/104778</td>\n",
       "      <td>Engaging narratives evoke similar neural activ...</td>\n",
       "      <td>Cohen, S. S.; Henin, S.; Parra, L. C.</td>\n",
       "      <td></td>\n",
       "      <td>THE CITY COLLEGE OF THE CITY UNIVERSITY OF NEW...</td>\n",
       "      <td>2017-01-31</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/01/...</td>\n",
       "      <td>10.1038/s41598-017-04402-4</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12153</th>\n",
       "      <td>10.1101/092171</td>\n",
       "      <td>Darwin: A Hardware-acceleration Framework for ...</td>\n",
       "      <td>Turakhia, Y.; Zheng, K. J.; Bejerano, G.; Dall...</td>\n",
       "      <td></td>\n",
       "      <td>STANFORD UNIVERSITY</td>\n",
       "      <td>2017-01-15</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/01/...</td>\n",
       "      <td>10.1109/MM.2019.2910009</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12154</th>\n",
       "      <td>10.1101/092171</td>\n",
       "      <td>Darwin: A Hardware-acceleration Framework for ...</td>\n",
       "      <td>Turakhia, Y.; Zheng, K. J.; Bejerano, G.; Dall...</td>\n",
       "      <td></td>\n",
       "      <td>STANFORD UNIVERSITY</td>\n",
       "      <td>2017-01-24</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/01/...</td>\n",
       "      <td>10.1109/MM.2019.2910009</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80783</th>\n",
       "      <td>10.1101/708800</td>\n",
       "      <td>CLEC-2 suppresses calcification in cultured os...</td>\n",
       "      <td>Kanai, T.; Sawa, Y.; Takara, K.; Kajiwara, K.;...</td>\n",
       "      <td></td>\n",
       "      <td>OKAYAMA UNIVERSITY GRADUATE SCHOOL OF MEDICINE...</td>\n",
       "      <td>2019-07-19</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/07/...</td>\n",
       "      <td>NA</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94799</th>\n",
       "      <td>10.1101/843763</td>\n",
       "      <td>STRESS-INDUCED GENETIC CHANGE IN FLAX REVEALS ...</td>\n",
       "      <td>Li, X.</td>\n",
       "      <td></td>\n",
       "      <td>CASE WESTERN RESERVE UNIVERSITY</td>\n",
       "      <td>2019-11-15</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/11/...</td>\n",
       "      <td>NA</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9616</th>\n",
       "      <td>10.1101/079533</td>\n",
       "      <td>Chiral vortex dynamics on membranes is an intr...</td>\n",
       "      <td>Diego Ramirez;Daniela A. Garcia-Soriano;Ana Ra...</td>\n",
       "      <td>Petra  Schwille</td>\n",
       "      <td>MAX PLANCK INSTITUTE FOR BIOCHEMISTRY</td>\n",
       "      <td>2016-10-07</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td></td>\n",
       "      <td>https://www.biorxiv.org/content/early/2016/10/...</td>\n",
       "      <td>NA</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17763</th>\n",
       "      <td>10.1101/146035</td>\n",
       "      <td>The Emergent Connectome in Caenorhabditis eleg...</td>\n",
       "      <td>- DevoWorm Group,  ; Alicea, B. J.</td>\n",
       "      <td>Bradly J. Alicea</td>\n",
       "      <td>ORTHOGONAL RESEARCH</td>\n",
       "      <td>2017-06-05</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/06/...</td>\n",
       "      <td>10.1016/j.biosystems.2018.09.016</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18592</th>\n",
       "      <td>10.1101/152553</td>\n",
       "      <td>Project MinE: study design and pilot analyses ...</td>\n",
       "      <td>- Project MinE Consortium,  ; Van Rheenen, W.;...</td>\n",
       "      <td>Jan H Veldink</td>\n",
       "      <td>DEPARTMENT OF NEUROLOGY, BRAIN CENTER RUDOLF M...</td>\n",
       "      <td>2017-06-20</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/06/...</td>\n",
       "      <td>10.1038/s41431-018-0177-4</td>\n",
       "      <td>37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19852</th>\n",
       "      <td>10.1101/143933</td>\n",
       "      <td>The Multiple Sclerosis Genomic Map: Role of pe...</td>\n",
       "      <td>- International Multiple Sclerosis Genetics Co...</td>\n",
       "      <td>Philip L. De Jager</td>\n",
       "      <td>COLUMBIA UNIVERSITY MEDICAL CENTER, NEW YORK, ...</td>\n",
       "      <td>2017-07-13</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/07/...</td>\n",
       "      <td>10.1126/science.aav7188</td>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20376</th>\n",
       "      <td>10.1101/167577</td>\n",
       "      <td>Genome-wide association analyses identify 44 r...</td>\n",
       "      <td>- Major Depressive Disorder Working Group of t...</td>\n",
       "      <td>Naomi R. Wray</td>\n",
       "      <td>UNIVERSITY OF QUEENSLAND</td>\n",
       "      <td>2017-07-24</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2017/07/...</td>\n",
       "      <td>10.1038/s41588-018-0090-3</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>105 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  DOI                                              Title  \\\n",
       "12673  10.1101/104778  Engaging narratives evoke similar neural activ...   \n",
       "12153  10.1101/092171  Darwin: A Hardware-acceleration Framework for ...   \n",
       "12154  10.1101/092171  Darwin: A Hardware-acceleration Framework for ...   \n",
       "80783  10.1101/708800  CLEC-2 suppresses calcification in cultured os...   \n",
       "94799  10.1101/843763  STRESS-INDUCED GENETIC CHANGE IN FLAX REVEALS ...   \n",
       "...               ...                                                ...   \n",
       "9616   10.1101/079533  Chiral vortex dynamics on membranes is an intr...   \n",
       "17763  10.1101/146035  The Emergent Connectome in Caenorhabditis eleg...   \n",
       "18592  10.1101/152553  Project MinE: study design and pilot analyses ...   \n",
       "19852  10.1101/143933  The Multiple Sclerosis Genomic Map: Role of pe...   \n",
       "20376  10.1101/167577  Genome-wide association analyses identify 44 r...   \n",
       "\n",
       "                                                 Authors  \\\n",
       "12673              Cohen, S. S.; Henin, S.; Parra, L. C.   \n",
       "12153  Turakhia, Y.; Zheng, K. J.; Bejerano, G.; Dall...   \n",
       "12154  Turakhia, Y.; Zheng, K. J.; Bejerano, G.; Dall...   \n",
       "80783  Kanai, T.; Sawa, Y.; Takara, K.; Kajiwara, K.;...   \n",
       "94799                                             Li, X.   \n",
       "...                                                  ...   \n",
       "9616   Diego Ramirez;Daniela A. Garcia-Soriano;Ana Ra...   \n",
       "17763                 - DevoWorm Group,  ; Alicea, B. J.   \n",
       "18592  - Project MinE Consortium,  ; Van Rheenen, W.;...   \n",
       "19852  - International Multiple Sclerosis Genetics Co...   \n",
       "20376  - Major Depressive Disorder Working Group of t...   \n",
       "\n",
       "      Corresponding_Authors  \\\n",
       "12673                         \n",
       "12153                         \n",
       "12154                         \n",
       "80783                         \n",
       "94799                         \n",
       "...                     ...   \n",
       "9616        Petra  Schwille   \n",
       "17763      Bradly J. Alicea   \n",
       "18592         Jan H Veldink   \n",
       "19852    Philip L. De Jager   \n",
       "20376         Naomi R. Wray   \n",
       "\n",
       "                                             Institution       Date Version  \\\n",
       "12673  THE CITY COLLEGE OF THE CITY UNIVERSITY OF NEW... 2017-01-31       1   \n",
       "12153                                STANFORD UNIVERSITY 2017-01-15       1   \n",
       "12154                                STANFORD UNIVERSITY 2017-01-24       2   \n",
       "80783  OKAYAMA UNIVERSITY GRADUATE SCHOOL OF MEDICINE... 2019-07-19       1   \n",
       "94799                    CASE WESTERN RESERVE UNIVERSITY 2019-11-15       1   \n",
       "...                                                  ...        ...     ...   \n",
       "9616               MAX PLANCK INSTITUTE FOR BIOCHEMISTRY 2016-10-07       1   \n",
       "17763                                ORTHOGONAL RESEARCH 2017-06-05       1   \n",
       "18592  DEPARTMENT OF NEUROLOGY, BRAIN CENTER RUDOLF M... 2017-06-20       1   \n",
       "19852  COLUMBIA UNIVERSITY MEDICAL CENTER, NEW YORK, ... 2017-07-13       1   \n",
       "20376                           UNIVERSITY OF QUEENSLAND 2017-07-24       1   \n",
       "\n",
       "              Type        Category  \\\n",
       "12673  new results    Neuroscience   \n",
       "12153  new results        Genomics   \n",
       "12154  new results        Genomics   \n",
       "80783  new results    Cell Biology   \n",
       "94799  new results        Genomics   \n",
       "...            ...             ...   \n",
       "9616   new results                   \n",
       "17763  new results  Bioinformatics   \n",
       "18592  new results        Genetics   \n",
       "19852  new results        Genetics   \n",
       "20376  new results        Genetics   \n",
       "\n",
       "                                                     Xml  \\\n",
       "12673  https://www.biorxiv.org/content/early/2017/01/...   \n",
       "12153  https://www.biorxiv.org/content/early/2017/01/...   \n",
       "12154  https://www.biorxiv.org/content/early/2017/01/...   \n",
       "80783  https://www.biorxiv.org/content/early/2019/07/...   \n",
       "94799  https://www.biorxiv.org/content/early/2019/11/...   \n",
       "...                                                  ...   \n",
       "9616   https://www.biorxiv.org/content/early/2016/10/...   \n",
       "17763  https://www.biorxiv.org/content/early/2017/06/...   \n",
       "18592  https://www.biorxiv.org/content/early/2017/06/...   \n",
       "19852  https://www.biorxiv.org/content/early/2017/07/...   \n",
       "20376  https://www.biorxiv.org/content/early/2017/07/...   \n",
       "\n",
       "                              Published Num_of_Authors  \n",
       "12673        10.1038/s41598-017-04402-4              3  \n",
       "12153           10.1109/MM.2019.2910009              4  \n",
       "12154           10.1109/MM.2019.2910009              4  \n",
       "80783                                NA              8  \n",
       "94799                                NA              1  \n",
       "...                                 ...            ...  \n",
       "9616                                 NA              7  \n",
       "17763  10.1016/j.biosystems.2018.09.016              2  \n",
       "18592         10.1038/s41431-018-0177-4             37  \n",
       "19852           10.1126/science.aav7188             39  \n",
       "20376         10.1038/s41588-018-0090-3              3  \n",
       "\n",
       "[105 rows x 12 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import re\n",
    "pd.options.mode.use_inf_as_na = True # this option check for empty strings as well\n",
    "\n",
    "jl_fix_df = pd.DataFrame(columns=journal_df.columns)\n",
    "for col in journal_df:\n",
    "    jl_fix_df = pd.concat([jl_fix_df, journal_df[journal_df[col].isna()]])\n",
    "    jl_fix_df = pd.concat([jl_fix_df, journal_df[journal_df[col] == '']])\n",
    "\n",
    "pattern = re.compile('^-|$-')\n",
    "#display(journal_df[journal_df.Authors.str.contains(pattern)])\n",
    "jl_fix_df = pd.concat([jl_fix_df, journal_df[journal_df.Authors.str.contains(pattern)]])\n",
    "jl_fix_df.to_csv('data/data_fixing.csv')\n",
    "display(jl_fix_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "d058bbe1-6ac7-46c4-bd4d-8a280b7e12a0",
   "metadata": {
    "id": "XpQfyBrGEvR_"
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "9317c34c-95b2-4e56-ab63-f78b17d92361",
   "metadata": {
    "id": "XpQfyBrGEvR_"
   },
   "outputs": [],
   "source": [
    "#fixed_df = pd.read_csv('')\n",
    "#journal_df.merge(fixed_df, 'left', left_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "WTnCG-hREvPS",
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 3436
    },
    "id": "WTnCG-hREvPS",
    "outputId": "14ddcf93-a489-4d2c-bddd-9a370ba88408"
   },
   "outputs": [],
   "source": [
    "display(journal_df[journal_df.Published =='NA'])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cprjkBEg3p8Q",
   "metadata": {
    "id": "cprjkBEg3p8Q"
   },
   "source": [
    "### Publication data for Prepublication data for the given time period"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ubb3vA1A_LYY",
   "metadata": {
    "id": "ubb3vA1A_LYY"
   },
   "outputs": [],
   "source": [
    "#from tqdm import tqdm_notebook, tnrange\n",
    "from tqdm.notebook import tqdm_notebook\n",
    "import time\n",
    "import math\n",
    "\n",
    "#variables\n",
    "#https://api.biorxiv.org/pubs/biorvix/10.1101/759530\n",
    "base_url: str = 'https://api.biorxiv.org'\n",
    "query_type: str = 'pubs'\n",
    "server: str ='biorxiv'\n",
    "url: str = f'{base_url}/{query_type}/{server}'\n",
    "path: str = f'pub_journal-{query_type}-{server}-{datetime.now()}'\n",
    "step = 100\n",
    "#doi = ['10.1101/856302']\n",
    "doi = journal_df[journal_df.Published != 'NA'].Published\n",
    "\n",
    "#create directory\n",
    "pathlib.Path(path).mkdir(parents=True, exist_ok=True)\n",
    "total = len(doi)\n",
    "#def process_doi_data(path:str, url:str, doi:pd.Series, meta: List, col_names: List, item:int, loop_list, disable:bool = False):\n",
    "                     \n",
    "args = [(path, url, doi, \n",
    "         [\"preprint_doi\", \"published_doi\", \"preprint_title\", \"preprint_authors\", \"preprint_author_corresponding\", \"preprint_author_corresponding_institution\", \"preprint_category\", \"published_journal\", \"preprint_date\", \"published_date\"],\n",
    "         [\"DOI\", \"pub_DOI\", \"Title\", \"Authors\", \"Corresponding_Authors\", \"Institution\", \"Category\", \"Journal\", \"Preprint_Date\", \"Published_Date\"],\n",
    "         item, True) for item in range(0, total, step)]\n",
    "print(f'total doi: {total} iter: {total/step}')\n",
    "#tq.thread_map(lambda p: process_doi_data(*p), args, desc='process_doi_data', max_workers=5, total=len(args))\n",
    "result = tq.thread_map(lambda p: process_doi_data(*p), args, desc='process_doi_data', total=len(args))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "y_FpBhO6UGfk",
   "metadata": {
    "id": "y_FpBhO6UGfk"
   },
   "outputs": [],
   "source": [
    "filtered_df = pd.read_parquet(pathlib.Path(path))\n",
    "print([item for item in filtered_df.Published_Date if len(item.strip()) != 10])\n",
    "filtered_df = create_published_df(filtered_df).reindex()\n",
    "filtered_df.rename(columns={'pub_DOI':'Published', 'Preprint_Date':'Date'}, inplace=True)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6XmsPB97TLdq",
   "metadata": {
    "id": "6XmsPB97TLdq"
   },
   "source": [
    "Finding and cleaning missing data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "AZC1_IG3UqWN",
   "metadata": {
    "id": "AZC1_IG3UqWN"
   },
   "outputs": [],
   "source": [
    "pd.options.mode.use_inf_as_na = True # this option check for empty strings as well\n",
    "fix_df = pd.DataFrame(columns=filtered_df.columns)\n",
    "for col in filtered_df:\n",
    "    fix_df = pd.concat([fix_df, filtered_df[filtered_df[col].isna()]])\n",
    "    fix_df = pd.concat([fix_df, filtered_df[filtered_df[col] == '']])\n",
    "    \n",
    "    \n",
    "display(\"Rows with incomplete data:\", fix_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1sty3ERPRrNJ",
   "metadata": {
    "id": "1sty3ERPRrNJ"
   },
   "outputs": [],
   "source": [
    "print(f'\\nPrepublished Artices w/ Publication Info that have missing info: {filtered_df.isnull().sum()}\\n')\n",
    "#print('Published data:\\n',[(name, journal_df[name].isnull().sum()) for name in journal_df.columns if journal_df[name].isnull().values.any()])\n",
    "#print('Prepublish data:\\n',[(name, filtered_df[name].isnull().sum()) for name in filtered_df.columns if filtered_df[name].isnull().values.any()])\n",
    "fix_df = filtered_df[filtered_df['Published_Date'].isna()]\n",
    "display(\"Rows with NaN:\", fix_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "BunF5o7mlrlj",
   "metadata": {
    "id": "BunF5o7mlrlj"
   },
   "outputs": [],
   "source": [
    "filtered_df.info()\n",
    "clean_df = filtered_df.dropna()\n",
    "merged_df = pd.merge(journal_df, clean_df, how='right', on=['DOI','Published','Title', 'Authors','Corresponding_Authors','Institution','Category','Date','Num_of_Authors'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1VIdng1xqjyy",
   "metadata": {
    "id": "1VIdng1xqjyy"
   },
   "outputs": [],
   "source": [
    "pd.options.mode.use_inf_as_na = True\n",
    "print(f'Prepublished Artices w/ Publication Info: {merged_df.isnull().sum()}')\n",
    "display(merged_df)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "Zde3NBti0r72",
   "metadata": {
    "id": "Zde3NBti0r72"
   },
   "source": [
    "#### General Descriptive Anaylsis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "68b9edbb",
   "metadata": {
    "id": "68b9edbb"
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding_Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Date</th>\n",
       "      <th>Version</th>\n",
       "      <th>Type</th>\n",
       "      <th>Category</th>\n",
       "      <th>Xml</th>\n",
       "      <th>Published</th>\n",
       "      <th>Num_of_Authors</th>\n",
       "      <th>NotPublished</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045</td>\n",
       "      <td>148045.000000</td>\n",
       "      <td>148045</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unique</th>\n",
       "      <td>107510</td>\n",
       "      <td>118008</td>\n",
       "      <td>114476</td>\n",
       "      <td>69080</td>\n",
       "      <td>34455</td>\n",
       "      <td>NaN</td>\n",
       "      <td>25</td>\n",
       "      <td>5</td>\n",
       "      <td>28</td>\n",
       "      <td>146177</td>\n",
       "      <td>72872</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>top</th>\n",
       "      <td>10.1101/617381</td>\n",
       "      <td>A general theory of individuated multicellularity</td>\n",
       "      <td>Osato, N.</td>\n",
       "      <td>Thierry  Mora</td>\n",
       "      <td>STANFORD UNIVERSITY</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/09/...</td>\n",
       "      <td>NA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>freq</th>\n",
       "      <td>26</td>\n",
       "      <td>19</td>\n",
       "      <td>27</td>\n",
       "      <td>48</td>\n",
       "      <td>1637</td>\n",
       "      <td>NaN</td>\n",
       "      <td>107529</td>\n",
       "      <td>144703</td>\n",
       "      <td>27758</td>\n",
       "      <td>4</td>\n",
       "      <td>45212</td>\n",
       "      <td>NaN</td>\n",
       "      <td>102833</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mean</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-03-22 06:08:02.496538112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>7.165983</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2013-11-07 00:00:00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2018-06-04 00:00:00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-07-01 00:00:00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>6.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-04-29 00:00:00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>max</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-12-31 00:00:00</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>64.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>std</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5.746177</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                   DOI                                              Title  \\\n",
       "count           148045                                             148045   \n",
       "unique          107510                                             118008   \n",
       "top     10.1101/617381  A general theory of individuated multicellularity   \n",
       "freq                26                                                 19   \n",
       "mean               NaN                                                NaN   \n",
       "min                NaN                                                NaN   \n",
       "25%                NaN                                                NaN   \n",
       "50%                NaN                                                NaN   \n",
       "75%                NaN                                                NaN   \n",
       "max                NaN                                                NaN   \n",
       "std                NaN                                                NaN   \n",
       "\n",
       "          Authors Corresponding_Authors          Institution  \\\n",
       "count      148045                148045               148045   \n",
       "unique     114476                 69080                34455   \n",
       "top     Osato, N.         Thierry  Mora  STANFORD UNIVERSITY   \n",
       "freq           27                    48                 1637   \n",
       "mean          NaN                   NaN                  NaN   \n",
       "min           NaN                   NaN                  NaN   \n",
       "25%           NaN                   NaN                  NaN   \n",
       "50%           NaN                   NaN                  NaN   \n",
       "75%           NaN                   NaN                  NaN   \n",
       "max           NaN                   NaN                  NaN   \n",
       "std           NaN                   NaN                  NaN   \n",
       "\n",
       "                                 Date Version         Type      Category  \\\n",
       "count                          148045  148045       148045        148045   \n",
       "unique                            NaN      25            5            28   \n",
       "top                               NaN       1  new results  Neuroscience   \n",
       "freq                              NaN  107529       144703         27758   \n",
       "mean    2019-03-22 06:08:02.496538112     NaN          NaN           NaN   \n",
       "min               2013-11-07 00:00:00     NaN          NaN           NaN   \n",
       "25%               2018-06-04 00:00:00     NaN          NaN           NaN   \n",
       "50%               2019-07-01 00:00:00     NaN          NaN           NaN   \n",
       "75%               2020-04-29 00:00:00     NaN          NaN           NaN   \n",
       "max               2020-12-31 00:00:00     NaN          NaN           NaN   \n",
       "std                               NaN     NaN          NaN           NaN   \n",
       "\n",
       "                                                      Xml Published  \\\n",
       "count                                              148045    148045   \n",
       "unique                                             146177     72872   \n",
       "top     https://www.biorxiv.org/content/early/2019/09/...        NA   \n",
       "freq                                                    4     45212   \n",
       "mean                                                  NaN       NaN   \n",
       "min                                                   NaN       NaN   \n",
       "25%                                                   NaN       NaN   \n",
       "50%                                                   NaN       NaN   \n",
       "75%                                                   NaN       NaN   \n",
       "max                                                   NaN       NaN   \n",
       "std                                                   NaN       NaN   \n",
       "\n",
       "        Num_of_Authors NotPublished  \n",
       "count    148045.000000       148045  \n",
       "unique             NaN            2  \n",
       "top                NaN        False  \n",
       "freq               NaN       102833  \n",
       "mean          7.165983          NaN  \n",
       "min           1.000000          NaN  \n",
       "25%           4.000000          NaN  \n",
       "50%           6.000000          NaN  \n",
       "75%           9.000000          NaN  \n",
       "max          64.000000          NaN  \n",
       "std           5.746177          NaN  "
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#summary of table\n",
    "journal_df.describe(include='all', datetime_is_numeric=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "fd9cf4bd-3afe-46d0-87e7-4d1579cf2218",
   "metadata": {
    "id": "OpkcxRKX2kLR"
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding_Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Date</th>\n",
       "      <th>Version</th>\n",
       "      <th>Type</th>\n",
       "      <th>Category</th>\n",
       "      <th>Xml</th>\n",
       "      <th>Published</th>\n",
       "      <th>Num_of_Authors</th>\n",
       "      <th>NotPublished</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/000026</td>\n",
       "      <td>A Population Genetic Signature of Polygenic Lo...</td>\n",
       "      <td>Jeremy J Berg;Graham Coop;</td>\n",
       "      <td>Graham  Coop</td>\n",
       "      <td>UNIVERSITY OF CALIFORNIA, DAVIS</td>\n",
       "      <td>2013-11-07</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2013/11/...</td>\n",
       "      <td>10.1371/journal.pgen.1004412</td>\n",
       "      <td>3</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/000042</td>\n",
       "      <td>Routes for breaching and protecting genetic pr...</td>\n",
       "      <td>Yaniv Erlich;Arvind Narayanan;</td>\n",
       "      <td>Yaniv  Erlich</td>\n",
       "      <td>WHITEHEAD INSTITUTE</td>\n",
       "      <td>2013-11-07</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2013/11/...</td>\n",
       "      <td>10.1038/nrg3723</td>\n",
       "      <td>3</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/000067</td>\n",
       "      <td>Genetics of single-cell protein abundance vari...</td>\n",
       "      <td>Frank Albert;Sebastian Treusch;Arthur H Shockl...</td>\n",
       "      <td>Leonid  Kruglyak</td>\n",
       "      <td>UCLA</td>\n",
       "      <td>2013-11-07</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2013/11/...</td>\n",
       "      <td>10.1038/nature12904</td>\n",
       "      <td>6</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/000075</td>\n",
       "      <td>A Scalable Formulation for Engineering Combina...</td>\n",
       "      <td>Vanessa Jonsson;Anders Rantzer;Richard M Murray;</td>\n",
       "      <td>Vanessa  Jonsson</td>\n",
       "      <td>CALTECH</td>\n",
       "      <td>2013-11-07</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Evolutionary Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2013/11/...</td>\n",
       "      <td>10.1109/ACC.2014.6859452</td>\n",
       "      <td>4</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/000091</td>\n",
       "      <td>Designing Robustness to Temperature in a Feedf...</td>\n",
       "      <td>Shaunak Sen;Jongmin Kim;Richard M. Murray;</td>\n",
       "      <td>Shaunak  Sen</td>\n",
       "      <td>INDIAN INSTITUTE OF TECHNOLOGY DELHI</td>\n",
       "      <td>2013-11-07</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Synthetic Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2013/11/...</td>\n",
       "      <td>NA</td>\n",
       "      <td>4</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107505</th>\n",
       "      <td>10.1101/872853</td>\n",
       "      <td>Variation in repeat copy number of epithelial ...</td>\n",
       "      <td>Raposo, C. J.; McElroy, K. A.; Fuchs, S. M.</td>\n",
       "      <td>Stephen M Fuchs</td>\n",
       "      <td>TUFTS UNIVERSITY</td>\n",
       "      <td>2019-12-11</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/12/...</td>\n",
       "      <td>10.1093/femsyr/foaa018</td>\n",
       "      <td>3</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107506</th>\n",
       "      <td>10.1101/872879</td>\n",
       "      <td>Structure of the processive human Pol δ holoen...</td>\n",
       "      <td>Lancey, C.; Tehseen, M.; Raducanu, V.-S.; Rash...</td>\n",
       "      <td>Alfredo  De Biasio</td>\n",
       "      <td>LEICESTER INSTITUTE OF STRUCTURAL &amp; CHEMICAL B...</td>\n",
       "      <td>2019-12-11</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Biochemistry</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/12/...</td>\n",
       "      <td>10.1038/s41467-020-14898-6</td>\n",
       "      <td>12</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107507</th>\n",
       "      <td>10.1101/872952</td>\n",
       "      <td>A key regulatory protein for flagellum length ...</td>\n",
       "      <td>Atkins, M.; Tyc, J.; Shafiq, S.; Ahmed, M.; Be...</td>\n",
       "      <td>Sue  Vaughan</td>\n",
       "      <td>OXFORD BROOKES UNIVERSITY</td>\n",
       "      <td>2019-12-11</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Cell Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/12/...</td>\n",
       "      <td>NA</td>\n",
       "      <td>10</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107508</th>\n",
       "      <td>10.1101/873109</td>\n",
       "      <td>FGF9 and FGF10 use distinct signaling pathways...</td>\n",
       "      <td>Yin, Y.; Ornitz, D. M.</td>\n",
       "      <td>David M Ornitz</td>\n",
       "      <td>WASHINGTON UNIVERSITY SCHOOL OF MEDICINE</td>\n",
       "      <td>2019-12-11</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/12/...</td>\n",
       "      <td>10.1126/scisignal.aay4353</td>\n",
       "      <td>2</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107509</th>\n",
       "      <td>10.1101/873232</td>\n",
       "      <td>Multiple requirements for Rab GTPases in the d...</td>\n",
       "      <td>Best, B. T.; Leptin, M.</td>\n",
       "      <td>Maria  Leptin</td>\n",
       "      <td>EUROPEAN MOLECULAR BIOLOGY LABORATORY</td>\n",
       "      <td>2019-12-11</td>\n",
       "      <td>1</td>\n",
       "      <td>new results</td>\n",
       "      <td>Developmental Biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2019/12/...</td>\n",
       "      <td>10.1534/g3.119.400967</td>\n",
       "      <td>2</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>107510 rows × 13 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   DOI                                              Title  \\\n",
       "0       10.1101/000026  A Population Genetic Signature of Polygenic Lo...   \n",
       "1       10.1101/000042  Routes for breaching and protecting genetic pr...   \n",
       "2       10.1101/000067  Genetics of single-cell protein abundance vari...   \n",
       "3       10.1101/000075  A Scalable Formulation for Engineering Combina...   \n",
       "4       10.1101/000091  Designing Robustness to Temperature in a Feedf...   \n",
       "...                ...                                                ...   \n",
       "107505  10.1101/872853  Variation in repeat copy number of epithelial ...   \n",
       "107506  10.1101/872879  Structure of the processive human Pol δ holoen...   \n",
       "107507  10.1101/872952  A key regulatory protein for flagellum length ...   \n",
       "107508  10.1101/873109  FGF9 and FGF10 use distinct signaling pathways...   \n",
       "107509  10.1101/873232  Multiple requirements for Rab GTPases in the d...   \n",
       "\n",
       "                                                  Authors  \\\n",
       "0                              Jeremy J Berg;Graham Coop;   \n",
       "1                          Yaniv Erlich;Arvind Narayanan;   \n",
       "2       Frank Albert;Sebastian Treusch;Arthur H Shockl...   \n",
       "3        Vanessa Jonsson;Anders Rantzer;Richard M Murray;   \n",
       "4              Shaunak Sen;Jongmin Kim;Richard M. Murray;   \n",
       "...                                                   ...   \n",
       "107505        Raposo, C. J.; McElroy, K. A.; Fuchs, S. M.   \n",
       "107506  Lancey, C.; Tehseen, M.; Raducanu, V.-S.; Rash...   \n",
       "107507  Atkins, M.; Tyc, J.; Shafiq, S.; Ahmed, M.; Be...   \n",
       "107508                             Yin, Y.; Ornitz, D. M.   \n",
       "107509                            Best, B. T.; Leptin, M.   \n",
       "\n",
       "       Corresponding_Authors  \\\n",
       "0               Graham  Coop   \n",
       "1              Yaniv  Erlich   \n",
       "2           Leonid  Kruglyak   \n",
       "3           Vanessa  Jonsson   \n",
       "4               Shaunak  Sen   \n",
       "...                      ...   \n",
       "107505       Stephen M Fuchs   \n",
       "107506    Alfredo  De Biasio   \n",
       "107507          Sue  Vaughan   \n",
       "107508        David M Ornitz   \n",
       "107509         Maria  Leptin   \n",
       "\n",
       "                                              Institution       Date Version  \\\n",
       "0                         UNIVERSITY OF CALIFORNIA, DAVIS 2013-11-07       1   \n",
       "1                                     WHITEHEAD INSTITUTE 2013-11-07       1   \n",
       "2                                                    UCLA 2013-11-07       1   \n",
       "3                                                 CALTECH 2013-11-07       1   \n",
       "4                    INDIAN INSTITUTE OF TECHNOLOGY DELHI 2013-11-07       1   \n",
       "...                                                   ...        ...     ...   \n",
       "107505                                   TUFTS UNIVERSITY 2019-12-11       1   \n",
       "107506  LEICESTER INSTITUTE OF STRUCTURAL & CHEMICAL B... 2019-12-11       1   \n",
       "107507                          OXFORD BROOKES UNIVERSITY 2019-12-11       1   \n",
       "107508           WASHINGTON UNIVERSITY SCHOOL OF MEDICINE 2019-12-11       1   \n",
       "107509              EUROPEAN MOLECULAR BIOLOGY LABORATORY 2019-12-11       1   \n",
       "\n",
       "               Type               Category  \\\n",
       "0       new results               Genetics   \n",
       "1       new results               Genomics   \n",
       "2       new results               Genomics   \n",
       "3       new results   Evolutionary Biology   \n",
       "4       new results      Synthetic Biology   \n",
       "...             ...                    ...   \n",
       "107505  new results           Cell Biology   \n",
       "107506  new results           Biochemistry   \n",
       "107507  new results           Cell Biology   \n",
       "107508  new results  Developmental Biology   \n",
       "107509  new results  Developmental Biology   \n",
       "\n",
       "                                                      Xml  \\\n",
       "0       https://www.biorxiv.org/content/early/2013/11/...   \n",
       "1       https://www.biorxiv.org/content/early/2013/11/...   \n",
       "2       https://www.biorxiv.org/content/early/2013/11/...   \n",
       "3       https://www.biorxiv.org/content/early/2013/11/...   \n",
       "4       https://www.biorxiv.org/content/early/2013/11/...   \n",
       "...                                                   ...   \n",
       "107505  https://www.biorxiv.org/content/early/2019/12/...   \n",
       "107506  https://www.biorxiv.org/content/early/2019/12/...   \n",
       "107507  https://www.biorxiv.org/content/early/2019/12/...   \n",
       "107508  https://www.biorxiv.org/content/early/2019/12/...   \n",
       "107509  https://www.biorxiv.org/content/early/2019/12/...   \n",
       "\n",
       "                           Published  Num_of_Authors  NotPublished  \n",
       "0       10.1371/journal.pgen.1004412               3         False  \n",
       "1                    10.1038/nrg3723               3         False  \n",
       "2                10.1038/nature12904               6         False  \n",
       "3           10.1109/ACC.2014.6859452               4         False  \n",
       "4                                 NA               4          True  \n",
       "...                              ...             ...           ...  \n",
       "107505        10.1093/femsyr/foaa018               3         False  \n",
       "107506    10.1038/s41467-020-14898-6              12         False  \n",
       "107507                            NA              10          True  \n",
       "107508     10.1126/scisignal.aay4353               2         False  \n",
       "107509         10.1534/g3.119.400967               2         False  \n",
       "\n",
       "[107510 rows x 13 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "published: 72951 (67.86%)\n",
      "not published: 34559 (32.14%)\n"
     ]
    }
   ],
   "source": [
    "# published vs unpublished\n",
    "def percentage (item1, item2) -> float:\n",
    "    return round(item1 / (item1 + item2) * 100, 2)\n",
    "\n",
    "journal_df['NotPublished'] = journal_df['Published'] == 'NA'\n",
    "tmp_df = journal_df.groupby('DOI', as_index=False).first()\n",
    "display(tmp_df)\n",
    "result = freq_count(tmp_df,'NotPublished')\n",
    "\n",
    "print(f'published: {result.iloc[0]} ({percentage(result.iloc[0],result.iloc[1])}%)')\n",
    "print(f'not published: {result.iloc[1]} ({percentage(result.iloc[1],result.iloc[0])}%)')      "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "FYItzFQromtZ",
   "metadata": {
    "id": "FYItzFQromtZ"
   },
   "outputs": [],
   "source": [
    "#freq count of Instituion\n",
    "freq_count(journal_df, 'Institution')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "aK2EySJTo-iP",
   "metadata": {
    "id": "aK2EySJTo-iP"
   },
   "outputs": [],
   "source": [
    "#freq count of number of version\n",
    "freq_count(journal_df, 'Version')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "QtuIfogBpJi6",
   "metadata": {
    "id": "QtuIfogBpJi6"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "new results              105228\n",
       "confirmatory results       1648\n",
       "contradictory results       632\n",
       "                              2\n",
       "withdrawn                     0\n",
       "Name: Type, dtype: int64"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count of Type\n",
    "freq_count(tmp_df, 'Type')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ccWCKmfmpbJI",
   "metadata": {
    "id": "ccWCKmfmpbJI"
   },
   "outputs": [],
   "source": [
    "#freq count Category\n",
    "freq_count(tmp_df, 'Category')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "kUz1VYpi_dQS",
   "metadata": {
    "id": "kUz1VYpi_dQS"
   },
   "source": [
    "#### Check"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ecaaea6d",
   "metadata": {
    "id": "ecaaea6d"
   },
   "outputs": [],
   "source": [
    "assert get_total(url) == journal_df.shape[0]\n",
    "\"Total number of papers submitted and the length of the dataframe match.\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "537b3cc3",
   "metadata": {
    "id": "537b3cc3"
   },
   "source": [
    "The above code runs through all papers that are submitted within selected time frame. Error discussed during the previous meeting has been fixed. "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "70483477",
   "metadata": {
    "id": "70483477"
   },
   "source": [
    "----"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6dPz3eLWwpri",
   "metadata": {
    "id": "6dPz3eLWwpri",
    "tags": []
   },
   "source": [
    "## Case 2: Published Articles query for a given timefame"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2lRrqLvbYI5g",
   "metadata": {
    "id": "2lRrqLvbYI5g"
   },
   "outputs": [],
   "source": [
    "# \"global\" variables\n",
    "#https://api.biorxiv.org/pubs/biorvix/2018-08-21/2018-08-28\n",
    "base_url: str = 'https://api.biorxiv.org'\n",
    "query_type: str = 'pubs'\n",
    "server: str = 'biorxiv'\n",
    "start_interval: str = '2011-01-01'\n",
    "end_interval: str = '2020-12-31'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "A8PpsTQs2YT-",
   "metadata": {
    "id": "A8PpsTQs2YT-"
   },
   "source": [
    "### BioRvix Server"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "9ZgDU19JYStP",
   "metadata": {
    "id": "9ZgDU19JYStP"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://api.biorxiv.org/pubs/biorxiv/2011-01-01/2020-12-31\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "9b6a17bb733c4a8ea0d3ad86844f7763",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "get_big_data:   0%|          | 0/560 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "key: preprint_author_corresponding journal: {'preprint_doi': '10.1101/026161', 'published_doi': '10.1242/jcs.195982', 'published_journal': 'Journal of Cell Science', 'preprint_platform': 'bioRxiv', 'preprint_title': 'Crucial Roles of the Arp2/3 Complex during Mammalian Corticogenesis', 'preprint_authors': 'Pei-Shan Wang; Fu-Sheng Chou; Fengli Guo; Praveen Suraneni; Sheng Xia; Sree Ramachandran; Rong Li', 'preprint_category': '', 'preprint_date': '2015-09-06', 'published_date': '2016-08-15'}\n",
      "'preprint_author_corresponding'\n",
      "key: preprint_author_corresponding_institution journal: {'preprint_doi': '10.1101/026161', 'published_doi': '10.1242/jcs.195982', 'published_journal': 'Journal of Cell Science', 'preprint_platform': 'bioRxiv', 'preprint_title': 'Crucial Roles of the Arp2/3 Complex during Mammalian Corticogenesis', 'preprint_authors': 'Pei-Shan Wang; Fu-Sheng Chou; Fengli Guo; Praveen Suraneni; Sheng Xia; Sree Ramachandran; Rong Li', 'preprint_category': '', 'preprint_date': '2015-09-06', 'published_date': '2016-08-15'}\n",
      "'preprint_author_corresponding_institution'\n"
     ]
    }
   ],
   "source": [
    "# \"local\" variables\n",
    "url: str = f'{base_url}/{query_type}/{server}/{start_interval}/{end_interval}'\n",
    "path: str = f'data/pub-{query_type}-{server}-{start_interval}!{end_interval}-{datetime.now()}'\n",
    "step = 100\n",
    "pubs_df = None\n",
    "\n",
    "#finding the number of records for a given time period\n",
    "\n",
    "pathlib.Path(path).mkdir(parents=True, exist_ok=True)\n",
    "print(url)\n",
    "\n",
    "multithread_processor(path, url, \n",
    "                      [\"preprint_doi\", \"published_doi\", \"preprint_title\", \"preprint_authors\", \"preprint_author_corresponding\", \"preprint_author_corresponding_institution\", \"preprint_category\", \"published_journal\", \"preprint_date\", \"published_date\"],\n",
    "                      [\"DOI\", \"pub_DOI\", \"Title\", \"Authors\", \"Corresponding_Authors\", \"Institution\", \"Category\", \"Journal\", \"Preprint_Date\", \"Published_Date\"],\n",
    "                      step, range(0, get_total(url), 100), True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "MVP_xYqnAiIU",
   "metadata": {
    "id": "MVP_xYqnAiIU"
   },
   "outputs": [],
   "source": [
    "pubs_df = pd.read_parquet(pathlib.Path(path)).sort_index(kind='mergesort', key=lambda x: x.astype(int))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "CbPhyHOlAo5p",
   "metadata": {
    "id": "CbPhyHOlAo5p"
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'pubs_df' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m/tmp/ipykernel_85815/3616842600.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mpubs_df\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mhead\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;31m#print([(index, value, datetime.strptime(value.split(':')[0], '%Y-%m-%d').date()) for index, value in enumerate(pubs_df.Published_Date) if len(value.strip()) != 10])\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mpubs_df\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mloc\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'DOI'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvalue\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mindex\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvalue\u001b[0m \u001b[0;32min\u001b[0m \u001b[0menumerate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpubs_df\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mPublished_Date\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mvalue\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstrip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;36m10\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'pubs_df' is not defined"
     ]
    }
   ],
   "source": [
    "pubs_df.head(1)\n",
    "#print([(index, value, datetime.strptime(value.split(':')[0], '%Y-%m-%d').date()) for index, value in enumerate(pubs_df.Published_Date) if len(value.strip()) != 10])\n",
    "print([(index, pubs_df.loc(index, 'DOI'), value) for index, value in enumerate(pubs_df.Published_Date) if len(value.strip()) != 10])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "A18hdPgAT5mk",
   "metadata": {
    "id": "A18hdPgAT5mk"
   },
   "outputs": [],
   "source": [
    "\n",
    "pubs_df = create_published_df(pubs_df)\n",
    "\n",
    "#display(pubs_df)\n",
    "#pubs_df.shape\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "ZTWyOr9wP_x_",
   "metadata": {
    "id": "ZTWyOr9wP_x_"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Index: 55979 entries, 0 to 55978\n",
      "Data columns (total 11 columns):\n",
      " #   Column                 Non-Null Count  Dtype         \n",
      "---  ------                 --------------  -----         \n",
      " 0   DOI                    55979 non-null  object        \n",
      " 1   pub_DOI                55979 non-null  object        \n",
      " 2   Title                  55979 non-null  object        \n",
      " 3   Authors                55979 non-null  object        \n",
      " 4   Corresponding_Authors  55979 non-null  object        \n",
      " 5   Institution            55979 non-null  category      \n",
      " 6   Category               55979 non-null  category      \n",
      " 7   Journal                55979 non-null  category      \n",
      " 8   Preprint_Date          55979 non-null  datetime64[ns]\n",
      " 9   Published_Date         55979 non-null  datetime64[ns]\n",
      " 10  Num_of_Authors         55979 non-null  int64         \n",
      "dtypes: category(3), datetime64[ns](2), int64(1), object(5)\n",
      "memory usage: 4.9+ MB\n",
      "False False\n",
      "0        92 days\n",
      "1        36 days\n",
      "2        78 days\n",
      "3        87 days\n",
      "4            NaT\n",
      "          ...   \n",
      "55974   174 days\n",
      "55975   510 days\n",
      "55976   371 days\n",
      "55977    41 days\n",
      "55978   687 days\n",
      "Length: 55979, dtype: timedelta64[ns]\n"
     ]
    }
   ],
   "source": [
    "pubs_df.info()\n",
    "print(pubs_df.Published_Date.isna().values.any(), pubs_df.Preprint_Date.isna().values.any())\n",
    "print(pubs_df.Published_Date.subtract(pubs_df.Preprint_Date).where(pubs_df.Published_Date >= pubs_df.Preprint_Date))\n",
    "#for index in range(pubs_df.shape[0]):\n",
    "#    try:\n",
    "#        pubs_df.loc(index, 'Published_Date') - pubs_df.loc(index, 'Preprint_Date')\n",
    "#    except Exception as e:\n",
    "        #print(e)\n",
    "        #print (f'index: {index} Published_Date: {')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "v2_8a0y02hSI",
   "metadata": {
    "id": "v2_8a0y02hSI"
   },
   "source": [
    "#### General Descriptive Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "0r7-MU3EAITY",
   "metadata": {
    "id": "0r7-MU3EAITY"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Index: 55979 entries, 0 to 55978\n",
      "Data columns (total 11 columns):\n",
      " #   Column                 Non-Null Count  Dtype         \n",
      "---  ------                 --------------  -----         \n",
      " 0   DOI                    55979 non-null  object        \n",
      " 1   pub_DOI                55979 non-null  object        \n",
      " 2   Title                  55979 non-null  object        \n",
      " 3   Authors                55979 non-null  object        \n",
      " 4   Corresponding_Authors  55979 non-null  object        \n",
      " 5   Institution            55979 non-null  category      \n",
      " 6   Category               55979 non-null  category      \n",
      " 7   Journal                55979 non-null  category      \n",
      " 8   Preprint_Date          55979 non-null  datetime64[ns]\n",
      " 9   Published_Date         55979 non-null  datetime64[ns]\n",
      " 10  Num_of_Authors         55979 non-null  int64         \n",
      "dtypes: category(3), datetime64[ns](2), int64(1), object(5)\n",
      "memory usage: 4.9+ MB\n"
     ]
    }
   ],
   "source": [
    "pubs_df.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "ecI9DRx0wlGW",
   "metadata": {
    "id": "ecI9DRx0wlGW"
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>pub_DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding_Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Category</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Preprint_Date</th>\n",
       "      <th>Published_Date</th>\n",
       "      <th>Num_of_Authors</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979</td>\n",
       "      <td>55979.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unique</th>\n",
       "      <td>55974</td>\n",
       "      <td>55906</td>\n",
       "      <td>55961</td>\n",
       "      <td>55396</td>\n",
       "      <td>39467</td>\n",
       "      <td>18942</td>\n",
       "      <td>28</td>\n",
       "      <td>3321</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>top</th>\n",
       "      <td>10.1101/756668</td>\n",
       "      <td>10.1016/j.neuroimage.2020.1165</td>\n",
       "      <td>Modeling Continuous Admixture</td>\n",
       "      <td>Chang, W. H.; Lai, A. G.</td>\n",
       "      <td>Jesse D Bloom</td>\n",
       "      <td>STANFORD UNIVERSITY</td>\n",
       "      <td>Neuroscience</td>\n",
       "      <td>Plos One</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>freq</th>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "      <td>2</td>\n",
       "      <td>7</td>\n",
       "      <td>30</td>\n",
       "      <td>644</td>\n",
       "      <td>9941</td>\n",
       "      <td>3408</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mean</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2018-10-20 10:04:06.049411328</td>\n",
       "      <td>2019-05-22 20:02:06.304506880</td>\n",
       "      <td>7.018418</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2013-11-07 00:00:00</td>\n",
       "      <td>2013-12-10 00:00:00</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2018-01-25 00:00:00</td>\n",
       "      <td>2018-09-03 00:00:00</td>\n",
       "      <td>4.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-01-09 00:00:00</td>\n",
       "      <td>2019-08-29 00:00:00</td>\n",
       "      <td>6.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-10-17 00:00:00</td>\n",
       "      <td>2020-05-28 00:00:00</td>\n",
       "      <td>9.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>max</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-12-30 00:00:00</td>\n",
       "      <td>2020-12-31 00:00:00</td>\n",
       "      <td>54.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>std</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5.391007</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                   DOI                         pub_DOI  \\\n",
       "count            55979                           55979   \n",
       "unique           55974                           55906   \n",
       "top     10.1101/756668  10.1016/j.neuroimage.2020.1165   \n",
       "freq                 2                               3   \n",
       "mean               NaN                             NaN   \n",
       "min                NaN                             NaN   \n",
       "25%                NaN                             NaN   \n",
       "50%                NaN                             NaN   \n",
       "75%                NaN                             NaN   \n",
       "max                NaN                             NaN   \n",
       "std                NaN                             NaN   \n",
       "\n",
       "                                Title                   Authors  \\\n",
       "count                           55979                     55979   \n",
       "unique                          55961                     55396   \n",
       "top     Modeling Continuous Admixture  Chang, W. H.; Lai, A. G.   \n",
       "freq                                2                         7   \n",
       "mean                              NaN                       NaN   \n",
       "min                               NaN                       NaN   \n",
       "25%                               NaN                       NaN   \n",
       "50%                               NaN                       NaN   \n",
       "75%                               NaN                       NaN   \n",
       "max                               NaN                       NaN   \n",
       "std                               NaN                       NaN   \n",
       "\n",
       "       Corresponding_Authors          Institution      Category   Journal  \\\n",
       "count                  55979                55979         55979     55979   \n",
       "unique                 39467                18942            28      3321   \n",
       "top            Jesse D Bloom  STANFORD UNIVERSITY  Neuroscience  Plos One   \n",
       "freq                      30                  644          9941      3408   \n",
       "mean                     NaN                  NaN           NaN       NaN   \n",
       "min                      NaN                  NaN           NaN       NaN   \n",
       "25%                      NaN                  NaN           NaN       NaN   \n",
       "50%                      NaN                  NaN           NaN       NaN   \n",
       "75%                      NaN                  NaN           NaN       NaN   \n",
       "max                      NaN                  NaN           NaN       NaN   \n",
       "std                      NaN                  NaN           NaN       NaN   \n",
       "\n",
       "                        Preprint_Date                 Published_Date  \\\n",
       "count                           55979                          55979   \n",
       "unique                            NaN                            NaN   \n",
       "top                               NaN                            NaN   \n",
       "freq                              NaN                            NaN   \n",
       "mean    2018-10-20 10:04:06.049411328  2019-05-22 20:02:06.304506880   \n",
       "min               2013-11-07 00:00:00            2013-12-10 00:00:00   \n",
       "25%               2018-01-25 00:00:00            2018-09-03 00:00:00   \n",
       "50%               2019-01-09 00:00:00            2019-08-29 00:00:00   \n",
       "75%               2019-10-17 00:00:00            2020-05-28 00:00:00   \n",
       "max               2020-12-30 00:00:00            2020-12-31 00:00:00   \n",
       "std                               NaN                            NaN   \n",
       "\n",
       "        Num_of_Authors  \n",
       "count     55979.000000  \n",
       "unique             NaN  \n",
       "top                NaN  \n",
       "freq               NaN  \n",
       "mean          7.018418  \n",
       "min           1.000000  \n",
       "25%           4.000000  \n",
       "50%           6.000000  \n",
       "75%           9.000000  \n",
       "max          54.000000  \n",
       "std           5.391007  "
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#summary of table\n",
    "pubs_df.describe(include='all', datetime_is_numeric=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "HorMz3hk1iWt",
   "metadata": {
    "id": "HorMz3hk1iWt"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4     7458\n",
       "3     7120\n",
       "5     6774\n",
       "6     5914\n",
       "2     4979\n",
       "7     4749\n",
       "8     3810\n",
       "9     2875\n",
       "10    2371\n",
       "11    1778\n",
       "12    1370\n",
       "13    1111\n",
       "14     868\n",
       "1      828\n",
       "15     665\n",
       "16     497\n",
       "17     451\n",
       "18     352\n",
       "19     272\n",
       "20     226\n",
       "21     182\n",
       "22     147\n",
       "23     128\n",
       "24     121\n",
       "25      98\n",
       "26      97\n",
       "27      63\n",
       "28      57\n",
       "30      56\n",
       "42      48\n",
       "29      45\n",
       "41      42\n",
       "31      41\n",
       "34      39\n",
       "32      37\n",
       "33      36\n",
       "43      36\n",
       "39      33\n",
       "38      32\n",
       "36      29\n",
       "40      28\n",
       "35      24\n",
       "46      20\n",
       "45      20\n",
       "44      15\n",
       "37      14\n",
       "47       9\n",
       "49       8\n",
       "48       3\n",
       "54       1\n",
       "50       1\n",
       "52       1\n",
       "Name: Num_of_Authors, dtype: int64"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count of Num_of_Authors\n",
    "freq_count(pubs_df,'Num_of_Authors')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "Ir0ALSY-2LTK",
   "metadata": {
    "id": "Ir0ALSY-2LTK"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Jesse D Bloom              30\n",
       "Thierry  Mora              29\n",
       "Antonis  Rokas             26\n",
       "Dave  Thirumalai           22\n",
       "Mark  Gerstein             21\n",
       "                           ..\n",
       "Takuma  Kobayashi           1\n",
       "Pauliina  Damdimopoulou     1\n",
       "Xuequn  Shang               1\n",
       "Gabi  Kastenmüller          1\n",
       "Naeha  Subramanian          1\n",
       "Name: Corresponding_Authors, Length: 39467, dtype: int64"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count \"Corresponding_Authors\"\n",
    "freq_count(pubs_df, 'Corresponding_Authors')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "-FNSHPcQ6GwP",
   "metadata": {
    "id": "-FNSHPcQ6GwP"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "STANFORD UNIVERSITY                                             644\n",
       "UNIVERSITY OF OXFORD                                            528\n",
       "UNIVERSITY OF CAMBRIDGE                                         486\n",
       "UNIVERSITY OF MICHIGAN                                          362\n",
       "UNIVERSITY OF WASHINGTON                                        344\n",
       "                                                               ... \n",
       "GUIZHOU UNIVERISTY OF CHINA                                       1\n",
       "GUIZHOU UNIVERSITY                                                1\n",
       "GUNMA UNIVERSITY GRADUATE SCHOOL OF HEALTH SCIENCES               1\n",
       "GURDON INSTITUTE - UNIVERSITY OF CAMBRIDGE                        1\n",
       "†DEPARTMENT OF PSYCHOLOGY, UNIVERSITY OF TURIN, TURIN, ITALY      1\n",
       "Name: Institution, Length: 18942, dtype: int64"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq_count Institution\n",
    "freq_count(pubs_df, 'Institution')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "KI8pfBWD6XNJ",
   "metadata": {
    "id": "KI8pfBWD6XNJ"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Neuroscience                              9941\n",
       "Microbiology                              5128\n",
       "Bioinformatics                            4987\n",
       "Genomics                                  3726\n",
       "Evolutionary Biology                      3630\n",
       "Genetics                                  3027\n",
       "Cell Biology                              2794\n",
       "Biophysics                                2747\n",
       "Ecology                                   2201\n",
       "Biochemistry                              2069\n",
       "Molecular Biology                         1856\n",
       "Cancer Biology                            1738\n",
       "Developmental Biology                     1673\n",
       "Plant Biology                             1653\n",
       "Immunology                                1522\n",
       "Systems Biology                           1431\n",
       "Bioengineering                            1192\n",
       "Epidemiology                              1006\n",
       "Animal Behavior And Cognition              857\n",
       "Physiology                                 720\n",
       "Pharmacology And Toxicology                533\n",
       "Synthetic Biology                          516\n",
       "Scientific Communication And Education     319\n",
       "Zoology                                    278\n",
       "Pathology                                  276\n",
       "Paleontology                                70\n",
       "Clinical Trials                             60\n",
       "                                            29\n",
       "Name: Category, dtype: int64"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq_count Category\n",
    "freq_count(pubs_df, 'Category')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "bwPUZ3yZ6nKT",
   "metadata": {
    "id": "bwPUZ3yZ6nKT"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Plos One                                                                               3408\n",
       "Elife                                                                                  2566\n",
       "Scientific Reports                                                                     2479\n",
       "Nature Communications                                                                  2099\n",
       "Proceedings Of The National Academy Of Sciences                                        1369\n",
       "                                                                                       ... \n",
       "International Journal Of Breast Cancer                                                    1\n",
       "International Journal Of Behavioral Nutrition And Physical Activity                       1\n",
       "International Journal Of Astrobiology                                                     1\n",
       "International Journal Of Artificial Intelligence And Machine Learning                     1\n",
       "Современные Проблемы Науки И Образования (Modern Problems Of Science And Education)       1\n",
       "Name: Journal, Length: 3321, dtype: int64"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count Jounral\n",
    "freq_count(pubs_df, 'Journal')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "tIQbFXeb69Hg",
   "metadata": {
    "id": "tIQbFXeb69Hg"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2019-05-24    139\n",
       "2019-07-02    127\n",
       "2019-09-05    122\n",
       "2019-06-21    120\n",
       "2018-12-13    117\n",
       "             ... \n",
       "2015-05-26      1\n",
       "2015-05-12      1\n",
       "2014-07-18      1\n",
       "2014-01-06      1\n",
       "2020-11-23      1\n",
       "Name: Preprint_Date, Length: 2368, dtype: int64"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count Preprint_Date\n",
    "freq_count(pubs_df, 'Preprint_Date')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "f8f46e5c-d1d2-43a1-b0eb-0d03c903502d",
   "metadata": {
    "id": "dos-OlYF7NnT"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2020-11-30    119\n",
       "2020-11-10    114\n",
       "2020-09-28    114\n",
       "2020-05-19    113\n",
       "2020-10-27    110\n",
       "             ... \n",
       "2016-04-16      1\n",
       "2016-03-26      1\n",
       "2016-04-03      1\n",
       "2016-02-27      1\n",
       "2019-12-22      1\n",
       "Name: Published_Date, Length: 2216, dtype: int64"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count Published_date\n",
    "freq_count(pubs_df, 'Published_Date')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "04a3d692-78d7-4c0b-bfee-7d81437abdf9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>count</th>\n",
       "      <th>mean</th>\n",
       "      <th>std</th>\n",
       "      <th>min</th>\n",
       "      <th>25%</th>\n",
       "      <th>50%</th>\n",
       "      <th>75%</th>\n",
       "      <th>max</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Category</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <td>29.0</td>\n",
       "      <td>9.804351</td>\n",
       "      <td>5.299783</td>\n",
       "      <td>0.985647</td>\n",
       "      <td>5.125362</td>\n",
       "      <td>8.903674</td>\n",
       "      <td>13.536212</td>\n",
       "      <td>20.370028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Animal Behavior And Cognition</th>\n",
       "      <td>856.0</td>\n",
       "      <td>7.360031</td>\n",
       "      <td>5.203498</td>\n",
       "      <td>0.164274</td>\n",
       "      <td>4.041151</td>\n",
       "      <td>6.078154</td>\n",
       "      <td>9.306146</td>\n",
       "      <td>55.524754</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Biochemistry</th>\n",
       "      <td>2060.0</td>\n",
       "      <td>5.388536</td>\n",
       "      <td>4.071484</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>2.718742</td>\n",
       "      <td>4.402554</td>\n",
       "      <td>6.833816</td>\n",
       "      <td>33.446272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Bioengineering</th>\n",
       "      <td>1188.0</td>\n",
       "      <td>6.111285</td>\n",
       "      <td>3.978310</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.285488</td>\n",
       "      <td>5.355346</td>\n",
       "      <td>7.893386</td>\n",
       "      <td>23.195548</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Bioinformatics</th>\n",
       "      <td>4975.0</td>\n",
       "      <td>7.928177</td>\n",
       "      <td>5.713281</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>4.172570</td>\n",
       "      <td>6.570977</td>\n",
       "      <td>10.217869</td>\n",
       "      <td>58.514549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Biophysics</th>\n",
       "      <td>2728.0</td>\n",
       "      <td>5.852493</td>\n",
       "      <td>4.438419</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.022649</td>\n",
       "      <td>4.829668</td>\n",
       "      <td>7.523768</td>\n",
       "      <td>55.294770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Cancer Biology</th>\n",
       "      <td>1732.0</td>\n",
       "      <td>7.330300</td>\n",
       "      <td>4.900762</td>\n",
       "      <td>0.065710</td>\n",
       "      <td>4.065792</td>\n",
       "      <td>6.275283</td>\n",
       "      <td>9.527916</td>\n",
       "      <td>40.608637</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Cell Biology</th>\n",
       "      <td>2789.0</td>\n",
       "      <td>6.813330</td>\n",
       "      <td>4.719810</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.646892</td>\n",
       "      <td>5.815314</td>\n",
       "      <td>8.706544</td>\n",
       "      <td>47.442453</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Clinical Trials</th>\n",
       "      <td>59.0</td>\n",
       "      <td>7.093871</td>\n",
       "      <td>4.517703</td>\n",
       "      <td>0.525678</td>\n",
       "      <td>3.811167</td>\n",
       "      <td>5.256781</td>\n",
       "      <td>9.495062</td>\n",
       "      <td>19.417236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Developmental Biology</th>\n",
       "      <td>1671.0</td>\n",
       "      <td>7.068244</td>\n",
       "      <td>4.952817</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.876876</td>\n",
       "      <td>5.946734</td>\n",
       "      <td>8.837964</td>\n",
       "      <td>42.054252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Ecology</th>\n",
       "      <td>2197.0</td>\n",
       "      <td>7.859181</td>\n",
       "      <td>5.509378</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>4.238280</td>\n",
       "      <td>6.538122</td>\n",
       "      <td>9.987885</td>\n",
       "      <td>62.424280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Epidemiology</th>\n",
       "      <td>1004.0</td>\n",
       "      <td>7.997972</td>\n",
       "      <td>4.717478</td>\n",
       "      <td>0.098565</td>\n",
       "      <td>4.829668</td>\n",
       "      <td>6.998090</td>\n",
       "      <td>10.160373</td>\n",
       "      <td>29.897945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Evolutionary Biology</th>\n",
       "      <td>3618.0</td>\n",
       "      <td>7.690458</td>\n",
       "      <td>5.501852</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>4.115074</td>\n",
       "      <td>6.406702</td>\n",
       "      <td>9.585412</td>\n",
       "      <td>50.826506</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Genetics</th>\n",
       "      <td>3020.0</td>\n",
       "      <td>7.250823</td>\n",
       "      <td>5.153974</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.745457</td>\n",
       "      <td>6.176718</td>\n",
       "      <td>9.363642</td>\n",
       "      <td>55.787593</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Genomics</th>\n",
       "      <td>3719.0</td>\n",
       "      <td>7.702134</td>\n",
       "      <td>5.311866</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>4.221853</td>\n",
       "      <td>6.505267</td>\n",
       "      <td>9.692191</td>\n",
       "      <td>42.021397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Immunology</th>\n",
       "      <td>1519.0</td>\n",
       "      <td>6.246689</td>\n",
       "      <td>4.765151</td>\n",
       "      <td>0.164274</td>\n",
       "      <td>3.252634</td>\n",
       "      <td>5.191072</td>\n",
       "      <td>7.967309</td>\n",
       "      <td>73.003552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Microbiology</th>\n",
       "      <td>5124.0</td>\n",
       "      <td>5.886814</td>\n",
       "      <td>4.406204</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>2.989794</td>\n",
       "      <td>4.796813</td>\n",
       "      <td>7.490914</td>\n",
       "      <td>45.306885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Molecular Biology</th>\n",
       "      <td>1853.0</td>\n",
       "      <td>6.290584</td>\n",
       "      <td>4.763736</td>\n",
       "      <td>0.065710</td>\n",
       "      <td>3.351198</td>\n",
       "      <td>5.256781</td>\n",
       "      <td>7.950882</td>\n",
       "      <td>68.042465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Neuroscience</th>\n",
       "      <td>9916.0</td>\n",
       "      <td>7.643903</td>\n",
       "      <td>5.210845</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>4.172570</td>\n",
       "      <td>6.505267</td>\n",
       "      <td>9.757901</td>\n",
       "      <td>50.235118</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Paleontology</th>\n",
       "      <td>70.0</td>\n",
       "      <td>6.898118</td>\n",
       "      <td>4.395757</td>\n",
       "      <td>0.755662</td>\n",
       "      <td>4.016510</td>\n",
       "      <td>6.045299</td>\n",
       "      <td>8.148011</td>\n",
       "      <td>24.805437</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pathology</th>\n",
       "      <td>276.0</td>\n",
       "      <td>6.494673</td>\n",
       "      <td>4.224366</td>\n",
       "      <td>0.952792</td>\n",
       "      <td>3.581182</td>\n",
       "      <td>5.289636</td>\n",
       "      <td>8.328713</td>\n",
       "      <td>22.604160</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pharmacology And Toxicology</th>\n",
       "      <td>533.0</td>\n",
       "      <td>5.956473</td>\n",
       "      <td>4.589790</td>\n",
       "      <td>0.197129</td>\n",
       "      <td>3.055504</td>\n",
       "      <td>4.796813</td>\n",
       "      <td>7.359494</td>\n",
       "      <td>48.132405</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Physiology</th>\n",
       "      <td>715.0</td>\n",
       "      <td>6.459132</td>\n",
       "      <td>4.288957</td>\n",
       "      <td>0.164274</td>\n",
       "      <td>3.367626</td>\n",
       "      <td>5.519621</td>\n",
       "      <td>8.509415</td>\n",
       "      <td>26.218197</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Plant Biology</th>\n",
       "      <td>1650.0</td>\n",
       "      <td>6.319647</td>\n",
       "      <td>4.435789</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.449763</td>\n",
       "      <td>5.273209</td>\n",
       "      <td>7.885172</td>\n",
       "      <td>33.216288</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Scientific Communication And Education</th>\n",
       "      <td>316.0</td>\n",
       "      <td>8.042688</td>\n",
       "      <td>6.052775</td>\n",
       "      <td>0.394259</td>\n",
       "      <td>4.665394</td>\n",
       "      <td>6.373848</td>\n",
       "      <td>9.659336</td>\n",
       "      <td>49.775149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Synthetic Biology</th>\n",
       "      <td>514.0</td>\n",
       "      <td>6.271000</td>\n",
       "      <td>4.645690</td>\n",
       "      <td>0.032855</td>\n",
       "      <td>3.170496</td>\n",
       "      <td>5.256781</td>\n",
       "      <td>7.909813</td>\n",
       "      <td>36.666051</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Systems Biology</th>\n",
       "      <td>1427.0</td>\n",
       "      <td>8.202992</td>\n",
       "      <td>5.684800</td>\n",
       "      <td>0.065710</td>\n",
       "      <td>4.402554</td>\n",
       "      <td>6.998090</td>\n",
       "      <td>10.644982</td>\n",
       "      <td>44.748352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Zoology</th>\n",
       "      <td>278.0</td>\n",
       "      <td>6.509640</td>\n",
       "      <td>4.609388</td>\n",
       "      <td>0.098565</td>\n",
       "      <td>3.564755</td>\n",
       "      <td>5.289636</td>\n",
       "      <td>7.942668</td>\n",
       "      <td>26.382472</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         count      mean       std       min  \\\n",
       "Category                                                                       \n",
       "                                          29.0  9.804351  5.299783  0.985647   \n",
       "Animal Behavior And Cognition            856.0  7.360031  5.203498  0.164274   \n",
       "Biochemistry                            2060.0  5.388536  4.071484  0.032855   \n",
       "Bioengineering                          1188.0  6.111285  3.978310  0.032855   \n",
       "Bioinformatics                          4975.0  7.928177  5.713281  0.032855   \n",
       "Biophysics                              2728.0  5.852493  4.438419  0.032855   \n",
       "Cancer Biology                          1732.0  7.330300  4.900762  0.065710   \n",
       "Cell Biology                            2789.0  6.813330  4.719810  0.032855   \n",
       "Clinical Trials                           59.0  7.093871  4.517703  0.525678   \n",
       "Developmental Biology                   1671.0  7.068244  4.952817  0.032855   \n",
       "Ecology                                 2197.0  7.859181  5.509378  0.032855   \n",
       "Epidemiology                            1004.0  7.997972  4.717478  0.098565   \n",
       "Evolutionary Biology                    3618.0  7.690458  5.501852  0.032855   \n",
       "Genetics                                3020.0  7.250823  5.153974  0.032855   \n",
       "Genomics                                3719.0  7.702134  5.311866  0.032855   \n",
       "Immunology                              1519.0  6.246689  4.765151  0.164274   \n",
       "Microbiology                            5124.0  5.886814  4.406204  0.032855   \n",
       "Molecular Biology                       1853.0  6.290584  4.763736  0.065710   \n",
       "Neuroscience                            9916.0  7.643903  5.210845  0.032855   \n",
       "Paleontology                              70.0  6.898118  4.395757  0.755662   \n",
       "Pathology                                276.0  6.494673  4.224366  0.952792   \n",
       "Pharmacology And Toxicology              533.0  5.956473  4.589790  0.197129   \n",
       "Physiology                               715.0  6.459132  4.288957  0.164274   \n",
       "Plant Biology                           1650.0  6.319647  4.435789  0.032855   \n",
       "Scientific Communication And Education   316.0  8.042688  6.052775  0.394259   \n",
       "Synthetic Biology                        514.0  6.271000  4.645690  0.032855   \n",
       "Systems Biology                         1427.0  8.202992  5.684800  0.065710   \n",
       "Zoology                                  278.0  6.509640  4.609388  0.098565   \n",
       "\n",
       "                                             25%       50%        75%  \\\n",
       "Category                                                                \n",
       "                                        5.125362  8.903674  13.536212   \n",
       "Animal Behavior And Cognition           4.041151  6.078154   9.306146   \n",
       "Biochemistry                            2.718742  4.402554   6.833816   \n",
       "Bioengineering                          3.285488  5.355346   7.893386   \n",
       "Bioinformatics                          4.172570  6.570977  10.217869   \n",
       "Biophysics                              3.022649  4.829668   7.523768   \n",
       "Cancer Biology                          4.065792  6.275283   9.527916   \n",
       "Cell Biology                            3.646892  5.815314   8.706544   \n",
       "Clinical Trials                         3.811167  5.256781   9.495062   \n",
       "Developmental Biology                   3.876876  5.946734   8.837964   \n",
       "Ecology                                 4.238280  6.538122   9.987885   \n",
       "Epidemiology                            4.829668  6.998090  10.160373   \n",
       "Evolutionary Biology                    4.115074  6.406702   9.585412   \n",
       "Genetics                                3.745457  6.176718   9.363642   \n",
       "Genomics                                4.221853  6.505267   9.692191   \n",
       "Immunology                              3.252634  5.191072   7.967309   \n",
       "Microbiology                            2.989794  4.796813   7.490914   \n",
       "Molecular Biology                       3.351198  5.256781   7.950882   \n",
       "Neuroscience                            4.172570  6.505267   9.757901   \n",
       "Paleontology                            4.016510  6.045299   8.148011   \n",
       "Pathology                               3.581182  5.289636   8.328713   \n",
       "Pharmacology And Toxicology             3.055504  4.796813   7.359494   \n",
       "Physiology                              3.367626  5.519621   8.509415   \n",
       "Plant Biology                           3.449763  5.273209   7.885172   \n",
       "Scientific Communication And Education  4.665394  6.373848   9.659336   \n",
       "Synthetic Biology                       3.170496  5.256781   7.909813   \n",
       "Systems Biology                         4.402554  6.998090  10.644982   \n",
       "Zoology                                 3.564755  5.289636   7.942668   \n",
       "\n",
       "                                              max  \n",
       "Category                                           \n",
       "                                        20.370028  \n",
       "Animal Behavior And Cognition           55.524754  \n",
       "Biochemistry                            33.446272  \n",
       "Bioengineering                          23.195548  \n",
       "Bioinformatics                          58.514549  \n",
       "Biophysics                              55.294770  \n",
       "Cancer Biology                          40.608637  \n",
       "Cell Biology                            47.442453  \n",
       "Clinical Trials                         19.417236  \n",
       "Developmental Biology                   42.054252  \n",
       "Ecology                                 62.424280  \n",
       "Epidemiology                            29.897945  \n",
       "Evolutionary Biology                    50.826506  \n",
       "Genetics                                55.787593  \n",
       "Genomics                                42.021397  \n",
       "Immunology                              73.003552  \n",
       "Microbiology                            45.306885  \n",
       "Molecular Biology                       68.042465  \n",
       "Neuroscience                            50.235118  \n",
       "Paleontology                            24.805437  \n",
       "Pathology                               22.604160  \n",
       "Pharmacology And Toxicology             48.132405  \n",
       "Physiology                              26.218197  \n",
       "Plant Biology                           33.216288  \n",
       "Scientific Communication And Education  49.775149  \n",
       "Synthetic Biology                       36.666051  \n",
       "Systems Biology                         44.748352  \n",
       "Zoology                                 26.382472  "
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#freq count by category\n",
    "tmp = pubs_df\n",
    "tmp['Time_Month'] = pubs_df.Published_Date.subtract(pubs_df.Preprint_Date).divide(np.timedelta64(1, 'M')).where(pubs_df.Published_Date > pubs_df.Preprint_Date)\n",
    "#tmp.Time_Month = tmp.\n",
    "tmp.groupby(['Category'], as_index=True).Time_Month.describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0f0ba865-ecce-4532-a015-b19ae807f954",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Time duration by Category - Journal\n",
    "tmp.groupby(['Category','Journal'], as_index=True).Time_Month.describe().to_csv('data/Category-Journal_duration.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "b5952934-e202-4b68-ae6e-5be4b69949b2",
   "metadata": {
    "id": "dos-OlYF7NnT"
   },
   "outputs": [],
   "source": [
    "#Time duration by Journal - Category\n",
    "tmp.groupby(['Journal','Category'], as_index=True).Time_Month.describe().to_csv('data/Journal-Category_duration.csv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9cAnosLu_NSA",
   "metadata": {
    "id": "9cAnosLu_NSA"
   },
   "source": [
    "#### Check"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "yNwqBAdIwkj-",
   "metadata": {
    "id": "yNwqBAdIwkj-"
   },
   "outputs": [],
   "source": [
    "'''\n",
    "                      columns=[\"pre_DOI\", \"pub_DOI\",\n",
    "                               \"Title\", \"Authors\", \"Corresponding_Authors\",\n",
    "                               \"Institution\",\n",
    "                               \"Category\", \"Journal\", \"Preprint_Date\", \"Published_Date\"])\n",
    "'''\n",
    "#assert get_total(f'{base_url}{query_type}{server}{start_interval}{end_interval}') == pubs_df.shape[0]\n",
    "\"Total number of papers published and the length of the dataframe does match.\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "WyvJwYLNscDP",
   "metadata": {
    "id": "WyvJwYLNscDP"
   },
   "source": [
    "----"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "LZHeXoT-Km87",
   "metadata": {
    "id": "LZHeXoT-Km87"
   },
   "source": [
    "### Detailed analysis of published articles for a given timeframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "AQMsAnVcYtwk",
   "metadata": {
    "id": "AQMsAnVcYtwk"
   },
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'multipledispatch'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "\u001b[0;32m/tmp/ipykernel_2770545/2392374002.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mmultipledispatch\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mdispatch\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m@\u001b[0m\u001b[0mdispatch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mDataFrame\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;32mdef\u001b[0m \u001b[0mget_values\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdf\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mpd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mDataFrame\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrow\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcol\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0mget_values\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mrow\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mcol\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'multipledispatch'"
     ]
    }
   ],
   "source": [
    "from multipledispatch import dispatch\n",
    "@dispatch(pd.DataFrame, int, str)\n",
    "def get_values(df: pd.DataFrame, row: int, col: str):\n",
    "    return get_values(df, [row], [col])\n",
    "\n",
    "@dispatch(pd.DataFrame, int, list)\n",
    "def get_values(df: pd.DataFrame, row: int, cols: List[str]):\n",
    "    return get_values(df, [row], cols)\n",
    "\n",
    "@dispatch(pd.DataFrame, list, str)\n",
    "def get_values(df: pd.DataFrame, rows: List[int], col: str):\n",
    "    return get_values(df, rows, [col])\n",
    "\n",
    "@dispatch(pd.DataFrame, list, list)\n",
    "def get_values(df: pd.DataFrame, rows: List[int], cols: List[str]) -> List[List[str]]:\n",
    "    return [[df.at(row, col) for col in cols] for row in rows]\n",
    "\n",
    "@dispatch(pd.DataFrame, str)\n",
    "def get_values(df: pd.DataFrame, col: str):\n",
    "    return df[col]\n",
    "\n",
    "@dispatch(pd.DataFrame, list)\n",
    "def get_values(df: pd.DataFrame, cols: List[str]):\n",
    "    return df[cols]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "pFH0AzJ1LJms",
   "metadata": {
    "id": "pFH0AzJ1LJms"
   },
   "source": [
    "                                                                                                                                                                          ### BioRvix Service"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "w_Tv0XvnKybY",
   "metadata": {
    "colab": {
     "background_save": true
    },
    "id": "w_Tv0XvnKybY"
   },
   "outputs": [],
   "source": [
    "#from tqdm import tqdm_notebook, tnrange\n",
    "from tqdm.notebook import tqdm_notebook\n",
    "import time\n",
    "import math\n",
    "\n",
    "#variables\n",
    "#https://api.biorxiv.org/pubs/biorvix/2018-08-21/2018-08-28\n",
    "#https://api.biorxiv.org/details/biorxiv/10.1101/759530\n",
    "#https://api.biorxiv.org/details/biorvix/10.1101/759530\n",
    "base_url: str = 'https://api.biorxiv.org'\n",
    "query_type: str = 'details'\n",
    "server: str ='biorxiv'\n",
    "url: str = f'{base_url}/{query_type}/{server}'\n",
    "path: str = f'data/prepub-{query_type}-{server}-{datetime.now()}'\n",
    "step = 100\n",
    "#doi = ['10.1101/856302']\n",
    "doi = pubs_df.DOI\n",
    "prepub_df = None\n",
    "\n",
    "'''\n",
    "for item in tqdm_notebook(range(0, total, step), desc='get_prepublish_data', \n",
    "                 total=math.ceil(total/step)):\n",
    "    #time.sleep(0.005)\n",
    "    results = get_data(None, \n",
    "                       url, \n",
    "                       doi[item:item+step])\n",
    "    tmp = list(results)\n",
    "    prepub_df = query_to_df([r for r in tmp], \n",
    "                            [\"doi\", \"title\", \"authors\", \"author_corresponding\", \"author_corresponding_institution\", \"date\", \"version\", \"type\", \"category\", \"published\"],\n",
    "                            range(item, item + (len(tmp) * step), step),\n",
    "#                            range(item, (item + ((len(tmp[counter]) - 1) * step)) * 10, step),\n",
    "                            [\"DOI\", \"Title\", \"Authors\", \"Corresponding_Authors\", \"Institution\", \"Date\", \"Version\", \"Type\", \"Category\", \"pub_DOI\"], False)\n",
    "    #prepub_df = create_prepublish_df(prepub_df)\n",
    "    prepub_df.to_parquet(pathlib.Path(f'{path}/{datetime.utcnow().timestamp()}.parquet'))\n",
    "'''\n",
    "\n",
    "#create directory\n",
    "pathlib.Path(path).mkdir(parents=True, exist_ok=True)\n",
    "total = len(doi)\n",
    "args = [(path, url, doi, item, step, True) for item in range(0, total, step)]\n",
    "#print(f'args: {len(args)}\\n{args}')\n",
    "print(url, f'total doi: {total} iter: {total/step}')\n",
    "#tq.thread_map(lambda p: process_doi_data(*p), args, desc='process_doi_data', max_workers=5, total=len(args))\n",
    "tq.thread_map(lambda p: process_doi_data(*p), args, desc='process_doi_data', total=len(args))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "aV_7o_yC4V5U",
   "metadata": {
    "id": "aV_7o_yC4V5U"
   },
   "outputs": [],
   "source": [
    "prepub_df = create_prepublish_df(pd.read_parquet(pathlib.Path(path)))\n",
    "\n",
    "prepub_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "eucJeZDX55IZ",
   "metadata": {
    "id": "eucJeZDX55IZ"
   },
   "outputs": [],
   "source": [
    "print('Published data:\\n',[(name, pubs_df[name].isnull().sum()) for name in pubs_df.columns if pubs_df[name].isnull().values.any()])\n",
    "print('Prepublish data:\\n',[(name, prepub_df[name].isnull().sum()) for name in prepub_df.columns if prepub_df[name].isnull().values.any()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "pnrO6Q_P3JDe",
   "metadata": {
    "id": "pnrO6Q_P3JDe"
   },
   "outputs": [],
   "source": [
    "combined_df = pd.merge(pubs_df,prepub_df, how=\"outer\", on=['DOI','Title', 'Authors', 'Corresponding_Authors', 'Institution', 'Category', 'Num_of_Authors', 'pub_DOI'])\n",
    "#combined_df = pd.merge(pubs_df,prepub_df, how=\"right\", on=['DOI','Title', 'Authors', 'Corresponding_Authors', 'Institution', 'Category', 'Num_of_Authors'])\n",
    "combined_df['Time_month'] = (combined_df['Published_Date'] - combined_df['Preprint_Date']) / np.timedelta64(1, 'M')\n",
    "combined_df['Keep'] = combined_df.Published_Date > combined_df.Preprint_Date\n",
    "#ref = combined_df.set_index(['pub_DOI'], inplace=False)\n",
    "combined_df.head(1)\n",
    "#display(combined_df.groupby('DOI').count())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "VTLoIyweE6F_",
   "metadata": {
    "id": "VTLoIyweE6F_"
   },
   "outputs": [],
   "source": [
    "print([(index, value) for index, value in enumerate(combined_df.groupby(['DOI'])['Time_month']) ])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b7xApG-iAst7",
   "metadata": {
    "id": "b7xApG-iAst7"
   },
   "outputs": [],
   "source": [
    "# checking for NA\n",
    "print([(name, combined_df[name].isnull().sum()) for name in combined_df.columns if combined_df[name].isnull().values.any()])\n",
    "print([(name, combined_df.groupby([name], dropna=False).size) for name in combined_df.columns if (combined_df.groupby([name], dropna=False)[name].count() > 0).any()])\n",
    "print([(name, combined_df.groupby([name], dropna=True).size) for name in combined_df.columns if (combined_df.groupby([name], dropna=False)[name].count() > 0).any()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "C_lYaBefgFPW",
   "metadata": {
    "id": "C_lYaBefgFPW"
   },
   "outputs": [],
   "source": [
    "#%%capture cap\n",
    "display(prepub_df.shape, pubs_df.shape, combined_df.shape, combined_df.groupby(['pub_DOI'], as_index=True).count().shape)\n",
    "#test = combined_df.groupby(['DOI', 'pub_DOI', 'Category', 'Institution', 'Corresponding_Authors', 'Journal', 'Preprint_Date', 'Published_Date', 'Authors', 'Num_of_Authors', 'Title', 'Date', 'Version'])\n",
    "test = combined_df.groupby(['DOI'], as_index = True)\n",
    "#display(test.iloc[test.Title.nunique().ne(1),:])\n",
    "#display(test.Title.nunique().ne(1).index)\n",
    "#display(test.Title.nunique().ne(1))\n",
    "#key = test.Title.nunique().ne(1)\n",
    "#display(ref.index.isin(key))\n",
    "#display(ref[ref.index.isin(key)])\n",
    "#display(test.Title.nunique().ne(1).shape)\n",
    "#test.Title.indices.\n",
    "#display(test.Title.indices)\n",
    "#df.at only get signal value so need to use list comprehension to get multiple columns in a particular row\n",
    "#print(test.Title.indices['10.1128/IAI.00353-19'])\n",
    "#display(combined_df .iloc[[1826, 1827]])\n",
    "#display(combined_df)\n",
    "combined_df.groupby(['pub_DOI'], as_index=False).Title.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "rQAZIomEWwYl",
   "metadata": {
    "id": "rQAZIomEWwYl"
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "iPFzy5JTitmc",
   "metadata": {
    "id": "iPFzy5JTitmc"
   },
   "outputs": [],
   "source": [
    "combined_df.groupby(['Category', 'Keep'], as_index=True).Time_month.describe()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "DyuqmfoqFL9v",
   "metadata": {
    "id": "DyuqmfoqFL9v"
   },
   "source": [
    "**NB:** There are some issues with the above table since there are 29 publications that have more than one enteries."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "mI2n4-JGDcEh",
   "metadata": {
    "id": "mI2n4-JGDcEh"
   },
   "outputs": [],
   "source": [
    "df = combined_df.loc[combined_df.reset_index().groupby(['pub_DOI'])['Time_month'].idxmax()]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f0rK3WF8bVfi",
   "metadata": {
    "id": "f0rK3WF8bVfi"
   },
   "outputs": [],
   "source": [
    "display(df.groupby(['Category']).Time_month.describe())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "RznQ2U0LrwCn",
   "metadata": {
    "id": "RznQ2U0LrwCn"
   },
   "outputs": [],
   "source": [
    "combined_df.groupby(['Institution'], as_index=True).pub_DOI.describe().to_csv(\"data/general.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "VnD7vbrqrYae",
   "metadata": {
    "id": "VnD7vbrqrYae"
   },
   "outputs": [],
   "source": [
    "combined_df.groupby(['Institution', 'Category'], as_index=True).Time_month.describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "exkYqa1ZVb1w",
   "metadata": {
    "id": "exkYqa1ZVb1w"
   },
   "outputs": [],
   "source": [
    "reduce_df = combined_df[combined_df.Keep == True]\n",
    "reduce_df.groupby(['Category', 'Keep'], as_index=True).Time_month.describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "Whgfy1V8rJVV",
   "metadata": {
    "id": "Whgfy1V8rJVV"
   },
   "outputs": [],
   "source": [
    "print(pubs_df.shape)\n",
    "print(prepub_df.shape)\n",
    "print(test.count().shape)\n",
    "print((test.Title.nunique() - 1 ).shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4NjU5tNt_XgO",
   "metadata": {
    "id": "4NjU5tNt_XgO"
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "hB2MtvlYa0qF",
   "metadata": {
    "id": "hB2MtvlYa0qF"
   },
   "outputs": [],
   "source": [
    "test.filter(lambda x: x['Title'].nunique() > 1).to_csv('data/title-change.csv')\n",
    "prepub_df.to_csv('data/pre&pub-data.csv')\n",
    "pubs_df.to_csv('data/pub_data.csv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2d11227e",
   "metadata": {
    "id": "2d11227e"
   },
   "source": [
    "# Junk Code\n",
    "Please disregard the codes below."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6j7ohr_84AdV",
   "metadata": {
    "id": "6j7ohr_84AdV"
   },
   "outputs": [],
   "source": [
    "'''\n",
    "for item in tqdm_notebook(range(0, total, step), desc='get_prepublish_data', \n",
    "                 total=math.ceil(total/step)):\n",
    "    #time.sleep(0.005)\n",
    "    results = get_data(None, \n",
    "                       url, \n",
    "                       doi[item:item+step])\n",
    "    tmp = list(results)\n",
    "    prepub_df = query_to_df([r for r in tmp], \n",
    "                            [\"doi\", \"title\", \"authors\", \"author_corresponding\", \"author_corresponding_institution\", \"date\", \"version\", \"type\", \"category\", \"published\"],\n",
    "                            range(item, item + (len(tmp) * step), step),\n",
    "#                            range(item, (item + ((len(tmp[counter]) - 1) * step)) * 10, step),\n",
    "                            [\"DOI\", \"Title\", \"Authors\", \"Corresponding_Authors\", \"Institution\", \"Date\", \"Version\", \"Type\", \"Category\", \"pub_DOI\"], False)\n",
    "    #prepub_df = create_prepublish_df(prepub_df)\n",
    "    prepub_df.to_parquet(pathlib.Path(f'{path}/{datetime.utcnow().timestamp()}.parquet'))\n",
    "'''\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "Raiv_IEv2gJk",
   "metadata": {
    "id": "Raiv_IEv2gJk"
   },
   "outputs": [],
   "source": [
    "# \"local\" variables\n",
    "url: str = f'{base_url}/{query_type}/{server}/{start_interval}/{end_interval}'\n",
    "path: str = f'pub-{query_type}-{server}-{start_interval}!{end_interval}-{datetime.datetime.now()}'\n",
    "step = 100\n",
    "pubs_df = None\n",
    "\n",
    "#finding the number of records for a given time period\n",
    "\n",
    "pathlib.Path(path).mkdir(parents=True, exist_ok=True)\n",
    "print(url)\n",
    "results = get_data(None, \n",
    "         url, \n",
    "         range(0, get_total(url), 100))\n",
    "\n",
    "tmp = list(results)\n",
    "for item in range (0, get_total(url), step):\n",
    "    result_list = tmp[item:(item+step)]\n",
    "    pubs_df = query_to_df(result_list, \n",
    "                          [\"preprint_doi\", \"published_doi\", \"preprint_title\", \"preprint_authors\", \"preprint_author_corresponding\", \"preprint_author_corresponding_institution\", \"preprint_category\", \"published_journal\", \"preprint_date\", \"published_date\"],\n",
    "                          range(item, len(result_list) * step, step),\n",
    "                          [\"DOI\", \"pub_DOI\", \"Title\", \"Authors\", \"Corresponding_Authors\", \"Institution\", \"Category\", \"Journal\", \"Preprint_Date\", \"Published_Date\"])\n",
    "    pubs_df.to_parquet(pathlib.Path(f'{path}/{datetime.datetime.utcnow().timestamp()}.parquet'))\n",
    "pubs_df = pd.read_parquet(pathlib.Path(path))\n",
    "pubs_df = create_published_df(pubs_df)\n",
    "\n",
    "display(pubs_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "Hw_gzl2w9Oga",
   "metadata": {
    "id": "Hw_gzl2w9Oga"
   },
   "outputs": [],
   "source": [
    "def get_publisher_prefix(doi: pd.Series) -> pd.Series:\n",
    "    return doi.apply(lambda x: x.split(\"/\")[0])\n",
    "\n",
    "query_type='/publisher'\n",
    "\n",
    "def get_publisher_data(url)\n",
    "get_data(None,f'{base_url}{query_type}'\n",
    "         get_publisher_prefix(pubs_df.pub_DOI).unique())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a01c1Bgp8HK9",
   "metadata": {
    "id": "a01c1Bgp8HK9"
   },
   "source": [
    "**Journal API**<br>\n",
    "https://www.nature.com/opensearch/<br>\n",
    "https://www.biorxiv.org/content/10.1101/339747v4<br>\n",
    "https://www.biorxiv.org/content/10.1101/339747v4.full.pdf<br>\n",
    "https://api.biorxiv.org/details/biorxiv/10.1101/099697\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "77a59f13",
   "metadata": {
    "id": "77a59f13"
   },
   "outputs": [],
   "source": [
    "journal_list = []\n",
    "\n",
    "# `license`,`abstract`, and `server` are excluded from the metrics. \n",
    "for journal in json_info[\"collection\"]:\n",
    "    journal_list.append([journal[\"doi\"], journal[\"title\"], journal[\"authors\"],\n",
    "                         journal[\"author_corresponding\"], \n",
    "                         journal[\"author_corresponding_institution\"],\n",
    "                         journal[\"date\"], journal[\"version\"], journal[\"type\"],\n",
    "                         journal[\"category\"], journal[\"jatsxml\"], journal[\"published\"]])\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "27bbfb07",
   "metadata": {
    "id": "27bbfb07"
   },
   "outputs": [],
   "source": [
    "journal_df = pd.DataFrame(data=journal_list,\n",
    "                         columns=[\"DOI\", \"Title\", \"Authors\", \"Corresponding Authors\",\n",
    "                                  \"Institution\",\n",
    "                                  \"Date\", \"Version\", \"Type\", \"Category\", \"Xml\", \"Published\"])\n",
    "journal_df.head()"
   ]
  }
 ],
 "metadata": {
  "colab": {
   "collapsed_sections": [
    "3E24xcwG1kXQ",
    "6dPz3eLWwpri"
   ],
   "include_colab_link": true,
   "machine_shape": "hm",
   "provenance": []
  },
  "gpuClass": "standard",
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  },
  "widgets": {
   "application/vnd.jupyter.widget-state+json": {
    "01b6eddd2404428bb81a4547d60444b3": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "1bbfab880f2149e685d407db49da94a1": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "49032de80ac44169b9a8337bed34ead2": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "4edf0b643db94dfeae6db7b8a0519cdb": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_7c45fe41b4dd4db7a0a1c964f8c68e7c",
       "IPY_MODEL_af6fac0308a34252af8aa82673b98406",
       "IPY_MODEL_5993c8519cc3465eae2f87b2318414c4"
      ],
      "layout": "IPY_MODEL_86847fee2c014f76a008a321f5209a4a"
     }
    },
    "543c5ba3a78946f58b91fe5bd0e7f1d1": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "5993c8519cc3465eae2f87b2318414c4": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_1bbfab880f2149e685d407db49da94a1",
      "placeholder": "​",
      "style": "IPY_MODEL_e4a56b5756c94cc68e6b4f9ba88f66e8",
      "value": " 1481/1481 [08:28&lt;00:00,  2.07it/s]"
     }
    },
    "7c45fe41b4dd4db7a0a1c964f8c68e7c": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_01b6eddd2404428bb81a4547d60444b3",
      "placeholder": "​",
      "style": "IPY_MODEL_aa058be5dc914ec28a81bff85de7168e",
      "value": "get_big_data: 100%"
     }
    },
    "86847fee2c014f76a008a321f5209a4a": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "aa058be5dc914ec28a81bff85de7168e": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "af6fac0308a34252af8aa82673b98406": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_543c5ba3a78946f58b91fe5bd0e7f1d1",
      "max": 1481,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_49032de80ac44169b9a8337bed34ead2",
      "value": 1481
     }
    },
    "e4a56b5756c94cc68e6b4f9ba88f66e8": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    }
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
