{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "309111f4",
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "from urllib.request import Request, urlopen\n",
    "from bs4 import BeautifulSoup\n",
    "for cursor in range(10):\n",
    "    url = f\"https://api.biorxiv.org/pubs/biorxiv/2000-03-01/2020-03-30/{cursor*100+1}\"\n",
    "    req = Request(url, headers={'User-Agent': 'Mozilla/5.0'})\n",
    "    webpage = urlopen(req).read()\n",
    "    soup = BeautifulSoup(webpage)\n",
    "    with open(f\"data-{cursor}.json\",'w',encoding='utf-8') as f:\n",
    "        f.write(soup.text)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "9d384d0e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<html>\\n <body>\\n  <p>\\n   {\"messages\":[{\"status\":\"ok\",\"interval\":\"2000-03-01:2020-03-30\",\"cursor\":\"1\",\"count\":100,\"total\":38531}], \"collection\":[{\"preprint_doi\":\"10.1101\\\\/001297\",\"published_doi\":\"10.1371\\\\/journal.pone.0085203\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Aerodynamic characteristics of a feathered dinosaur measured using physical models. Effects of form on static stability and control effectiveness.\",\"preprint_authors\":\"Dennis Evangelista;Griselda Cardona;Eric Guenther-Gleason;Tony Huynh;Austin Kwong;Dylan Marks;Neil Ray;Adrian Tisbe;Kyle Tse;Mimi Kohl;\",\"preprint_category\":\"Biophysics \",\"preprint_date\":\"2013-12-10\",\"published_date\":\"2014-01-15\",\"preprint_abstract\":\"We report the effects of posture and morphology on the static aerodynamic stability and control effectiveness of physical models based on the feathered dinosaur, {dagger}Microraptor gui, from the Cretaceous of China. Postures had similar lift and drag coefficients and were broadly similar when simplified metrics of gliding were considered, but they exhibited different stability characteristics depending on the position of the legs and the presence of feathers on the legs and the tail. Both stability and the function of appendages in generating maneuvering forces and torques changed as the glide angle or angle of attack were changed. These are significant because they represent an aerial environment that may have shifted during the evolution of directed aerial descent and other aerial behaviors. Certain movements were particularly effective (symmetric movements of the wings and tail in pitch, asymmetric wing movements, some tail movements). Other appendages altered their function from creating yaws at high angle of attack to rolls at low angle of attack, or reversed their function entirely. While {dagger}M. gui lived after {dagger}Archaeopteryx and likely represents a side experiment with feathered morphology, the general patterns of stability and control effectiveness suggested from the manipulations of forelimb, hindlimb and tail morphology here may help understand the evolution of flight control aerodynamics in vertebrates. Though these results rest on a single specimen, as further fossils with different morphologies tested, the findings here could be applied in a phylogenetic context to reveal biomechanical constraints on extinct flyers arising from the need to maneuver.\",\"preprint_author_corresponding\":\"Dennis  Evangelista\",\"preprint_author_corresponding_institution\":\"UC Berkeley\"},{\"preprint_doi\":\"10.1101\\\\/000422\",\"published_doi\":\"10.3389\\\\/fgene.2014.00013\",\"published_journal\":\"Frontiers in Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"On the optimal trimming of high-throughput mRNAseq data\",\"preprint_authors\":\"Matthew D MacManes;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2013-11-14\",\"published_date\":\"2014-01-31\",\"preprint_abstract\":\"The widespread and rapid adoption of high-throughput sequencing technologies has afforded researchers the opportunity to gain a deep understanding of genome level processes that underlie evolutionary change, and perhaps more importantly, the links between genotype and phenotype. In particular, researchers interested in functional biology and adaptation have used these technologies to sequence mRNA transcriptomes of specific tissues, which in turn are often compared to other tissues, or other individuals with different phenotypes. While these techniques are extremely powerful, careful attention to data quality is required. In particular, because high-throughput sequencing is more error-prone than traditional Sanger sequencing, quality trimming of sequence reads should be an important step in all data processing pipelines. While several software packages for quality trimming exist, no general guidelines for the specifics of trimming have been developed. Here, using empirically derived sequence data, I provide general recommendations regarding the optimal strength of trimming, specifically in mRNA-Seq studies. Although very aggressive quality trimming is common, this study suggests that a more gentle trimming, specifically of those nucleotides whose PO_SCPLOWHREDC_SCPLOW score &lt;2 or &lt;5, is optimal for most studies across a wide variety of metrics.\",\"preprint_author_corresponding\":\"Matthew D MacManes\",\"preprint_author_corresponding_institution\":\"University of New Hampshire\"},{\"preprint_doi\":\"10.1101\\\\/001396\",\"published_doi\":\"10.1162\\\\/NECO_a_00568\",\"published_journal\":\"Neural Computation\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Parametric inference in the large data limit using maximally informative models\",\"preprint_authors\":\"Justin B. Kinney;Gurinder S. Atwal;\",\"preprint_category\":\"Biophysics \",\"preprint_date\":\"2013-12-13\",\"published_date\":\"2014-03-10\",\"preprint_abstract\":\"Motivated by data-rich experiments in transcriptional regulation and sensory neuro-science, we consider the following general problem in statistical inference. When exposed to a high-dimensional signal S, a system of interest computes a representation R of that signal which is then observed through a noisy measurement M. From a large number of signals and measurements, we wish to infer the \\\\\\\\\\\\\"filter\\\\\\\\\\\\\" that maps S to R. However, the standard method for solving such problems, likelihood-based inference, requires perfect a priori knowledge of the \\\\\\\\\\\\\"noise function\\\\\\\\\\\\\" mapping R to M. In practice such noise functions are usually known only approximately, if at all, and using an incorrect noise function will typically bias the inferred filter. Here we show that, in the large data limit, this need for a pre-characterized noise function can be circumvented by searching for filters that instead maximize the mutual information I[M; R] between observed measurements and predicted representations. Moreover, if the correct filter lies within the space of filters being explored, maximizing mutual information becomes equivalent to simultaneously maximizing every dependence measure that satisfies the Data Processing Inequality. It is important to note that maximizing mutual information will typically leave a small number of directions in parameter space unconstrained. We term these directions \\\\\\\\\\\\\"diffeomorphic modes\\\\\\\\\\\\\" and present an equation that allows these modes to be derived systematically. The presence of diffeomorphic modes reflects a fundamental and nontrivial substructure within parameter space, one that is obscured by standard likelihood-based inference.\",\"preprint_author_corresponding\":\"Justin B. Kinney\",\"preprint_author_corresponding_institution\":\"Cold Spring Harbor Laboratory\"},{\"preprint_doi\":\"10.1101\\\\/002980\",\"published_doi\":\"10.1016\\\\/j.bpj.2014.01.012\",\"published_journal\":\"Biophysical Journal\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Genetic drift suppresses bacterial conjugation in spatially structured populations\",\"preprint_authors\":\"Peter D. Freese;Kirill S. Korolev;Jose I Jimenez;Irene A. Chen;\",\"preprint_category\":\"Biophysics \",\"preprint_date\":\"2014-02-24\",\"published_date\":\"2014-02-18\",\"preprint_abstract\":\"Conjugation is the primary mechanism of horizontal gene transfer that spreads antibiotic resistance among bacteria. Although conjugation normally occurs in surface-associated growth (e.g., biofilms), it has been traditionally studied in well-mixed liquid cultures lacking spatial structure, which is known to affect many evolutionary and ecological processes. Here we visualize spatial patterns of gene transfer mediated by F plasmid conjugation in a colony of Escherichia coli growing on solid agar, and we develop a quantitative understanding by spatial extension of traditional mass-action models. We found that spatial structure suppresses conjugation in surface-associated growth because strong genetic drift leads to spatial isolation of donor and recipient cells, restricting conjugation to rare boundaries between donor and recipient strains. These results suggest that ecological strategies, such as enforcement of spatial structure and enhancement of genetic drift, could complement molecular strategies in slowing the spread of antibiotic resistance genes.\",\"preprint_author_corresponding\":\"Irene A. Chen\",\"preprint_author_corresponding_institution\":\"Univ. of California - Santa Barbara\"},{\"preprint_doi\":\"10.1101\\\\/001693\",\"published_doi\":\"10.1103\\\\/PhysRevLett.112.068101\",\"published_journal\":\"Physical Review Letters\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A statistical mechanics model for the collective epigenetic histone modification dynamics\",\"preprint_authors\":\"Hang Zhang;XIAO-JUN TIAN;Abhishek Mukhopadhyay;Kenneth S Kim;Jianhua Xing;\",\"preprint_category\":\"Biophysics \",\"preprint_date\":\"2014-01-08\",\"published_date\":\"2014-02-10\",\"preprint_abstract\":\"Epigenetic histone modifications play an important role in the maintenance of different cell phenotypes. The exact molecular mechanism for inheritance of the modification patterns over cell generations remains elusive. We construct a Potts-type model based on experimentally observed nearest-neighbor enzyme lateral interactions and nucleosome covalent modification state biased enzyme recruitment. The model can lead to effective nonlocal interactions among nucleosomes suggested in previous theoretical studies, and epigenetic memory is robustly inheritable against stochastic cellular processes.\\\\\\\\n\\\\\\\\nPACS numbers: 82.39.Rt, 87.17.Aa, 87.16.Yc, 87.16.A\",\"preprint_author_corresponding\":\"Jianhua  Xing\",\"preprint_author_corresponding_institution\":\"Virginia Tech\"},{\"preprint_doi\":\"10.1101\\\\/000141\",\"published_doi\":\"10.1371\\\\/journal.pcbi.1003433\",\"published_journal\":\"PLOS Computational Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Microenvironmental variables must influence intrinsic phenotypic parameters of cancer stem cells to affect tumourigenicity\",\"preprint_authors\":\"Jacob Scott;Anita Hjelmeland;Prakash Chinnaiyan;Alexander R A Anderson;David Basanta;\",\"preprint_category\":\"Cancer Biology \",\"preprint_date\":\"2013-11-07\",\"published_date\":\"2014-01-16\",\"preprint_abstract\":\"Since the discovery of tumour initiating cells (TICs) in solid tumours, studies focussing on their role in cancer initiation and progression have abounded. The biological interrogation of these cells continues to yield volumes of information on their pro-tumourigenic behaviour, but actionable generalised conclusions have been scarce. Further, new information suggesting a dependence of tumour composition and growth on the microenvironment has yet to be studied theoretically. To address this point, we created a hybrid, discrete\\\\/continuous computational cellular automaton model of a generalised stem-cell driven tissue with a simple microenvironment. Using the model we explored the phenotypic traits inherent to the tumour initiating cells and the effect of the microenvironment on tissue growth. We identify the regions in phenotype parameter space where TICs are able to cause a disruption in homeostasis, leading to tissue overgrowth and tumour maintenance. As our parameters and model are non-specific, they could apply to any tissue TIC and do not assume specific genetic mutations. Targeting these phenotypic traits could represent a generalizable therapeutic strategy across cancer types. Further, we find that the microenvironmental variable does not strongly effect the outcomes, suggesting a need for direct feedback from the microenvironment onto stem-cell behaviour in future modelling endeavours.\\\\\\\\n\\\\\\\\nAuthor SummaryIn this paper, we present a mathematical\\\\/computational model of a tumour growing according to the canonical cancer stem-cell hypothesis with a simplified microenvironment. We explore the parameters of this model and find good agreement between our model and other theoretical models in terms of the intrinsic cellular parameters, which are difficult to study biologically. We find, however, disagreement between novel biological data and our model in terms of the microenvironmental changes. We conclude that future theoretical models of stem-cell driven tumours must include specific feedback from the microenvironment onto the individual cellular behavior. Further, we identify several cell intrinsic parameters which govern loss of homeostasis into a state of uncontrolled growth.\",\"preprint_author_corresponding\":\"Jacob  Scott\",\"preprint_author_corresponding_institution\":\"H. Lee Moffitt Cancer Center and Research Institute and University of Oxford\"},{\"preprint_doi\":\"10.1101\\\\/001602\",\"published_doi\":\"10.7554\\\\/eLife.01856.001\",\"published_journal\":\"eLife\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"p97-dependent retrotranslocation and proteolytic processing govern formation of active Nrf1 upon proteasome inhibition\",\"preprint_authors\":\"Senthil K Radhakrishnan;Willem den Besten;Raymond J Deshaies;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2013-12-29\",\"published_date\":\"2014-01-21\",\"preprint_abstract\":\"Proteasome inhibition elicits an evolutionarily conserved response wherein proteasome subunit mRNAs are upregulated, resulting in recovery of proteasome activity. We previously demonstrated that the transcription factor Nrf1 mediates this homeostatic response in mammalian cells. We show here that Nrf1 is initially translocated into the lumen of the ER, but is rapidly and efficiently retrotranslocated to the cytosolic side of the membrane in a manner that depends on p97\\\\/VCP. Normally, retrotranslocated Nrf1 is degraded by the proteasome and active species do not accumulate. However, in cells with compromised proteasomes, retrotranslocated Nrf1 escapes degradation and is cleaved N-terminal to Leu-104 to yield a fragment that is no longer tethered to the ER-membrane. Importantly, this cleavage event is essential for Nrf1-dependent activation of proteasome gene expression upon proteasome inhibition. Our data uncover an unexpected role for p97 in activation of a transcription factor by relocalizing it from the ER lumen to the cytosol.\",\"preprint_author_corresponding\":\"Raymond J Deshaies\",\"preprint_author_corresponding_institution\":\"California Institute of Technology\"},{\"preprint_doi\":\"10.1101\\\\/001701\",\"published_doi\":\"10.7554\\\\/eLife.00829\",\"published_journal\":\"eLife\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"An unmet actin requirement explains the mitotic inhibition of clathrin-mediated endocytosis\",\"preprint_authors\":\"Satdip Kaur;Andrew B Fielding;Gisela Gassner;Nicholas J Carter;Stephen J Royle;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-01-09\",\"published_date\":\"2014-02-18\",\"preprint_abstract\":\"Clathrin-mediated endocytosis (CME) is the major internalisation route for many different receptor types in mammalian cells. CME is shut down during early mitosis, but the mechanism of this inhibition is unclear. Here we show that the mitotic shutdown is due to an unmet requirement for actin in CME. In mitotic cells, membrane tension is increased and this invokes a requirement for the actin cytoskeleton to assist the CME machinery to overcome the increased load. However, the actin cytoskeleton is engaged in the formation of a rigid cortex in mitotic cells and is therefore unavailable for deployment. We demonstrate that CME can be \\\\\\\\\\\\\"restarted\\\\\\\\\\\\\" in mitotic cells despite high membrane tension, by allowing actin to engage in endocytosis. Mitotic phosphorylation of endocytic proteins is maintained in mitotic cells with restored CME, indicating that direct phosphorylation of the CME machinery does not account for shutdown.\",\"preprint_author_corresponding\":\"Stephen J Royle\",\"preprint_author_corresponding_institution\":\"University of Warwick\"},{\"preprint_doi\":\"10.1101\\\\/001842\",\"published_doi\":\"10.1098\\\\/rspb.2013.3127\",\"published_journal\":\"Proceedings B\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"The\\\\u00a0emergence\\\\u00a0of\\\\u00a0the\\\\u00a0rescue\\\\u00a0effect\\\\u00a0from\\\\u00a0explicit\\\\u00a0within- and\\\\u00a0between-patch\\\\u00a0dynamics\\\\u00a0in\\\\u00a0a metapopulation\",\"preprint_authors\":\"Anders Eriksson;Federico El\\\\u00edas-Wolff;Bernhard Mehlig;Andrea Manica;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2014-01-15\",\"published_date\":\"2014-04-07\",\"preprint_abstract\":\"Immigration can rescue local populations from extinction, helping to stabilise a metapopulation. Local population dynamics is important for determining the strength of this rescue effect, but the mechanistic link between local demographic parameters and the rescue effect at the metapopulation level has received very little attention by modellers. We develop an analytical framework that allows us to describe the emergence of the rescue effect from interacting local stochastic dynamics. We show this framework to be applicable to a wide range of spatial scales, providing a powerful and convenient alternative to individual-based models for making predictions concerning the fate of metapopulations. We show that the rescue effect plays an important role in minimising the increase in local extinction probability associated with high demographic stochasticity, but its role is more limited in the case of high local environmental stochasticity of recruitment or survival. While most models postulate the rescue effect, our framework provides an explicit mechanistic link between local dynamics and the emergence of the rescue effect, and more generally the stability of the whole metapopulation.\",\"preprint_author_corresponding\":\"Anders  Eriksson\",\"preprint_author_corresponding_institution\":\"University of Cambridge\"},{\"preprint_doi\":\"10.1101\\\\/003012\",\"published_doi\":\"10.1073\\\\/pnas.1313285111\",\"published_journal\":\"Proceedings of the National Academy of Sciences\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Genetic drift opposes mutualism during spatial population expansion\",\"preprint_authors\":\"Melanie JI Mueller;Beverly I Neugeboren;David R Nelson;Andrew W Murray;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-02-25\",\"published_date\":\"2014-01-06\",\"preprint_abstract\":\"Mutualistic interactions benefit both partners, promoting coexistence and genetic diversity. Spatial structure can promote cooperation, but spatial expansions may also make it hard for mutualistic partners to stay together, since genetic drift at the expansion front creates regions of low genetic and species diversity. To explore the antagonism between mutualism and genetic drift, we grew cross-feeding strains of the budding yeast S. cerevisiae on agar surfaces as a model for mutualists undergoing spatial expansions. By supplying varying amounts of the exchanged nutrients, we tuned strength and symmetry of the mutualistic interaction. Strong mutualism suppresses genetic demixing during spatial expansions and thereby maintains diversity, but weak or asymmetric mutualism is overwhelmed by genetic drift even when mutualism is still beneficial, slowing growth and reducing diversity. Theoretical modeling using experimentally measured parameters predicts the size of demixed regions and how strong mutualism must be to survive a spatial expansion.\",\"preprint_author_corresponding\":\"Melanie JI Mueller\",\"preprint_author_corresponding_institution\":\"Harvard University\"},{\"preprint_doi\":\"10.1101\\\\/000315\",\"published_doi\":\"10.1371\\\\/journal.pone.0087782\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"On the Reproducibility of TCGA Ovarian Cancer MicroRNA Profiles\",\"preprint_authors\":\"Ying-Wooi Wan;Claire Mach;Genevera I. Allen;Matthew Anderson;Zhandong Liu;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2013-11-13\",\"published_date\":\"2014-01-29\",\"preprint_abstract\":\"Dysregulated microRNA (miRNA) expression is a well-established feature of human cancer. However, the role of specific miRNAs in determining cancer outcomes remains unclear. Using Level 3 expression data from the Cancer Genome Atlas (TCGA), we identified 61 miRNAs that are associated with overall survival in 469 ovarian cancers profiled by microarray (p&lt;0.01). We also identified 12 miRNAs that are associated with survival when miRNAs were profiled in the same specimens using Next Generation Sequencing (miRNA-Seq) (p&lt;0.01). Surprisingly, only 1 miRNA transcript is associated with ovarian cancer survival in both datasets. Our analyses indicate that this discrepancy is due to the fact that miRNA levels reported by the two platforms correlate poorly, even after correcting for potential issues inherent to signal detection algorithms. Further investigation is warranted.\",\"preprint_author_corresponding\":\"Zhandong  Liu\",\"preprint_author_corresponding_institution\":\"Baylor College of Medicine\"},{\"preprint_doi\":\"10.1101\\\\/000067\",\"published_doi\":\"10.1038\\\\/nature12904\",\"published_journal\":\"Nature\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Genetics of single-cell protein abundance variation in large yeast populations\",\"preprint_authors\":\"Frank Albert;Sebastian Treusch;Arthur H Shockley;Joshua S Bloom;Leonid Kruglyak;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2013-11-07\",\"published_date\":\"2014-02-27\",\"preprint_abstract\":\"Many DNA sequence variants influence phenotypes by altering gene expression. Our understanding of these variants is limited by sample sizes of current studies and by measurements of mRNA rather than protein abundance. We developed a powerful method for identifying genetic loci that influence protein expression in very large populations of the yeast Saccharomyes cerevisiae. The method measures single-cell protein abundance through the use of green-fluorescent-protein tags. We applied this method to 160 genes and detected many more loci per gene than previous studies. We also observed closer correspondence between loci that influence protein abundance and loci that influence mRNA abundance of a given gene. Most loci cluster at hotspot locations that influence multiple proteins--in some cases, more than half of those examined. The variants that underlie these hotspots have profound effects on the gene regulatory network and provide insights into genetic variation in cell physiology between yeast strains.\",\"preprint_author_corresponding\":\"Leonid  Kruglyak\",\"preprint_author_corresponding_institution\":\"UCLA\"},{\"preprint_doi\":\"10.1101\\\\/001776\",\"published_doi\":\"10.1186\\\\/1471-2164-15-82\",\"published_journal\":\"BMC Genomics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens\",\"preprint_authors\":\"Haoyang Cai;Nitin Kumar;Homayoun C Bagheri;Christian von Mering;Mark Robinson;Michael Baudis;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2014-01-13\",\"published_date\":\"2014-01-29\",\"preprint_abstract\":\"BackgroundChromothripsis is a recently discovered phenomenon of genomic rearrangement, possibly arising during a single genome-shattering event. This could provide an alternative paradigm in cancer development, replacing the gradual accumulation of genomic changes with a \\\\\\\\\\\\\"one-off\\\\\\\\\\\\\" catastrophic event. However, the term has been used with varying operational definitions, with the minimal consensus being a large number of locally clustered copy number aberrations. The mechanisms underlying these chromothripsis-like patterns (CTLP) and their specific impact on tumorigenesis are still poorly understood.\\\\\\\\n\\\\\\\\nResultsHere, we identified CTLP in 918 cancer samples, from a dataset of more than 22,000 oncogenomic arrays covering 132 cancer types. Fragmentation hotspots were found to be located on chromosome 8, 11, 12 and 17. Among the various cancer types, soft-tissue tumors exhibited particularly high CTLP frequencies. Genomic context analysis revealed that CTLP rearrangements frequently occurred in genomes that additionally harbored multiple copy number aberrations (CNAs). An investigation into the affected chromosomal regions showed a large proportion of arm-level pulverization and telomere related events, which would be compatible to a number of underlying mechanisms. We also report evidence that these genomic events may be correlated with patient age, stage and survival rate.\\\\\\\\n\\\\\\\\nConclusionsThrough a large-scale analysis of oncogenomic array data sets, this study characterized features associated with genomic aberrations patterns, compatible to the spectrum of \\\\\\\\\\\\\"chromothripsis\\\\\\\\\\\\\"-definitions as previously used. While quantifying clustered genomic copy number aberrations in cancer samples, our data indicates an underlying biological heterogeneity behind these chromothripsis-like patterns, beyond a well defined \\\\\\\\\\\\\"chromthripsis\\\\\\\\\\\\\" phenomenon.\",\"preprint_author_corresponding\":\"Michael  Baudis\",\"preprint_author_corresponding_institution\":\"University of Zurich\"},{\"preprint_doi\":\"10.1101\\\\/000968\",\"published_doi\":\"10.3389\\\\/fnhum.2013.00880\",\"published_journal\":\"Frontiers in Human Neuroscience\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Functional connectivity networks with and without global signal correction\",\"preprint_authors\":\"Satoru Hayasaka;\",\"preprint_category\":\"Neuroscience \",\"preprint_date\":\"2013-11-26\",\"published_date\":\"2013-12-18\",\"preprint_abstract\":\"In functional connectivity analyses in BOLD (blood oxygenation level dependent) fMRI data, there is an ongoing debate on whether to correct global signals in fMRI time series data. Although the discussion has been ongoing in the fMRI community since the early days of fMRI data analyses, this subject has gained renewed attention in recent years due to the surging popularity of functional connectivity analyses, in particular graph theory-based network analyses. However, the impact of correcting (or not correcting) for global signals has not been systematically characterized in the context of network analyses. Thus, in this work, I examined the effect of global signal correction on an fMRI network analysis. In particular, voxel-based resting-state fMRI networks were constructed with and without global signal correction. The resulting functional connectivity networks were compared. Without global signal correction, the distributions of the correlation coefficients were positively biased. I also found that, without global signal correction, nodes along the interhemisphic fissure were highly connected whereas some nodes and subgraphs around white-matter tracts became disconnected from the rest of the network. These results from this study show differences between the networks with or without global signal correction.\",\"preprint_author_corresponding\":\"Satoru  Hayasaka\",\"preprint_author_corresponding_institution\":\"Wake Forest School of Medicine\"},{\"preprint_doi\":\"10.1101\\\\/002410\",\"published_doi\":\"10.1038\\\\/nn.3669\",\"published_journal\":\"Nature Neuroscience\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Broadly tuned and respiration-independent inhibition in the olfactory bulb of awake mice\",\"preprint_authors\":\"Brittany N Cazakoff;Billy Y B Lau;Kerensa L Crump;Heike Demmer;Stephen David Shea;\",\"preprint_category\":\"Neuroscience \",\"preprint_date\":\"2014-02-06\",\"published_date\":\"2014-04-01\",\"preprint_abstract\":\"Olfactory representations are shaped by both brain state and respiration; however, the interaction and circuit substrates of these influences are poorly understood. Granule cells (GCs) in the main olfactory bulb (MOB) are presumed to sculpt activity that reaches the olfactory cortex via inhibition of mitral\\\\/tufted cells (MTs). GCs may potentially sparsen ensemble activity by facilitating lateral inhibition among MTs, and\\\\/or they may enforce temporally-precise activity locked to breathing. Yet, the selectivity and temporal structure of GC activity during wakefulness are unknown. We recorded GCs in the MOB of anesthetized and awake mice and reveal pronounced state-dependent features of odor coding and temporal patterning. Under anesthesia, GCs exhibit sparse activity and are strongly and synchronously coupled to the respiratory cycle. Upon waking, GCs desynchronize, broaden their odor responses, and typically fire without regard for the respiratory rhythm. Thus during wakefulness, GCs exhibit stronger odor responses with less temporal structure. Based on these observations, we propose that during wakefulness GCs likely predominantly shape MT odor responses through broadened lateral interactions rather than respiratory synchronization.\",\"preprint_author_corresponding\":\"Stephen David Shea\",\"preprint_author_corresponding_institution\":\"Cold Spring Harbor Laboratory\"},{\"preprint_doi\":\"10.1101\\\\/000950\",\"published_doi\":\"10.1093\\\\/mp\\\\/ssu003\",\"published_journal\":\"Molecular Plant\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"An XA21-Associated Kinase (OsSERK2) regulates immunity mediated by the XA21 and XA3 immune receptors\",\"preprint_authors\":\"Xuewei Chen;Shimin Zuo;Benjamin Schwessinger;Mawsheng Chern;Patrick Canlas;Deling Ruan;Arsalan Daudi;Xiaogang Zhang;Jing Wang;Christopher Petzold;Joshua Heazlewood;Pamela C Ronald;\",\"preprint_category\":\"Plant Biology \",\"preprint_date\":\"2013-11-26\",\"published_date\":\"2014-01-30\",\"preprint_abstract\":\"The rice XA21 immune receptor kinase and the structurally related XA3 receptor, confer immunity to Xanthomonas oryzae pv. oryzae (Xoo), the causal agent of bacterial leaf blight. Here we report the isolation of OsSERK2 (rice somatic embryogenesis receptor kinase 2) and demonstrate that OsSERK2 positively regulates immunity mediated by XA21 and XA3 as well as the rice immune receptor FLS2 (OsFLS2). Rice plants silenced for OsSerk2 display altered morphology and reduced sensitivity to the hormone brassinolide. OsSERK2 interacts with the intracellular domains of each immune receptor in the yeast-two-hybrid system in a kinase activity dependent manner. OsSERK2 undergoes bidirectional trans-phosphorylation with XA21 in vitro and forms a constitutive complex with XA21 in vivo. These results demonstrate an essential role for OsSERK2 in the function of three rice immune receptors and suggest that direct interaction with the rice immune receptors is critical for their function. Taken together, our findings suggest that the mechanism of OsSERK2-meditated regulation of rice XA21, XA3 and FLS2 differs from that of AtSERK3\\\\/BAK1-mediated regulation of Arabidopsis FLS2 and EFR.\",\"preprint_author_corresponding\":\"Pamela C Ronald\",\"preprint_author_corresponding_institution\":\"Department of Plant Pathology and the Genome Center, University of California, Davis, California 956\"},{\"preprint_doi\":\"10.1101\\\\/003335\",\"published_doi\":\"10.1021\\\\/sb400206c\",\"published_journal\":\"ACS Synthetic Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Rapidly characterizing the fast dynamics of RNA genetic circuitry with cell-free transcription-translation (TX-TL) systems\",\"preprint_authors\":\"Melissa K Takahashi;James Chappell;Clarmyra A Hayes;Zachary Z Sun;Vipul Singhal;Kevin J Spring;Shaima Al-Khabouri;Christopher P Fall;Vincent Noireaux;Richard M Murray;Julius B Lucks;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2014-03-17\",\"published_date\":\"2014-03-28\",\"preprint_abstract\":\"RNA regulators are emerging as powerful tools to engineer synthetic genetic networks or rewire existing ones. A potential strength of RNA networks is that they may be able to propagate signals on timescales that are set by the fast degradation rates of RNAs. However, a current bottleneck to verifying this potential is the slow design-build-test cycle of evaluating these networks in vivo. Here we adapt an Escherichia coli-based cell-free transcription-translation (TX-TL) system for rapidly prototyping RNA networks. We used this system to measure the response time of an RNA transcription cascade to be approximately five minutes per step of the cascade. We also show that this response time can be adjusted with temperature and regulator threshold response tuning. Finally we use TX-TL to prototype a new RNA network, an RNA single input module, and show that this network temporally stages the expression of two genes in vivo.\",\"preprint_author_corresponding\":\"Julius B Lucks\",\"preprint_author_corresponding_institution\":\"Cornell University\"},{\"preprint_doi\":\"10.1101\\\\/001172\",\"published_doi\":\"10.1093\\\\/sysbio\\\\/syu018\",\"published_journal\":\"Systematic Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Species Delimitation using Genome-Wide SNP Data\",\"preprint_authors\":\"Adam Leache;Matthew Fujita;Vladimir Minin;Remco Bouckaert;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2013-12-05\",\"published_date\":\"2014-03-12\",\"preprint_abstract\":\"The multi-species coalescent has provided important progress for evolutionary inferences, including increasing the statistical rigor and objectivity of comparisons among competing species delimitation models. However, Bayesian species delimitation methods typically require brute force integration over gene trees via Markov chain Monte Carlo (MCMC), which introduces a large computation burden and precludes their application to genomic-scale data. Here we combine a recently introduced dynamic programming algorithm for estimating species trees that bypasses MCMC integration over gene trees with sophisticated methods for estimating marginal likelihoods, needed for Bayesian model selection, to provide a rigorous and computationally tractable technique for genome-wide species delimitation. We provide a critical yet simple correction that brings the likelihoods of different species trees, and more importantly their corresponding marginal likelihoods, to the same common denominator, which enables direct and accurate comparisons of competing species delimitation models using Bayes factors. We test this approach, which we call Bayes factor delimitation (*with genomic data; BFD*), using common species delimitation scenarios with computer simulations. Varying the numbers of loci and the number of samples suggest that the approach can distinguish the true model even with few loci and limited samples per species. Misspecification of the prior for population size{theta} has little impact on support for the true model. We apply the approach to West African forest geckos (Hemidactylus fasciatus complex) using genome-wide SNP data data. This new Bayesian method for species delimitation builds on a growing trend for objective species delimitation methods with explicit model assumptions that are easily tested.\",\"preprint_author_corresponding\":\"Adam  Leache\",\"preprint_author_corresponding_institution\":\"University of Washington\"},{\"preprint_doi\":\"10.1101\\\\/002741\",\"published_doi\":\"10.7554\\\\/eLife.01917.001\",\"published_journal\":\"eLife\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Fluorescent sensors for activity and regulation of the nitrate transceptor CHL1\\\\/NRT1.1 and oligopeptide transporters\",\"preprint_authors\":\"Cheng Hsun Ho;Wolf B. Frommer;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-02-18\",\"published_date\":\"2014-03-12\",\"preprint_abstract\":\"To monitor nitrate and peptide transport activity in vivo, we converted the dual-affinity nitrate transceptor CHL1\\\\/NRT1.1\\\\/NPF6.3 and four related oligopeptide transporters PTR1, 2, 4 and 5 into fluorescence activity sensors (NiTrac1, PepTrac). Substrate addition to yeast expressing transporter fusions with yellow fluorescent protein and mCerulean triggered substrate-dependent donor quenching or resonance energy transfer. Fluorescence changes were nitrate\\\\/peptide-specific, respectively. Like CHL1, NiTrac1 had biphasic kinetics. Mutation of T101A eliminated high-affinity transport and blocked the fluorescence response to low nitrate. NiTrac was used for characterizing side chains considered important for substrate interaction, proton coupling, and regulation. We observed a striking correlation between transport activity and sensor output. Coexpression of NiTrac with known calcineurin-like proteins (CBL1, 9; CIPK23) and candidates identified in an interactome screen (CBL1, KT2, WNKinase 8) blocked NiTrac1 responses, demonstrating the suitability for in vivo analysis of activity and regulation. The new technology is applicable in plant and medical research.\",\"preprint_author_corresponding\":\"Wolf B. Frommer\",\"preprint_author_corresponding_institution\":\"Carnegie Institution for Science\"},{\"preprint_doi\":\"10.1101\\\\/001800\",\"published_doi\":\"10.1371\\\\/journal.pone.0091819\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"The Toxoplasma Acto-MyoA Motor Complex Is Important but Not Essential for Gliding Motility and Host Cell Invasion\",\"preprint_authors\":\"Saskia Egarter;Nicole Andenmatten;Allison J Jackson;Jamie A Whitelaw;Gurmann Pall;Jennifer A Black;David JP Ferguson;Isabelle Tardieux;Alex Mogilner;Markus Meissner;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-01-15\",\"published_date\":\"2014-03-14\",\"preprint_abstract\":\"Apicomplexan parasites are thought to actively invade the host cell by gliding motility. This movement is powered by the parasite own actomyosin system and depends on the regulated polymerisation and depolymerisation of actin to generate the force for gliding and host cell penetration. Recent studies demonstrated that Toxoplasma gondii can invade the host cell in the absence of several core components of the invasion machinery, such as the motor protein myosin A (MyoA), the microneme proteins MIC2 and AMA1 and actin, indicating the presence of alternative invasion mechanisms. Here the roles of MyoA, MLC1, GAP45 and Act1, core components of the gliding machinery, are re-dissected in detail. Although important roles of these components for gliding motility and host cell invasion are verified, mutant parasites remain invasive and do not show a block of gliding motility, suggesting that other mechanisms must be in place to enable the parasite to move and invade the host cell. A novel, hypothetical model for parasite gliding motility and invasion is presented based on osmotic forces generated in the cytosol of the parasite that are converted into motility.\",\"preprint_author_corresponding\":\"Markus  Meissner\",\"preprint_author_corresponding_institution\":\"University of Glasgow\"},{\"preprint_doi\":\"10.1101\\\\/002659\",\"published_doi\":\"10.1038\\\\/nmat3912\",\"published_journal\":\"Nature Materials\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Synthesis and patterning of tunable multiscale materials with engineered cells\",\"preprint_authors\":\"Allen Y Chen;Urartu O.S. Seker;Michelle Y Lu;Robert J Citorik;Timothy Lu;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2014-02-14\",\"published_date\":\"2014-03-23\",\"preprint_abstract\":\"A major challenge in materials science is to create self-assembling, functional, and environmentally responsive materials which can be patterned across multiple length scales. Natural biological systems, such as biofilms, shells, and skeletal tissues, implement dynamic regulatory programs to assemble complex multiscale materials comprised of living and non-living components1-9. Such systems can provide inspiration for the design of heterogeneous functional systems which integrate biotic and abiotic materials via hierarchical self-assembly. Here, we present a synthetic-biology platform for synthesizing and patterning self-assembled functional amyloid materials across multiple length scales with bacterial biofilms. We engineered Escherichia coli curli amyloid production under the tight control of synthetic regulatory circuits and interfaced amyloids with inorganic materials to create a biofilm-based electrical switch whose conductance can be selectively toggled by specific environmental signals. Furthermore, we externally tuned synthetic biofilms to build nanoscale amyloid biomaterials with different structure and composition through the controlled expression of their constituent subunits with artificial gene circuits. By using synthetic cell-cell communication, our engineered biofilms can also autonomously manufacture dynamic materials whose structure and composition change with time. In addition, we show that by combining subunit-level protein engineering, controlled genetic expression of self-assembling subunit proteins, and macroscale spatial gradients, synthetic biofilms can pattern protein biomaterials across multiple length scales. This work lays a foundation for synthesizing, patterning, and controlling composite materials with engineered biological systems. We envision that this approach can be expanded to other cellular and biomaterials contexts for the construction of self-organizing, environmentally responsive, and tunable multiscale composite materials with heterogeneous functionalities.\",\"preprint_author_corresponding\":\"Timothy  Lu\",\"preprint_author_corresponding_institution\":\"Massachusetts Institute of Technology\"},{\"preprint_doi\":\"10.1101\\\\/000455\",\"published_doi\":\"10.1186\\\\/2047-217X-3-3\",\"published_journal\":\"GigaScience\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A data repository and analysis framework for spontaneous neural activity recordings in developing retina\",\"preprint_authors\":\"Stephen Eglen;Michael Weeks;Mark Jessop;Jennifer Simonotto;Tom Jackson;Evelyne Sernagor;\",\"preprint_category\":\"Neuroscience \",\"preprint_date\":\"2013-11-15\",\"published_date\":\"2014-03-26\",\"preprint_abstract\":\"BackgroundDuring early development, neural circuits fire spontaneously, generating activity episodes with complex spatiotemporal patterns. Recordings of spontaneous activity have been made in many parts of the nervous system over the last 25 years, reporting developmental changes in activity patterns and the effects of various genetic perturbations.\\\\\\\\n\\\\\\\\nResultsWe present a curated repository of multielectrode array recordings of spontaneous activity in developing mouse and ferret retina. The data have been annotated with minimal metadata and converted into HDF5. This paper describes the structure of the data, along with examples of reproducible research using these data files. We also demonstrate how these data can be analysed in the CARMEN workflow system. This article is written as a literate programming document; all programs and data described here are freely available.\\\\\\\\n\\\\\\\\nConclusions1. We hope this repository will lead to novel analysis of spontaneous activity recorded in different laboratories. 2. We encourage published data to be added to the repository. 3. This repository serves as an example of how multielectrode array recordings can be stored for long-term reuse.\",\"preprint_author_corresponding\":\"Stephen  Eglen\",\"preprint_author_corresponding_institution\":\"University of Cambridge\"},{\"preprint_doi\":\"10.1101\\\\/001032\",\"published_doi\":\"10.1371\\\\/journal.pone.0091768\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Mechanism of \\\\u03b2-Aminobutyric Acid-Induced Resistance in Wheat to the Grain Aphid, Sitobion avenae\",\"preprint_authors\":\"He-He Cao;Meng Zhang;Hui Zhao;Yi Zhang;Xing-Xing Wang;Shan-Shan Guo;Zhan-Feng Zhang;Tong-Xian Liu;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2013-12-02\",\"published_date\":\"2014-03-20\",\"preprint_abstract\":\"The non-protein amino acid {beta}-aminobutyric acid (BABA) could induce plant resistance to a broad spectrum of biotic and abiotic stresses. However, BABA-induced plant resistance to insects is less well-studied, especially its underlying mechanism. In this research, we applied BABA to wheat seedlings and tested its effects on Sitobion avenae. When applied as a soil drench, BABA significantly reduced weight of S. avenae, whereas foliar spray and seed treatment had no such effects. BABA-mediated suppression of S. avenae growth is dose dependent and could last at least for 7 days. The aminobutyric acid concentration in phloem sap of BABA-treated plants accumulated to high levels and increased with BABA concentrations applied. Moreover, after 10 days of treatment, the aminobutyric acid content in BABA-treated plants was still higher than that in control treatment. S. avenae could not discriminate artificial diet containing BABA from standard diet, indicating that BABA itself is not a deterrent to this aphid. Also S. avenae did not show preference for control plants or BABA-treated plants. Consistent with choice test results, S. avenae had similar feeding activities on control and BABA-treated plants, suggesting that BABA did not induce antifeedants in wheat seedlings. In addition, aminobutyric acid concentration in S. avenae feeding on BABA-treated plants was significantly higher than those feeding on control palnts. S. avenae growth rate was reduced on artificial diet containing BABA, indicating direct toxic effects of BABA to this aphid. These results suggest that BABA application could enhance wheat plant resistance to S. avenae and the mechanism is possibly due to direct toxicity of high BABA contents in plant phloem.\",\"preprint_author_corresponding\":\"He-He  Cao\",\"preprint_author_corresponding_institution\":\"Northwest A&amp;F University\"},{\"preprint_doi\":\"10.1101\\\\/001040\",\"published_doi\":\"10.1371\\\\/journal.pone.0090714\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"High Genetic Diversity and Adaptive Potential of Two Simian Hemorrhagic Fever Viruses in a Wild Primate Population\",\"preprint_authors\":\"Adam L. Bailey;Michael Lauck;Andrea Weiler;Samuel D. Sibley;Jorge M. Dinis;Zachary Bergman;Chase W. Nelson;Michael Correll;Michael Gleicher;David Hyeroba;Alex Tumukunde;Geoffrey Weny;Colin Chapman;Jens Kuhn;Austin Hughes;Thomas C. Friedrich;Tony L. Goldberg;David H. O\\'Connor;\",\"preprint_category\":\"Microbiology \",\"preprint_date\":\"2013-12-03\",\"published_date\":\"2014-03-20\",\"preprint_abstract\":\"Key biological properties such as high genetic diversity and high evolutionary rate enhance the potential of certain RNA viruses to adapt and emerge. Identifying viruses with these properties in their natural hosts could dramatically improve disease forecasting and surveillance. Recently, we discovered two novel members of the viral family Arteriviridae: simian hemorrhagic fever virus (SHFV)-krc1 and SHFV-krc2, infecting a single wild red colobus (Procolobus rufomitratus tephrosceles) in Kibale National Park, Uganda. Nearly nothing is known about the biological properties of SHFVs in nature, although the SHFV type strain, SHFV-LVR, has caused devastating outbreaks of viral hemorrhagic fever in captive macaques. Here we detected SHFV-krc1 and SHFV-krc2 in 40% and 47% of 60 wild red colobus tested, respectively. We found viral loads in excess of 106 - 107 RNA copies per milliliter of blood plasma for each of these viruses. SHFV-krc1 and SHFV-krc2 also showed high genetic diversity at both the inter- and intra-host levels. Analyses of synonymous and non-synonymous nucleotide diversity across viral genomes revealed patterns suggestive of positive selection in SHFV open reading frames (ORF) 5 (SHFV-krc2 only) and 7 (SHFV-krc1 and SHFV-krc2). Thus, these viruses share several important properties with some of the most rapidly evolving, emergent RNA viruses.\",\"preprint_author_corresponding\":\"David H. O\\'Connor\",\"preprint_author_corresponding_institution\":\"University of Wisconsin - Madison\"},{\"preprint_doi\":\"10.1101\\\\/000885\",\"published_doi\":\"10.1021\\\\/sb400203p\",\"published_journal\":\"ACS Synthetic Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Resource usage and gene circuit performance characterization in a cell-free ?breadboard?\",\"preprint_authors\":\"Dan Siegal-Gaskins;Zoltan A. Tuza;Jongmin Kim;Vincent Noireaux;Richard M. Murray;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2013-11-25\",\"published_date\":\"2014-04-07\",\"preprint_abstract\":\"The many successes of synthetic biology have come in a manner largely different from those in other engineering disciplines; in particular, without well-characterized and simplified prototyping environments to play a role analogous to wind-tunnels in aerodynamics and breadboards in electrical engineering. However, as the complexity of synthetic circuits increases, the benefits--in cost savings and design cycle time--of a more traditional engineering approach can be significant. We have recently developed an in vitro  breadboard prototyping platform based on E. coli cell extract that allows biocircuits to operate in an environment considerably simpler than but functionally similar to in vivo. The simplicity of this system makes it a promising tool for rapid biocircuit design and testing, as well as for probing fundamental aspects of gene circuit operation normally masked by cellular complexity. In this work we characterize the cell-free breadboard using real-time and simultaneous measurements of transcriptional and translational activities of a small set of reporter genes and a transcriptional activation cascade. We determine the effects of promoter strength, gene concentration, and nucleoside triphosphate concentration on biocircuit properties, and we isolate the specific contributions of essential biomolecular resources--core RNA polymerase and ribosomes--to overall performance. Importantly, we show how limits on resources, particularly those involved in translation, are manifested as reduced expression in the presence of orthogonal genes that serve as additional loads on the system.\",\"preprint_author_corresponding\":\"Dan  Siegal-Gaskins\",\"preprint_author_corresponding_institution\":\"California Institute of Technology\"},{\"preprint_doi\":\"10.1101\\\\/003368\",\"published_doi\":\"10.1098\\\\/rsif.2013.1052\",\"published_journal\":\"Journal of the Royal Society Interface\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Hydrogen peroxide thermochemical oscillator as driver for primordial RNA replication\",\"preprint_authors\":\"Rowena Ball;John Brindley;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2014-03-17\",\"published_date\":\"2014-06-06\",\"preprint_abstract\":\"This paper presents and tests a previously unrecognised mechanism for driving a replicating molecular system on the prebiotic earth. It is proposed that cell-free RNA replication in the primordial soup may have been driven by self-sustained oscillatory thermochemical reactions. To test this hypothesis a well-characterised hydrogen peroxide oscillator was chosen as the driver and complementary RNA strands with known association and melting kinetics were used as the substrate. An open flow system model for the self-consistent, coupled evolution of the temperature and concentrations in a simple autocatalytic scheme is solved numerically, and it is shown that thermochemical cycling drives replication of the RNA strands. For the (justifiably realistic) values of parameters chosen for the simulated example system, the mean amount of replicant produced at steady state is 6.56 times the input amount, given a constant supply of substrate species. The spontaneous onset of sustained thermochemical oscillations via slowly drifting parameters is demonstrated, and a scheme is given for prebiotic production of complementary RNA strands on rock surfaces.\",\"preprint_author_corresponding\":\"Rowena  Ball\",\"preprint_author_corresponding_institution\":\"The Australian National University\"},{\"preprint_doi\":\"10.1101\\\\/000943\",\"published_doi\":\"10.3389\\\\/fncir.2014.00028\",\"published_journal\":\"Frontiers in Neural Circuits\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Simultaneous optogenetic manipulation and calcium imaging in freely moving C. elegans\",\"preprint_authors\":\"Frederick B. Shipley;Christopher M. Clark;Mark J. Alkema;Andrew M. Leifer;\",\"preprint_category\":\"Neuroscience \",\"preprint_date\":\"2013-11-27\",\"published_date\":\"2014-03-24\",\"preprint_abstract\":\"Editor:\\\\\\\\n\\\\\\\\nA fundamental goal of systems neuroscience is to probe the dynamics of neural activity that generate behavior. Here we present an instrument to simultaneously manipulate and monitor neural activity and behavior in the freely moving nematode Caenorhabditis elegans. We use the instrument to directly observe the relationship between sensory neuron activation, interneuron dynamics and locomotion in the mechanosensory circuit.\\\\\\\\n\\\\\\\\nPreviously in this journal, we presented an optogenetic illumination system capable of real-time light delivery with high spatial resolution to stimulate or inhibit specified targets in freely moving C. elegans [1]. This \\\\\\\\\\\\\"Colbert\\\\\\\\\\\\\" system and others like it [2] have been instrumental in defining neural coding of several behaviors in C. elegans including chemotaxis [3], nociception [4] and the escape response [5]. Here we integrate the Colbert s ...\",\"preprint_author_corresponding\":\"Andrew M. Leifer\",\"preprint_author_corresponding_institution\":\"Princeton University\"},{\"preprint_doi\":\"10.1101\\\\/003772\",\"published_doi\":\"10.1088\\\\/1478-3975\\\\/11\\\\/2\\\\/026005\",\"published_journal\":\"Physical Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Comparison of the theoretical and real-world evolutionary potential of a genetic circuit.\",\"preprint_authors\":\"Manuel Razo-Mejia;James Boedicker;Daniel Jones;Alexander de Luna;Justin Block Kinney;Rob Phillips;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-04-02\",\"published_date\":\"2014-04-01\",\"preprint_abstract\":\"With the development of next-generation sequencing technologies, many large scale experimental efforts aim to map genotypic variability among individuals. This natural variability in populations fuels many fundamental biological processes, ranging from evolutionary adaptation and speciation to the spread of genetic diseases and drug resistance. An interesting and important component of this variability is present within the regulatory regions of genes. As these regions evolve, accumulated mutations lead to modulation of gene expression, which may have consequences for the phenotype. A simple model system where the link between genetic variability, gene regulation and function can be studied in detail is missing. In this article we develop a model to explore how the sequence of the wild-type lac promoter dictates the fold change in gene expression. The model combines single-base pair resolution maps of transcription factor and RNA polymerase binding energies with a comprehensive thermodynamic model of gene regulation. The model was validated by predicting and then measuring the variability of lac operon regulation in a collection of natural isolates. We then implement the model to analyze the sensitivity of the promoter sequence to the regulatory output, and predict the potential for regulation to evolve due to point mutations in the promoter region.\",\"preprint_author_corresponding\":\"Rob  Phillips\",\"preprint_author_corresponding_institution\":\"California Institute of Technology\"},{\"preprint_doi\":\"10.1101\\\\/000752\",\"published_doi\":\"10.1016\\\\/j.ajhg.2014.03.004\",\"published_journal\":\"American Journal of Human Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Joint analysis of functional genomic data and genome-wide association studies of 18 human traits\",\"preprint_authors\":\"Joseph Pickrell;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2013-11-19\",\"published_date\":\"2014-03-04\",\"preprint_abstract\":\"Annotations of gene structures and regulatory elements can inform genome-wide association studies (GWAS). However, choosing the relevant annotations for interpreting an association study of a given trait remains challenging. We describe a statistical model that uses association statistics computed across the genome to identify classes of genomic element that are enriched or depleted for loci that influence a trait. The model naturally incorporates multiple types of annotations. We applied the model to GWAS of 18 human traits, including red blood cell traits, platelet traits, glucose levels, lipid levels, height, BMI, and Crohns disease. For each trait, we evaluated the relevance of 450 different genomic annotations, including protein-coding genes, enhancers, and DNase-I hypersensitive sites in over a hundred tissues and cell lines. We show that the fraction of phenotype-associated SNPs that influence protein sequence ranges from around 2% (for platelet volume) up to around 20% (for LDL cholesterol); that repressed chromatin is significantly depleted for SNPs associated with several traits; and that cell type-specific DNase-I hypersensitive sites are enriched for SNPs associated with several traits (for example, the spleen in platelet volume). Finally, by re-weighting each GWAS using information from functional genomics, we increase the number of loci with high-confidence associations by around 5%.\",\"preprint_author_corresponding\":\"Joseph  Pickrell\",\"preprint_author_corresponding_institution\":\"New York Genome Center\"},{\"preprint_doi\":\"10.1101\\\\/001255\",\"published_doi\":\"10.1007\\\\/s10750-014-1870-0\",\"published_journal\":\"Hydrobiologia\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Broad-scale spatial patterns of canopy cover and pond morphology affect the structure of a Neotropical amphibian metacommunity\",\"preprint_authors\":\"Diogo B. Provete;Thiago Gon\\\\u00e7alves-Souza;Michel Garey;Itamar A. Martins;Denise Rossa-Feres;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2013-12-10\",\"published_date\":\"2014-04-09\",\"preprint_abstract\":\"Spatial and environmental processes influence species composition at distinct scales. Previous studies suggested that the landscape-scale distribution of larval anurans is influenced by environmental gradients related to adult breeding site selection, such as pond canopy cover, but not water chemistry. However, the combined effects of spatial, pond morphology, and water chemistry variables on metacommunity structure of larval anurans have not been analyzed. We used a partial redundancy analysis with variation partitioning to analyze the relative influence of pond morphology (e.g., depth, area, and aquatic vegetation), water chemistry, and spatial variables on a tadpole metacommunity from southeastern Brazil. We predict that the metacommunity will be spatially structured at broad spatial scales, while environmental variables, mainly related to adult habitat selection, would play a larger role at fine spatial scales. We found that broad-scale spatial patterns of pond canopy cover and pond morphology strongly influenced metacommunity structure. Additionally, species composition was spatially autocorrelated at short distances. We suggest that the reproductive behavior of adult anurans is driving tadpole metacommunity dynamics, since pond morphology, but not water chemistry affects breeding site selection by adults. Our results contribute to the understanding of amphibian species diversity in tropical environments.\",\"preprint_author_corresponding\":\"Diogo B. Provete\",\"preprint_author_corresponding_institution\":\"Universidade Federal de Goi\\\\u00e1s\"},{\"preprint_doi\":\"10.1101\\\\/003459\",\"published_doi\":\"10.1073\\\\/pnas.1404354111\",\"published_journal\":\"Proceedings of the National Academy of Sciences\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Mpl expression on megakaryocytes and platelets is dispensable for thrombopoiesis but essential to prevent myeloproliferation\",\"preprint_authors\":\"Ashley P Ng;Maria Kauppi;Donald Metcalf;Craig D Hyland;Emma C Josefsson;Marion Lebois;Jian-Guo Zhang;Ladina Di Rago;Tracey Baldwin;Douglas J Hilton;Warren S Alexander;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-03-19\",\"published_date\":\"2014-04-07\",\"preprint_abstract\":\"Thrombopoietin (TPO) acting via its receptor Mpl is the major cytokine regulator of platelet number. To precisely define the role of specific hematopoietic cells in TPO dependent hematopoiesis, we generated mice that express the Mpl receptor normally on stem\\\\/progenitor cells but lack expression on megakaryocytes and platelets (MplPF4cre\\\\/PF4cre). MplPF4cre\\\\/PF4cre mice displayed profound megakaryocytosis and thrombocytosis with a remarkable expansion of megakaryocyte-committed and multipotential progenitor cells, the latter displaying biological responses and a gene expression signature indicative of chronic TPO over-stimulation as the underlying causative mechanism, despite a normal circulating TPO level. Thus, TPO signaling in megakaryocytes is dispensable for platelet production; its key role in control of platelet number is via generation and stimulation of the bipotential megakaryocyte precursors. Nevertheless, Mpl expression on megakaryocytes and platelets is essential to prevent megakaryocytosis and myeloproliferation by restricting the amount of TPO available to stimulate the production of megakaryocytes from the progenitor cell pool.\\\\\\\\n\\\\\\\\nSignificance statementBlood platelets, the small circulating cells that co-ordinate hemostasis, are produced by specialized bone marrow cells called megakaryocytes. The cytokine thrombopoietin (TPO) is a key regulator of platelet production acting via its specific cell receptor, Mpl. Via genetic modification of the Mpl allele in mice, we precisely define the bone marrow cells that express Mpl and, by genetically removing Mpl from megakaryocytes and platelets, we show TPO signaling via Mpl is not required in megakaryocytes for their expansion, maturation or platelet production. Rather, Mpl expression on megakaryocytes is essential for regulating TPO availability in the bone marrow microenvironment to prevent myeloproliferation, a model we suggest is important for human disease.\",\"preprint_author_corresponding\":\"Ashley P Ng\",\"preprint_author_corresponding_institution\":\"The Walter and Eliza Hall Institute of Medical Research\"},{\"preprint_doi\":\"10.1101\\\\/003707\",\"published_doi\":\"10.1111\\\\/evo.12420\",\"published_journal\":\"Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"An evolutionary resolution of manipulation conflict\",\"preprint_authors\":\"Mauricio Gonz\\\\u00e1lez-Forero;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-03-31\",\"published_date\":\"2014-05-21\",\"preprint_abstract\":\"Individuals can manipulate the behavior of social partners. However, manipulation may conflict with the fitness interests of the manipulated individuals. Manipulated individuals can then be favored to resist manipulation, possibly reducing or eliminating the manipulated behavior in the long run. I use a mathematical model to show that conflicts where manipulation and resistance coevolve can disappear as a result of the coevolutionary process. I find that while manipulated individuals are selected to resist, they can simultaneously be favored to express the manipulated behavior at higher efficiency (i.e., providing increasing fitness effects to recipients of the manipulated behavior). Efficiency can increase to a point at which selection for resistance disappears. This process yields an efficient social behavior that is induced by social partners, and over which the inducing and induced individuals are no longer in conflict. A necessary factor is costly inefficiency. I develop the model to address the evolution of advanced eusociality via maternal manipulation (AEMM). The model predicts AEMM to be particularly likely in taxa with ancestrally imperfect resistance to maternal manipulation. Costly inefficiency occurs if the cost of delayed dispersal is larger than the benefit of exploiting the maternal patch. I discuss broader implications of the process.\",\"preprint_author_corresponding\":\"Mauricio  Gonz\\\\u00e9lez-Forero\",\"preprint_author_corresponding_institution\":\"University of Tennessee, Knoxville, and National Institute for Mathematical and Biological Synthesis\"},{\"preprint_doi\":\"10.1101\\\\/003939\",\"published_doi\":\"10.1007\\\\/s13235-014-0106-2\",\"published_journal\":\"Dynamic Games and Applications\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Evolutionary Multiplayer Games\",\"preprint_authors\":\"Chaitanya S Gokhale;Arne Traulsen;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-04-07\",\"published_date\":\"2014-03-27\",\"preprint_abstract\":\"Evolutionary game theory has become one of the most diverse and far reaching theories in biology. Applications of this theory range from cell dynamics to social evolution. However, many applications make it clear that inherent non-linearities of natural systems need to be taken into account. One way of introducing such non-linearities into evolutionary games is by the inclusion of multiple players. An example is of social dilemmas, where group benefits could e.g. increase less than linear with the number of cooperators. Such multiplayer games can be introduced in all the fields where evolutionary game theory is already well established. However, the inclusion of non-linearities can help to advance the analysis of systems which are known to be complex, e.g. in the case of non-Mendelian inheritance. We review the diachronic theory and applications of multiplayer evolutionary games and present the current state of the field. Our aim is a summary of the theoretical results from well-mixed populations in infinite as well as finite populations. We also discuss examples from three fields where the theory has been successfully applied, ecology, social sciences and population genetics. In closing, we probe certain future directions which can be explored using the complexity of multiplayer games while preserving the promise of simplicity of evolutionary games.\",\"preprint_author_corresponding\":\"Arne  Traulsen\",\"preprint_author_corresponding_institution\":\"Max Planck Institute for Evolutionary Biology\"},{\"preprint_doi\":\"10.1101\\\\/002675\",\"published_doi\":\"10.1242\\\\/bio.20147955\",\"published_journal\":\"Biology Open\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Non-specificity of Pitstop 2 in clathrin-mediated endocytosis\",\"preprint_authors\":\"Anna K Willox;Yasmina M.E. Sahraoui;Stephen J Royle;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-02-13\",\"published_date\":\"2014-04-04\",\"preprint_abstract\":\"Small molecule inhibitors of clathrin-mediated endocytosis are highly desired for the dissection of membrane trafficking pathways in the lab and for potential use as anti-infectives in the clinic. One inhibition strategy is to prevent clathrin from contacting adaptor proteins so that clathrin-mediated endocytosis cannot occur. \\\\\\\\\\\\\"Pitstop\\\\\\\\\\\\\" compounds have been developed which block only one of the four functional interaction sites on the N-terminal domain of clathrin heavy chain. Despite this limitation, Pitstop 2 causes profound inhibition of clathrin-mediated endocytosis. In this study, we probed for non-specific activity of Pitstop 2 by examining its action in cells expressing clathrin heavy chain harbouring mutations in the N-terminal domain interaction sites. We conclude that the inhibition observed with this compound is due to non-specificity, i.e. it causes inhibition away from its proposed mode of action. We recommend that these compounds be used with caution in cells and that they should not be used to conclude anything of the function of clathrins N-terminal domain.\",\"preprint_author_corresponding\":\"Stephen J Royle\",\"preprint_author_corresponding_institution\":\"University of Warwick\"},{\"preprint_doi\":\"10.1101\\\\/003160\",\"published_doi\":\"10.1007\\\\/s00439-014-1441-5\",\"published_journal\":\"Human Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Conditions for the validity of SNP-based heritability estimation\",\"preprint_authors\":\"James J Lee;Carson C Chow;\",\"preprint_category\":\"Genetics \",\"preprint_date\":\"2014-03-03\",\"published_date\":\"2014-04-18\",\"preprint_abstract\":\"The heritability of a trait (h2) is the proportion of its population variance caused by genetic differences, and estimates of this parameter are important for interpreting the results of genome-wide association studies (GWAS). In recent years, researchers have adopted a novel method for estimating a lower bound on heritability directly from GWAS data that uses realized genetic similarities between nominally unrelated individuals. The quantity estimated by this method is purported to be the contribution to heritability that could in principle be recovered from association studies employing the given panel of SNPs [Formula] Thus far the validity of this approach has mostly been tested empirically. Here, we provide a mathematical explication and show that the method should remain a robust means of obtaining [Formula] under circumstances wider than those under which it has so far been derived.\",\"preprint_author_corresponding\":\"Carson C Chow\",\"preprint_author_corresponding_institution\":\"NIDDK, NIH\"},{\"preprint_doi\":\"10.1101\\\\/001743\",\"published_doi\":\"10.1242\\\\/bio.20147633\",\"published_journal\":\"Biology Open\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"OPPOSING MICROTUBULE MOTORS CONTROL MOTILITY, MORPHOLOGY, AND CARGO SEGREGATION DURING ER-TO-GOLGI TRANSPORT.\",\"preprint_authors\":\"Anna K Brown;Sylvie D Hunt;David J Stephens;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-01-10\",\"published_date\":\"2014-03-28\",\"preprint_abstract\":\"We recently demonstrated that dynein and kinesin motors drive multiple aspects of endosomal function in mammalian cells. These functions include driving motility, maintaining morphology (notably through providing longitudinal tension to support vesicle fission), and driving cargo sorting. Microtubule motors drive bidirectional motility during traffic between the endoplasmic reticulum (ER) and Golgi. Here, we have examined the role of microtubule motors in transport carrier motility, morphology, and domain organization during ER-to-Golgi transport. We show that consistent with our findings for endosomal dynamics, microtubule motor function during ER-to-Golgi transport of secretory is required for motility, morphology of, and cargo sorting within vesicular tubular carriers en route to the Golgi. Our data are consistent with previous findings that defined roles for dynein-1 and kinesin-1 (KIF5B) and kinesin-2 in this trafficking step. Our high resolution tracking data identify some intriguing aspects. Depletion of kinesin-1 reduces the number of motile structures seen which is in line with other findings relating to the role of kinesin-1 in ER export. However, those transport carriers that were produced had a much greater run length suggesting that this motor can act as a brake on anterograde motility. Kinesin-2 depletion did not significantly reduce the number of motile transport carriers but did cause a similar increase in run length. These data suggest that kinesins act as negative regulators of ER-to-Golgi transport. Depletion of dynein not only reduced the number of motile carriers formed but also caused tubulation of carriers similar to that seen for SNX-coated early endosomes. Our data indicated that the previously observed anterograde-retrograde polarity of transport carriers in transit to the Golgi from the ER is maintained by microtubule motor function.\",\"preprint_author_corresponding\":\"David J Stephens\",\"preprint_author_corresponding_institution\":\"University of Bristol\"},{\"preprint_doi\":\"10.1101\\\\/002238\",\"published_doi\":\"10.1371\\\\/journal.pone.0093766\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Fast Principal Component Analysis of Large-Scale Genome-Wide Data\",\"preprint_authors\":\"Gad Abraham;Michael Inouye;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2014-01-30\",\"published_date\":\"2014-04-09\",\"preprint_abstract\":\"Principal component analysis (PCA) is routinely used to analyze genome-wide single-nucleotide polymorphism (SNP) data, for detecting population structure and potential outliers. However, the size of SNP datasets has increased immensely in recent years and PCA of large datasets has become a time consuming task. We have developed flashpca, a highly efficient PCA implementation based on randomized algorithms, which delivers identical accuracy in extracting the top principal components compared with existing tools, in substantially less time. We demonstrate the utility of flashpca on both HapMap3 and on a large Immunochip dataset. For the latter, flashpca performed PCA of 15,000 individuals up to 125 times faster than existing tools, with identical results, and PCA of 150,000 individuals using flashpca completed in 4 hours. The increasing size of SNP datasets will make tools such as flashpca essential as traditional approaches will not adequately scale. This approach will also help to scale other applications that leverage PCA or eigen-decomposition to substantially larger datasets.\",\"preprint_author_corresponding\":\"Gad  Abraham\",\"preprint_author_corresponding_institution\":\"University of Melbourne\"},{\"preprint_doi\":\"10.1101\\\\/000430\",\"published_doi\":\"10.1021\\\\/sb400157z\",\"published_journal\":\"ACS Synthetic Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Negative autoregulation matches production and demand in synthetic transcriptional networks\",\"preprint_authors\":\"Elisa Franco;Giulia Giordano;Per-Ola Forsberg;Richard M Murray;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2013-11-14\",\"published_date\":\"2014-04-03\",\"preprint_abstract\":\"We propose a negative feedback architecture that regulates activity of artificial genes, or \\\\\\\\\\\\\"genelets\\\\\\\\\\\\\", to meet their output downstream demand, achieving robustness with respect to uncertain open-loop output production rates. In particular, we consider the case where the outputs of two genelets interact to form a single assembled product. We show with analysis and experiments that negative autoregulation matches the production and demand of the outputs: the magnitude of the regulatory signal is proportional to the \\\\\\\\\\\\\"error\\\\\\\\\\\\\" between the circuit output concentration and its actual demand. This two-device system is experimentally implemented using in vitro transcriptional networks, where reactions are systematically designed by optimizing nucleic acid sequences with publicly available software packages. We build a predictive ordinary differential equation (ODE) model that captures the dynamics of the system, and can be used to numerically assess the scalability of this architecture to larger sets of interconnected genes. Finally, with numerical simulations we contrast our negative autoregulation scheme with a cross-activation architecture, which is less scalable and results in slower response times.\",\"preprint_author_corresponding\":\"Elisa  Franco\",\"preprint_author_corresponding_institution\":\"University of California at Riverside\"},{\"preprint_doi\":\"10.1101\\\\/003525\",\"published_doi\":\"10.1007\\\\/s11886-014-0488-1\",\"published_journal\":\"Current Cardiology Reports\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Towards a molecular systems model of coronary artery disease\",\"preprint_authors\":\"Gad Abraham;Oneil G Bhalala;Paul IW de Bakker;Samuli Ripatti;Michael Inouye;\",\"preprint_category\":\"Systems Biology \",\"preprint_date\":\"2014-03-23\",\"published_date\":\"2014-04-18\",\"preprint_abstract\":\"Coronary artery disease (CAD) is a complex disease driven by myriad interactions of genetics and environmental factors. Traditionally, studies have analyzed only one disease factor at a time, providing useful but limited understanding of the underlying etiology. Recent advances in cost-effective and high-throughput technologies, such as single nucleotide polymorphism (SNP) genotyping, exome\\\\/genome sequencing, gene expression microarrays and metabolomics assays have enabled the collection of millions of data points in many thousands of individuals. In order to make sense of such  omics data, effective analytical methods are needed. We review and highlight some of the main results in this area, focusing on integrative approaches that consider multiple modalities simultaneously. Such analyses have the potential to uncover the genetic basis of CAD, produce genomic risk scores (GRS) for disease prediction, disentangle the complex interactions underlying disease, and predict response to treatment.\",\"preprint_author_corresponding\":\"Michael  Inouye\",\"preprint_author_corresponding_institution\":\"Medical Systems Biology, Department of Pathology and Department of Microbiology &amp; Immunology, The Un\"},{\"preprint_doi\":\"10.1101\\\\/001495\",\"published_doi\":\"10.1371\\\\/journal.pcbi.1003569\",\"published_journal\":\"PLOS Computational Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A coarse-grained elastic network atom contact model and its use in the simulation of protein dynamics and the prediction of the effect of mutations\",\"preprint_authors\":\"Vincent Frappier;Rafael Najmanovich;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2013-12-20\",\"published_date\":\"2014-04-24\",\"preprint_abstract\":\"Normal mode analysis (NMA) methods are widely used to study dynamic aspects of protein structures. Two critical components of NMA methods are coarse-graining in the level of simplification used to represent protein structures and the choice of potential energy functional form. There is a trade-off between speed and accuracy in different choices. In one extreme one finds accurate but slow molecular-dynamics based methods with all-atom representations and detailed atom potentials. On the other extreme, fast elastic network model (ENM) methods with Conly representations and simplified potentials that based on geometry alone, thus oblivious to protein sequence. Here we present ENCoM, an Elastic Network Contact Model that employs a potential energy function that includes a pairwise atom-type non-bonded interaction term and thus makes it possible to consider the effect of the specific nature of amino-acids on dynamics within the context of NMA. ENCoM is as fast as existing ENM methods and outperforms such methods in the generation of conformational ensembles. Here we introduce a new application for NMA methods with the use of ENCoM in the prediction of the effect of mutations on protein stability. While existing methods are based on machine learning or enthalpic considerations, the use of ENCoM, based on vibrational normal modes, is based on entropic considerations. This represents a novel area of application for NMA methods and a novel approach for the prediction of the effect of mutations. We compare ENCoM to a large number of methods in terms of accuracy and self-consistency. We show that the accuracy of ENCoM is comparable to that of the best existing methods. We show that existing methods are biased towards the prediction of destabilizing mutations and that ENCoM is less biased at predicting stabilizing mutations.\",\"preprint_author_corresponding\":\"Rafael  Najmanovich\",\"preprint_author_corresponding_institution\":\"Universit\\\\u00e9 de Sherbrooke\"},{\"preprint_doi\":\"10.1101\\\\/000984\",\"published_doi\":\"10.1371\\\\/journal.pone.0098122\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A Bayesian Method to Incorporate Hundreds of Functional Characteristics with Association Evidence to Improve Variant Prioritization\",\"preprint_authors\":\"Sarah A Gagliano;Michael R Barnes;Michael E Weale;Jo Knight;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2013-12-04\",\"published_date\":\"2014-05-20\",\"preprint_abstract\":\"The increasing quantity and quality of functional genomic information motivate the assessment and integration of these data with association data, including data originating from genome-wide association studies (GWAS). We used previously described GWAS signals (\\\\\\\\\\\\\"hits\\\\\\\\\\\\\") to train a regularized logistic model in order to predict SNP causality on the basis of a large multivariate functional dataset. We show how this model can be used to derive Bayes factors for integrating functional and association data into a combined Bayesian analysis. Functional characteristics were obtained from the Encyclopedia of DNA Elements (ENCODE), from published expression quantitative trait loci (eQTL), and from other sources of genome-wide characteristics. We trained the model using all GWAS signals combined, and also using phenotype specific signals for autoimmune, brain-related, cancer, and cardiovascular disorders. The non-phenotype specific and the autoimmune GWAS signals gave the most reliable results. We found SNPs with higher probabilities of causality from functional characteristics showed an enrichment of more significant p-values compared to all GWAS SNPs in three large GWAS studies of complex traits. We investigated the ability of our Bayesian method to improve the identification of true causal signals in a psoriasis GWAS dataset and found that combining functional data with association data improves the ability to prioritise novel hits. We used the predictions from the penalized logistic regression model to calculate Bayes factors relating to functional characteristics and supply these online alongside resources to integrate these data with association data.\\\\\\\\n\\\\\\\\nAuthor SummaryLarge-scale genetic studies have had success identifying genes that play a role in complex traits. Advanced statistical procedures suggest that there are still genetic variants to be discovered, but these variants are difficult to detect. Incorporating biological information that affect the amount of protein or other product produced can be used to prioritise the genetic variants in order to identify which are likely to be causal. The method proposed here uses such biological characteristics to predict which genetic variants are most likely to be causal for complex traits.\",\"preprint_author_corresponding\":\"Jo  Knight\",\"preprint_author_corresponding_institution\":\"Centre for Addiction and Mental Health\"},{\"preprint_doi\":\"10.1101\\\\/005512\",\"published_doi\":\"10.1186\\\\/1471-2148-14-98\",\"published_journal\":\"BMC Evolutionary Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Dynamics of a combined medea-underdominant population transformation system\",\"preprint_authors\":\"Chaitanya Gokhale;Richard Guy Reeves;Floyd A Reed;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-05-27\",\"published_date\":\"2014-05-07\",\"preprint_abstract\":\"BackgroundTransgenic constructs intended to be stably established at high frequencies in wild populations have been demonstrated to \\\\\\\\\\\\\"drive\\\\\\\\\\\\\" from low frequencies in experimental insect populations. Linking such population transformation constructs to genes which render them unable to transmit pathogens could eventually be used to stop the spread of vector-borne diseases like malaria and dengue.\\\\\\\\n\\\\\\\\nResultsGenerally, population transformation constructs with only a single transgenic drive mechanism have been envisioned. Using a theoretical modelling approach we describe the predicted properties of a construct combining autosomal Medea and underdominant population transformation systems. We show that when combined they can exhibit synergistic properties which in broad circumstances surpass those of the single systems.\\\\\\\\n\\\\\\\\nConclusionWith combined systems, intentional population transformation and its reversal can be achieved readily. Combined constructs also enhance the capacity to geographically restrict transgenic constructs to targeted populations. It is anticipated that these properties are likely to be of particular value in attracting regulatory approval and public acceptance of this novel technology.\",\"preprint_author_corresponding\":\"Chaitanya  Gokhale\",\"preprint_author_corresponding_institution\":\"Massey University\"},{\"preprint_doi\":\"10.1101\\\\/003608\",\"published_doi\":\"10.1371\\\\/journal.pone.0096303\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A Comparison of Peak Callers Used for DNase-seq Data\",\"preprint_authors\":\"Hashem Koohy;Thomas Down;Mikhail Spivakov;Tim Hubbard;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2014-03-27\",\"published_date\":\"2014-05-08\",\"preprint_abstract\":\"Genome-wide profiling of open chromatin regions using DNase I and high-throughput sequencing (DNase-seq) is an increasingly popular approach for finding and studying regulatory elements. A variety of algorithms have been developed to identify regions of open chromatin from raw sequence-tag data, which has motivated us to assess and compare their performance.\\\\\\\\n\\\\\\\\nIn this study, four published, publicly available peak calling algorithms used for DNase-seq data analysis (F-seq, Hotspot, MACS and ZINBA) are assessed at a range of signal thresholds on two published DNase-seq datasets for three cell types. The results were benchmarked against an independent dataset of regulatory regions derived from ENCODE in vivo transcription factor binding data for each particular cell type. The level of overlap between peak regions reported by each algorithm and this ENCODE-derived reference set was used to assess sensitivity and specificity of the algorithms.\\\\\\\\n\\\\\\\\nOur study suggests that F-seq has a slightly higher sensitivity than the next best algorithms. Hotspot and the ChIP-seq oriented method, MACS, both perform competitively when used with their default parameters. However the generic peak finder ZINBA appears to be less sensitive than the other three.\\\\\\\\n\\\\\\\\nWe also assess accuracy of each algorithm over a range of signal thresholds. In particular, we show that the accuracy of F-Seq can be considerably improved by using a threshold setting that is different from the default value.\",\"preprint_author_corresponding\":\"Hashem  Koohy\",\"preprint_author_corresponding_institution\":\"Babraham Institute\"},{\"preprint_doi\":\"10.1101\\\\/003277\",\"published_doi\":\"10.7554\\\\/eLife.02409\",\"published_journal\":\"eLife\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Filament formation by metabolic enzymes is a specific adaptation to an advanced state of cellular starvation\",\"preprint_authors\":\"Ivana Petrovska;Elisabeth N\\\\u00fcske;Matthias C Munder;Gayathrie Kulasegaran;Liliana Malinovska;Sonja Kroschwald;Doris Richter;Karim Fahmy;Kimberley Gibson;Jean-Marc Verbavatz;Simon Alberti;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-03-07\",\"published_date\":\"2014-04-25\",\"preprint_abstract\":\"One of the key questions in biology is how the metabolism of a cell responds to changes in the environment. In budding yeast, starvation causes a drop in intracellular pH, but the functional role of this pH change is not well understood. Here, we show that the enzyme glutamine synthetase (Gln1) forms filaments at low pH and that filament formation leads to enzyme inactivation. Filament formation by Gln1 is a highly cooperative process, strongly dependent on macromolecular crowding, and involves back-to-back stacking of cylindrical homo-decamers into filaments that associate laterally to form higher order fibrils. Other metabolic enzymes also assemble into filaments at low pH. Hence, we propose that filament formation is a general mechanism to inactivate and store key metabolic enzymes during a state of advanced cellular starvation. These findings have broad implications for understanding the interplay between nutritional stress, the metabolism and the physical organization of a cell.\",\"preprint_author_corresponding\":\"Simon  Alberti\",\"preprint_author_corresponding_institution\":\"Max Planck Institute of Molecular Cell Biology and Genetics\"},{\"preprint_doi\":\"10.1101\\\\/002048\",\"published_doi\":\"10.1016\\\\/j.bpj.2014.02.009\",\"published_journal\":\"Biophysical Journal\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Bayesian Energy Landscape Tilting: Towards Concordant Models of Molecular Ensembles\",\"preprint_authors\":\"Kyle Beauchamp;Vijay Pande;Rhiju Das;\",\"preprint_category\":\"Biophysics \",\"preprint_date\":\"2014-01-24\",\"published_date\":\"2014-03-18\",\"preprint_abstract\":\"Predicting biological structure has remained challenging for systems such as disordered proteins that take on myriad conformations. Hybrid simulation\\\\/experiment strategies have been undermined by difficulties in evaluating errors from computa- tional model inaccuracies and data uncertainties. Building on recent proposals from maximum entropy theory and nonequilibrium thermodynamics, we address these issues through a Bayesian Energy Landscape Tilting (BELT) scheme for computing Bayesian \\\\\\\\\\\\\"hyperensembles\\\\\\\\\\\\\" over conformational ensembles. BELT uses Markov chain Monte Carlo to directly sample maximum-entropy conformational ensembles consistent with a set of input experimental observables. To test this framework, we apply BELT to model trialanine, starting from disagreeing simulations with the force fields ff96, ff99, ff99sbnmr-ildn, CHARMM27, and OPLS-AA. BELT incorporation of limited chemical shift and 3J measurements gives convergent values of the peptides , {beta}, and PPII conformational populations in all cases. As a test of predictive power, all five BELT hyperensembles recover set-aside measurements not used in the fitting and report accu- rate errors, even when starting from highly inaccurate simulations. BELTs principled fxramework thus enables practical predictions for complex biomolecular systems from discordant simulations and sparse data.\",\"preprint_author_corresponding\":\"Rhiju  Das\",\"preprint_author_corresponding_institution\":\"Stanford University\"},{\"preprint_doi\":\"10.1101\\\\/000307\",\"published_doi\":\"10.1371\\\\/journal.pgen.1004293\",\"published_journal\":\"PLOS Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Drosophila embryogenesis scales uniformly across temperature and developmentally diverse species\",\"preprint_authors\":\"Steven Gregory Kuntz;Michael B Eisen;\",\"preprint_category\":\"Developmental Biology \",\"preprint_date\":\"2013-11-12\",\"published_date\":\"2014-04-24\",\"preprint_abstract\":\"Temperature affects both the timing and outcome of animal development, but the detailed effects of temperature on the progress of early development have been poorly characterized. To determine the impact of temperature on the order and timing of events during Drosophila melanogaster embryogenesis, we used time-lapse imaging to track the progress of embryos from shortly after egg laying through hatching at seven precisely maintained temperatures between 17.5{degrees}C and 32.5{degrees}C. We employed a combination of automated and manual annotation to determine when 36 milestones occurred in each embryo. D. melanogaster embryogenesis takes 33 hours at 17.5{degrees}C, and accelerates with increasing temperature to a low of 16 hours at 27.5{degrees}C, above which embryogenesis slows slightly. Remarkably, while the total time of embryogenesis varies over two fold, the relative timing of events from cellularization through hatching is constant across temperatures. To further explore the relationship between temperature and embryogenesis, we expanded our analysis to cover ten additional Drosophila species of varying climatic origins. Six of these species, like D. melanogaster, are of tropical origin, and embryogenesis time at different temperatures was similar for them all. D. mojavensis, a sub-tropical fly, develops slower than the tropical species at lower temperatures, while D. virilis, a temperate fly, exhibits slower development at all temperatures. The alpine sister species D. persimilis and D. pseudoobscura develop as rapidly as tropical flies at cooler temperatures, but exhibit diminished acceleration above 22.5{degrees}C and have drastically slowed development by 30{degrees}C. Despite ranging from 13 hours for D. erecta at 30{degrees}C to 46 hours for D. virilis at 17.5{degrees}C, the relative timing of events from cellularization through hatching is constant across all of the species and temperatures examined here, suggesting the existence of a previously unrecognized timer controlling the progress of embryogenesis that has been tuned by natural selection in response to the thermal environment in which each species lives.\\\\\\\\n\\\\\\\\nAuthor SummaryTemperature profoundly impacts the rate of development of \\\\\\\\\\\\\"cold-blooded\\\\\\\\\\\\\" animals, which proceeds far faster when it is warm. There is, however, no universal relationship. Closely related species can develop at markedly different speeds at the same temperature, likely resulting from environmental adaptation. This creates a major challenge when comparing development among species, as it is unclear whether they should be compared at the same temperature or under different conditions to maintain the same developmental rate. Facing this challenge while working with flies (Drosophila species), we found there was little data to inform this decision. So, using time-lapse imaging, precise temperature-control, and computational and manual video-analysis, we tracked the complex process of embryogenesis in 11 species at seven different temperatures. There was over a three-fold difference in developmental rate between the fastest species at its fastest temperature and the slowest species at its slowest temperature. However, our finding that the timing of events within development all scaled uniformly across species and temperatures astonished us. This is good news for developmental biologists, since we can induce species to develop nearly identically by growing them at different temperatures. But it also means flies must possess some unknown clock-like molecular mechanism driving embryogenesis forward.\",\"preprint_author_corresponding\":\"Steven Gregory Kuntz\",\"preprint_author_corresponding_institution\":\"University of California, Berkeley\"},{\"preprint_doi\":\"10.1101\\\\/000760\",\"published_doi\":\"10.1111\\\\/cobi.12308\",\"published_journal\":\"Conservation Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Linking indices for biodiversity monitoring to extinction risk theory\",\"preprint_authors\":\"Michael A. McCarthy;Alana L. Moore;Jochen Krauss;John W Morgan;Christopher F. Clements;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2013-11-21\",\"published_date\":\"2014-05-12\",\"preprint_abstract\":\"Biodiversity indices often combine data from different species when used in monitoring programs. Heuristic properties can suggest preferred indices, but we lack objective ways to discriminate between indices with similar heuristics. Biodiversity indices can be evaluated by determining how well they reflect management objectives that a monitoring program aims to support. For example, the Convention on Biological Diversity requires reporting about extinction rates, so simple indices that reflect extinction risk would be valuable. Here we develop three biodiversity indices that are based on simple models of population viability that relate extinction risk to abundance. The first index is based on the geometric mean abundance of species. A second uses a more general power mean. A third integrates both the geometric mean abundance and trend. These indices require the same data as previous indices, but they also relate directly to extinction risk. Field data for butterflies and woodland plants, and experimental studies of protozoan communities show that the indices correlate with local extinction rates. Applying the index based on the geometric mean to global data on changes in avian abundance suggests that the average extinction probability of birds has increased approximately 1% from 1970 to 2009.\",\"preprint_author_corresponding\":\"Michael A. McCarthy\",\"preprint_author_corresponding_institution\":\"The University of Melbourne\"},{\"preprint_doi\":\"10.1101\\\\/000521\",\"published_doi\":\"10.1111\\\\/evo.12438\",\"published_journal\":\"Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Pathways to social evolution: reciprocity, relatedness, and synergy\",\"preprint_authors\":\"Jeremy Van Cleve;Erol Akcay;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2013-11-16\",\"published_date\":\"2014-06-03\",\"preprint_abstract\":\"Many organisms live in populations structured by space and by class, exhibit plastic responses to their social partners, and are subject to non-additive ecological and fitness effects. Social evolution theory has long recognized that all of these factors can lead to different selection pressures but has only recently attempted to synthesize how these factors interact. Using models for both discrete and continuous phenotypes, we show that analyzing these factors in a consistent framework reveals that they interact with one another in ways previously overlooked. Specifically, behavioral responses (reciprocity), genetic relatedness, and synergy interact in non-trivial ways that cannot be easily captured by simple summary indices of assortment. We demonstrate the importance of these interactions by showing how they have been neglected in previous synthetic models of social behavior both within and between species. These interactions also affect the level of behavioral responses that can evolve in the long run; proximate biological mechanisms are evolutionarily stable when they generate enough responsiveness relative to the level of responsiveness that exactly balances the ecological costs and benefits. Given the richness of social behavior across taxa, these interactions should be a boon for empirical research as they are likely crucial for describing the complex relationship linking ecology, demography, and social behavior.\",\"preprint_author_corresponding\":\"Jeremy  Van Cleve\",\"preprint_author_corresponding_institution\":\"National Evolutionary Synthesis Center\"},{\"preprint_doi\":\"10.1101\\\\/000208\",\"published_doi\":\"10.1101\\\\/gr.169292.113\",\"published_journal\":\"Genome Research\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Population genomics of parallel hybrid zones in the mimetic butterflies, H. melpomene and H. erato\",\"preprint_authors\":\"Nicola Nadeau;Mayte Ruiz;Patricio Salazar;Brian Counterman;Jose Alejandro Medina;Humberto Ortiz-Zuazaga;Anna Morrison;W. Owen McMillan;Chri Jiggins;Riccardo Papa;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2013-11-12\",\"published_date\":\"2014-05-13\",\"preprint_abstract\":\"Hybrid zones can be valuable tools for studying evolution and identifying genomic regions responsible for adaptive divergence and underlying phenotypic variation. Hybrid zones between subspecies of Heliconius butterflies can be very narrow and are maintained by strong selection acting on colour pattern. The co-mimetic species H. erato and H. melpomene have parallel hybrid zones where both species undergo a change from one colour pattern form to another. We use restriction associated DNA sequencing to obtain several thousand genome wide sequence markers and use these to analyse patterns of population divergence across two pairs of parallel hybrid zones in Peru and Ecuador. We compare two approaches for analysis of this type of data; alignment to a reference genome and de novo assembly, and find that alignment gives the best results for species both closely (H. melpomene) and distantly (H. erato, ~15% divergent) related to the reference sequence. Our results confirm that the colour pattern controlling loci account for the majority of divergent regions across the genome, but we also detect other divergent regions apparently unlinked to colour pattern differences. We also use association mapping to identify previously unmapped colour pattern loci, in particular the Ro locus. Finally, we identify within our sample a new cryptic population of H. timareta in Ecuador, which occurs at relatively low altitude and is mimetic with H. melpomene malleti.\",\"preprint_author_corresponding\":\"Chri  Jiggins\",\"preprint_author_corresponding_institution\":\"Cambridge\"},{\"preprint_doi\":\"10.1101\\\\/000042\",\"published_doi\":\"10.1038\\\\/nrg3723\",\"published_journal\":\"Nature Reviews Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Routes for breaching and protecting genetic privacy\",\"preprint_authors\":\"Yaniv Erlich;Arvind Narayanan;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2013-11-07\",\"published_date\":\"2014-06-01\",\"preprint_abstract\":\"We are entering the era of ubiquitous genetic information for research, clinical care, and personal curiosity. Sharing these datasets is vital for rapid progress in understanding the genetic basis of human diseases. However, one growing concern is the ability to protect the genetic privacy of the data originators. Here, we technically map threats to genetic privacy and discuss potential mitigation strategies for privacy-preserving dissemination of genetic data.\\\\\\\\n\\\\\\\\nAbout the AuthorsYaniv Erlich is a Fellow at the Whitehead Institute for Biomedical Research. Erlich received his Ph.D. from Cold Spring Harbor Laboratory in 2010 and B.Sc. from Tel-Aviv University in 2006. Prior to that, Erlich worked in computer security and was responsible for conducting penetration tests on financial institutes and commercial companies. Dr. Erlichs research involves developing new algorithms for computational human genetics.\\\\\\\\n\\\\\\\\nArvind Narayanan is an Assistant Professor in the Department of Computer Science and the Center for Information Technology and Policy at Princeton. He studies information privacy and security. His research has shown that data anonymization is broken in fundamental ways, for which he jointly received the 2008 Privacy Enhancing Technologies Award. His current research interests include building a platform for privacy-preserving data sharing.\\\\\\\\n\\\\\\\\nSummaryO_LIBroad data dissemination is essential for advancements in genetics, but also brings to light concerns regarding privacy.\\\\\\\\nC_LIO_LIPrivacy breaching techniques work by cross-referencing two or more pieces of information to gain new, potentially undesirable knowledge on individuals or their families.\\\\\\\\nC_LIO_LIBroadly speaking, the main routes to breach privacy are identity tracing, attribute disclosure, and completion of sensitive DNA information.\\\\\\\\nC_LIO_LIIdentity tracing exploits quasi-identifiers in the DNA data or metadata to uncover the identity of an unknown genetic dataset.\\\\\\\\nC_LIO_LIAttribute disclosure techniques work on known DNA datasets. They use the DNA information to link the identity of a person with a sensitive phenotype.\\\\\\\\nC_LIO_LICompletion techniques also work on known DNA data. They try to uncover sensitive genomic areas that were masked to protect the participant.\\\\\\\\nC_LIO_LIIn the last few years, we have witnessed a rapid growth in the range of techniques and tools to conduct these privacy-breaching attacks. Currently, most of the techniques are beyond the reach of the general public, but can be executed by trained persons with varying degrees of effort.\\\\\\\\nC_LIO_LIThere is considerable debate regarding risk management. One camp supports a pragmatic, ad-hoc approach of privacy by obscurity and the other supports a systematic, mathematically-backed approach of privacy by design.\\\\\\\\nC_LIO_LIPrivacy by design algorithms include access control, differential privacy, and cryptographic techniques. So far, data custodians of genetic databases mainly adopted access control as a mitigation strategy.\\\\\\\\nC_LIO_LINew developments in cryptographic techniques may usher in an additional arsenal of security by design techniques.\\\\\\\\nC_LI\",\"preprint_author_corresponding\":\"Yaniv  Erlich\",\"preprint_author_corresponding_institution\":\"Whitehead Institute\"},{\"preprint_doi\":\"10.1101\\\\/004739\",\"published_doi\":\"10.1016\\\\/j.patcog.2013.10.014\",\"published_journal\":\"Pattern Recognition\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Automatic Classification of Human Epithelial Type 2 Cell Indirect Immunofluorescence Images using Cell Pyramid Matching\",\"preprint_authors\":\"Arnold Wiliem;Conrad Sanderson;Yongkang Wong;Peter Hobson;Rodney Minchin;Brian Lovell;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2014-05-05\",\"published_date\":\"2014-07-01\",\"preprint_abstract\":\"This paper describes a novel system for automatic classification of images obtained from Anti-Nuclear Antibody (ANA) pathology tests on Human Epithelial type 2 (HEp-2) cells using the Indirect Immunofluorescence (IIF) protocol. The IIF protocol on HEp-2 cells has been the hallmark method to identify the presence of ANAs, due to its high sensitivity and the large range of antigens that can be detected. However, it suffers from numerous shortcomings, such as being subjective as well as time and labour intensive. Computer Aided Diagnostic (CAD) systems have been developed to address these problems, which automatically classify a HEp-2 cell image into one of its known patterns (eg. speckled, homogeneous). Most of the existing CAD systems use handpicked features to represent a HEp-2 cell image, which may only work in limited scenarios. We propose a novel automatic cell image classification method termed Cell Pyramid Matching (CPM), which is comprised of regional histograms of visual words coupled with the Multiple Kernel Learning framework. We present a study of several variations of generating histograms and show the efficacy of the system on two publicly available datasets: the ICPR HEp-2 cell classification contest dataset and the SNPHEp-2 dataset.\",\"preprint_author_corresponding\":\"Conrad  Sanderson\",\"preprint_author_corresponding_institution\":\"NICTA\"},{\"preprint_doi\":\"10.1101\\\\/001958\",\"published_doi\":\"10.1089\\\\/cmb.2014.0029\",\"published_journal\":\"Journal of Computational Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Joint variant and de novo mutation identification on pedigrees from high-throughput sequencing data\",\"preprint_authors\":\"John G Cleary;Ross Braithwaite;Kurt Gaastra;Brian S Hilbush;Stuart Inglis;Sean A Irvine;Alan Jackson;Richard Littin;Sahar Nohzadeh-Malakshah;Minita Shah;Mehul Rathod;David Ware;Len Trigg;Francisco M De La Vega;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2014-01-22\",\"published_date\":\"2014-05-29\",\"preprint_abstract\":\"The analysis of whole-genome or exome sequencing data from trios and pedigrees has being successfully applied to the identification of disease-causing mutations. However, most methods used to identify and genotype genetic variants from next-generation sequencing data ignore the relationships between samples, resulting in significant Mendelian errors, false positives and negatives. Here we present a Bayesian network framework that jointly analyses data from all members of a pedigree simultaneously using Mendelian segregation priors, yet providing the ability to detect de novo mutations in offspring, and is scalable to large pedigrees. We evaluated our method by simulations and analysis of WGS data from a 17 individual, 3-generation CEPH pedigree sequenced to 50X average depth. Compared to singleton calling, our family caller produced more high quality variants and eliminated spurious calls as judged by common quality metrics such as Ti\\\\/Tv, Het\\\\/Hom ratios, and dbSNP\\\\/SNP array data concordance. We developed a ground truth dataset to further evaluate our calls by identifying recombination cross-overs in the pedigree and testing variants for consistency with the inferred phasing, and we show that our method significantly outperforms singleton and population variant calling in pedigrees. We identify all previously validated de novo mutations in NA12878, concurrent with a 7X precision improvement. Our results show that our method is scalable to large genomics and human disease studies and allows cost optimization by rational sequencing capacity distribution.\",\"preprint_author_corresponding\":\"Francisco M De La Vega\",\"preprint_author_corresponding_institution\":\"Real Time Genomics, Inc.\"},{\"preprint_doi\":\"10.1101\\\\/002469\",\"published_doi\":\"10.1186\\\\/1471-2105-15-126\",\"published_journal\":\"BMC Bioinformatics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Automated ensemble assembly and validation of microbial genomes\",\"preprint_authors\":\"Sergey Koren;Todd J Treangen;Christopher M Hill;Mihai Pop;Adam M Phillippy;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2014-02-07\",\"published_date\":\"2014-05-03\",\"preprint_abstract\":\"BackgroundThe continued democratization of DNA sequencing has sparked a new wave of development of genome assembly and assembly validation methods. As individual research labs, rather than centralized centers, begin to sequence the majority of new genomes, it is important to establish best practices for genome assembly. However, recent evaluations such as GAGE and the Assemblathon have concluded that there is no single best approach to genome assembly. Instead, it is preferable to generate multiple assemblies and validate them to determine which is most useful for the desired analysis; this is a labor-intensive process that is often impossible or unfeasible.\\\\\\\\n\\\\\\\\nResultsTo encourage best practices supported by the community, we present iMetAMOS, an automated ensemble assembly pipeline; iMetAMOS encapsulates the process of running, validating, and selecting a single assembly from multiple assemblies. iMetAMOS packages several leading open-source tools into a single binary that automates parameter selection and execution of multiple assemblers, scores the resulting assemblies based on multiple validation metrics, and annotates the assemblies for genes and contaminants. We demonstrate the utility of the ensemble process on 225 previously unassembled Mycobacterium tuberculosis genomes as well as a Rhodobacter sphaeroides benchmark dataset. On these real data, iMetAMOS reliably produces validated assemblies and identifies potential contamination without user intervention. In addition, intelligent parameter selection produces assemblies of R. sphaeroides that exceed the quality of those from the GAGE-B evaluation, affecting the relative ranking of some assemblers.\\\\\\\\n\\\\\\\\nConclusionsEnsemble assembly with iMetAMOS provides users with multiple, validated assemblies for each genome. Although computationally limited to small or mid-sized genomes, this approach is the most effective and reproducible means for generating high-quality assemblies and enables users to select an assembly best tailored to their specific needs.\",\"preprint_author_corresponding\":\"Sergey  Koren\",\"preprint_author_corresponding_institution\":\"National Biodefense Analysis and Countermeasures Center\"},{\"preprint_doi\":\"10.1101\\\\/000158\",\"published_doi\":\"10.1186\\\\/1471-2164-15-398\",\"published_journal\":\"BMC Genomics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Functional Annotation Signatures of Disease Susceptibility Loci Improve SNP Association Analysis\",\"preprint_authors\":\"Edwin S Iversen;Gary Lipton;Merlise A. Clyde;Alvaro N. A. Monteiro;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2013-11-11\",\"published_date\":\"2014-05-24\",\"preprint_abstract\":\"We describe the development and application of a Bayesian statistical model for the prior probability of phenotype-genotype association that incorporates data from past association studies and publicly available functional annotation data regarding the susceptibility variants under study. The model takes the form of a binary regression of association status on a set of annotation variables whose coefficients were estimated through an analysis of associated SNPs housed in the GWAS Catalog (GC). The set of functional predictors we examined includes measures that have been demonstrated to correlate with the association status of SNPs in the GC and some whose utility in this regard is speculative: summaries of the UCSC Human Genome Browser ENCODE super-track data, dbSNP function class, sequence conservation summaries, proximity to genomic variants included in the Database of Genomic Variants (DGV) and known regulatory elements included in the Open Regulatory Annotation database (ORegAnno), PolyPhen-2 probabilities and RegulomeDB categories. Because we expected that only a fraction of the annotation variables would contribute to predicting association, we employed a penalized likelihood method to reduce the impact of non-informative predictors and evaluated the models ability to predict GC SNPs not used to construct the model. We show that the functional data alone are predictive of a SNPs presence in the GC. Further, using data from a genome-wide study of ovarian cancer, we demonstrate that their use as prior data when testing for association is practical at the genome-wide scale and improves power to detect associations.\",\"preprint_author_corresponding\":\"Edwin S Iversen\",\"preprint_author_corresponding_institution\":\"Duke University\"},{\"preprint_doi\":\"10.1101\\\\/004457\",\"published_doi\":\"10.1103\\\\/PhysRevLett.112.208102\",\"published_journal\":\"Physical Review Letters\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Cell size regulation in bacteria\",\"preprint_authors\":\"Ariel Amir;\",\"preprint_category\":\"Biophysics \",\"preprint_date\":\"2014-04-23\",\"published_date\":\"2014-05-23\",\"preprint_abstract\":\"Various bacteria such as the canonical gram negative Escherichia coli or the well-studied gram positive Bacillus subtilis divide symmetrically after they approximately double their volume. Their size at division is not constant, but is typically distributed over a narrow range. Here, we propose an analytically tractable model for cell size control, and calculate the cell size and inter-division time distributions, and the correlations between these variables. We suggest ways of extracting the model parameters from experimental data, and show that existing data for E. coli supports partial size control, and a particular explanation: a cell attempts to add a constant volume from the time of initiation of DNA replication to the next initiation event. This hypothesis accounts for the experimentally observed correlations between mother and daughter cells as well as the exponential dependence of size on growth rate.\\\\\\\\n\\\\\\\\nPACS numbers: 87.17.Ee, 87.17.Aa, 87.10.Mn, 87.81Tt\",\"preprint_author_corresponding\":\"Ariel  Amir\",\"preprint_author_corresponding_institution\":\"Harvard University\"},{\"preprint_doi\":\"10.1101\\\\/002196\",\"published_doi\":\"10.12688\\\\/f1000research.3986\",\"published_journal\":\"F1000Research\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Cytoplasmic nanojunctions between lysosomes and sarcoplasmic reticulum are required for specific calcium signaling\",\"preprint_authors\":\"Nicola Fameli;Oluseye A. Ogunbayo;Cornelis van Breemen;A. Mark Evans;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-01-28\",\"published_date\":\"2014-04-22\",\"preprint_abstract\":\"Herein we demonstrate how nanojunctions between lysosomes and sarcoplasmic reticulum (L-SR junctions) serve to couple lysosomal activation to regenerative, ryanodine receptor-mediated cellular Ca2+ waves. In pulmonary artery smooth muscle cells (PASMCs) it has been proposed that nicotinic acid adenine dinucleotide phosphate (NAADP) triggers increases in cytoplasmic Ca2+ via L-SR junctions, in a manner that requires initial Ca2+ release from lysosomes and subsequent Ca2+-induced Ca2+ release (CICR) via ryanodine receptor (RyR) subtype 3 on the SR membrane proximal to lysosomes. L-SR junction membrane separation has been estimated to be &lt; 400 nm and thus beyond the resolution of light microscopy, which has restricted detailed investigations of the junctional coupling process. The present study utilizes standard and tomographic transmission electron microscopy to provide a thorough ultrastructural characterization of the L-SR junctions in PASMCs. We show that L-SR nanojunctions are prominent features within these cells and estimate that the junctional membrane separation and extension are about 15 nm and 300 nm, respectively. Furthermore, we develop a quantitative model of the L-SR junction using these measurements, prior kinetic and specific Ca2+ signal information as input data. Simulations of NAADP-dependent junctional Ca2+ transients demonstrate that the magnitude of these signals can breach the threshold for CICR via RyR3. By correlation analysis of live cell Ca2+ signals and simulated Ca2+ transients within L-SR junctions, we estimate that \\\\\\\\\\\\\"trigger zones\\\\\\\\\\\\\" with a 60-100 junctions are required to confer a signal of similar magnitude. This is compatible with the 130 lysosomes\\\\/cell estimated from our ultrastructural observations. Most importantly, our model shows that increasing the L-SR junctional width above 50 nm lowers the magnitude of junctional [Ca2+] such that there is a failure to breach the threshold for CICR via RyR3. L-SR junctions are therefore a pre-requisite for efficient Ca2+ signal coupling and may contribute to cellular function in health and disease.\",\"preprint_author_corresponding\":\"Nicola  Fameli\",\"preprint_author_corresponding_institution\":\"University of British Columbia\"},{\"preprint_doi\":\"10.1101\\\\/002790\",\"published_doi\":\"10.1371\\\\/journal.pone.0093987\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"ESCRT-0 is not required for ectopic Notch activation and tumor suppression in Drosophila\",\"preprint_authors\":\"Emiliana Tognon;Nadine Wollscheid;Katia Cortese;Carlo Tacchetti;thomas vaccari;\",\"preprint_category\":\"Developmental Biology \",\"preprint_date\":\"2014-02-18\",\"published_date\":\"2014-04-09\",\"preprint_abstract\":\"Multivesicular endosome (MVE) sorting depends on proteins of the Endosomal Sorting Complex Required for Transport (ESCRT) family. These are organized in four complexes (ESCRT-0, -I, -II, -III) that act in a sequential fashion to deliver ubiquitylated cargoes into the internal luminal vesicles (ILVs) of the MVE. Drosophila genes encoding ESCRT-I, -II, -III components function in sorting signaling receptors, including Notch and the JAK\\\\/STAT signaling receptor Domeless. Loss of ESCRT-I, -II, -III in Drosophila epithelia causes altered signaling and cell polarity, suggesting that ESCRTs genes are tumor suppressors. However, the nature of the tumor suppressive function of ESCRTs, and whether tumor suppression is linked to receptor sorting is unclear. Unexpectedly, a null mutant in Hrs, encoding one of components of the ESCRT-0 complex, which acts upstream of ESCRT-I, -II, -III in MVE sorting is dispensable for tumor suppression. Here, we report that two Drosophila epithelia lacking activity of Stam, the other known components of the ESCRT-0 complex, or of both Hrs and Stam fail to degrade signaling receptors. However, mutant tissue surprisingly maintains normal apico-basal polarity and proliferation control and does not display ectopic Notch signaling activation, unlike cells that lack ESCRT-I, -II, -III activity. Overall, our in vivo data indicate that the ESCRT-0 complex plays no crucial role in regulation of tumor suppression, and suggest re-evaluation of the relationship of signaling modulation in endosomes and tumorigenesis.\",\"preprint_author_corresponding\":\"thomas  vaccari\",\"preprint_author_corresponding_institution\":\"IFOM, Istituto FIRC di Oncologia Molecolare\"},{\"preprint_doi\":\"10.1101\\\\/004820\",\"published_doi\":\"10.1111\\\\/1365-2745.12259\",\"published_journal\":\"Journal of Ecology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Some plants don&amp;#146;t play games: An ideal free distribution explains the root production of plants that do not engage in a tragedy of the commons game.\",\"preprint_authors\":\"Gordon McNickle;Joel S Brown;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2014-05-06\",\"published_date\":\"2014-05-07\",\"preprint_abstract\":\"This is the pre-peer-reviewed version of the following article: G.G. McNickle and J.S. Brown. (2014) An ideal free distribution explains the root production of plants that do not engage in a tragedy of the commons game. Journal of Ecology. DOI: 10.1111\\\\/1365-2745.12259, which has been published in final form at http:\\\\/\\\\/onlinelibrary.wiley.com\\\\/doi\\\\/10.1111\\\\/1365-2745.12259\\\\/abstract\\\\\\\\n\\\\\\\\nO_LIGame theoretic models that seek to predict the most competitive strategy plants use for competition in soil are clear; they generally predict that over-proliferation of roots is the only evolutionarily stable strategy. However, empirical studies are equally clear that not all plants employ this strategy of over-proliferation of roots. Here, our goal was to develop and test an alternative non-game theoretic model that can be used to develop alternative hypotheses for plants that do not appear to play games.\\\\\\\\nC_LIO_LIThe model is similar to previous models, but does not use a game theoretic optimization criterion. Instead, plants use only nutrient availability to select a root allocation strategy, ignoring neighbours. To test the model we compare root allocation and seed yield of plants grown either alone or with neighbours.\\\\\\\\nC_LIO_LIThe model predicted plants that do not sense neighbours (or ignore neighbours) should allocate roots relative to resource availability following an ideal free distribution. This means that if a soil volume of quality R contains x roots, then a soil volume of quality R\\\\/n will contain x\\\\/n roots. The experimental data were consistent with this prediction. That is, plants grown with 1.2g of slow release fertilizer resources produced 0.043 g of roots, while plants grown with neighbours, or plants grown with half as much fertilizer produced half as much root mass (0.026g, and 0.24g respectively). Seed yield followed a similar pattern.\\\\\\\\nC_LIO_LIThis model presents an alternative predictive framework for those plant species that do not seem to play a tragedy of the commons game for belowground competition.\\\\\\\\nC_LIO_LISynthesis: It remains unclear why some plants do not engage in belowground games for competition. Models suggest over-proliferation is an unbeatable evolutionary stable strategy, yet plants that do not play the game apparently coexist with plants that do. We suggest that a greater understanding of trade-offs among traits that are important for other biotic interactions (above-ground competition, enemy defence, mutualisms) will lead to a greater understanding of why some species over-proliferate roots when in competition but other species do not.\\\\\\\\nC_LI\",\"preprint_author_corresponding\":\"Gordon  McNickle\",\"preprint_author_corresponding_institution\":\"Wilfrid Laurier University\"},{\"preprint_doi\":\"10.1101\\\\/001941\",\"published_doi\":\"10.1111\\\\/2041-210X.12206\",\"published_journal\":\"Methods in Ecology and Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Particle size distribution and optimal capture of aqueous macrobial eDNA\",\"preprint_authors\":\"Cameron R. Turner;Matthew A. Barnes;Charles C.Y. Xu;Stuart E. Jones;Christopher L. Jerde;David M. Lodge;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2014-01-21\",\"published_date\":\"2014-05-01\",\"preprint_abstract\":\"O_LIDetecting aquatic macroorganisms with environmental DNA (eDNA) is a new survey method with broad applicability. However, the origin, state, and fate of aqueous macrobial eDNA - which collectively determine how well eDNA can serve as a proxy for directly observing organisms and how eDNA should be captured, purified, and assayed - are poorly understood.\\\\\\\\nC_LIO_LIThe size of aquatic particles provides clues about their origin, state, and fate. We used sequential filtration size fractionation to measure, for the first time, the particle size distribution (PSD) of macrobial eDNA, specifically Common Carp (hereafter referred to as Carp) eDNA. We compared it to the PSDs of total eDNA (from all organisms) and suspended particle matter (SPM). We quantified Carp mitochondrial eDNA using a custom qPCR assay, total eDNA with fluorometry, and SPM with gravimetric analysis.\\\\\\\\nC_LIO_LIIn a lake and a pond, we found Carp eDNA in particles from &gt;180 to &lt;0.2 m, but it was most abundant from 1-10 m. Total eDNA was most abundant below 0.2 m and SPM was most abundant above 100 m. SPM was []0.1% total eDNA, and total eDNA was []0.0004% Carp eDNA. 0.2 m filtration maximized Carp eDNA capture (85%{+\\\\/-}6%) while minimizing total (i.e., non-target) eDNA capture (48%{+\\\\/-}3%), but filter clogging limited this pore size to a volume &lt;250 mL. To mitigate this limitation we estimated a continuous PSD model for Carp eDNA and derived an equation for calculating isoclines of pore size and water volume that yield equivalent amounts of Carp eDNA.\\\\\\\\nC_LIO_LIOur results suggest that aqueous macrobial eDNA predominantly exists inside mitochondria or cells, and that settling plays an important role in its fate. For optimal eDNA capture, we recommend 0.2 m filtration or a combination of larger pore size and water volume that exceeds the 0.2 m isocline. In situ filtration of large volumes could maximize detection probability when surveying large habitats for rare organisms. Our method for eDNA particle size analysis enables future research to compare the PSDs of eDNA from other organisms and environments, and to easily apply them for ecological monitoring.\\\\\\\\nC_LI\",\"preprint_author_corresponding\":\"Cameron R. Turner\",\"preprint_author_corresponding_institution\":\"University of Notre Dame\"},{\"preprint_doi\":\"10.1101\\\\/004135\",\"published_doi\":\"10.1016\\\\/j.evolhumbehav.2014.04.003\",\"published_journal\":\"Evolution and Human Behavior\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Direct Reciprocity Under Uncertainty Does Not Explain One-shot Cooperation, but Demonstrates the Benefits of a Norm Psychology\",\"preprint_authors\":\"Matthew Zefferman;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-04-11\",\"published_date\":\"2014-04-27\",\"preprint_abstract\":\"Humans in many societies cooperate in economic experiments at much higher levels than would be expected if their goal was maximizing economic returns, even when their interactions are anonymous and one-shot. This is a puzzle because paying a cost to benefit another in one-shot interactions gives no direct or indirect benefits to the cooperator. This paper explores the logic of two competing evolutionary hypotheses to explain this behavior. The \\\\\\\\\\\\\"norm psychology\\\\\\\\\\\\\" hypothesis holds that a player\\'s choice of strategy reflects socially-learned cultural norms. Its premise is that over the course of human evolutionary history, cultural norms varied considerably across human societies and through a process of gene-culture co-evolution, humans evolved mechanisms to learn and adopt the norms that are successful in their particular society. The \\\\\\\\\\\\\"mismatch\\\\\\\\\\\\\" hypothesis holds that pro-social preferences evolved genetically in our hunter-gatherer past where one-shot anonymous interactions were rare and these preferences are misapplied in modern laboratory settings. I compare these hypotheses by adopting a well-known model of the mismatch hypothesis and show that selection for one-shot cooperation in the model is an artifact of agents being constrained to only two strategies: Tit-for-Tat and Always Defect. Allowing for repentant and forgiving strategies reverses selection away from one-shot cooperation under all environmental parameters. Direct reciprocity does not necessarily lead to cooperation, but instead generates many different normative equilibria depending on a groups idiosyncratic evolutionary history. Therefore, an agent whose behavior is evoked solely from non-cultural environmental cues will be disadvantaged relative to an agent who learns the locally successful norms. Cooperation in one-shot laboratory experiments is thus more easily explained as the result of a psychology evolved for learning social norms than as a genetic mismatch.\",\"preprint_author_corresponding\":\"Matthew  Zefferman\",\"preprint_author_corresponding_institution\":\"National Institute for Mathematical and Biological Synthesis, University of Tennessee\"},{\"preprint_doi\":\"10.1101\\\\/001933\",\"published_doi\":\"10.1111\\\\/mec.12755\",\"published_journal\":\"Molecular Ecology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Coalescence 2.0: a multiple branching of recent theoretical developments and their applications\",\"preprint_authors\":\"Aurelien Tellier;Christophe Lemaire;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-01-21\",\"published_date\":\"2014-05-20\",\"preprint_abstract\":\"Population genetics theory has laid the foundations for genomics analyses including the recent burst in genome scans for selection and statistical inference of past demographic events in many prokaryote, animal and plant species. Identifying SNPs under natural selection and underpinning species adaptation relies on disentangling the respective contribution of random processes (mutation, drift, migration) from that of selection on nucleotide variability. Most theory and statistical tests have been developed using the Kingmans coalescent theory based on the Wright-Fisher population model. However, these theoretical models rely on biological and life-history assumptions which may be violated in many prokaryote, fungal, animal or plant species. Recent theoretical developments of the so called multiple merger coalescent models are reviewed here ({Lambda}-coalescent, beta-coalescent, Bolthausen-Snitzman, {Xi}-coalescent). We explicit how these new models take into account various pervasive ecological and biological characteristics, life history traits or life cycles which were not accounted in previous theories such as 1) the skew in offspring production typical of marine species, 2) fast adapting microparasites (virus, bacteria and fungi) exhibiting large variation in population sizes during epidemics, 3) the peculiar life cycles of fungi and bacteria alternating sexual and asexual cycles, and 4) the high rates of extinction-recolonization in spatially structured populations. We finally discuss the relevance of multiple merger models for the detection of SNPs under selection in these species, for population genomics of very large sample size and advocate to potentially examine the conclusion of previous population genetics studies.\",\"preprint_author_corresponding\":\"Aurelien  Tellier\",\"preprint_author_corresponding_institution\":\"Technische Universit?t M?nchen\"},{\"preprint_doi\":\"10.1101\\\\/005223\",\"published_doi\":\"10.1016\\\\/j.evolhumbehav.2014.04.001\",\"published_journal\":\"Evolution and Human Behavior\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Strategic Social Learning and the Population Dynamics of Human Behavior: The Game of Go\",\"preprint_authors\":\"Bret A Beheim;Calvin Thigpen;Richard McElreath;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-05-16\",\"published_date\":\"2014-04-13\",\"preprint_abstract\":\"Human culture is widely believed to undergo evolution, via mechanisms rooted in the nature of human cognition. A number of theories predict the kinds of human learning strategies, as well as the population dynamics that result from their action. There is little work, however, that quantitatively examines the evidence for these strategies and resulting cultural evolution within human populations. One of the obstacles is the lack of individual-level data with which to link transmission events to larger cultural dynamics. Here, we address this problem with a rich quantitative database from the East Asian board game known as Go. We draw from a large archive of Go games spanning the last six decades of professional play, and find evidence that the evolutionary dynamics of particular cultural variants are driven by a mix of individual and social learning processes. Particular players vary dramatically in their sensitivity to population knowledge, which also varies by age and nationality. The dynamic patterns of opening Go moves are consistent with an ancient, ongoing arms race within the game itself.\",\"preprint_author_corresponding\":\"Bret A Beheim\",\"preprint_author_corresponding_institution\":\"University of New Mexico\"},{\"preprint_doi\":\"10.1101\\\\/002063\",\"published_doi\":\"10.1371\\\\/journal.pone.0086169\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Holsteins Favor Heifers, Not Bulls: Biased Milk Production Programmed during Pregnancy as a Function of Fetal Sex\",\"preprint_authors\":\"Katie Hinde;Abigail J Carpenter;John C Clay;Barry J Bradford;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-01-24\",\"published_date\":\"2014-02-03\",\"preprint_abstract\":\"Mammalian females pay high energetic costs for reproduction, the greatest of which is imposed by lactation. The synthesis of milk requires, in part, the mobilization of bodily reserves to nourish developing young. Numerous hypotheses have been advanced to predict how mothers will differentially invest in sons and daughters, however few studies have addressed sex-biased milk synthesis. Here we leverage the dairy cow model to investigate such phenomena. Using 2.39 million lactation records from 1.49 million dairy cows, we demonstrate that the sex of the fetus influences the capacity of the mammary gland to synthesize milk during lactation. Cows favor daughters, producing significantly more milk for daughters than for sons across lactation. Using a sub-sample of this dataset (N = 113,750 subjects) we further demonstrate that the effects of fetal sex interact dynamically across parities, whereby the sex of the fetus being gestated can enhance or diminish the production of milk during an established lactation. Moreover the sex of the fetus gestated on the first parity has persistent consequences for milk synthesis on the subsequent parity. Specifically, gestation of a daughter on the first parity increases milk production by [~]445 kg over the first two lactations. Our results identify a dramatic and sustained programming of mammary function by offspring in utero. Nutritional and endocrine conditions in utero are known to have pronounced and long-term effects on progeny, but the ways in which the progeny has sustained physiological effects on the dam have received little attention to date.\",\"preprint_author_corresponding\":\"Katie  Hinde\",\"preprint_author_corresponding_institution\":\"Harvard University\"},{\"preprint_doi\":\"10.1101\\\\/001073\",\"published_doi\":\"10.1534\\\\/genetics.114.164350\",\"published_journal\":\"Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Variational Inference of Population Structure in Large SNP Datasets\",\"preprint_authors\":\"Anil Raj;Matthew Stephens;Jonathan K Pritchard;\",\"preprint_category\":\"Genetics \",\"preprint_date\":\"2013-12-02\",\"published_date\":\"2014-04-02\",\"preprint_abstract\":\"Tools for estimating population structure from genetic data are now used in a wide variety of applications in population genetics. However, inferring population structure in large modern data sets imposes severe computational challenges. Here, we develop efficient algorithms for approximate inference of the model underlying the STRUCTURE program using a variational Bayesian framework. Variational methods pose the problem of computing relevant posterior distributions as an optimization problem, allowing us to build on recent advances in optimization theory to develop fast inference tools. In addition, we propose useful heuristic scores to identify the number of populations represented in a dataset and a new hierarchical prior to detect weak population structure in the data. We test the variational algorithms on simulated data, and illustrate using genotype data from the CEPH-Human Genome Diversity Panel. The variational algorithms are almost two orders of magnitude faster than STRUCTURE and achieve accuracies comparable to those of ADMIXTURE. Furthermore, our results show that the heuristic scores for choosing model complexity provide a reasonable range of values for the number of populations represented in the data, with minimal bias towards detecting structure when it is very weak. Our algorithm, fastSTRUCTURE, is freely available online at http:\\\\/\\\\/pritchardlab.stanford.edu\\\\/structure.html.\",\"preprint_author_corresponding\":\"Anil  Raj\",\"preprint_author_corresponding_institution\":\"Stanford University\"},{\"preprint_doi\":\"10.1101\\\\/004853\",\"published_doi\":\"10.1371\\\\/journal.pgen.1004402\",\"published_journal\":\"PLOS Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"MOSAIC EPIGENETIC DYSREGULATION OF ECTODERMAL CELLS IN AUTISM SPECTRUM DISORDER\",\"preprint_authors\":\"Esther R. Berko;Masako Suzuki;Faygel Beren;Christophe Lemetre;Christine M. Alaimo;R. Brent Calder;Karen Ballaban-Gil;Batya Gounder;Kaylee Kampf;Jill Kirschen;Shahina B. Maqbool;Zeineen Momin;David M. Reynolds;Natalie Russo;Lisa Shulman;Edyta Stasiek;Jessica Tozour;Maria Valicenti-McDermott;Shenglong Wang;Brett S. Abrahams;Joseph Hargitai;Dov Inbar;Zhengdong Zhang;Joseph D. Buxbaum;Sophie Molholm;John J. Foxe;Robert W. Marion;Adam Auton;John Greally;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2014-05-06\",\"published_date\":\"2014-05-29\",\"preprint_abstract\":\"DNA mutational events are increasingly being identified in autism spectrum disorder (ASD), but the potential additional role of dysregulation of the epigenome in the pathogenesis of the condition remains unclear. The epigenome is of interest as a possible mediator of environmental effects during development, encoding a cellular memory reflected by altered function of progeny cells. Advanced maternal age (AMA) is associated with an increased risk of having a child with ASD for reasons that are not understood. To explore whether AMA involves covert aneuploidy or epigenetic dysregulation leading to ASD in the offspring, we tested an homogeneous ectodermal cell type from 47 individuals with ASD compared with 48 typically developing (TD) controls born to mothers of []35 years, using a quantitative genome-wide DNA methylation assay. We show that DNA methylation patterns are dysregulated in ectodermal cells in these individuals, having accounted for confounding effects due to subject age, sex and ancestral haplotype. We did not find mosaic aneuploidy or copy number variability to occur at differentially-methylated regions in these subjects. Of note, the loci with distinctive DNA methylation were found at genes expressed in the brain and encoding protein products significantly enriched for interactions with those produced by known ASD-causing genes, representing a perturbation by epigenomic dysregulation of the same networks compromised by DNA mutational mechanisms. The results indicate the presence of a mosaic subpopulation of epigenetically-dysregulated, ectodermally-derived cells in subjects with ASD. The epigenetic dysregulation observed in these ASD subjects born to older mothers may be associated with aging parental gametes, environmental influences during embryogenesis or could be the consequence of mutations of the chromatin regulatory genes increasingly implicated in ASD. The results indicate that epigenetic dysregulatory mechanisms may complement and interact with DNA mutations in the pathogenesis of the disorder.\\\\\\\\n\\\\\\\\nAUTHOR SUMMARYOlder mothers have a higher than expected risk of having a child with an autism spectrum disorder (ASD). The reason for this increased risk is unknown. The eggs of older mothers are more prone to abnormalities of chromosome numbers, suggesting this as one possible mechanism of the increased ASD risk. Age is also associated with a loss of control of epigenetic regulatory patterns that govern gene expression, indicating a second potential mechanism. To test both possibilities, we sampled cells from the same developmental origin as the brain, and performed genome-wide tests looking for unusual chromosome numbers and DNA methylation patterns. The studies were performed on individuals with ASD and typically developing controls, all born to mothers at least 35 years of age at the time of birth. We found the cells from individuals with ASD to have changes in DNA methylation at a number of loci, especially near genes encoding proteins known to interact with those already implicated in ASD. We conclude that epigenetic dysregulation occurring in gametes or early embryonic life may be one of the contributors to the development of ASD.\",\"preprint_author_corresponding\":\"John  Greally\",\"preprint_author_corresponding_institution\":\"Albert Einstein College of Medicine\"},{\"preprint_doi\":\"10.1101\\\\/001388\",\"published_doi\":\"10.1093\\\\/molbev\\\\/msu121\",\"published_journal\":\"Molecular Biology and Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Bayesian inference of infectious disease transmission from whole genome sequence data\",\"preprint_authors\":\"Xavier Didelot;Jennifer Gardy;Caroline Colijn;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2013-12-16\",\"published_date\":\"2014-04-08\",\"preprint_abstract\":\"Genomics is increasingly being used to investigate disease outbreaks, but an important question remains unanswered - how well do genomic data capture known transmission events, particularly for pathogens with long carriage periods or large within-host population sizes? Here we present a novel Bayesian approach to reconstruct densely-sampled outbreaks from genomic data whilst considering within-host diversity. We infer a time-labelled phylogeny using BEAST, then infer a transmission network via a Monte-Carlo Markov Chain. We find that under a realistic model of within-host evolution, reconstructions of simulated outbreaks contain substantial uncertainty even when genomic data reflect a high substitution rate. Reconstruction of a real-world tuberculosis outbreak displayed similar uncertainty, although the correct source case and several clusters of epidemiologically linked cases were identified. We conclude that genomics cannot wholly replace traditional epidemiology, but that Bayesian reconstructions derived from sequence data may form a useful starting point for a genomic epidemiology investigation.\",\"preprint_author_corresponding\":\"Xavier  Didelot\",\"preprint_author_corresponding_institution\":\"Imperial College London\"},{\"preprint_doi\":\"10.1101\\\\/003533\",\"published_doi\":\"10.1016\\\\/j.bbrc.2014.04.093\",\"published_journal\":\"Biochemical and Biophysical Research Communications\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Rice BiP3 regulates immunity mediated by the PRRs XA3 and XA21 but not immunity mediated by the NB-LRR protein, Pi5\",\"preprint_authors\":\"Chang-Jin Park;Min-Young Song;Chi-Yeol Kim;Jong-Seong Jeon;Pamela C. Ronald;\",\"preprint_category\":\"Plant Biology \",\"preprint_date\":\"2014-03-26\",\"published_date\":\"2014-05-23\",\"preprint_abstract\":\"Plant innate immunity is mediated by pattern recognition receptors (PRRs) and intracellular NB-LRR (nucleotide-binding domain and leucine-rich repeat) proteins. Overexpression of the endoplasmic reticulum (ER) chaperone, luminal-binding protein 3 (BiP3) compromises resistance to Xanthomonas oryzae pv. oryzae (Xoo) mediated by the rice PRR XA21 (Park et al., PLoS ONE 5(2): e9262). Here we show that BiP3 overexpression also compromises resistance mediated by rice XA3, a PRR that provides broad-spectrum resistance to Xoo. In contrast, BiP3 overexpression has no effect on resistance mediated by rice Pi5, an NB-LRR protein that confers resistance to the fungal pathogen Magnaporthe oryzae (M. oryzae). Our results suggest that rice BiP3 regulates membrane-resident PRR-mediated immunity.\",\"preprint_author_corresponding\":\"Pamela C. Ronald\",\"preprint_author_corresponding_institution\":\"Department of Plant Pathology and the Genome Center, University of California Davis\"},{\"preprint_doi\":\"10.1101\\\\/000448\",\"published_doi\":\"10.1021\\\\/sb500024b\",\"published_journal\":\"ACS Synthetic Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Design and implementation of a synthetic biomolecular concentration tracker\",\"preprint_authors\":\"Victoria Hsiao;Emmanuel LC de los Santos;Weston R Whitaker;John E Dueber;Richard M Murray;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2013-11-15\",\"published_date\":\"2014-05-15\",\"preprint_abstract\":\"As a field, synthetic biology strives to engineer increasingly complex artificial systems in living cells. Active feedback in closed loop systems offers a dynamic and adaptive way to ensure constant relative activity independent of intrinsic and extrinsic noise. In this work, we design, model, and implement a biomolecular concentration tracker, in which an output protein tracks the concentration of an input protein. Synthetic modular protein scaffold domains are used to colocalize a two-component system, and a single negative feedback loop modulates the production of the output protein. Using a combination of model and experimental work, we show that the circuit achieves real-time protein concentration tracking in Escherichia coli and that steady state outputs can be tuned.\",\"preprint_author_corresponding\":\"Victoria  Hsiao\",\"preprint_author_corresponding_institution\":\"California Institute of Technology\"},{\"preprint_doi\":\"10.1101\\\\/000380\",\"published_doi\":\"10.1016\\\\/j.biochi.2012.05.024\",\"published_journal\":\"Biochimie\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A model of flux regulation in the cholesterol biosynthesis pathway: Immune mediated graduated flux reduction versus statin-like led stepped flux reduction\",\"preprint_authors\":\"Steven Watterson;Maria-Luisa Guerriero;Mathieu Blanc;Alexander Mazein;Laurence Loewe;Kevin Robertson;Holly Gibbs;Guanghou Shui;Markus Wenk;Jane Hillston;Peter Ghazal;\",\"preprint_category\":\"Systems Biology \",\"preprint_date\":\"2013-11-14\",\"published_date\":\"2014-03-01\",\"preprint_abstract\":\"Graphical Abstract\\\\\\\\n\\\\\\\\nO_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=37 SRC=\\\\\\\\\\\\\"FIGDIR\\\\/small\\\\/000380_ufig1.gif\\\\\\\\\\\\\" ALT=\\\\\\\\\\\\\"Figure 1\\\\\\\\\\\\\"&gt;\\\\\\\\nView larger version (14K):\\\\\\\\norg.highwire.dtl.DTLVardef@197f26org.highwire.dtl.DTLVardef@1eac3f3org.highwire.dtl.DTLVardef@1e698f1org.highwire.dtl.DTLVardef@42fb4b_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIWe model the cholesterol biosynthesis pathway and its regulation\\\\\\\\nC_LIO_LIThe innate immune response leads to a suppression of flux through the pathway\\\\\\\\nC_LIO_LIStatin inhibitors show a different mode of suppression to the immune response\\\\\\\\nC_LIO_LIStatin inhibitor suppression is less robust and less specific than immune suppression\\\\\\\\nC_LI\\\\\\\\n\\\\\\\\nAsbtractThe cholesterol biosynthesis pathway has recently been shown to play an important role in the innate immune response to viral infection with host protection occurring through a coordinate down regulation of the enzymes catalyzing each metabolic step. In contrast, statin based drugs, which form the principle pharmaceutical agents for decreasing the activity of this pathway, target a single enzyme. Here, we build an ordinary differential equation model of the cholesterol biosynthesis pathway in order to investigate how the two regulatory strategies impact upon the behaviour of the pathway. We employ a modest set of assumptions: that the pathway operates away from saturation, that each metabolite is involved in multiple cellular interactions and that mRNA levels reflect enzyme concentrations. Using data taken from primary bone marrow derived macrophage cells infected with murine cytomegalovirus infection or treated with IFN{gamma}, we show that, under these assumptions, coordinate down regulation of enzyme activity imparts a graduated reduction in flux along the pathway. In contrast, modelling a statin-like treatment that achieves the same degree of down-regulation in cholesterol production, we show that this delivers a step change in flux along the pathway. The graduated reduction mediated by physiological coordinate regulation of multiple enzymes supports a mechanism that allows a greater level of specificity, altering cholesterol levels with less impact upon interactions branching from the pathway, than pharmacological step reductions. We argue that coordinate regulation is likely to show a long-term evolutionary advantage over single enzyme regulation. Finally, the results from our models have implications for future pharmaceutical therapies intended to target cholesterol production with greater specificity and fewer off target effects, suggesting that this can be achieved by mimicking the coordinated down-regulation observed in immunological responses.\",\"preprint_author_corresponding\":\"Steven  Watterson\",\"preprint_author_corresponding_institution\":\"University of Ulster\"},{\"preprint_doi\":\"10.1101\\\\/000927\",\"published_doi\":\"10.1371\\\\/journal.pone.0097272\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Investigating the relation between stochastic differentiation and homeostasis in intestinal crypts via multiscale modeling\",\"preprint_authors\":\"Alex Graudenzi;Giulio Caravagna;Giovanni De Matteis;Marco Antoniotti;\",\"preprint_category\":\"Systems Biology \",\"preprint_date\":\"2013-11-25\",\"published_date\":\"2014-05-28\",\"preprint_abstract\":\"Colorectal tumors originate and develop within intestinal crypts. Even though some of the essential phenomena that characterize crypt structure and dynamics have been effectively described in the past, the relation between the differentiation process and the overall crypt homeostasis is still partially understood. We here investigate this relation and other important biological phenomena by introducing a novel multiscale model that combines a morphological description of the crypt with a gene regulation model: the emergent dynamical behavior of the underlying gene regulatory network drives cell growth and differentiation processes, linking the two distinct spatio-temporal levels. The model relies on a few a priori assumptions, yet accounting for several key processes related to crypt functioning, such as: dynamic gene activation patterns, stochastic differentiation, signaling pathways ruling cell adhesion properties, cell displacement, cell growth, mitosis, apoptosis and the presence of biological noise.\\\\\\\\n\\\\\\\\nWe show that this modeling approach captures the major dynamical phenomena that characterize the regular physiology of crypts, such as cell sorting, coordinate migration, dynamic turnover, stem cell niche maintenance and clonal expansion. All in all, the model suggests that the process of stochastic differentiation might be sufficient to drive the crypt to homeostasis, under certain crypt configurations. Besides, our approach allows to make precise quantitative inferences that, when possible, were matched to the current biological knowledge and it permits to investigate the role of gene-level perturbations, with reference to cancer development. We also remark the theoretical framework is general and may applied to different tissues, organs or organisms.\",\"preprint_author_corresponding\":\"Alex  Graudenzi\",\"preprint_author_corresponding_institution\":\"Dept. of Informatics, Systems and Communication\"},{\"preprint_doi\":\"10.1101\\\\/004317\",\"published_doi\":\"10.1093\\\\/nar\\\\/gku511\",\"published_journal\":\"Nucleic Acids Research\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Comprehensive mutational scanning of a kinase in vivo reveals substrate-dependent fitness landscapes\",\"preprint_authors\":\"Alexandre Melnikov;Peter Rogov;Li Wang;Andreas Gnirke;Tarjei S Mikkelsen;\",\"preprint_category\":\"Molecular Biology \",\"preprint_date\":\"2014-04-18\",\"published_date\":\"2014-06-09\",\"preprint_abstract\":\"Deep mutational scanning has emerged as a promising tool for mapping sequence-activity relationships in proteins1-4, RNA5 and DNA6-8. In this approach, diverse variants of a sequence of interest are first ranked according to their activities in a relevant pooled assay, and this ranking is then used to infer the shape of the fitness landscape around the wild-type sequence. Little is currently know, however, about the degree to which such fitness landscapes are dependent on the specific assay conditions from which they are inferred. To explore this issue, we performed deep mutational scanning of APH(3)II, a Tn5 transposon-derived kinase that confers resistance to aminoglycoside antibiotics9, in E. coli under selection with each of six structurally diverse antibiotics at a range of inhibitory concentrations. We found that the resulting fitness landscapes showed significant dependence on both antibiotic structure and concentration. This shows that the notion of essential amino acid residues is context-dependent, but also that this dependence can be exploited to guide protein engineering. Specifically, we found that differential analysis of fitness landscapes allowed us to generate synthetic APH(3)II variants with orthogonal substrate specificities.\",\"preprint_author_corresponding\":\"Tarjei S Mikkelsen\",\"preprint_author_corresponding_institution\":\"Broad Institute\"},{\"preprint_doi\":\"10.1101\\\\/002717\",\"published_doi\":\"10.7717\\\\/peerj.434\",\"published_journal\":\"PeerJ\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Immune stimulation reduces sleep and memory ability in Drosophila melanogaster\",\"preprint_authors\":\"Eamonn Mallon;Akram Alghamdi;Robert Holdbrook;Ezio Rosato;\",\"preprint_category\":\"Zoology \",\"preprint_date\":\"2014-02-14\",\"published_date\":\"2014-06-10\",\"preprint_abstract\":\"Psychoneuroimmunology studies the increasing number of connections between neurobiology and immunology. We establish Drosophila melanogaster as a tractable model in this field by demonstrating the effects of the immune response on two fundamental behaviours: sleep and memory ability.\\\\\\\\n\\\\\\\\nWe used the Geneswitch system to upregulate peptidoglycan receptor protein (PGRP) expression, thereby stimulating the immune system in the absence of infection. Geneswitch was activated by feeding the steroid RU486, to the flies. Importantly, by stimulating the immune system of adult flies in the absence of infection we have avoided the added complications of developmental and disease effects that have confounded other studies. We used an aversive classical conditioning paradigm to quantify memory and measures of activity to infer sleep.\\\\\\\\n\\\\\\\\nImmune stimulated flies exhibited reduced levels of sleep, which could not be explained by a generalised increase in waking activity. The effects on sleep were more pronounced for day compared to night sleep. Immune stimulated flies also showed a reduction in memory abilities.\\\\\\\\n\\\\\\\\nThese are important results as they establish Drosophila as a model for immune-neural interactions and provide a possible role for sleep in the interplay between the immune response and memory.\",\"preprint_author_corresponding\":\"Eamonn  Mallon\",\"preprint_author_corresponding_institution\":\"University of Leicester\"},{\"preprint_doi\":\"10.1101\\\\/002071\",\"published_doi\":\"10.1007\\\\/s12035-014-8770-1\",\"published_journal\":\"Molecular Neurobiology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"A microRNA profile in Fmr1 knockout mice reveals microRNA expression alterations with possible roles in fragile X syndrome\",\"preprint_authors\":\"Ting Liu;Rui-Ping Wan;Ling-Jia Tang;Shu-Jing Liu;Hai-Jun Li;Qi-Hua Zhao;Wei-Ping Liao;Xiao-Fang Sun;Yong-Hong Yi;Yue-Sheng Long;\",\"preprint_category\":\"Neuroscience \",\"preprint_date\":\"2014-01-26\",\"published_date\":\"2014-06-07\",\"preprint_abstract\":\"Fragile X syndrome (FXS), a common form of inherited mental retardation, is caused by a loss of expression of the fragile X mental retardation protein (FMRP). FMRP is involved in brain functions by interacting with mRNAs and microRNAs (miRNAs) that selectively control gene expression at translational level. However, little is known about the role of FMRP in regulating miRNA expression. Here, we found a development-dependant dynamic expression of Fmr1 mRNA (encoding FMRP) in mouse hippocampus with a small peak at postnatal day 7 (P7). MiRNA microarray analysis showed that the levels of 38 miRNAs showed a significant increase with about 15~250 folds and the levels of 26 miRNAs showed a significant decrease with only about 2~4 folds in the hippocampus of P7 Fmr1 KO mice. Q-PCR assay showed that 9 of the most increased miRNAs (&gt;100 folds in microarrays) were increased about 40~70 folds and their pre-miRNAs were increased about 5~10 folds, but no significant difference in their pri-miRNA levels was observed, suggesting a role of FMRP in regulating miRNA processing from pri-miRNA to pre-miRNA. We further demonstrated that a set of protein-coding mRNAs, potentially targeted by the 9 miRNAs were down-regulated in the hippocampus of Fmr1 KO mice. Finally, luciferase assays demonstrated that miR-34b, miR-340, miR-148a could down-regulate the reporter gene expression by interacting with the Met 3\\' UTR. Taken together these findings suggest that the miRNA expression alterations resulted from the absence of FMRP might contribute to molecular pathology of FXS.\",\"preprint_author_corresponding\":\"Yue-Sheng  Long\",\"preprint_author_corresponding_institution\":\"The Second Affiliated Hospital of Guangzhou Medical University\"},{\"preprint_doi\":\"10.1101\\\\/005561\",\"published_doi\":\"10.1016\\\\/j.mbs.2014.06.002\",\"published_journal\":\"Mathematical Biosciences\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Quantifying the effects of anagenetic and cladogenetic evolution\",\"preprint_authors\":\"Krzysztof Bartoszek;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-05-28\",\"published_date\":\"2014-06-14\",\"preprint_abstract\":\"An ongoing debate in evolutionary biology is whether phenotypic change occurs predominantly around the time of speciation or whether it instead accumulates gradually over time. In this work I propose a general framework incorporating both types of change, quantify the effects of speciational change via the correlation between species and attribute the proportion of change to each type. I discuss results of parameter estimation of Hominoid body size in this light. I derive mathematical formulae related to this problem, the probability generating functions of the number of speciation events along a randomly drawn lineage and from the most recent common ancestor of two randomly chosen tip species for a conditioned Yule tree. Additionally I obtain in closed form the variance of the distance from the root to the most recent common ancestor of two randomly chosen tip species.\",\"preprint_author_corresponding\":\"Krzysztof  Bartoszek\",\"preprint_author_corresponding_institution\":\"Uppsala University\"},{\"preprint_doi\":\"10.1101\\\\/002808\",\"published_doi\":\"10.1371\\\\/journal.pone.0099117\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Migration and interaction in a contact zone: mtDNA variation among Bantu-speakers in southern Africa\",\"preprint_authors\":\"Chiara Barbieri;M\\\\u00e1rio Vicente;Sandra Oliveira;Koen Bostoen;Jorge Rocha;Mark Stoneking;Brigitte Pakendorf;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-02-18\",\"published_date\":\"2014-06-05\",\"preprint_abstract\":\"Bantu speech communities expanded over large parts of sub-Saharan Africa within the last 4000-5000 years, reaching different parts of southern Africa 1200-2000 years ago. The Bantu languages subdivide in several major branches, with languages belonging to the Eastern and Western Bantu branches spreading over large parts of Central, Eastern, and Southern Africa. There is still debate whether this linguistic divide is correlated with a genetic distinction between Eastern and Western Bantu speakers. During their expansion, Bantu speakers would have come into contact with diverse local populations, such as the Khoisan hunter-gatherers and pastoralists of southern Africa, with whom they may have intermarried. In this study, we analyze complete mtDNA genome sequences from over 900 Bantu-speaking individuals from Angola, Zambia, Namibia, and Botswana to investigate the demographic processes at play during the last stages of the Bantu expansion. Our results show that most of these Bantu-speaking populations are genetically very homogenous, with no genetic division between speakers of Eastern and Western Bantu languages. Most of the mtDNA diversity in our dataset is due to different degrees of admixture with autochthonous populations. Only the pastoralist Himba and Herero stand out due to high frequencies of particular L3f and L3d lineages; the latter are also found in the neighboring Damara, who speak a Khoisan language and were foragers and small-stock herders. In contrast, the close cultural and linguistic relatives of the Herero and Himba, the Kuvale, are genetically similar to other Bantu-speakers. Nevertheless, as demonstrated by resampling tests, the genetic divergence of Herero, Himba, and Kuvale is compatible with a common shared ancestry with high levels of drift and differential female admixture with local pre-Bantu populations.\",\"preprint_author_corresponding\":\"Chiara  Barbieri\",\"preprint_author_corresponding_institution\":\"Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig, Germany, 04103\"},{\"preprint_doi\":\"10.1101\\\\/003640\",\"published_doi\":\"10.1093\\\\/bioinformatics\\\\/btu390\",\"published_journal\":\"Bioinformatics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"The Scramble Conversion Tool\",\"preprint_authors\":\"James K Bonfield;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2014-03-28\",\"published_date\":\"2014-06-14\",\"preprint_abstract\":\"MotivationThe reference CRAM file format implementation is in Java. We present \\\\\\\\\\\\\"Scramble\\\\\\\\\\\\\": a new C implementation of SAM, BAM and CRAM file I\\\\/O.\\\\\\\\n\\\\\\\\nResultsThe C API for CRAM is 1.5-1.7x slower than BAM at decoding, but 1.8-2.6x faster at encoding. We see file size savings of 40-50%.\\\\\\\\n\\\\\\\\nAvailabilitySource code is available from http:\\\\/\\\\/sourceforge.net\\\\/projects\\\\/staden\\\\/files\\\\/io_lib\\\\/\",\"preprint_author_corresponding\":\"James K Bonfield\",\"preprint_author_corresponding_institution\":\"Wellcome Trust Sanger Institute\"},{\"preprint_doi\":\"10.1101\\\\/006072\",\"published_doi\":\"10.1093\\\\/icb\\\\/icu080\",\"published_journal\":\"Integrative and Comparative Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Gene co-expression modules underlying polymorphic and monomorphic zooids in the colonial hydrozoan, Hydractinia symbiolongicarpus\",\"preprint_authors\":\"David C Plachetzki;M. Sabrina Pankey;Brian R Johnson;Eric J Ronne;Artyom Kopp;Richard K Grosberg;\",\"preprint_category\":\"Developmental Biology \",\"preprint_date\":\"2014-06-07\",\"published_date\":\"2014-06-15\",\"preprint_abstract\":\"Advances in sequencing technology have forced a quantitative revolution in Evolutionary Biology. One important feature of this renaissance is that comprehensive genomic resources can be obtained quickly for almost any taxon, thus speeding the development of new model organisms. Here we analyze 20 RNA-seq libraries from morphologically, sexually, and genetically distinct polyp types from the gonochoristic colonial hydrozoan, Hydractinia symbiolongicarpus (Cnidaria). Analyses of these data using Weighted Gene Co-expression Networks highlights deeply conserved genetic elements of animal spermatogenesis and demonstrate the utility of these methods in identifying modules of genes that correlate with different zooid types across various statistical contrasts. RNAseq data and analytical scripts described here are deposited in publicly available databases.\",\"preprint_author_corresponding\":\"David C Plachetzki\",\"preprint_author_corresponding_institution\":\"The University of New Hampshire\"},{\"preprint_doi\":\"10.1101\\\\/002899\",\"published_doi\":\"10.1093\\\\/molbev\\\\/msu173\",\"published_journal\":\"Molecular Biology and Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"An experimentally determined evolutionary model dramatically improves phylogenetic fit\",\"preprint_authors\":\"Jesse D Bloom;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-02-20\",\"published_date\":\"2014-05-24\",\"preprint_abstract\":\"All modern approaches to molecular phylogenetics require a quantitative model for how genes evolve. Unfortunately, existing evolutionary models do not realistically represent the site-heterogeneous selection that governs actual sequence change. Attempts to remedy this problem have involved augmenting these models with a burgeoning number of free parameters. Here I demonstrate an alternative: experimental determination of a parameter-free evolutionary model via mutagenesis, functional selection, and deep sequencing. Using this strategy, I create an evolutionary model for influenza nucleoprotein that describes the gene phylogeny far better than existing models with dozens or even hundreds of free parameters. Emerging high-throughput experimental strategies such as the one employed here provide fundamentally new information that has the potential to transform the sensitivity of phylogenetic and genetic analyses.\",\"preprint_author_corresponding\":\"Jesse D Bloom\",\"preprint_author_corresponding_institution\":\"Fred Hutchinson Cancer Research Center\"},{\"preprint_doi\":\"10.1101\\\\/003194\",\"published_doi\":\"10.1093\\\\/emph\\\\/eou018\",\"published_journal\":\"Evolution, Medicine, and Public Health\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Phylogenetic tree shapes resolve disease transmission patterns\",\"preprint_authors\":\"Jennifer Gardy;Caroline Colijn;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-03-05\",\"published_date\":\"2014-06-09\",\"preprint_abstract\":\"Whole genome sequencing is becoming popular as a tool for understanding outbreaks of communicable diseases, with phylogenetic trees being used to identify individual transmission events or to characterize outbreak-level overall transmission dynamics. Existing methods to infer transmission dynamics from sequence data rely on well-characterised infectious periods, epidemiological and clinical meta-data which may not always be available, and typically require computationally intensive analysis focussing on the branch lengths in phylogenetic trees. We sought to determine whether the topological structures of phylogenetic trees contain signatures of the overall transmission patterns underyling an outbreak. Here we use simulated outbreaks to train and then test computational classifiers. We test the method on data from two real-world outbreaks. We find that different transmission patterns result in quantitatively different phylogenetic tree shapes. We describe five topological features that summarize a phylogenys structure and find that computational classifiers based on these are capable of predicting an outbreaks transmission dynamics. The method is robust to variations in the transmission parameters and network types, and recapitulates known epidemiology of previously characterized real-world outbreaks. We conclude that there are simple structural properties of phylogenetic trees which, when combined, can distinguish communicable disease outbreaks with a super-spreader, homogeneous transmission, and chains of transmission. This is possible using genome data alone, and can be done during an outbreak. We discuss the implications for management of outbreaks.\",\"preprint_author_corresponding\":\"Caroline  Colijn\",\"preprint_author_corresponding_institution\":\"Imperial College London\"},{\"preprint_doi\":\"10.1101\\\\/005710\",\"published_doi\":\"10.1093\\\\/hmg\\\\/ddu285\",\"published_journal\":\"Human Molecular Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Inhibition of protein translation by the DISC1-Boymaw fusion gene from a Scottish family with major psychiatric disorders\",\"preprint_authors\":\"Baohu Ji;Kerin Higa;Minjung Kim;Lynn Zhou;Jared Young;Mark Geyer;Xianjin Zhou;\",\"preprint_category\":\"Neuroscience \",\"preprint_date\":\"2014-05-31\",\"published_date\":\"2014-06-19\",\"preprint_abstract\":\"The t(1; 11) translocation appears to be the causal genetic lesion with 70% penetrance for schizophrenia, major depression, and other psychiatric disorders in a Scottish family. Molecular studies identified the disruption of the DISC1 (disrupted-in-schizophrenia 1) gene by chromosome translocation at chromosome 1q42. Our previous studies, however, revealed that the translocation also disrupted another gene, Boymaw (also termed DISC1FP1), on chromosome 11. After translocation, two fusion genes (the DISC1-Boymaw (DB7) and the Boymaw-DISC1 (BD13)) are generated between the DISC1 and Boymaw genes. In the present study, we report that expression of the DB7 fusion gene inhibits both intracellular NADH oxidoreductase activities and protein translation. We generated humanized DISC1-Boymaw mice with gene targeting to examine the in vivo functions of the fusion genes. Consistent with the in vitro studies on the DB7 fusion gene, protein translation activity is decreased in the hippocampus and in cultured primary neurons from the brains of the humanized mice. Expression of Gad67, Nmdar1, and Psd95 proteins are also reduced. The humanized mice display prolonged and increased responses to the NMDA receptor antagonist, ketamine, on various mouse genetic backgrounds. Abnormal information processing of acoustic startle and depressive-like behaviors are also observed. In addition, the humanized mice display abnormal erythropoiesis, which was reported to associate with depression in humans. Expression of the DB7 fusion gene may reduce protein translation to impair brain functions and thereby contribute to the pathogenesis of major psychiatric disorders.\",\"preprint_author_corresponding\":\"Xianjin  Zhou\",\"preprint_author_corresponding_institution\":\"University of California, San Diego\"},{\"preprint_doi\":\"10.1101\\\\/001917\",\"published_doi\":\"10.1098\\\\/rsif.2014.0320\",\"published_journal\":\"Journal of the Royal Society Interface\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Modelling and analysis of bacterial tracks suggest an active reorientation mechanism in Rhodobacter sphaeroides\",\"preprint_authors\":\"Gabriel Rosser;Ruth E. Baker;Judith P. Armitage;Alexander George Fletcher;\",\"preprint_category\":\"Microbiology \",\"preprint_date\":\"2014-01-21\",\"published_date\":\"2014-08-06\",\"preprint_abstract\":\"Most free-swimming bacteria move in approximately straight lines, interspersed with random reorientation phases. A key open question concerns varying mechanisms by which reorientation occurs. We combine mathematical modelling with analysis of a large tracking dataset to study the poorly-understood reorientation mechanism in the monoflagellate species Rhodobacter sphaeroides. The flagellum on this species rotates counterclockwise to propel the bacterium, periodically ceasing rotation to enable reorientation. When rotation restarts the cell body usually points in a new direction. It has been assumed that the new direction is simply the result of Brownian rotation. We consider three variants of a self-propelled particle model of bacterial motility. The first considers rotational diffusion only, corresponding to a non-chemotactic mutant strain. A further two models also include stochastic reorientations, describing  run-and-tumble motility. We derive expressions for key summary statistics and simulate each model using a stochastic computational algorithm. We also discuss the effect of cell geometry on rotational diffusion. Working with a previously published tracking dataset, we compare predictions of the models with data on individual stopping events in R. sphaeroides. This provides strong evidence that this species undergoes some form of active reorientation rather than simple reorientation by Brownian rotation.\",\"preprint_author_corresponding\":\"Alexander George Fletcher\",\"preprint_author_corresponding_institution\":\"University of Oxford\"},{\"preprint_doi\":\"10.1101\\\\/000653\",\"published_doi\":\"10.1242\\\\/bio.20148409\",\"published_journal\":\"Biology Open\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"An interplay between extracellular signalling and the dynamics of the exit from pluripotency drives cell fate decisions in mouse ES cells\",\"preprint_authors\":\"David A Turner;Jamie Trott;Penelope Hayward;Pau Ru\\\\u00e9;Alfonso Martinez Arias;\",\"preprint_category\":\"Developmental Biology \",\"preprint_date\":\"2013-11-18\",\"published_date\":\"2014-06-20\",\"preprint_abstract\":\"Embryonic Stem cells derived from the epiblast tissue of the mammalian blastocyst, retain the capability to differentiate into any adult cell type and are able to self-renew indefinitely under appropriate culture conditions. Despite the large amount of knowledge that we have accumulated to date about the regulation and control of self-renewal, efficient directed differentiation into specific tissues remains elusive. In this work, we have analyzed in a systematic manner the interaction between the dynamics of loss of pluripotency and Activin\\\\/Nodal, BMP4 and Wnt signalling in fate assignment during the early stages of differentiation of mouse ES cells in culture. During the initial period of differentiation cells exit from pluripotency and enter an Epi-like state. Following this transient stage, and under the influence of Activin\\\\/Nodal and BMP signalling, cells face a fate choice between differentiating into neuroectoderm and contributing to Primitive Streak fates. We find that Wnt signalling does not suppress neural development as previously thought and that it aids both fates in a context dependent manner. Our results suggest that as cells exit pluripotency they are endowed with a primary neuroectodermal fate and that the potency to become endomesodermal rises with time. We suggest that this situation translates into a \\\\\\\\\\\\\"race for fates\\\\\\\\\\\\\" at the level of single cells in which the neuroectodermal fate has an advantage.\",\"preprint_author_corresponding\":\"Alfonso  Martinez Arias\",\"preprint_author_corresponding_institution\":\"Department of Genetics. University of Cambridge. Cambridge CB2 3EH. UK.\"},{\"preprint_doi\":\"10.1101\\\\/001545\",\"published_doi\":\"10.1186\\\\/1471-2105-15-220\",\"published_journal\":\"BMC Bioinformatics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Exploring community structure in biological networks with random graphs\",\"preprint_authors\":\"Pratha Sah;Lisa O. Singh;Aaron Clauset;Shweta Bansal;\",\"preprint_category\":\"Bioinformatics \",\"preprint_date\":\"2013-12-22\",\"published_date\":\"2014-06-25\",\"preprint_abstract\":\"BackgroundCommunity structure is ubiquitous in biological networks. There has been an increased interest in unraveling the community structure of biological systems as it may provide important insights into a systems functional components and the impact of local structures on dynamics at a global scale. Choosing an appropriate community detection algorithm to identify the community structure in an empirical network can be difficult, however, as the many algorithms available are based on a variety of cost functions and are difficult to validate. Even when community structure is identified in an empirical system, disentangling the effect of community structure from other network properties such as clustering coefficient and assortativity can be a challenge.\\\\\\\\n\\\\\\\\nResultsHere, we develop a generative model to produce undirected, simple, connected graphs with a specified degrees and pattern of communities, while maintaining a graph structure that is as random as possible. Additionally, we demonstrate two important applications of our model: (a) to generate networks that can be used to benchmark existing and new algorithms for detecting communities in biological networks; and (b) to generate null models to serve as random controls when investigating the impact of complex network features beyond the byproduct of degree and modularity in empirical biological networks.\\\\\\\\n\\\\\\\\nConclusionOur model allows for the systematic study of the presence of community structure and its impact on network function and dynamics. This process is a crucial step in unraveling the functional consequences of the structural properties of biological systems and uncovering the mechanisms that drive these systems.\",\"preprint_author_corresponding\":\"Shweta  Bansal\",\"preprint_author_corresponding_institution\":\"Georgetown University\"},{\"preprint_doi\":\"10.1101\\\\/005017\",\"published_doi\":\"10.1371\\\\/journal.pgen.1004434\",\"published_journal\":\"PLOS Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Background selection as baseline for nucleotide variation across the Drosophila genome\",\"preprint_authors\":\"Josep M Comeron;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-05-09\",\"published_date\":\"2014-06-26\",\"preprint_abstract\":\"The constant removal of deleterious mutations by natural selection causes a reduction in neutral diversity and efficacy of selection at genetically linked sites (a process called Background Selection, BGS). Population genetic studies, however, often ignore BGS effects when investigating demographic events or the presence of other types of selection. To obtain a more realistic evolutionary expectation that incorporates the unavoidable consequences of deleterious mutations, we generated high-resolution landscapes of variation across the Drosophila melanogaster genome under a BGS scenario independent of polymorphism data. We find that BGS plays a significant role in shaping levels of variation across the entire genome, including long introns and intergenic regions distant from annotated genes. We also find that a very large percentage of the observed variation in diversity across autosomes can be explained by BGS alone, up to 70% across individual chromosome arms, thus indicating that BGS predictions can be used as baseline to infer additional types of selection and demographic events. This approach allows detecting several outlier regions with signal of recent adaptive events and selective sweeps. The use of a BGS baseline, however, is particularly appropriate to investigate the presence of balancing selection and our study exposes numerous genomic regions with the predicted signature of higher polymorphism than expected when a BGS context is taken into account. Importantly, we show that these conclusions are robust to the mutation and selection parameters of the BGS model. Finally, analyses of protein evolution together with previous comparisons of genetic maps between Drosophila species, suggest temporally variable recombination landscapes and thus, local BGS effects that may differ between extant and past phases. Because genome-wide BGS and temporal changes in linkage effects can skew approaches to estimate demographic and selective events, future analyses should incorporate BGS predictions and capture local recombination variation across genomes and along lineages.\",\"preprint_author_corresponding\":\"Josep M Comeron\",\"preprint_author_corresponding_institution\":\"University of Iowa\"},{\"preprint_doi\":\"10.1101\\\\/000109\",\"published_doi\":\"10.1371\\\\/journal.pgen.1004410\",\"published_journal\":\"PLOS Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Speciation and introgression between Mimulus nasutus and Mimulus guttatus\",\"preprint_authors\":\"Yaniv Brandvain;Amanda M Kenney;Lex Fagel;Graham Coop;Andrea L Sweigart;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2013-11-07\",\"published_date\":\"2014-06-26\",\"preprint_abstract\":\"Mimulus guttatus and M. nasutus are an evolutionary and ecological model sister species pair differentiated by ecology, mating system, and partial reproductive isolation. Despite extensive research on this system, the history of divergence and differentiation in this sister pair is unclear. We present and analyze a novel population genomic data set which shows that M. nasutus \"budded\" off of a central Californian M. guttatus population within the last 200 to 500 thousand years. In this time, the M. nasutus genome has accrued numerous genomic signatures of the transition to predominant selfing. Despite clear biological differentiation, we document ongoing, bidirectional introgression. We observe a negative relationship between the recombination rate and divergence between M. nasutus and sympatric M. guttatus samples, suggesting that selection acts against M. nasutus ancestry in M. guttatus.\",\"preprint_author_corresponding\":\"Yaniv  Brandvain\",\"preprint_author_corresponding_institution\":\"Department of Evolution and Ecology &amp; Center for Population Biology, University of California -Davis\"},{\"preprint_doi\":\"10.1101\\\\/002493\",\"published_doi\":\"10.1371\\\\/journal.pone.0100163\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"The roles of standing genetic variation and evolutionary history in determining the evolvability of anti-predator strategies\",\"preprint_authors\":\"Jordan Fish;Daniel R O\\'Donnell;Abhijna Parigi;Ian Dworkin;Aaron P Wagner;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-02-07\",\"published_date\":\"2014-06-23\",\"preprint_abstract\":\"Standing genetic variation and the historical environment in which that variation arises (evolutionary history) are both potentially significant determinants of a populations capacity for evolutionary response to a changing environment. We evaluated the relative importance of these two factors in influencing the evolutionary trajectories in the face of sudden environmental change. We used the open-ended digital evolution software Avida to examine how historic exposure to predation pressures, different levels of genetic variation, and combinations of the two, impact anti-predator strategies and competitive abilities evolved in the face of threats from new, invasive, predator populations. We show that while standing genetic variation plays some role in determining evolutionary responses, evolutionary history has the greater influence on a populations capacity to evolve effective anti-predator traits. This adaptability likely reflects the relative ease of repurposing existing, relevant genes and traits, and the broader potential value of the generation and maintenance of adaptively flexible traits in evolving populations.\",\"preprint_author_corresponding\":\"Daniel R O\\'Donnell\",\"preprint_author_corresponding_institution\":\"Michigan State University\"},{\"preprint_doi\":\"10.1101\\\\/003004\",\"published_doi\":\"10.1007\\\\/s00239-014-9628-x\",\"published_journal\":\"Journal of Molecular Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Neanderthals had our de novo genes.\",\"preprint_authors\":\"John Stewart Taylor;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-02-25\",\"published_date\":\"2014-06-01\",\"preprint_abstract\":\"Gene duplication provides a profusion of raw material for evolutionary innovation (Lynch and Conery, 2000). While most duplicates rapidly become unrecognizable some, e.g., those that are immediately useful or those that after a period of relaxed selection gain unique roles, are retained and thereby expand a genomes protein-coding repertoire. Ohno (1970) famously remarked that without gene duplication, the creation of metazoans, vertebrates, and mammals from unicellular organisms would have been impossible. Such big leaps in evolution, he argued, required the creation of new gene loci with previously nonexistent functions (Taylor and Raes, 2004). Recently, another source of new genes has been recognized: Though rare, it seems clear that new genes can emerge from formerly non-coding DNA, the  de novo protein coding genes (Zhao et al., 2014, and references therein).\\\\\\\\n\\\\\\\\nIn 2009 Knowles ...\",\"preprint_author_corresponding\":\"John Stewart Taylor\",\"preprint_author_corresponding_institution\":\"Department of Biology, University of Victoria, Victoria, BC, Canada\"},{\"preprint_doi\":\"10.1101\\\\/001784\",\"published_doi\":\"10.1126\\\\/science.1250939\",\"published_journal\":\"Science\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Global Epistasis Makes Adaptation Predictable Despite Sequence-Level Stochasticity\",\"preprint_authors\":\"Sergey Kryazhimskiy;Daniel Paul Rice;Elizabeth Jerison;Michael M Desai;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-01-13\",\"published_date\":\"2014-06-27\",\"preprint_abstract\":\"Epistasis can make adaptation highly unpredictable, rendering evolutionary trajectories contingent on the chance effects of initial mutations. We used experimental evolution in Saccharomyces cerevisiae to quantify this effect, finding dramatic differences in adaptability between 64 closely related genotypes. Despite these differences, sequencing of 105 evolved clones showed no significant effect of initial genotype on future sequence-level evolution. Instead, reconstruction experiments revealed a consistent pattern of diminishing returns epistasis. Our results suggest that many beneficial mutations affecting a variety of biological processes are globally coupled: they interact strongly, but only through their combined effect on fitness. Sequence-level adaptation is thus highly stochastic. Nevertheless, fitness evolution is strikingly predictable because differences in adaptability are determined only by global fitness-mediated epistasis, not by the identity of individual mutations.\",\"preprint_author_corresponding\":\"Michael M Desai\",\"preprint_author_corresponding_institution\":\"Harvard University\"},{\"preprint_doi\":\"10.1101\\\\/005348\",\"published_doi\":\"10.1038\\\\/ng.3015\",\"published_journal\":\"Nature Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Inferring human population size and separation history from multiple genome sequences\",\"preprint_authors\":\"Stephan Schiffels;Richard Durbin;\",\"preprint_category\":\"Genetics \",\"preprint_date\":\"2014-05-21\",\"published_date\":\"2014-05-30\",\"preprint_abstract\":\"The availability of complete human genome sequences from populations across the world has given rise to new population genetic inference methods that explicitly model their ancestral relationship under recombination and mutation. So far, application of these methods to evolutionary history more recent than 20-30 thousand years ago and to population separations has been limited. Here we present a new method that overcomes these shortcomings. The Multiple Sequentially Markovian Coalescent (MSMC) analyses the observed pattern of mutations in multiple individuals, focusing on the first coalescence between any two individuals. Results from applying MSMC to genome sequences from nine populations across the world suggest that the genetic separation of non-African ancestors from African Yoruban ancestors started long before 50,000 years ago, and give information about human population history as recently as 2,000 years ago, including the bottleneck in the peopling of the Americas, and separations within Africa, East Asia and Europe.\",\"preprint_author_corresponding\":\"Richard  Durbin\",\"preprint_author_corresponding_institution\":\"Sanger Institute\"},{\"preprint_doi\":\"10.1101\\\\/004614\",\"published_doi\":\"10.1186\\\\/1471-2164-15-494\",\"published_journal\":\"BMC Genomics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Characterizing a collective and dynamic component of chromatin immunoprecipitation enrichment profiles in yeast\",\"preprint_authors\":\"Lucas D. Ward;Junbai Wang;Harmen J. Bussemaker;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2014-04-29\",\"published_date\":\"2014-06-19\",\"preprint_abstract\":\"Recent chromatin immunoprecipitation (ChIP) experiments in fly, mouse, and human have revealed the existence of high-occupancy target (HOT) regions or hotspots that show enrichment across many assayed DNA-binding proteins. Similar co-enrichment observed in yeast so far has been treated as artifactual, and has not been fully characterized. Here we reanalyze ChIP data from both array-based and sequencing-based experiments to show that in the yeast S. cerevisiae, the collective enrichment phenomenon is strongly associated with proximity to noncoding RNA genes and with nucleosome depletion. DNA sequence motifs that confer binding affinity for the proteins are largely absent from these hotspots, suggesting that protein-protein interactions play a prominent role. The hotspots are condition-specific, suggesting that they reflect a chromatin state or protein state, and are not a static feature of underlying sequence. Additionally, only a subset of all assayed factors is associated with these loci, suggesting that the co-enrichment cannot be simply explained by a chromatin state that is universally more prone to immunoprecipitation. Together our results suggest that the co-enrichment patterns observed in yeast represent transcription factor co-occupancy. More generally, they make clear that great caution must be used when interpreting ChIP enrichment profiles for individual factors in isolation, as they will include factor-specific as well as collective contributions.\",\"preprint_author_corresponding\":\"Harmen J. Bussemaker\",\"preprint_author_corresponding_institution\":\"Columbia University\"},{\"preprint_doi\":\"10.1101\\\\/004341\",\"published_doi\":\"10.1073\\\\/pnas.1409572111\",\"published_journal\":\"Proceedings of the National Academy of Sciences\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Quantifying selection in immune receptor repertoires\",\"preprint_authors\":\"Yuval Elhanati;Anand Murugan;Curtis G Callan Jr.;Thierry Mora;Aleksandra Walczak;\",\"preprint_category\":\"Immunology \",\"preprint_date\":\"2014-04-21\",\"published_date\":\"2014-05-22\",\"preprint_abstract\":\"The efficient recognition of pathogens by the adaptive immune system relies on the diversity of receptors displayed at the surface of immune cells. T-cell receptor diversity results from an initial random DNA editing process, called VDJ recombination, followed by functional selection of cells according to the interaction of their surface receptors with self and foreign antigenic peptides. To quantify the effect of selection on the highly variable elements of the receptor, we apply a probabilistic maximum likelihood approach to the analysis of high-throughput sequence data from the {beta}-chain of human T-cell receptors. We quantify selection factors for V and J gene choice, and for the length and amino-acid composition of the variable region. Our approach is necessary to disentangle the effects of selection from biases inherent in the recombination process. Inferred selection factors differ little between donors, or between naive and memory repertoires. The number of sequences shared between donors is well-predicted by the model, indicating a purely stochastic origin of such \\\\\\\\\\\\\"public\\\\\\\\\\\\\" sequences. We find a significant correlation between biases induced by VDJ recombination and our inferred selection factors, together with a reduction of diversity during selection. Both effects suggest that natural selection acting on the recombination process has anticipated the selection pressures experienced during somatic evolution.\",\"preprint_author_corresponding\":\"Thierry  Mora\",\"preprint_author_corresponding_institution\":\"Ecole normale sup?rieure\"},{\"preprint_doi\":\"10.1101\\\\/004382\",\"published_doi\":\"10.1016\\\\/j.meegid.2012.11.009\",\"published_journal\":\"Infection, Genetics and Evolution\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Lack of association between Toll Like Receptor-2 &amp;amp; Toll Like Receptor-4 Gene Polymorphisms and Iranian Asthmatics risk or features\",\"preprint_authors\":\"hamid bahrami;saeed daneshmandi;Ali Akbar Pourfathollah;hasan haidarnazhad;\",\"preprint_category\":\"Immunology \",\"preprint_date\":\"2014-04-28\",\"published_date\":\"2014-03-01\",\"preprint_abstract\":\"BackgroundAsthma as chronic inflammatory airway disease is considered to be the most common chronic disease that is involving genetic and environmental factors. Toll like receptors (TLRs) and other inflammatory mediators are important in modulation of inflammation. In this study we evaluated the role of TLR2 Arg753Gln and TLR4 Asp299Gly polymorphisms in the asthma susceptibility, progress, control levels and lung functions in Iranian subjects.\\\\\\\\n\\\\\\\\nMethodsOn 99 asthmatic patients and 120 normal subjects, TLR2 Arg753Gln and TLR4 Asp299Gly polymorphism were evaluated by PCR-RFLP method recruiting Msp1 and Nco1 restriction enzymes, respectively. IgE serum levels by ELISA technique were determined and asthma diagnosis, treatment and control levels were considered using standard schemes and criteria.\\\\\\\\n\\\\\\\\nResultsOur results indicated that the genotype and allele frequencies of the TLR2 Arg753Gln and TLR4 Asp299Gly polymorphisms were not significantly different between control subjects and asthmatics (p &gt; 0.05) or even in asthma features such as IgE levels, asthma history and pulmonary factors (p &gt; 0.05).\\\\\\\\n\\\\\\\\nConclusionsMeanwhile some previous studies indicated TLRs and their polymorphisms role in asthma incidence and features, our data demonstrated that TLR2 Arg753Gln and TLR4 Asp299Gly gene variants were not risk factor of asthma or its features in Iranian patients. Genetic complexity, ethnicity, influence of other genes or polymorphisms may overcome these polymorphisms in our asthmatics.\",\"preprint_author_corresponding\":\"Ali Akbar  Pourfathollah\",\"preprint_author_corresponding_institution\":\"tarbiat modares university\"},{\"preprint_doi\":\"10.1101\\\\/005371\",\"published_doi\":\"10.1186\\\\/gb-2014-15-6-r86\",\"published_journal\":\"Genome Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"IVT-seq reveals extreme bias in RNA-sequencing\",\"preprint_authors\":\"Nicholas F Lahens;Ibrahim Halil Kavakli;Ray Zhang;Katharina Hayer;Michael B Black;Hannah Dueck;Angel Pizarro;Junhyong Kim;Rafael A Irizarry;Russell S Thomas;Gregory R Grant;John B Hogenesch;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2014-05-21\",\"published_date\":\"2014-06-30\",\"preprint_abstract\":\"BackgroundRNA sequencing (RNA-seq) is a powerful technique for identifying and quantifying transcription and splicing events, both known and novel. However, given its recent development and the proliferation of library construction methods, understanding the bias it introduces is incomplete but critical to realizing its value.\\\\\\\\n\\\\\\\\nResultsHere we present a method, in vitro transcription sequencing (IVT-seq), for identifying and assessing the technical biases in RNA-seq library generation and sequencing at scale. We created a pool of &gt; 1000 in vitro transcribed (IVT) RNAs from a full-length human cDNA library and sequenced them with poly-A and total RNA-seq, the most common protocols. Because each cDNA is full length and we show IVT is incredibly processive, each base in each transcript should be equivalently represented. However, with common RNA-seq applications and platforms, we find [~]50% of transcripts have &gt; 2-fold and [~]10% have &gt; 10-fold differences in within-transcript sequence coverage. Strikingly, we also find &gt; 6% of transcripts have regions of high, unpredictable sequencing coverage, where the same transcript varies dramatically in coverage between samples, confounding accurate determination of their expression. To get at causal factors, we used a combination of experimental and computational approaches to show that rRNA depletion is responsible for the most significant variability in coverage and that several sequence determinants also strongly influence representation.\\\\\\\\n\\\\\\\\nConclusionsIn sum, these results show the utility of IVT-seq in promoting better understanding of bias introduced by RNA-seq and suggest caution in its interpretation. Furthermore, we find that rRNA-depletion is responsible for substantial, unappreciated biases in coverage. Perhaps most importantly, these coverage biases introduced during library preparation suggest exon level expression analysis may be inadvisable.\",\"preprint_author_corresponding\":\"John B Hogenesch\",\"preprint_author_corresponding_institution\":\"University of Pennsylvania\"},{\"preprint_doi\":\"10.1101\\\\/005462\",\"published_doi\":\"10.1186\\\\/gb-2014-15-6-r88\",\"published_journal\":\"Genome Biology\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Human genomic regions with exceptionally high or low levels of population differentiation identified from 911 whole-genome sequences\",\"preprint_authors\":\"Vincenza Colonna;Qasim Ayub;Yuan Chen;Luca Pagani;Pierre Luisi;Marc Pybus;Erik Garrison;Yali Xue;Chris Tyler-Smith;\",\"preprint_category\":\"Genomics \",\"preprint_date\":\"2014-05-23\",\"published_date\":\"2014-06-30\",\"preprint_abstract\":\"BackgroundPopulation differentiation has proved to be effective for identifying loci under geographically-localized positive selection, and has the potential to identify loci subject to balancing selection. We have previously investigated the pattern of genetic differentiation among human populations at 36.8 million genomic variants to identify sites in the genome showing high frequency differences. Here, we extend this dataset to include additional variants, survey sites with low levels of differentiation, and evaluate the extent to which highly differentiated sites are likely to result from selective or other processes.\\\\\\\\n\\\\\\\\nResultsWe demonstrate that while sites of low differentiation represent sampling effects rather than balancing selection, sites showing extremely high population differentiation are enriched for positive selection events and that one half may be the result of classic selective sweeps. Among these, we rediscover known examples, where we actually identify the established functional SNP, and discover novel examples including the genes ABCA12, CALD1 and ZNF804, which we speculate may be linked to adaptations in skin, calcium metabolism and defense, respectively.\\\\\\\\n\\\\\\\\nConclusionsWe have identified known and many novel candidate regions for geographically restricted positive selection, and suggest several directions for further research.\",\"preprint_author_corresponding\":\"Vincenza  Colonna\",\"preprint_author_corresponding_institution\":\"The Wellcome Trust Sanger Institute\"},{\"preprint_doi\":\"10.1101\\\\/003541\",\"published_doi\":\"10.1073\\\\/pnas.1405500111\",\"published_journal\":\"Proceedings of the National Academy of Sciences\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"An optimized CRISPR\\\\/Cas toolbox for efficient germline and somatic genome engineering in Drosophila\",\"preprint_authors\":\"Fillip Port;Hui-Min Chen;Tzumin Lee;Simon L Bullock;\",\"preprint_category\":\"Genetics \",\"preprint_date\":\"2014-03-24\",\"published_date\":\"2014-07-07\",\"preprint_abstract\":\"The type II CRISPR\\\\/Cas system has recently emerged as a powerful method to manipulate the genomes of various organisms. Here, we report a novel toolbox for high efficiency genome engineering of Drosophila melanogaster consisting of transgenic Cas9 lines and versatile guide RNA (gRNA) expression plasmids. Systematic evaluation reveals Cas9 lines with ubiquitous or germline restricted patterns of activity. We also demonstrate differential activity of the same gRNA expressed from different U6 snRNA promoters, with the previously untested U6:3 promoter giving the most potent effect. Choosing an appropriate combination of Cas9 and gRNA allows targeting of essential and non-essential genes with transmission rates ranging from 25% - 100%. We also provide evidence that our optimized CRISPR\\\\/Cas tools can be used for offset nicking-based mutagenesis and, in combination with oligonucleotide donors, to precisely edit the genome by homologous recombination with efficiencies that do not require the use of visible markers. Lastly, we demonstrate a novel application of CRISPR\\\\/Cas-mediated technology in revealing loss-of-function phenotypes in somatic cells following efficient biallelic targeting by Cas9 expressed in a ubiquitous or tissue-restricted manner. In summary, our CRISPR\\\\/Cas tools will facilitate the rapid evaluation of mutant phenotypes of specific genes and the precise modification of the genome with single nucleotide precision. Our results also pave the way for high throughput genetic screening with CRISPR\\\\/Cas.\",\"preprint_author_corresponding\":\"Fillip  Port\",\"preprint_author_corresponding_institution\":\"MRC-LMB\"},{\"preprint_doi\":\"10.1101\\\\/003202\",\"published_doi\":\"10.1093\\\\/hmg\\\\/ddu359\",\"published_journal\":\"Human Molecular Genetics\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"TCF7L2 is a master regulator of insulin production and processing\",\"preprint_authors\":\"Yuedan Zhou;Soo-Young Park;Jing Su;Kathleen Bailey;Emilia Ottosson-Laakso;Liliya Shcerbina;Nikolay Oskolkov;Enming Zhang;Thomas Thevenin;Jo?o Fadista;Hedvig Bennet;Petter Vikman;Nils Wierup;Malin Fex;Johan Rung;Claes Wollheim;Marcelo Nobrega;Erik Renstr?m;Leif Groop;Ola Hansson;\",\"preprint_category\":\"Cell Biology \",\"preprint_date\":\"2014-03-05\",\"published_date\":\"2014-07-11\",\"preprint_abstract\":\"Although variants in the T-cell factor 7-like 2 gene (TCF7L2) confer the strongest risk of type 2 diabetes (T2D) by presumed effects on islet function, the underlying mechanisms are not well understood. We have identified TCF7L2-target genes and described the regulatory network downstream of TCF7L2 responsible for its effect on insulin secretion in rodents and human pancreatic islets. ISL1 is a direct target of TCF7L2 and regulates proinsulin production and processing via MAFA, PDX1, NKX6.1, PCSK1 and PCSK2 and possibly clearance of proinsulin via SLC30A8. Taken together, these results demonstrate that not only synthesis of proinsulin is regulated by TCF7L2, but also processing and possibly clearance of proinsulin and insulin in a genotype dependent manner. These multiple targets in key pathways may explain why TCF7L2 has emerged as the gene showing the strongest association with T2D.\",\"preprint_author_corresponding\":\"Ola  Hansson\",\"preprint_author_corresponding_institution\":\"Lund University\"},{\"preprint_doi\":\"10.1101\\\\/003269\",\"published_doi\":\"10.1371\\\\/journal.pone.0101851\",\"published_journal\":\"PLOS ONE\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Early warning signs in social-ecological networks\",\"preprint_authors\":\"Samir Suweis;Paolo D\\'Odorico;\",\"preprint_category\":\"Ecology \",\"preprint_date\":\"2014-03-12\",\"published_date\":\"2014-07-11 \\\\t\",\"preprint_abstract\":\"A number of social-ecological systems exhibit complex behaviour associated with nonlinearities, bifurcations, and interaction with stochastic drivers. These systems are often prone to abrupt and unexpected instabilities and state shifts that emerge as a discontinuous response to gradual changes in environmental drivers. Predicting such behaviours is crucial to the prevention of or preparation for unwanted regime shifts. Recent research in ecology has investigated early warning signs that anticipate the divergence of univariate ecosystem dynamics from a stable attractor. To date, leading indicators of instability in systems with multiple interacting components have remained poorly investigated. This is a major limitation in the understanding of the dynamics of complex social-ecological networks. Here, we develop a theoretical framework to demonstrate that rising variance - measured, for example, by the maximum element of the covariance matrix of the network - is an effective leading indicator of network instability. We show that its reliability and robustness depend more on the sign of the interactions within the network than the network structure or noise intensity. Mutualistic, scale free and small world networks are less stable than their antagonistic or random counterparts but their instability is more reliably predicted by this leading indicator. These results provide new advances in multidimensional early warning analysis and offer a framework to evaluate the resilience of social-ecological networks.\",\"preprint_author_corresponding\":\"Paolo  D\\'Odorico\",\"preprint_author_corresponding_institution\":\"University of Virginia\"},{\"preprint_doi\":\"10.1101\\\\/002683\",\"published_doi\":\"10.1098\\\\/rstb.2013.0252\",\"published_journal\":\"Proceedings B\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"Evolutionary rates for multivariate traits: the role of selection and genetic variation\",\"preprint_authors\":\"William Pitchers;Jason B. Wolf;Tom Tregenza;John Hunt;Ian Dworkin;\",\"preprint_category\":\"Evolutionary Biology \",\"preprint_date\":\"2014-02-14\",\"published_date\":\"2014-07-07\",\"preprint_abstract\":\"A fundamental question in evolutionary biology is the relative importance of selection and genetic architecture in determining evolutionary rates. Adaptive evolution can be described by the multivariate breeders equation [Formula], which predicts evolutionary change for a suite of phenotypic traits [Formula] as a product of directional selection acting on them ({beta}) and the genetic variance-covariance matrix for those traits (G). Despite being empirically challenging to estimate, there are enough published estimates of G and {beta} to allow for synthesis of general patterns across species. We use published estimates to test the hypotheses that there are systematic differences in the rate of evolution among trait types, and that these differences are in part due to genetic architecture. We find evidence some evidence that sexually selected traits exhibit faster rates of evolution compared to life-history or morphological traits. This difference does not appear to be related to stronger selection on sexually selected traits. Using numerous proposed approaches to quantifying the shape, size and structure of G we examine how these parameters relate to one another, and how they vary among taxonomic and trait groupings. Despite considerable variation, they do not explain the observed differences in evolutionary rates.\",\"preprint_author_corresponding\":\"Ian  Dworkin\",\"preprint_author_corresponding_institution\":\"Michigan State University\"},{\"preprint_doi\":\"10.1101\\\\/005041\",\"published_doi\":\"10.7554\\\\/eLife.03300\",\"published_journal\":\"eLife\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"The inherent mutational tolerance and antigenic evolvability of influenza hemagglutinin\",\"preprint_authors\":\"Bargavi Thyagarajan;Jesse D Bloom;\",\"preprint_category\":\"Microbiology \",\"preprint_date\":\"2014-05-12\",\"published_date\":\"2014-07-08 \\\\t\",\"preprint_abstract\":\"Influenza is notable for its evolutionary capacity to escape immunity targeting the viral hemagglutinin. We used deep mutational scanning to examine the extent to which a high inherent mutational tolerance contributes to this antigenic evolvability. We created mutant viruses that incorporate most of the {approx} 104 amino-acid mutations to hemagglutinin from A\\\\/WSN\\\\/1933 (H1N1) influenza. After passaging these viruses in tissue culture to select for functional variants, we used deep sequencing to quantify mutation frequencies before and after selection. These data enable us to infer the preference for each amino acid at each site in hemagglutinin. These inferences are consistent with existing knowledge about the proteins structure and function, and can be used to create a model that describes hemagglutinins evolution far better than existing phylogenetic models. We show that hemagglutinin has a high inherent tolerance for mutations at antigenic sites, suggesting that this is one factor contributing to influenzas antigenic evolution.\",\"preprint_author_corresponding\":\"Jesse D Bloom\",\"preprint_author_corresponding_institution\":\"Fred Hutchinson Cancer Research Center\"},{\"preprint_doi\":\"10.1101\\\\/004499\",\"published_doi\":\"10.1093\\\\/nar\\\\/gku604\",\"published_journal\":\"Nucleic Acids Research\",\"preprint_platform\":\"bioRxiv\",\"preprint_title\":\"In vivo generation of DNA sequence diversity for cellular barcoding\",\"preprint_authors\":\"Ian Peikon;Diana Gizatullina;Anthony Zador;\",\"preprint_category\":\"Synthetic Biology \",\"preprint_date\":\"2014-04-24\",\"published_date\":\"2014-07-10 \\\\t\",\"preprint_abstract\":\"Heterogeneity is a ubiquitous feature of biological systems. A complete understanding of such systems requires a method for uniquely identifying and tracking individual components and their interactions with each other. We have developed a novel method of uniquely tagging individual cells in vivo with a genetic \\\\\\\\\\\\\"barcode\\\\\\\\\\\\\" that can be recovered by DNA sequencing. We demonstrate the feasibility of this technique in bacterial cells. This method should prove useful in tracking interactions of cells within a network, and\\\\/or heterogeneity within complex biological samples.\",\"preprint_author_corresponding\":\"Anthony  Zador\",\"preprint_author_corresponding_institution\":\"Cold Spring Harbor Laboratory\"}]}\\n  </p>\\n </body>\\n</html>'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.prettify()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "18fb58d3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Expecting ',' delimiter: line 1 column 191691 (char 191690)\n",
      " \"Sl\n"
     ]
    }
   ],
   "source": [
    "i = 4\n",
    "try:\n",
    "    with open(f\"data-{i}.json\") as f:\n",
    "        data = json.load(f)\n",
    "except Exception as e:\n",
    "    print(e)\n",
    "    with open(f\"data-{i}.json\", 'r') as f:\n",
    "        errIndex = int(str(e).split()[-1][:-1])\n",
    "        print(f.read()[errIndex-2:errIndex+2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "e6ddd375",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'messages': [{'status': 'ok',\n",
       "   'interval': '2000-03-01:2020-03-30',\n",
       "   'cursor': '901',\n",
       "   'count': 100,\n",
       "   'total': 38531}],\n",
       " 'collection': [{'preprint_doi': '10.1101/027797',\n",
       "   'published_doi': '10.1534/g3.115.023119',\n",
       "   'published_journal': 'G3: Genes|Genomes|Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Tying down loose ends in the Chlamydomonas genome',\n",
       "   'preprint_authors': 'Frederick R. Cross;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-09-29',\n",
       "   'published_date': '2015-12-23',\n",
       "   'preprint_abstract': 'The Chlamydomonas genome has been sequenced, assembled and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. The annotated genome is very rich in open reading frames upstream of the annotated coding sequence ( uORFs): almost three quarters of the assigned transcripts have at least one uORF, and frequently more than one. This is problematic with respect to the standard  scanning model for eukaryotic translation initiation. These uORFs can be grouped into three classes: class 1, initiating in-frame with the coding sequence (cds) (thus providing a potential in-frame N-terminal extension); class 2, initiating in the 5UT and terminating out-of-frame in the cds; and class 3, initiating and terminating within the 5UT. Multiple bioinformatics criteria (including analysis of Kozak consensus sequence agreement and BLASTP comparisons to the closely related Volvox genome, and statistical comparison to cds and to random-sequence controls) indicate that of ~4000 class 1 uORFs, approximately half are likely in vivo translation initiation sites. The proposed resulting N-terminal extensions in many cases will sharply alter the predicted biochemical properties of the encoded proteins. These results suggest significant modifications in ~2000 of the ~20,000 transcript models with respect to translation initiation and encoded peptides. In contrast, class 2 uORFs may be subject to purifying selection, and the existent ones (surviving selection) are likely inefficiently translated. Class 3 uORFs are remarkably similar to random sequence expectations with respect to size, number and composition and therefore may be largely selectively neutral; their very high abundance (found in more than half of transcripts, frequently with multiple uORFs per transcript) nevertheless suggests the possibility of translational regulation on a wide scale.',\n",
       "   'preprint_author_corresponding': 'Frederick R. Cross',\n",
       "   'preprint_author_corresponding_institution': 'Rockefeller University'},\n",
       "  {'preprint_doi': '10.1101/027896',\n",
       "   'published_doi': '10.7554/eLife.11750',\n",
       "   'published_journal': 'eLife',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'TRPA1 tunes mosquito thermotaxis to host temperatures',\n",
       "   'preprint_authors': 'Roman A Corfas;Leslie B Vosshall;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-09-30',\n",
       "   'published_date': '2015-12-15',\n",
       "   'preprint_abstract': 'While most ectotherms thermotax only to regulate their temperature, female mosquitoes are attracted to human body heat during pursuit of a blood meal. Here we elucidate the basic rules of Aedes aegypti thermotaxis and test the function of candidate thermoreceptors in this important behavior. We show that host-seeking mosquitoes are maximally attracted to thermal stimuli approximating host body temperatures, seeking relative warmth while avoiding both relative cooling and stimuli exceeding host body temperature. We found that the cation channel TRPA1, in addition to playing a conserved role in thermoregulation and chemosensation, is required for this specialized host-selective thermotaxis in mosquitoes. During host-seeking, AaegTRPA1-/- mutants failed to avoid stimuli exceeding host temperature, and were unable to discriminate between host-temperature and high-temperature stimuli. TRPA1-dependent tuning of thermotaxis is likely critical for mosquitoes host-seeking in a complex thermal environment in which humans are warmer than ambient air, but cooler than surrounding sun-warmed surfaces.',\n",
       "   'preprint_author_corresponding': 'Leslie B Vosshall',\n",
       "   'preprint_author_corresponding_institution': 'The Rockefeller University'},\n",
       "  {'preprint_doi': '10.1101/027938',\n",
       "   'published_doi': '10.1371/journal.pgen.1005732',\n",
       "   'published_journal': 'PLOS Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Understanding biases in ribosome profiling experiments reveals signatures of translation dynamics in yeast',\n",
       "   'preprint_authors': 'Jeffrey A Hussmann;Stephanie Patchett;Arlen Johnson;Sara Sawyer;William H Press;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-09-30',\n",
       "   'published_date': '2015-12-11',\n",
       "   'preprint_abstract': 'Ribosome profiling produces snapshots of the locations of actively translating ribosomes on messenger RNAs. These snapshots can be used to make inferences about translation dynamics. Recent ribosome profiling studies in yeast, however, have reached contradictory conclusions regarding the average translation rate of each codon. Some experiments have used cycloheximide (CHX) to stabilize ribosomes before measuring their positions, and these studies all counterintuitively report a weak negative correlation between the translation rate of a codon and the abundance of its cognate tRNA. In contrast, some experiments performed without CHX report strong positive correlations. To explain this contradiction, we identify unexpected patterns in ribosome density downstream of each type of codon in experiments that use CHX. These patterns are evidence that elongation continues to occur in the presence of CHX but with dramatically altered codon-specific elongation rates. The measured positions of ribosomes in these experiments therefore do not reflect the amounts of time ribosomes spend at each position in vivo. These results suggest that conclusions from experiments in yeast using CHX may need reexamination. In particular, we show that in all such experiments, codons decoded by less abundant tRNAs were in fact being translated more slowly before the addition of CHX disrupted these dynamics.',\n",
       "   'preprint_author_corresponding': 'Jeffrey A Hussmann',\n",
       "   'preprint_author_corresponding_institution': 'University of Texas at Austin'},\n",
       "  {'preprint_doi': '10.1101/028241',\n",
       "   'published_doi': '10.1038/gim.2015.161',\n",
       "   'published_journal': 'Genetics in Medicine',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Gene discovery for Mendelian conditions via social networking: de novo variants in KDM1A cause developmental delay and distinctive facial features',\n",
       "   'preprint_authors': 'Jessica Chong;Joon-Ho Yu;Peter Lorentzen;Karen Park;Seema M Jamal;Holly K Tabor;Anita Rauch;Margarita Sifuentes Saenz;Eugen Boltshauser;Karynne E Patterson;Deborah A Nickerson;University of Washington Center for Mendelian Geno;Michael J Bamshad;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-10-03',\n",
       "   'published_date': '2015-12-10',\n",
       "   'preprint_abstract': 'Purpose:The pace of Mendelian gene discovery is slowed by the \\\\\"n-of-1 problem\\\\\" - the difficulty of establishing causality of a putatively pathogenic variant in a single person or family. Identification of an unrelated person with an overlapping phenotype and suspected pathogenic variant in the same gene can overcome this barrier but is often impeded by lack of a convenient or widely-available way to share data on candidate variants / genes among families, clinicians and researchers.\\\\n\\\\nMethods:Social networking among families, clinicians and researchers was used to identify three children with variants of unknown significance in KDM1A and similar phenotypes.\\\\n\\\\nResults:De novo variants in KDM1A underlie a new syndrome characterized by developmental delay and distinctive facial features.\\\\n\\\\nConclusion:Social networking is a potentially powerful strategy to discover genes for rare Mendelian conditions, particularly those with non-specific phenotypic features. To facilitate the efforts of families to share phenotypic and genomic information with each other, clinicians, and researchers, we developed the Repository for Mendelian Genomics Family Portal (RMD-FP). Design and development of a web-based tool, MyGene2, that enables families, clinicians and researchers to search for gene matches based on analysis of phenotype and exome data deposited into the RMD-FP is underway.',\n",
       "   'preprint_author_corresponding': 'Jessica  Chong',\n",
       "   'preprint_author_corresponding_institution': 'University of Washington'},\n",
       "  {'preprint_doi': '10.1101/029496',\n",
       "   'published_doi': '10.1534/g3.115.025668',\n",
       "   'published_journal': 'G3: Genes|Genomes|Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'FASTmC: a suite of predictive models for non-reference-based estimations of DNA methylation',\n",
       "   'preprint_authors': 'Adam J Bewick;Brigitte T Hofmeister;Kevin J Lee;Xiaoyu Zhang;Dave W Hall;Robert J Schmitz;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-10-20',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'We describe a suite of predictive models, coined FASTmC, for non-reference, cost-effective exploration and comparative analysis of context-specific DNA methylation levels. Accurate estimations of true DNA methylation levels can be obtained from as few as several thousand short-reads generated from whole genome bisulfite sequencing. These models make high-resolution time course or developmental, and large diversity studies practical regardless of species, genome size and availability of a reference genome.',\n",
       "   'preprint_author_corresponding': 'Robert J Schmitz',\n",
       "   'preprint_author_corresponding_institution': 'University of Georgia'},\n",
       "  {'preprint_doi': '10.1101/021006',\n",
       "   'published_doi': '10.1534/genetics.115.179473',\n",
       "   'published_journal': 'Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Signatures of Dobzhansky-Muller Incompatibilities in the Genomes of Recombinant Inbred Lines',\n",
       "   'preprint_authors': 'Maria Colom-Tatch;Frank Johannes;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-06-16',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'In the construction of Recombinant Inbred Lines (RILs) from two divergent inbred parents certain genotype (or epigenotype) combinations may be functionally \\\\\"incompatible\\\\\" when brought together in the genomes of the progeny, thus resulting in sterility or lower fertility. Natural selection against these epistatic combinations during inbreeding can change haplotype frequencies and distort linkage disequilibrium (LD) relations between loci within and across chromosomes. These LD distortions have received increased experimental attention, because they point to genomic regions that may drive Dobzhansky-Muller-type of reproductive isolation and, ultimately, speciation in the wild. Here we study the selection signatures of two-locus epistatic incompatibility models and quantify their impact on the genetic composition of the genomes of 2-way RILs obtained by selfing. We also consider the biases introduced by breeders when trying to counteract the loss of lines by selectively propagating only viable seeds. Building on our theoretical results, we develop model-based maximum likelihood (ML) tests which can be employed in pairwise genome scans for incompatibility loci using multi-locus genotype data. We illustrate this ML approach in the context of two published A.thaliana RIL panels. Our work lays the theoretical foundation for studying more complex systems such as RILs obtained by sibling mating and/or from multi-parental crosses.',\n",
       "   'preprint_author_corresponding': 'Frank  Johannes',\n",
       "   'preprint_author_corresponding_institution': 'University of Groningen'},\n",
       "  {'preprint_doi': '10.1101/021105',\n",
       "   'published_doi': '10.1093/bioinformatics/btv733',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'PaxtoolsR: Pathway Analysis in R Using Pathway Commons',\n",
       "   'preprint_authors': 'Augustin Luna;zgn Babur;Blent Arman Aksoy;Emek Demir;Chris Sander;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-06-18',\n",
       "   'published_date': '2015-12-18',\n",
       "   'preprint_abstract': 'PurposePaxtoolsR package enables access to pathway data represented in the BioPAX format and made available through the Pathway Commons webservice for users of the R language. Features include the extraction, merging, and validation of pathway data represented in the BioPAX format. This package also provides novel pathway datasets and advanced querying features for R users through the Pathway Commons webservice allowing users to query, extract, and retrieve data and integrate this data with local BioPAX datasets.\\\\n\\\\nAvailabilityThe PaxtoolsR package is compatible with R 3.1.1 on Windows, Mac OS X, and Linux using Bioconductor 3.0 and is available through the Bioconductor R package repository along with source code and a tutorial vignette describing common tasks, such as data visualization and gene set enrichment analysis. Source code and documentation are at http://bioconductor.org/packages/release/bioc/html/paxtoolsr.html. This plugin is free, open-source and licensed under the GNU Lesser General Public License (LGPL) v3.0.\\\\n\\\\nContactpaxtools@cbio.mskcc.org',\n",
       "   'preprint_author_corresponding': 'Augustin  Luna',\n",
       "   'preprint_author_corresponding_institution': 'Computational Biology Center, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA'},\n",
       "  {'preprint_doi': '10.1101/021428',\n",
       "   'published_doi': '10.1371/journal.pcbi.1004648',\n",
       "   'published_journal': 'PLOS Computational Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Simple Plans or Sophisticated Habits? State, Transition and Learning Interactions in the Two-step Task.',\n",
       "   'preprint_authors': 'Thomas Akam;Rui Costa;Peter Dayan;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-06-24',\n",
       "   'published_date': '2015-12-11',\n",
       "   'preprint_abstract': 'The recently developed  two-step behavioural task promises to differentiate model-based or goal-directed from model-free or habitual reinforcement learning, while generating neurophysiologically-friendly decision datasets with parametric variation of decision variables. These desirable features have prompted widespread adoption of the task. However, the signatures of model-based control can be elusive - here, we investigate model-free learning methods that, depending on the analysis strategy, can masquerade as being model-based. We first show that unadorned model-free reinforcement learning can induce correlations between action values at the start of the trial and the subsequent trial events in such a way that analysis based on comparing successive trials can lead to erroneous conclusions. We also suggest a correction to the analysis that can alleviate this problem. We then consider model-free reinforcement learning strategies based on different state representations from those envisioned by the experimenter, which generate behaviour that appears model-based under these, and also more sophisticated, analyses. The existence of such strategies is of particular relevance to the design and interpretation of animal studies using the two-step task, as extended training and a sharp contrast between good and bad options are likely to promote their use.\\\\n\\\\nAuthor SummaryPlanning is the use of a predictive model of the consequences of actions to guide decision making. Planning plays a critical role in human behaviour but isolating its contribution is challenging because it is complemented by control systems which learn values of actions directly from the history of reinforcement, resulting in automatized mappings from states to actions often termed habits. Our study examined a recently developed behavioural task which uses choices in a multi-step decision tree to differentiate planning from value-based control. Using simulation, we demonstrated the existence of strategies which produce behaviour that resembles planning but in fact arises as a fixed mapping from particular sorts of states to actions. These results show that when a planning problem is faced repeatedly, sophisticated automatization strategies may be developed which identify that there are in fact a limited number of relevant states of the world each with an appropriate fixed or habitual response. Understanding such strategies is important for the design and interpretation of tasks which aim to isolate the contribution of planning to behaviour. Such strategies are also of independent scientific interest as they may contribute to automatization of behaviour in complex environments.',\n",
       "   'preprint_author_corresponding': 'Thomas  Akam',\n",
       "   'preprint_author_corresponding_institution': 'Champalimaud Neuroscience Program'},\n",
       "  {'preprint_doi': '10.1101/021964',\n",
       "   'published_doi': '10.1371/journal.pcbi.1004615',\n",
       "   'published_journal': 'PLOS Computational Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'How obstacles perturb population fronts and alter their genetic structure',\n",
       "   'preprint_authors': 'Wolfram Moebius;Andrew W. Murray;David R. Nelson;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-07-05',\n",
       "   'published_date': '2015-12-22',\n",
       "   'preprint_abstract': \"As populations spread into new territory, environmental heterogeneities can shape the population front and genetic composition. We study here the effect of one important building block of inhomogeneous environments, compact obstacles. With a combination of experiments, theory, and simulation, we show how isolated obstacles both create long-lived distortions of the front shape and amplify the effect of genetic drift. A system of bacteriophage T7 spreading on a spatially heterogeneous Escherichia coli lawn serves as an experimental model system to study population expansions. Using an inkjet printer, we create well-defined replicates of the lawn and quantitatively study the population expansion manifested in plaque growth. The transient perturbations of the plaque boundary found in the experiments are well described by a model in which the front moves with constant speed. Independent of the precise details of the expansion, we show that obstacles create a kink in the front that persists over large distances and is insensitive to the details of the obstacle's shape. The small deviations between experimental findings and the predictions of the constant speed model can be understood with a more general reaction-diffusion model, which reduces to the constant speed model when the obstacle size is large compared to the front width. Using this framework, we demonstrate that frontier alleles that just graze the side of an isolated obstacle increase in abundance, a phenomenon we call 'geometry-enhanced genetic drift', complementary to the founder effect associated with spatial bottlenecks. Bacterial range expansions around nutrient-poor barriers and stochastic simulations confirm this prediction, the latter highlight as well the effect of the obstacle on the genealogy of individuals at the front. We argue that related ideas and experimental techniques are applicable to a wide variety of more complex environments, leading to a better understanding of how environmental heterogeneities affect population range expansions.\",\n",
       "   'preprint_author_corresponding': 'Wolfram  Moebius',\n",
       "   'preprint_author_corresponding_institution': 'Harvard University'},\n",
       "  {'preprint_doi': '10.1101/023275',\n",
       "   'published_doi': '10.1038/ncomms10179',\n",
       "   'published_journal': 'Nature Communications',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Effect of Localization on the Stability of Mutualistic Ecological Networks',\n",
       "   'preprint_authors': 'Samir Suweis;Jacopo Grilli;Jayanth Banavar;Stefano Allesina;Amos Maritan;',\n",
       "   'preprint_category': 'Ecology ',\n",
       "   'preprint_date': '2015-07-27',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'The relationships between the core-periphery architecture of the species interaction network and the mechanisms ensuring the stability in mutualistic ecological communities are still unclear. In particular, most studies have focused their attention on asymptotic resilience or persistence, neglecting how perturbations propagate through the system. Here we develop a theoretical framework to evaluate the relationship between architecture of the interaction networks and the impact of perturbations by studying localization, a measure describing the ability of the perturbation to propagate through the network. We show that mutualistic ecological communities are localized, and localization reduces perturbation propagation and attenuates its impact on species abundance. Localization depends on the topology of the interaction networks, and it positively correlates with the variance of the weighted degree distribution, a signature of the network topological hetereogenity. Our results provide a different perspective on the interplay between the architecture of interaction networks in mutualistic communities and their stability.',\n",
       "   'preprint_author_corresponding': 'Samir  Suweis',\n",
       "   'preprint_author_corresponding_institution': 'University of Padova'},\n",
       "  {'preprint_doi': '10.1101/024208',\n",
       "   'published_doi': '10.12688/f1000research.7221.1',\n",
       "   'published_journal': 'F1000Research',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Oxygen changes drive non-uniform scaling in Drosophila melanogaster embryogenesis',\n",
       "   'preprint_authors': 'Steven Gregory Kuntz;Michael B. Eisen;',\n",
       "   'preprint_category': 'Developmental Biology ',\n",
       "   'preprint_date': '2015-08-12',\n",
       "   'published_date': '2015-10-23',\n",
       "   'preprint_abstract': 'In Drosophila embryogenesis, increasing either oxygen concentration or temperature accelerates development. Having previously investigated temperatures impact on embryogenesis, we characterized developmental response to oxygen levels using time-lapse imaging. Changing oxygen concentrations greatly impact survival, with developmental rate changes that are dwarfed by those induced by temperature. While extreme temperatures increase early embryo mortality, mild hypoxia increases arrest and death during mid-embryogenesis and mild hyperoxia increases survival over normoxia. Though not independent, the reactions to temperature and oxygen are fundamentally different, with developmental time being inversely proportional to oxygen concentration but logarithmically related to temperature. Most notably, while development scales uniformly with temperature, oxygen changes drive developmental heterochrony. Morphological processes change with oxygen concentration at different rates. Gut formation is more severely slowed by decreases in oxygen, while head involution and syncytial development are less impacted than the rest of development. These data reveal that uniform scaling, seen with changes in temperature, is not the default result of adjusting developmental rate.',\n",
       "   'preprint_author_corresponding': 'Steven Gregory Kuntz',\n",
       "   'preprint_author_corresponding_institution': 'University of California, Berkeley'},\n",
       "  {'preprint_doi': '10.1101/007906',\n",
       "   'published_doi': '10.1109/TCBB.2015.2509992',\n",
       "   'published_journal': 'IEEE/ACM Transactions On Computational Biology And Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Gene regulatory network inference from perturbed time-series expression data via ordered dynamical expansion of non-steady state actors',\n",
       "   'preprint_authors': 'Mahdi Zamanighomi;Mostafa Zamanian;Michael J Kimber;Zhengdao Wang;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2014-09-29',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'The reconstruction of gene regulatory networks from gene expression data has been the subject of intense research activity over the past many decades. A variety of models and methods have been developed to address different aspects of this important problem. However, these techniques are often difficult to scale, are narrowly focused on particular biological and experimental platforms, and require experimental data that are typically unavailable and difficult to ascertain. The more recent availability of higher-throughput sequencing platforms, combined with more precise modes of genetic perturbation, presents an opportunity to formulate more robust and comprehensive approaches to gene network inference. Here, we propose a step-wise framework for identifying gene-gene regulatory interactions that expand from a known point of genetic or chemical perturbation using time series gene expression data. This novel approach sequentially identifies non-steady state genes post-perturbation and incorporates them into a growing series of low-complexity optimization problems. The governing ordinary differential equations of this model are rooted in the biophysics of stochastic molecular events that underlie gene regulation, delineating roles for both protein and RNA-mediated gene regulation. We show the successful application of our core algorithms for network inference using simulated and real datasets.',\n",
       "   'preprint_author_corresponding': 'Zhengdao  Wang',\n",
       "   'preprint_author_corresponding_institution': 'Iowa State University'},\n",
       "  {'preprint_doi': '10.1101/009803',\n",
       "   'published_doi': '10.1086/682948',\n",
       "   'published_journal': 'The American Naturalist',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The role of standing variation in geographic convergent adaptation',\n",
       "   'preprint_authors': 'Peter L. Ralph;Graham Coop;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2014-09-30',\n",
       "   'published_date': '2015-10-01',\n",
       "   'preprint_abstract': 'The extent to which populations experiencing shared selective pressures adapt through a shared genetic response is relevant to many questions in evolutionary biology. In a number of well studied traits and species, it appears that convergent evolution within species is common. In this paper, we explore how standing, genetic variation contributes to convergent genetic responses in a geographically spread population, extending our previous work on the topic. Geographically limited dispersal slows the spread of each selected allele, hence allowing other alleles - newly arisen mutants or present as standing variation - to spread before any one comes to dominate the population. When such alleles meet, their progress is substantially slowed - if the alleles are selectively equivalent, they mix slowly, dividing the species range into a random tessellation, which can be well understood by analogy to a Poisson process model of crystallization. In this framework, we derive the geographic scale over which a typical allele is expected to dominate, the time it takes the species to adapt as a whole, and the proportion of adaptive alleles that arise from standing variation. Finally, we explore how negative pleiotropic effects of alleles before an environment change can bias the subset of alleles that contribute to the species adaptive response. We apply the results to the many geographically localized G6PD deficiency alleles thought to confer resistance to malaria, where the large mutational target size makes it a likely candidate for adaptation from standing variation, despite the selective cost of G6PD deficiency alleles in the absence of malaria. We find the numbers and geographic spread of these alleles matches our predictions reasonably well, consistent with the view that they arose from a combination of standing variation and new mutations since the advent of malaria. Our results suggest that much of adaptation may be geographically local even when selection pressures are homogeneous. Therefore, we argue that caution must be exercised when arguing that strongly geographically restricted alleles are necessarily the outcome of local adaptation. We close by discussing the implications of these results for ideas of species coherence and the nature of divergence between species.',\n",
       "   'preprint_author_corresponding': 'Graham  Coop',\n",
       "   'preprint_author_corresponding_institution': 'UC Davis'},\n",
       "  {'preprint_doi': '10.1101/018465',\n",
       "   'published_doi': '10.1186/s13059-015-0834-7',\n",
       "   'published_journal': 'Genome Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Metagenome-assembled genomes uncover a global brackish microbiome',\n",
       "   'preprint_authors': 'Luisa W Hugerth;John Larsson;Johannes Alneberg;Markus V Lindh;Catherine Legrand;Jarone Pinhassi;Anders F Andersson;',\n",
       "   'preprint_category': 'Microbiology ',\n",
       "   'preprint_date': '2015-04-23',\n",
       "   'published_date': '2015-12-14',\n",
       "   'preprint_abstract': 'Microbes are main drivers of biogeochemical cycles in oceans and lakes, yet surprisingly few bacterioplankton genomes have been sequenced, partly due to difficulties in cultivating them. Here we used automatic binning to reconstruct a large number of bacterioplankton genomes from a metagenomic time-series from the Baltic Sea. The genomes represent novel species within freshwater and marine clades, including clades not previously genome-sequenced. Their seasonal dynamics followed phylogenetic patterns, but with fine-grained lineage specific adaptations. Signs of streamlining were evident in most genomes, and estimated genome sizes correlated with abundance variation across filter size fractions. Comparing the genomes with globally distributed aquatic metagenomes suggested the existence of a global brackish metacommunity whose populations diverged from freshwater and marine relatives >100,000 years ago, hence long before the Baltic Sea was formed (8000 years). This markedly contrasts to most Baltic Sea multicellular organisms that are locally adapted populations of fresh- or marine counterparts.',\n",
       "   'preprint_author_corresponding': 'Anders F Andersson',\n",
       "   'preprint_author_corresponding_institution': 'KTH Royal Institute of Technology'},\n",
       "  {'preprint_doi': '10.1101/018788',\n",
       "   'published_doi': '10.1038/ng.3467',\n",
       "   'published_journal': 'Nature Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Fine-mapping cellular QTLs with RASQUAL and ATAC-seq',\n",
       "   'preprint_authors': 'Natsuhiko Kumasaka;Andrew Knights;Daniel Gaffney;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-04-30',\n",
       "   'published_date': '2015-12-14',\n",
       "   'preprint_abstract': 'When cellular traits are measured using high-throughput DNA sequencing quantitative trait loci (QTLs) manifest at two levels: population level differences between individuals and allelic differences between cis-haplotypes within individuals. We present RASQUAL (Robust Allele Specific QUAntitation and quality controL), a novel statistical approach for association mapping that integrates genetic effects and robust modelling of biases in next generation sequencing (NGS) data within a single, probabilistic framework. RASQUAL substantially improves causal variant localisation and sensitivity of association detection over existing methods in RNA-seq, DNaseI-seq and ChIP-seq data. We illustrate how RASQUAL can be used to maximise association detection by generating the first map of chromatin accessibility QTLs (caQTLs) in a European population using ATAC-seq. Despite a modest sample size, we identified 2,706 independent caQTLs (FDR 10%) and illustrate how RASQUALs improved causal variant localisation provides powerful information for fine-mapping disease-associated variants. We also map \\\\\"multipeak\\\\\" caQTLs, identical genetic associations found across multiple, independent open chromatin regions and illustrate how genetic signals in ATAC-seq data can be used to link distal regulatory elements with gene promoters. Our results highlight how joint modelling of population and allele-specific genetic signals can improve functional interpretation of noncoding variation.',\n",
       "   'preprint_author_corresponding': 'Daniel  Gaffney',\n",
       "   'preprint_author_corresponding_institution': 'Wellcome Trust Sanger Institute'},\n",
       "  {'preprint_doi': '10.1101/013995',\n",
       "   'published_doi': '10.1186/1471-2105-16-S18-S6',\n",
       "   'published_journal': 'BMC Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Privacy-preserving search for chemical compound databases',\n",
       "   'preprint_authors': 'Kana Shimizu;Koji Nuida;Hiromi Arai;Shigeo Mitsunari;Nuttapong Attrapadung;Michiaki Hamada;Koji Tsuda;Takatsugu Hirokawa;Jun Sakuma;Goichiro Hanaoka;Kiyoshi Asai;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-01-19',\n",
       "   'published_date': '2015-12-09',\n",
       "   'preprint_abstract': 'BackgroundSearching for similar compounds in a database is the most important process for in-silico drug screening. Since a query compound is an important starting point for the new drug, a query holder, who is afraid of the query being monitored by the database server, usually downloads all the records in the database and uses them in a closed network. However, a serious dilemma arises when the database holder also wants to output no information except for the search results, and such a dilemma prevents the use of many important data resources.\\\\n\\\\nResultsIn order to overcome this dilemma, we developed a novel cryptographic protocol that enables database searching while keeping both the query holders privacy and database holders privacy. Generally, the application of cryptographic techniques to practical problems is difficult because versatile techniques are computationally expensive while computationally inexpensive techniques can perform only trivial computation tasks. In this study, our protocol is successfully built only from an additive-homomorphic cryptosystem, which allows only addition performed on encrypted values but is computationally efficient compared with versatile techniques such as general purpose multi-party computation. In an experiment searching ChEMBL, which consists of more than 1,200,000 compounds, the proposed method was 36,900 times faster in CPU time and 12,000 times as efficient in communication size compared with general purpose multi-party computation.\\\\n\\\\nConclusionWe proposed a novel privacy-preserving protocol for searching chemical compound databases. The proposed method, easily scaling for large-scale databases, may help to accelerate drug discovery research by making full use of unused but valuable data that includes sensitive information.',\n",
       "   'preprint_author_corresponding': 'Kana  Shimizu',\n",
       "   'preprint_author_corresponding_institution': 'National Institute of Advanced Industrial Science and Technology (AIST)'},\n",
       "  {'preprint_doi': '10.1101/027458',\n",
       "   'published_doi': '10.7554/eLife.11147',\n",
       "   'published_journal': 'eLife',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A Serial Multiplex Immunogold Labeling Method for Identifying Peptidergic Neurons in Connectomes',\n",
       "   'preprint_authors': 'Reza Shahidi;Elizabeth Williams;Markus Conzelmann;Albina Asadulina;Csaba Veraszto;Sanja Jasek;Luis A Bezares-Calderon;Gaspar Jekely;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-09-23',\n",
       "   'published_date': '2015-12-15',\n",
       "   'preprint_abstract': 'Electron microscopy-based connectomics aims to comprehensively map synaptic connections in neural tissue. However, current approaches are limited in their capacity to directly assign molecular identities to neurons. Here, we use serial multiplex immunogold labeling (siGOLD) and serial-section transmission electron microscopy (ssTEM) to identify multiple peptidergic neurons in a connectome. The high immunogenicity of neuropeptides and their broad distribution along axons, allowed us to identify distinct neurons by immunolabeling small subsets of sections within larger series. We demonstrate the scalability of siGOLD by using 11 neuropeptide antibodies on a full-body larval ssTEM dataset of the annelid Platynereis. We also reconstruct a peptidergic circuitry comprising the sensory nuchal organs, found by siGOLD to express pigment-dispersing factor, a circadian neuropeptide. Our approach enables the direct overlaying of chemical neuromodulatory maps onto synaptic connectomic maps in the study of nervous systems.',\n",
       "   'preprint_author_corresponding': 'Gaspar  Jekely',\n",
       "   'preprint_author_corresponding_institution': 'Max Planck Institute for Developmental Biology'},\n",
       "  {'preprint_doi': '10.1101/029140',\n",
       "   'published_doi': '10.1016/j.bbamem.2015.12.014',\n",
       "   'published_journal': 'Biochimica et Biophysica Acta (BBA) - Biomembranes',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Molecular Simulation of Nonfacilitated Membrane Permeation',\n",
       "   'preprint_authors': 'Ernest Awoonor-Williams;Christopher Rowley;',\n",
       "   'preprint_category': 'Biophysics ',\n",
       "   'preprint_date': '2015-10-14',\n",
       "   'published_date': '2015-12-16',\n",
       "   'preprint_abstract': 'This is a review. Non-electrolytic compounds typically cross cell membranes by passive diffusion. The rate of permeation is dependent on the chemical properties of the solute and the composition of the lipid bilayer membrane. Predicting the permeability coefficient of a solute is important in pharmaceutical chemistry and toxicology. Molecular simulation has proven to be a valuable tool for modeling permeation of solutes through a lipid bilayer. In particular, the solubility-diffusion model has allowed for the quantitative calculation of permeability coefficients. The underlying theory and computational methods used to calculate membrane permeability are reviewed. We also discuss applications of these methods to examine the permeability of solutes and the effect of membrane composition on permeability. The application of coarse grain and polarizable models is discussed.',\n",
       "   'preprint_author_corresponding': 'Christopher  Rowley',\n",
       "   'preprint_author_corresponding_institution': 'Memorial University of Newfoundland'},\n",
       "  {'preprint_doi': '10.1101/031591',\n",
       "   'published_doi': '10.1109/TCBB.2015.2505278',\n",
       "   'published_journal': 'IEEE/ACM Transactions On Computational Biology And Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Towards extracting supporting information about predicted protein-protein interactions',\n",
       "   'preprint_authors': 'Adam Roth;Sandeep Subramanian;Madhavi Ganapathiraju;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-11-14',\n",
       "   'published_date': '2015-12-07',\n",
       "   'preprint_abstract': 'One of the goals of relation extraction is to identify protein-protein interactions (PPIs) in biomedical literature. Current systems are capturing binary relations and also the direction and type of an interaction. Besides assisting in the curation PPIs into databases, there has been little real-world application of these algorithms. We describe UPSITE, a text mining tool for extracting evidence in support of a hypothesized interaction. Given a predicted PPI, UPSITE uses a binary relation detector to check whether a PPI is found in abstracts in PubMed. If it is not found, UPSITE retrieves documents relevant to each of the two proteins separately, and extracts contextual information about biological events surrounding each protein, and calculates semantic similarity of the two proteins to provide evidential support for the predicted PPI. In evaluations, relation extraction achieved an Fscore of 0.88 on the HPRD50 corpus, and semantic similarity measured with angular distance was found to be statistically significant. With the development of PPI prediction algorithms, the burden of interpreting the validity and relevance of novel PPIs is on biologists. We suggest that presenting annotations of the two proteins in a PPI side-by-side and a score that quantifies their similarity lessens this burden to some extent.',\n",
       "   'preprint_author_corresponding': 'Madhavi  Ganapathiraju',\n",
       "   'preprint_author_corresponding_institution': 'University of Pittsburgh'},\n",
       "  {'preprint_doi': '10.1101/033886',\n",
       "   'published_doi': '10.1111/jeb.12807',\n",
       "   'published_journal': 'Journal of Evolutionary Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Fitness effects of new mutations in Chlamydomonas reinhardtii across two stress gradients',\n",
       "   'preprint_authors': 'Susanne A Kraemer;Andrew D Morgan;Robert W Ness;Peter D Keightley;Nick Colegrave;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-12-07',\n",
       "   'published_date': '2015-12-11',\n",
       "   'preprint_abstract': 'Most spontaneous mutations affecting fitness are likely to be deleterious, but the strength of selection acting on them might be impacted by environmental stress. Such stress-dependent selection could expose hidden genetic variation, which in turn might increase the adaptive potential of stressed populations. On the other hand, this variation might represent a genetic load and thus lead to population extinction under stress. Previous studies to determine the link between stress and mutational effects on fitness, however, have produced inconsistent results. Here, we determined the net change in fitness in 29 genotypes of the green algae Chlamydomonas reinhardtii that accumulated mutations in the near absence of selection for approximately 1,000 generations across two stress gradients, increasing NaCl and decreasing phosphate. We found mutational effects to be magnified under extremely stressful conditions, but such effects were specific both to the type of stress as well as to the genetic background. The detection of stress-dependent fitness effects of mutations depended on accurately scaling relative fitness measures by generation times, thus offering an explanation for the inconsistencies among previous studies.',\n",
       "   'preprint_author_corresponding': 'Susanne A Kraemer',\n",
       "   'preprint_author_corresponding_institution': 'University of Edinburgh'},\n",
       "  {'preprint_doi': '10.1101/033985',\n",
       "   'published_doi': '10.1007/s10822-015-9888-6',\n",
       "   'published_journal': 'Journal of Computer-Aided Molecular Design',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Modeling error in experimental assays using the bootstrap principle: Understanding discrepancies between assays using different dispensing technologies',\n",
       "   'preprint_authors': 'Sonya M Hanson;Sean Ekins;John D Chodera;',\n",
       "   'preprint_category': 'Biochemistry ',\n",
       "   'preprint_date': '2015-12-09',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'All experimental assay data contains error, but the magnitude, type, and primary origin of this error is often not obvious. Here, we describe a simple set of assay modeling techniques based on the bootstrap principle that allow sources of error and bias to be simulated and propagated into assay results. We demonstrate how deceptively simple operations--such as the creation of a dilution series with a robotic liquid handler--can significantly amplify imprecision and even contribute substantially to bias. To illustrate these techniques, we review an example of how the choice of dispensing technology can impact assay measurements, and show how large contributions to discrepancies between assays can be easily understood and potentially corrected for. These simple modeling techniques--illustrated with an accompanying IPython notebook--can allow modelers to understand the expected error and bias in experimental datasets, and even help experimentalists design assays to more effectively reach accuracy and imprecision goals.',\n",
       "   'preprint_author_corresponding': 'John D Chodera',\n",
       "   'preprint_author_corresponding_institution': 'Memorial Sloan Kettering Cancer Center'},\n",
       "  {'preprint_doi': '10.1101/034538',\n",
       "   'published_doi': '10.1371/journal.pmed.1001918',\n",
       "   'published_journal': 'PLOS Medicine',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Diversity in Clinical and Biomedical Research: A Promise Yet to Be Fulfilled',\n",
       "   'preprint_authors': 'Sam S Oh;Joshua Galanter;Neeta Thakur;Maria Pino-Yanes;Nicolas E Barcelo;Marquitta J White;Danille M de Bruin;Ruth M Greenblatt;Kirsten Bibbins-Domingo;Alan H.B. Wu;Luisa N Borrell;Chris Gunter;Neil R Powe;Esteban G Burchard;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-12-16',\n",
       "   'published_date': '2015-12-15',\n",
       "   'preprint_abstract': 'Summary PointsO_LIHealth disparities persist across race/ethnicity for the majority of Healthy People 2010 health indicators.\\\\nC_LIO_LIMost physicians and scientists are informed by research extrapolated from a largely homogenous population, usually White and male.\\\\nC_LIO_LIA growing proportion of Americans are not fully benefiting from clinical and biomedical advances since racial and ethnic minorities make up nearly 40% of the U.S. population.\\\\nC_LIO_LIIgnoring the racial/ethnic diversity of the U.S. population is a missed scientific opportunity to fully understand the factors that lead to disease or health.\\\\nC_LIO_LIU.S. biomedical research and study populations must better reflect the countrys changing demographics. Adequate representation of diverse populations in scientific research is imperative as a matter of social justice, economics, and science.\\\\nC_LI',\n",
       "   'preprint_author_corresponding': 'Esteban G Burchard',\n",
       "   'preprint_author_corresponding_institution': 'University of California, San Francisco'},\n",
       "  {'preprint_doi': '10.1101/004002',\n",
       "   'published_doi': '10.1086/682022',\n",
       "   'published_journal': 'The American Naturalist',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Model adequacy and the macroevolution of angiosperm functional traits',\n",
       "   'preprint_authors': 'Matthew Pennell;Richard G FitzJohn;William K Cornwell;Luke J Harmon;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2014-04-07',\n",
       "   'published_date': '2015-08-01',\n",
       "   'preprint_abstract': 'Making meaningful inferences from phylogenetic comparative data requires a meaningful model of trait evolution. It is thus important to determine whether the model is appropriate for the data and the question being addressed. One way to assess this is to ask whether the model provides a good statistical explanation for the variation in the data. To date, researchers have focused primarily on the explanatory power of a model relative to alternative models. Methods have been developed to assess the adequacy, or absolute explanatory power, of phylogenetic trait models but these have been restricted to specific models or questions. Here we present a general statistical framework for assessing the adequacy of phylogenetic trait models. We use our approach to evaluate the statistical performance of commonly used trait models on 337 comparative datasets covering three key Angiosperm functional traits. In general, the models we tested often provided poor statistical explanations for the evolution of these traits. This was true for many different groups and at many different scales. Whether such statistical inadequacy will qualitatively alter inferences draw from comparative datasets will depend on the context. Regardless, assessing model adequacy can provide interesting biological insights -- how and why a model fails to describe variation in a dataset gives us clues about what evolutionary processes may have driven trait evolution across time.',\n",
       "   'preprint_author_corresponding': 'Matthew  Pennell',\n",
       "   'preprint_author_corresponding_institution': 'University of Idaho'},\n",
       "  {'preprint_doi': '10.1101/013755',\n",
       "   'published_doi': '10.1093/molbev/msv331',\n",
       "   'published_journal': 'Molecular Biology and Evolution',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The Time-Scale of Recombination Rate Evolution in Great Apes',\n",
       "   'preprint_authors': 'Laurie S Stevison;August E Woerner;Jeffrey M Kidd;Joanna L Kelley;Krishna R Veeramah;Kimberly F McManus;Carlos D Bustamante;Michael F Hammer;Jeffrey D Wall;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-01-13',\n",
       "   'published_date': '2015-12-15',\n",
       "   'preprint_abstract': 'We present three linkage-disequilibrium (LD)-based recombination maps generated using whole-genome sequencing data of 10 Nigerian chimpanzees, 13 bonobos, and 15 western gorillas, collected as part of the Great Ape Genome Project (Prado-Martinez et al. 2013). Using species-specific PRDM9 sequences to predict potential binding sites, we identified an important role for PRDM9 in predicting recombination rate variation broadly across great apes. Our results are contrary to previous research that PRDM9 is not associated with recombination in western chimpanzees (Auton et al. 2012). Additionally, we show that fewer hotspots are shared among chimpanzee subspecies than within human populations, further narrowing the time-scale of complete hotspot turnover. We quantified the variation in the biased distribution of recombination rates towards recombination hotspots across great apes. We found that correlations between broad-scale recombination rates decline more rapidly than nucleotide divergence between species. We also compared the skew of recombination rates at centromeres and telomeres between species and show a skew from chromosome means extending as far as 10-15 Mb from chromosome ends. Further, we examined broad-scale recombination rate changes near a translocation in gorillas and found minimal differences as compared to other great ape species perhaps because the coordinates relative to the chromosome ends were unaffected. Finally, based on multiple linear regression analysis, we found that various correlates of recombination rate persist throughout primates including repeats, diversity, divergence and local effective population size (Ne). Our study is the first to analyze within-and between-species genome-wide recombination rate variation in several close relatives.',\n",
       "   'preprint_author_corresponding': 'Laurie S Stevison',\n",
       "   'preprint_author_corresponding_institution': 'University of California San Francisco'},\n",
       "  {'preprint_doi': '10.1101/023002',\n",
       "   'published_doi': '10.3389/fmicb.2015.01392',\n",
       "   'published_journal': 'Frontiers in Microbiology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Sulfur Isotope Effects of Dissimilatory Sulfite Reductase',\n",
       "   'preprint_authors': 'William D. Leavitt;Alexander S. Bradley;Andr A. Santos;Ins A.C. Pereira;David T. Johnston;',\n",
       "   'preprint_category': 'Microbiology ',\n",
       "   'preprint_date': '2015-07-22',\n",
       "   'published_date': '2015-12-24',\n",
       "   'preprint_abstract': 'The precise interpretation of environmental sulfur isotope records requires a quantitative understanding of the biochemical controls on sulfur isotope fractionation by the principle isotope-fractionating process within the S cycle, microbial sulfate reduction (MSR). Here we provide the only direct observation of the major (34S/32S) and minor (33S/32S, 36S/32S) sulfur isotope fractionations imparted by a central enzyme in the energy metabolism of sulfate reducers, dissimilatory sulfite reductase (DsrAB). Results from in vitro sulfite reduction experiments allow us to calculate the in vitro DsrAB isotope effect in 34S/32S (hereafter, 34{varepsilon}DsrAB) to be 15.3{+/-}2{per thousand}, 2{sigma}.The accompanying minor isotope effect in 33S, described as 33{lambda}DsrAB, is calculated to be 0.5150{+/-}0.0012, 2{sigma}. These observations facilitate a rigorous evaluation of the isotopic fractionation associated with the dissimilatory MSR pathway, as well as of the environmental variables that govern the overall magnitude of fractionation by natural communities of sulfate reducers. The isotope effect induced by DsrAB upon sulfite reduction is a factor of 0.3 to 0.6 times prior indirect estimates, which have ranged from 25 to 53{per thousand} in 34{varepsilon}DsrAB. The minor isotope fractionation observed from DsrAB is consistent with a kinetic or equilibrium effect. Our in vitro constraints on the magnitude of 34{varepsilon}DsrAB is similar to the median value of experimental observations compiled from all known published work, where 34{varepsilon}r-p = 16.1{per thousand} (r - p indicates reactant versus product, n = 648). This value closely matches those of MSR operating at high sulfate reduction rates in both laboratory chemostat experiments (34{varepsilon}SO4-H2S = 17.3{+/-}1.5{per thousand}) and in modern marine sediments (34{varepsilon}SO4-H2S = 17.3{+/-}3.8{per thousand}). Targeting the direct isotopic consequences of a specific enzymatic processes is a fundamental step toward a biochemical foundation for reinterpreting the biogeochemical and geobiological sulfur isotope records in modern and ancient environments.',\n",
       "   'preprint_author_corresponding': 'William D. Leavitt',\n",
       "   'preprint_author_corresponding_institution': 'Department of Earth and Planetary Sciences, Washington University in St. Louis'},\n",
       "  {'preprint_doi': '10.1101/021204',\n",
       "   'published_doi': '10.1126/science.1500816',\n",
       "   'published_journal': 'Science Advances',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Decoding the Population Activity of Grid Cells for Spatial Localization and Goal-Directed Navigation',\n",
       "   'preprint_authors': 'Martin Stemmler;Alexander Mathis;Andreas VM Herz;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-06-19',\n",
       "   'published_date': '2015-12-18',\n",
       "   'preprint_abstract': 'Mammalian grid cells fire whenever an animal crosses the points of an imaginary, hexagonal grid tessellating the environment. Here, we show how animals can localize themselves and navigate by reading-out a simple population vector of grid cell activity across multiple scales, even though this activity is intrinsically stochastic. This theory of dead reckoning explains why grid cells are organized into modules with equal lattice scale and orientation. Computing the homing vector is least error-prone when the ratio of successive grid scales is around 3/2. Silencing intermediate-scale modules should cause systematic errors in navigation, while knocking out the module at the smallest scale will only affect navigational precision. Read-out neurons should behave like goal-vector cells subject to nonlinear gain fields.',\n",
       "   'preprint_author_corresponding': 'Andreas VM Herz',\n",
       "   'preprint_author_corresponding_institution': 'Ludwig-Maximilians-University'},\n",
       "  {'preprint_doi': '10.1101/031625',\n",
       "   'published_doi': '10.1038/nmeth.3688',\n",
       "   'published_journal': 'Nature Methods',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A spectral analysis approach to detect actively translated open reading frames in high-resolution ribosome profiling data',\n",
       "   'preprint_authors': 'Lorenzo Calviello;Neelanjan Mukherjee;Emanuel Wyler;Henrik Zauber;Antje Hirsekorn;Matthias Selbach;Markus Landthaler;Benedikt Obermayer;Uwe Ohler;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-11-13',\n",
       "   'published_date': '2015-12-14',\n",
       "   'preprint_abstract': 'RNA sequencing protocols allow for quantifying gene expression regulation at each individual step, from transcription to protein synthesis. Ribosome Profiling (Ribo-seq) maps the positions of translating ribosomes over the entire transcriptome. Despite its great potential, a rigorous statistical approach to identify translated regions by means of the characteristic three-nucleotide periodicity of Ribo-seq data is not yet available. To fill this gap, we developed RiboTaper, which quantifies the significance of periodic Ribo-seq reads via spectral analysis methods.\\\\n\\\\nWe applied RiboTaper on newly generated, deep Ribo-seq data in HEK293 cells, to derive an extensive map of translation that covers Open Reading Frame (ORF) annotations for more than 11,000 protein-coding genes. We also find distinct ribosomal signatures for several hundred detected upstream ORFs and ORFs in annotated non-coding genes (ncORFs). Mass spectrometry data confirms that RiboTaper achieves excellent coverage of the cellular proteome and validates dozens of novel peptide products. Collectively, RiboTaper (available at https://ohlerlab.mdc-berlin.de/software/) is a powerful method for comprehensive de novo identification of actively used ORFs in the human genome.',\n",
       "   'preprint_author_corresponding': 'Uwe  Ohler',\n",
       "   'preprint_author_corresponding_institution': 'Berlin Institute for Medical Systems Biology, Max Delbrueck Center for Molecular Medicine'},\n",
       "  {'preprint_doi': '10.1101/017426',\n",
       "   'published_doi': '10.1016/j.molliq.2015.08.054',\n",
       "   'published_journal': 'Journal of Molecular Liquids',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'On the Optimal Use of Isotherm Models for the Characterization of Biosorption of Lead Onto Algae',\n",
       "   'preprint_authors': 'F. Brouers;Tariq Al-Musawi;',\n",
       "   'preprint_category': 'Bioengineering ',\n",
       "   'preprint_date': '2015-04-01',\n",
       "   'published_date': '2015-12-21',\n",
       "   'preprint_abstract': 'In this study the experimental isotherm data of biosorption of Pb(II) onto algae was modeled using several models. These models are: Langmuir, Hill-Sips, Brouers-Sotolongo, Brouers-Gaspard, and Redlich-Peterson models. The coefficients of each model were determined by non-linear fitting using Mathematica9 program. The maximum Pb(II) removal rate increased with the increase of temperature and reached the maximum value (98%) at the temperature of 40 {degrees}C. Even if the R2 error quantity is not the unique and always the best measure for nonlinear fitting, the Brouers-Sotolongo model gives in all cases the best fit and is definitely the most suitable one to satisfactorily describe bioisorption of Pb(II) on the algal biomass. In addition, this study shows that a complete set of data is necessary to have a good representation of the isotherm.',\n",
       "   'preprint_author_corresponding': 'Tariq  Al-Musawi',\n",
       "   'preprint_author_corresponding_institution': 'University of Baghdad/ Environmental Engineering Department'},\n",
       "  {'preprint_doi': '10.1101/011296',\n",
       "   'published_doi': '10.1371/journal.pcbi.1004640',\n",
       "   'published_journal': 'PLOS Computational Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Wheres the noise? Key features of neuronal variability and inference emerge from self-organized learning',\n",
       "   'preprint_authors': 'Christoph Hartmann;Andreea Lazar;Jochen Triesch;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2014-11-10',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'AbstractTrial-to-trial variability and spontaneous activity of cortical recordings have been suggested to reflect intrinsic noise. This view is currently challenged by mounting evidence for structure in these phenomena: Trial-to-trial variability decreases following stimulus onset and can be predicted by previous spontaneous activity. This spontaneous activity is similar in magnitude and structure to evoked activity and can predict decisions. Allof the observed neuronal properties described above can be accounted for, at an abstract computational level, by the sampling-hypothesis, according to which response variability reflects stimulus uncertainty. However, a mechanistic explanation at the level of neural circuit dynamics is still missing.\\\\n\\\\nIn this study, we demonstrate that all of these phenomena can be accounted for by a noise-free self-organizing recurrent neural network model (SORN). It combines spike-timing dependent plasticity (STDP) and homeostatic mechanisms in a deterministic network of excitatory and inhibitory McCulloch-Pitts neurons. The network self-organizes to spatio-temporally varying input sequences.\\\\n\\\\nWe find that the key properties of neural variability mentioned above develop in this model as the network learns to perform sampling-like inference. Importantly, the model shows high trial-to-trial variability although it is fully deterministic. This suggests that the trial-to-trial variability in neural recordings may not reflect intrinsic noise. Rather, it may reflect a deterministic approximation of sampling-like learning and inference. The simplicity of the model suggests that these correlates of the sampling theory are canonical properties of recurrent networks that learn with a combination of STDP and homeostatic plasticity mechanisms.\\\\n\\\\nAuthor SummaryNeural recordings seem very noisy. If the exact same stimulus is shown to an animal multiple times, the neural response will vary. In fact, the activity of a single neuron shows many features of a stochastic process. Furthermore, in the absence of a sensory stimulus, cortical spontaneous activity has a magnitude comparable to the activity observed during stimulus presentation. These findings have led to a widespread belief that neural activity is indeed very noisy. However, recent evidence indicates that individual neurons can operate very reliably and that the spontaneous activity in the brain is highly structured, suggesting that much of the noise may in fact be signal. One hypothesis regarding this putative signal is that it reflects a form of probabilistic inference through sampling. Here we show that the key features of neural variability can be accounted for in a completely deterministic network model through self-organization. As the network learns a model of its sensory inputs, the deterministic dynamics give rise to sampling-like inference. Our findings show that the notorious variability in neural recordings does not need to be seen as evidence for a noisy brain. Instead it may reflect sampling-like inference emerging from a self-organized learning process.',\n",
       "   'preprint_author_corresponding': 'Christoph  Hartmann',\n",
       "   'preprint_author_corresponding_institution': 'Frankfurt Institute for Advanced Studies (FIAS)'},\n",
       "  {'preprint_doi': '10.1101/016469',\n",
       "   'published_doi': '10.1534/genetics.115.183814',\n",
       "   'published_journal': 'Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Efficient strategies for calculating blockwise likelihoods under the coalescent',\n",
       "   'preprint_authors': 'Konrad Lohse;Martin Chmelik;Simon Henry Martin;Nick H Barton;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-03-13',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'The inference of demographic history from genome data is hindered by a lack of efficient computational approaches. In particular, it has proven difficult to exploit the information contained in the distribution of genealogies across the genome. We have previously shown that the generating function (GF) of genealogies can be used to analytically compute likelihoods of demographic models from configurations of mutations in short sequence blocks (Lohse et al., 2011). Although the GF has a simple, recursive form, the size of such likelihood computations explodes quickly with the number of individuals and applications of this framework have so far been limited to small samples (pairs and triplets) for which the GF can be written down by hand. Here we investigate several strategies for exploiting the inherent symmetries of the coalescent. In particular, we show that the GF of genealogies can be decomposed into a set of equivalence classes which allows likelihood calculations from non-trivial samples. Using this strategy, we used Mathematica to automate block-wise likelihood calculations based on the GF for a very general set of demographic scenarios that may involve population size changes, continuous migration, discrete divergence and admixture between multiple populations. To give a concrete example, we calculate the likelihood for a model of isolation with migration (IM), assuming two diploid samples without phase and outgroup information, and compare the power of our approach to that of minimal pairwise samples. We demonstrate the new inference scheme with an analysis of two individual butterfly genomes from the sister species Heliconius melpomene rosina and Heliconius cyndo.',\n",
       "   'preprint_author_corresponding': 'Konrad  Lohse',\n",
       "   'preprint_author_corresponding_institution': 'University of Edinburgh'},\n",
       "  {'preprint_doi': '10.1101/019711',\n",
       "   'published_doi': '10.1073/pnas.1510805112',\n",
       "   'published_journal': 'Proceedings of the National Academy of Sciences',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Distance from Sub-Saharan Africa Predicts Mutational Load in Diverse Human Genomes',\n",
       "   'preprint_authors': 'Brenna M. Henn;Laura R Botigue;Stephan Peischl;Isabelle Dupanloup;Mikhail Lipatov;Brian K Maples;Alicia R Martin;Shaila Musharoff;Howard Cann;Michael Snyder;Laurent Excoffier;Jeffrey Kidd;Carlos D Bustamante;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-05-22',\n",
       "   'published_date': '2015-12-28',\n",
       "   'preprint_abstract': 'The Out-of-Africa (OOA) dispersal ~50,000 years ago is characterized by a series of founder events as modern humans expanded into multiple continents. Population genetics theory predicts an increase of mutational load in populations undergoing serial founder effects during range expansions. To test this hypothesis, we have sequenced full genomes and high-coverage exomes from 7 geographically divergent human populations from Namibia, Congo, Algeria, Pakistan, Cambodia, Siberia and Mexico. We find that individual genomes vary modestly in the overall number of predicted deleterious alleles. We show via spatially explicit simulations that the observed distribution of deleterious allele frequencies is consistent with the OOA dispersal, particularly under a model where deleterious mutations are recessive. We conclude that there is a strong signal of purifying selection at conserved genomic positions within Africa, but that many predicted deleterious mutations have evolved as if they were neutral during the expansion out of Africa. Under a model where selection is inversely related to dominance, we show that OOA populations are likely to have a higher mutation load due to increased allele frequencies of nearly neutral variants that are recessive or partially recessive.',\n",
       "   'preprint_author_corresponding': 'Jeffrey  Kidd',\n",
       "   'preprint_author_corresponding_institution': 'University of Michigan'},\n",
       "  {'preprint_doi': '10.1101/019976',\n",
       "   'published_doi': '10.1111/pce.12702',\n",
       "   'published_journal': 'Plant, Cell and Environment',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Expression of E. coli glycogen branching enzyme in an Arabidopsis mutant devoid of endogenous starch branching enzymes induces the synthesis of starch-like polyglucans',\n",
       "   'preprint_authors': \"Laura Boyer;Xavier Roussel;Adeline Courseaux;Ofilia Mvundza Ndjindji;Christine Lancelon-Pin;Jean-Luc Putaux;Ian Tetlow;Michael Emes;Bruno Pontoire;Christophe D'Hulst;Fabrice Wattebled;\",\n",
       "   'preprint_category': 'Plant Biology ',\n",
       "   'preprint_date': '2015-05-29',\n",
       "   'published_date': '2015-12-30',\n",
       "   'preprint_abstract': 'Starch synthesis requires several enzymatic activities including branching enzymes (BEs) responsible for the formation of (1[->]6) linkages. Distribution and number of these linkages are further controlled by debranching enzymes (DBEs) that cleave some of them, rendering the polyglucan water-insoluble and semi-crystalline. Although the activity of BEs and DBEs is mandatory to sustain normal starch synthesis, the relative importance of each in the establishment of the plant storage polyglucan (i.e. water-insolubility, crystallinity, presence of amylose) is still debated. Here, we have substituted the activity of BEs in Arabidopsis with that of the Escherichia coli glycogen branching enzyme (GlgB). The latter is the BE counterpart in the metabolism of glycogen, a highly branched water-soluble and amorphous storage polyglucan. GlgB was expressed in the be2 be3 double mutant of Arabidopsis that is devoid of BE activity and consequently free of starch. The synthesis of a water-insoluble, partly crystalline, amylose-containing starch-like polyglucan was restored in GlgB-expressing plants, suggesting that BEs origin only have a limited impact on establishing essential characteristics of starch. Moreover, the balance between branching and debranching is crucial for the synthesis of starch, as an excess of branching activity results in the formation of highly branched, water-soluble, poorly crystalline polyglucan.',\n",
       "   'preprint_author_corresponding': 'Fabrice  Wattebled',\n",
       "   'preprint_author_corresponding_institution': 'Universite de Lille'},\n",
       "  {'preprint_doi': '10.1101/025205',\n",
       "   'published_doi': '10.1534/g3.115.025106',\n",
       "   'published_journal': 'G3: Genes|Genomes|Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A novel recombinant DNA system for high efficiency affinity purification of proteins in Saccharomyces cerevisiae',\n",
       "   'preprint_authors': 'Brian H Carrick;Lixuan HaO;Philip J Smaldino;David R Engelke;',\n",
       "   'preprint_category': 'Molecular Biology ',\n",
       "   'preprint_date': '2015-08-21',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'AbstractIsolation of endogenous proteins from Saccharomyces cerevisiae has been facilitated by inserting encoding polypeptide affinity tags at the C-termini of chromosomal open reading frames (ORFs) using homologous recombination of DNA fragments. The tagged protein isolation is limited by a number of factors, including high cost of affinity resins for bulk isolation and low concentration of ligands on the resin surface, leading to low isolation efficiencies and trapping of contaminants. To address this we have created a recombinant \\\\\"CelTag\\\\\" DNA construct from which PCR fragments can be created to easily tag C-termini of S. cerevisiae ORFs using selection for a nat1 marker. The tag has a C-terminal cellulose binding module to be used in the first affinity step. Microgranular cellulose is very inexpensive and has an effectively continuous ligand on its surface, allowing rapid, highly efficient purification with minimal background in a single step. Cellulose-bound proteins are released by specific cleavage of an included site for TEV protease, giving nearly pure product. The tag can be lifted from the recombinant DNA construct either with or without a 13x myc epitope tag between the target ORF and the TEV protease site. Binding of CelTag protein fusions to cellulose is stable to high salt, nonionic detergents, and 1 M urea, allowing stringent washing conditions to remove loosely associated components, as needed, before specific elution. It is anticipated that this reagent will allow isolation of rare or unstable protein complexes from large quantities of yeast extract, including soluble, membrane-bound, or chromatin-associated assemblies.',\n",
       "   'preprint_author_corresponding': 'David R Engelke',\n",
       "   'preprint_author_corresponding_institution': 'University of Michigan'},\n",
       "  {'preprint_doi': '10.1101/025338',\n",
       "   'published_doi': '10.1371/journal.pntd.0004252',\n",
       "   'published_journal': 'PLOS Neglected Tropical Diseases',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Whole genome sequencing of field isolates reveals extensive genetic diversity in Plasmodium vivax from Colombia',\n",
       "   'preprint_authors': 'David J Winter;Maria A Pacheco;Andres Felipe A Vallejo;Rachel S. Schwartz;Myriam Arevalo-Herrera;Socrates Herrera;Reed Cartwright;Ananias Escalante;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-08-23',\n",
       "   'published_date': '2015-12-28',\n",
       "   'preprint_abstract': 'Plasmodium vivax is the most prevalent malarial species in South America and exerts a substantial burden on the populations it affects. The control and eventual elimination of P. vivax are global health priorities. Genomic research contributes to this objective by improving our understanding of the biology of P. vivax and through the development of new genetic markers that can be used to monitor efforts to reduce malaria transmission.\\\\n\\\\nHere we analyze whole-genome data from eight field samples from a region in Cordoba, Colombia where malaria is endemic. We find considerable genetic diversity within this population, a result that contrasts with earlier studies suggesting that P. vivax had limited diversity in the Americas. We also identify a selective sweep around a substitution known to confer resistance to sulphadoxine-pyrimethamine (SP). This is the first observation of a selective sweep for SP resistance in this species. These results indicate that P. vivax has been exposed to SP pressure even when the drug is not in use as a first line treatment for patients afflicted by this parasite. We identify multiple non-synonymous substitutions in three other genes known to be involved with drug resistance in Plasmodium species. Finally, we found extensive microsatellite polymorphisms. Using this information we developed 18 polymorphic and easy to score microsatellite loci that can be used in epidemiological investigations in South America.\\\\n\\\\nAuthor SummaryAlthough P. vivax is not as deadly as the more widely studied P. falciparum, it remains a pressing global health problem. Here we report the results of a whole-genome study of P. vivax from Cordoba, Colombia, in South America. This parasite is the most prevalent in this region. We show that the parasite population is genetically diverse, which is contrary to expectations from earlier studies from the Americas. We also find molecular evidence that resistance to an anti-malarial drug has arisen recently in this region. This selective sweep indicates that the parasite has been exposed to a drug that is not used as first-line treatment for this malaria parasite. In addition to extensive single nucleotide and microsatellite polymorphism, we report 18 new genetic loci that might be helpful for fine-scale studies of this species in the Americas.',\n",
       "   'preprint_author_corresponding': 'David J Winter',\n",
       "   'preprint_author_corresponding_institution': 'Arizona State University'},\n",
       "  {'preprint_doi': '10.1101/026625',\n",
       "   'published_doi': '10.1534/g3.115.024349',\n",
       "   'published_journal': 'G3: Genes|Genomes|Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The effects of both recent and long-term selection and genetic drift are readily evident in North American barley breeding populations',\n",
       "   'preprint_authors': 'Ana M Poets;Mohsen Mohammadi;Kiran Seth;Hongyun Wang;Thomas J.Y. Kono;Zhou Fang;Gary J Muehlbauer;Kevin P Smith;Peter L Morrell;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-09-11',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Barley was introduced to North America [~]400 years ago but adaptation to modern production environments is more recent. Comparisons of allele frequencies among different growth habits and inflorescence types in North America indicate significant genetic differentiation has accumulated in a relatively short evolutionary time span. Allele frequency differentiation is greatest among barley with two-row versus six-row inflorescences, and then by spring versus winter growth habit. Large changes in allele frequency among breeding programs suggest a major contribution of genetic drift and linked selection on genetic variation. Despite this, comparisons of 3,613 modern North American cultivated breeding lines that differ for row type and growth habit permit the discovery of 183 SNP outliers putatively linked to targets of selection. For example, SNPs within the Cbf4, Ppd-H1, and Vrn-H1 loci which have previously been associated with agronomically-adaptive phenotypes, are identified as outliers. Analysis of extended haplotype-sharing identifies genomic regions shared within and among breeding programs, suggestive of a number of genomic regions subject to recent selection. Finally, we are able to identify recent bouts of gene flow between breeding programs that could point to the sharing of agronomically-adaptive variation. These results are supported by pedigrees and breeders understanding of germplasm sharing.',\n",
       "   'preprint_author_corresponding': 'Peter L Morrell',\n",
       "   'preprint_author_corresponding_institution': 'University of Minnesota'},\n",
       "  {'preprint_doi': '10.1101/027995',\n",
       "   'published_doi': '10.1016/j.jtbi.2015.12.008',\n",
       "   'published_journal': 'Journal of Theoretical Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Modelling the role of immunity in reversion of viral antigenic sites',\n",
       "   'preprint_authors': 'Carmen H. S. Chan;Lloyd P. Sanders;Mark M. Tanaka;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-09-30',\n",
       "   'published_date': '2015-12-23',\n",
       "   'preprint_abstract': 'Antigenic sites in viral pathogens exhibit distinctive evolutionary dynamics due to their role in evading recognition by host immunity. Antigenic selection is known to drive higher rates of non-synonymous substitution; less well understood is why differences are observed between viruses in their propensity to mutate to a novel or previously encountered amino acid. Here, we present a model to explain patterns of antigenic reversion and forward substitution in terms of the epidemiological and molecular processes of the viral population. We develop an analytical three-strain model and extend the analysis to a multi-site model to predict characteristics of observed sequence samples. Our model provides insight into how the balance between selection to escape immunity and to maintain viability is affected by the rate of mutational input. We also show that while low probabilities of reversion may be due to either a low cost of immune escape or slowly decaying host immunity, these two scenarios can be differentiated by the frequency patterns at antigenic sites. Comparison between frequency patterns of human influenza A (H3N2) and human RSV-A suggests that the increased rates of antigenic reversion in RSV-A is due to faster decaying immunity and not higher costs of escape.',\n",
       "   'preprint_author_corresponding': 'Carmen H. S. Chan',\n",
       "   'preprint_author_corresponding_institution': 'University of New South Wales'},\n",
       "  {'preprint_doi': '10.1101/028076',\n",
       "   'published_doi': '10.1371/journal.pcbi.1004636',\n",
       "   'published_journal': 'PLOS Computational Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Complete firing-rate response of neurons with complex intrinsic dynamics',\n",
       "   'preprint_authors': 'Maximilian A Puelma Touzel;Fred Wolf;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-10-02',\n",
       "   'published_date': '2015-12-31',\n",
       "   'preprint_abstract': 'The response of a neuronal population over a space of inputs depends on the intrinsic properties of its constituent neurons. Two main modes of single neuron dynamics-integration and resonance-have been distinguished. While resonator cell types exist in a variety of brain areas, few models incorporate this feature and fewer have investigated its effects. To understand better how a resonators frequency preference emerges from its intrinsic dynamics and contributes to its local areas population firing rate dynamics, we analyze the dynamic gain of an analytically solvable two-degree of freedom neuron model. In the Fokker-Planck approach, the dynamic gain is intractable. The alternative Gauss-Rice approach lifts the resetting of the voltage after a spike. This allows us to derive a complete expression for the dynamic gain of a resonator neuron model in terms of a cascade of filters on the input. We find six distinct response types and use them to fully characterize the routes to resonance across all values of the relevant timescales. We find that resonance arises primarily due to slow adaptation with an intrinsic frequency acting to sharpen and adjust the location of the resonant peak. We determine the parameter regions for the existence of an intrinsic frequency and for subthreshold and spiking resonance, finding all possible intersections of the three. The expressions and analysis presented here provide an account of how intrinsic neuron dynamics shape dynamic population response properties and can facilitate the construction of an exact theory of correlations and stability of population activity in networks containing populations of resonator neurons.\\\\n\\\\nAuthor SummaryDynamic gain, the amount by which features at specific frequencies in the input to a neuron are amplified or attenuated in its output spiking, is fundamental for the encoding of information by neural populations. Most studies of dynamic gain have focused on neurons without intrinsic degrees of freedom exhibiting integrator-type subthreshold dynamics. Many neuron types in the brain, however, exhibit complex subthreshold dynamics such as resonance, found for instance in cortical interneurons, stellate cells, and mitral cells. A resonator neuron has at least two degrees of freedom for which the classical Fokker-Planck approach to calculating the dynamic gain is largely intractable. Here, we lift the voltage-reset rule after a spike, allowing us to derive for the first time a complete expression of the dynamic gain of a resonator neuron model. We find the gain can exhibit only six shapes. The resonant ones have peaks that become large due to intrinsic adaptation and become sharp due to an intrinsic frequency. A resonance can nevertheless result from either property. The analysis presented here helps explain how intrinsic neuron dynamics shape population-level response properties and provides a powerful tool for developing theories of inter-neuron correlations and dynamic responses of neural populations.',\n",
       "   'preprint_author_corresponding': 'Maximilian A Puelma Touzel',\n",
       "   'preprint_author_corresponding_institution': 'Max Planck Institute for Dynamics and SelfOrganization (MPIDS), Goettingen, Germany'},\n",
       "  {'preprint_doi': '10.1101/028837',\n",
       "   'published_doi': '10.1534/genetics.115.183665',\n",
       "   'published_journal': 'Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'RAD Capture (Rapture): Flexible and efficient sequence-based genotyping.',\n",
       "   'preprint_authors': \"Omar A Ali;Sean M O'Rourke;Stephen J Amish;Mariah H Meek;Gordon Luikart;Carson Jeffres;Michael R Miller;\",\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-10-11',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Massively parallel sequencing has revolutionized many areas of biology but sequencing large amounts of DNA in many individuals is cost-prohibitive and unnecessary for many studies. Genomic complexity reduction techniques such as sequence capture and restriction enzyme-based methods enable the analysis of many more individuals per unit cost. Despite their utility, current complexity reduction methods have limitations, especially when large numbers of individuals are analyzed. Here we develop a much improved restriction site associated DNA (RAD) sequencing protocol and a new combinatorial method called Rapture (RAD capture). The new RAD protocol improves versatility by separating RAD tag isolation and sequencing library preparation into two distinct steps. This protocol also recovers more unique (non-clonal) RAD fragments and produces better data for both standard RAD and Rapture analysis. Rapture then uses an in-solution capture of chosen RAD tags to target sequencing reads to desired loci. Rapture combines the benefits of both RAD and sequence capture, i.e. very inexpensive and rapid library preparation for many individuals as well as high specificity in the number and location of genomic loci analyzed. Our results demonstrate that Rapture is a rapid and flexible technology capable of analyzing a very large number of individuals with minimal sequencing and library preparation cost. The methods presented here should improve the efficiency of genetic analysis for many aspects of agricultural, environmental, and biomedical science.',\n",
       "   'preprint_author_corresponding': 'Michael R Miller',\n",
       "   'preprint_author_corresponding_institution': 'Department of Animal Science, University of California, Davis CA 95616, USA'},\n",
       "  {'preprint_doi': '10.1101/028969',\n",
       "   'published_doi': '10.1016/j.meegid.2015.11.030',\n",
       "   'published_journal': 'Infection, Genetics and Evolution',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Phylogenomic Reconstruction Supports Supercontinent Origins for Leishmania',\n",
       "   'preprint_authors': 'Kelly M Harkins;Rachel S Schwartz;Reed Cartwright;Anne Stone;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-10-13',\n",
       "   'published_date': '2015-12-18',\n",
       "   'preprint_abstract': 'Leishmania, a genus of parasites transmitted to human hosts and mammalian/reptilian reservoirs by an insect vector, is the causative agent of the human disease complex leishmaniasis. The evolutionary relationships within the genus Leishmania and its origins are the source of ongoing debate, reflected in conflicting phylogenetic and biogeographic reconstructions. This study employs a recently described bioinformatics method, SISRS, to identify over 200,000 informative sites across the genome from newly sequenced and publicly available Leishmania data. This dataset is used to reconstruct the evolutionary relationships of this genus. Additionally, we constructed a large multi-gene dataset; we used this dataset to reconstruct the phylogeny and estimate divergence dates for species. We conclude that the genus Leishmania evolved at least 90-100 million years ago. Our results support the hypothesis that Leishmania clades separated prior to, and during, the breakup of Gondwana. Additionally, we confirm that reptile-infecting Leishmania are derived from mammalian forms, and that the species that infect porcupines and sloths form a clade long separated from other species. We also firmly place the guinea-pig infecting species, L. enrietti, the globally dispersed L. siamensis, and the newly identified Australia species from kangaroos as sibling species whose distribution arises from the ancient connection between Australia, Antarctica, and South America.',\n",
       "   'preprint_author_corresponding': 'Kelly M Harkins',\n",
       "   'preprint_author_corresponding_institution': 'University of California Santa Cruz, Arizona State University'},\n",
       "  {'preprint_doi': '10.1101/029207',\n",
       "   'published_doi': '10.1371/journal.pgen.1005751',\n",
       "   'published_journal': 'PLOS Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Dissecting the genetic basis of a complex cis-regulatory adaptation',\n",
       "   'preprint_authors': 'Santiago Naranjo;Justin Smith;Carlo Artieri;Mian Zhang;Yiqi Zhou;Michael Palmer;Hunter Fraser;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-10-15',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Although single genes underlying several evolutionary adaptations have been identified, the genetic basis of complex, polygenic adaptations has been far more challenging to pinpoint. Here we report that the budding yeast Saccharomyces paradoxus has recently evolved resistance to citrinin, a naturally occurring mycotoxin. Applying a genome-wide test for selection on cis-regulation, we identified five genes involved in the citrinin response that are constitutively up-regulated in S. paradoxus. Four of these genes are necessary for resistance, and are also sufficient to increase the resistance of a sensitive strain when over-expressed. Moreover, cis-regulatory divergence in the promoters of these genes contributes to resistance, while exacting a cost in the absence of citrinin. Our results demonstrate how the subtle effects of individual regulatory elements can be combined, via natural selection, into a complex adaptation. Our approach can be applied to dissect the genetic basis of polygenic adaptations in a wide range of species.\\\\n\\\\nAuthor SummaryAdaptation via natural selection has been a subject of great interest for well over a century, yet we still have little understanding of its molecular basis. What are the genetic changes that are actually being selected? While single genes underlying several adaptations have been identified, the genetic basis of complex, polygenic adaptations has been far more challenging to pinpoint. The complex trait that we study here is the resistance of Saccharomyces yeast to a mycotoxin called citrinin, which is produced by many other species of fungi, and is a common food contaminant for both humans and livestock. We found that sequence changes in the promoters of at least three genes have contributed to citrinin resistance, by up-regulating their transcription even in the absence of citrinin. Higher expression of these genes confers a fitness advantage in the presence of citrinin, while exacting a cost in its absence-- a fitness tradeoff. Our results provide a detailed view of a complex adaptation, and our approach can be applied to polygenic adaptations in a wide range of species.',\n",
       "   'preprint_author_corresponding': 'Hunter  Fraser',\n",
       "   'preprint_author_corresponding_institution': 'Stanford'},\n",
       "  {'preprint_doi': '10.1101/021303',\n",
       "   'published_doi': '10.1093/sysbio/syv121',\n",
       "   'published_journal': 'Systematic Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Machine Learning Biogeographic Processes from Biotic Patterns: A New Trait-Dependent Dispersal and Diversification Model with Model-Choice By Simulation-Trained Discriminant Analysis',\n",
       "   'preprint_authors': 'Jeet Sukumaran;Evan P. Economo;L. Lacey Knowles;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-06-22',\n",
       "   'published_date': '2015-12-28',\n",
       "   'preprint_abstract': 'Current statistical biogeographical analysis methods are limited in the ways ecology can be related to the processes of diversification and geographical range evolution, requiring conflation of geography and ecology, and/or assuming ecologies that are uniform across all lineages and invariant in time. This precludes the possibility of studying a broad class of macro-evolutionary biogeographical theories that relate geographical and species histories through iineage-specific ecological and evolutionary dynamics, such as taxon cycle theory. Here we present a new model that generates phylogenies under a complex of superpositioned geographical range evolution, trait evolution, and diversification processes that can communicate with each other. This means that, under our model, the diversification and transition of the states of a lineage through geographical space is separate from, yet is conditional on, the state of the lineage in trait space (or vice versa). We present a likelihood-free method of inference under our model using discriminant analysis of principal components of summary statistics calculated on phylogenies, with the discriminant functions trained on data generated by simulations under our model. This approach of model classification is shown to be efficient, robust, and performant over a broad range of parameter space defined by the relative rates of dispersal, trait evolution, and diversification processes. We apply our method to a case study of the taxon cycle, i.e. testing for habitat and trophic-level constraints in the dispersal regimes of the Wallacean avifaunal radiation.',\n",
       "   'preprint_author_corresponding': 'Jeet  Sukumaran',\n",
       "   'preprint_author_corresponding_institution': 'University of Michigan, Ann Arbor'},\n",
       "  {'preprint_doi': '10.1101/021923',\n",
       "   'published_doi': '10.1073/pnas.1512977112',\n",
       "   'published_journal': 'Proceedings of the National Academy of Sciences',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Fluctuating fitness shapes the clone size distribution of immune repertoires',\n",
       "   'preprint_authors': 'Jonathan Desponds;Thierry Mora;Aleksandra Walczak;',\n",
       "   'preprint_category': 'Immunology ',\n",
       "   'preprint_date': '2015-07-05',\n",
       "   'published_date': '2015-12-28',\n",
       "   'preprint_abstract': 'The adaptive immune system relies on the diversity of receptors expressed on the surface of B and T-cells to protect the organism from a vast amount of pathogenic threats. The proliferation and degradation dynamics of different cell types (B cells, T cells, naive, memory) is governed by a variety of antigenic and environmental signals, yet the observed clone sizes follow a universal power law distribution. Guided by this reproducibility we propose effective models of somatic evolution where cell fate depends on an effective fitness. This fitness is determined by growth factors acting either on clones of cells with the same receptor responding to specific antigens, or directly on single cells with no regards for clones. We identify fluctuations in the fitness acting specifically on clones as the essential ingredient leading to the observed distributions. Combining our models with experiments we characterize the scale of fluctuations in antigenic environments and we provide tools to identify the relevant growth signals in different tissues and organisms. Our results generalize to any evolving population in a fluctuating environment.',\n",
       "   'preprint_author_corresponding': 'Thierry  Mora',\n",
       "   'preprint_author_corresponding_institution': 'Ecole normale suprieure'},\n",
       "  {'preprint_doi': '10.1101/022301',\n",
       "   'published_doi': '10.1093/bioinformatics/btv722',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Fast and efficient QTL mapper for thousands of molecular phenotypes',\n",
       "   'preprint_authors': 'Halit Ongen;Alfonso Buil;Andrew Brown;Emmanouil Dermitzakis;Olivier Delaneau;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-08-07',\n",
       "   'published_date': '2015-12-26',\n",
       "   'preprint_abstract': 'MotivationIn order to discover quantitative trait loci (QTLs), multi-dimensional genomic data sets combining DNA-seq and ChiP-/RNA-seq require methods that rapidly correlate tens of thousands of molecular phenotypes with millions of genetic variants while appropriately controlling for multiple testing.\\\\n\\\\nResultsWe have developed FastQTL, a method that implements a popular cis-QTL mapping strategy in a user- and cluster-friendly tool. FastQTL also proposes an efficient permutation procedure to control for multiple testing. The outcome of permutations is modeled using beta distributions trained from a few permutations and from which adjusted p-values can be estimated at any level of significance with little computational cost. The Geuvadis & GTEx pilot data sets can be now easily analyzed an order of magnitude faster than previous approaches.\\\\n\\\\nAvailabilitySource code, binaries and comprehensive documentation of FastQTL are freely available to download at http://fastqtl.sourceforge.net/.\\\\n\\\\nContactolivier.delaneau@unige.ch',\n",
       "   'preprint_author_corresponding': 'Olivier  Delaneau',\n",
       "   'preprint_author_corresponding_institution': 'University of Geneva'},\n",
       "  {'preprint_doi': '10.1101/022731',\n",
       "   'published_doi': '10.1186/s12864-015-2313-7',\n",
       "   'published_journal': 'BMC Genomics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Whats in your next-generation sequence data? An exploration of unmapped DNA and RNA sequence reads from the bovine reference individual',\n",
       "   'preprint_authors': 'Lynsey K. Whitacre;Polyana C. Tizioto;JaeWoo Kim;Tad S. Sonstegard;Steven G. Schroeder;Leeson J. Alexander;Juan F. Medrano;Robert D. Schnabel;Jeremy F. Taylor;Jared E. Decker;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-07-17',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Next-generation sequencing projects commonly commence by aligning reads to a reference genome assembly. While improvements in alignment algorithms and computational hardware have greatly enhanced the efficiency and accuracy of alignments, a significant percentage of reads often remain unmapped. We generated de novo assemblies of unmapped reads from the DNA and RNA sequencing of the Bos taurus reference individual and identified the closest matching sequence to each contig by alignment to the NCBI non-redundant nucleotide database using BLAST. As expected, many of these contigs represent vertebrate sequence that is absent, incomplete, or misassembled in the UMD3.1 reference assembly. However, numerous additional contigs represent invertebrate species. Most prominent were several species of Spirurid nematodes and a blood-borne parasite, Babesia bigemina. These species are not known to infect taurine cattle and the reference animal appears to have been host to unsequenced sister species. We demonstrate the importance of exploring unmapped reads to ascertain sequences that are either absent or misassembled in the reference assembly and for detecting sequences indicative of infectious or symbiotic organisms.',\n",
       "   'preprint_author_corresponding': 'Jared E. Decker',\n",
       "   'preprint_author_corresponding_institution': 'University of Missouri'},\n",
       "  {'preprint_doi': '10.1101/022962',\n",
       "   'published_doi': '10.1016/j.celrep.2015.11.051',\n",
       "   'published_journal': 'Cell Reports',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Nanoscale architecture of the axon initial segment reveals an organized and robust scaffold',\n",
       "   'preprint_authors': 'Christophe Leterrier;Jean Potier;Ghislaine Caillol;Claire Debarnot;Fanny Rueda Boroni;Bndicte Dargent;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-07-21',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'The Axon Initial Segment [AIS], located within the first 30 m of the axon, has two essential roles in generating action potentials and maintaining axonal identity. AIS assembly depends on a {beta}IV-spectrin / ankyrin G scaffold, but its macromolecular arrangement is not well understood. Here we quantitatively determined the AIS nanoscale architecture using STo-chastic Optical Reconstruction Microscopy [STORM]. First we directly demonstrate that the 190-nm periodicity of the AIS submembrane lattice results from longitudinal, head-to-head {beta}IV-spectrin molecules connecting actin rings. Using multicolor 3D-STORM, we resolve the nanoscale organization of ankyrin G: its aminoterminus associates with the submembrane lattice, whereas the carboxyterminus radially extends (~32 nm on average) toward the cytosol. This AIS nano-architecture is highly resistant to cytoskeletal perturbations, advocating its role in structural stabilization. Our findings provide a comprehensive view of the AIS molecular architecture, and will help understanding the crucial physiological functions of this compartment.',\n",
       "   'preprint_author_corresponding': 'Christophe  Leterrier',\n",
       "   'preprint_author_corresponding_institution': 'Aix Marseille Universit, CNRS, CRN2M UMR7286, 13344 cedex 15, Marseille, France'},\n",
       "  {'preprint_doi': '10.1101/023184',\n",
       "   'published_doi': '10.1371/journal.pcbi.1004679',\n",
       "   'published_journal': 'PLOS Computational Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Capabilities and Limitations of Tissue Size Control Through Passive Mechanical Forces',\n",
       "   'preprint_authors': 'Jochen Kursawe;Pavel A. Brodskiy;Jeremiah J. Zartman;Ruth E. Baker;Alexander George Fletcher;',\n",
       "   'preprint_category': 'Developmental Biology ',\n",
       "   'preprint_date': '2015-07-24',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Embryogenesis is an extraordinarily robust process, exhibiting the ability to control tissue size and repair patterning defects in the face of environmental and genetic perturbations. The size and shape of a developing tissue is a function of the number and size of its constituent cells as well as their geometric packing. How these cellular properties are coordinated at the tissue level to ensure developmental robustness remains a mystery; understanding this process requires studying multiple concurrent processes that make up morphogenesis, including the spatial patterning of cell fates and apoptosis, as well as cell intercalations. In this work, we develop a computational model that aims to understand aspects of the robust pattern repair mechanisms of the Drosophila embryonic epidermal tissues. Size control in this system has previously been shown to rely on the regulation of apoptosis rather than proliferation; however, to date little work has been done to understand the role of cellular mechanics in this process. We employ a vertex model of an embryonic segment to test hypotheses about the emergence of this size control. Comparing the model to previously published data across wild type and genetic perturbations, we show that passive mechanical forces suffice to explain the observed size control in the posterior (P) compartment of a segment. However, observed asymmetries in cell death frequencies across the segment are demonstrated to require patterning of cellular properties in the model. Finally, we show that distinct forms of mechanical regulation in the model may be distinguished by differences in cell shapes in the P compartment, as quantified through experimentally accessible summary statistics, as well as by the tissue recoil after laser ablation experiments.\\\\n\\\\nAuthor SummaryDeveloping embryos are able to grow organs of the correct size even in the face of significant external perturbations. Such robust size control is achieved via tissue-level coordination of cell growth, proliferation, death and rearrangement, through mechanisms that are not well understood. Here, we employ computational modelling to test hypotheses of size control in the developing fruit fly. Segments in the surface tissues of the fruit fly embryo have been shown to achieve the same size even if the number of cells in each segment is perturbed genetically. We show that simple mechanical interactions between the cells of this tissue can recapitulate previously gathered data on tissue sizes and cell numbers. However, this simple model does not capture the experimentally observed spatial variation in cell death rates in this tissue, which may be explained through several adaptations to the model. These distinct adaptations may be distinguished through summary statistics of the tissue behaviour, such as statistics of cell shapes or tissue recoil after cutting. This work demonstrates how computational modelling can help investigate the complex mechanical interactions underlying tissue size and shape, which are important for understanding the underlying causes of birth defects and diseases driven by uncontrolled growth.',\n",
       "   'preprint_author_corresponding': 'Alexander George Fletcher',\n",
       "   'preprint_author_corresponding_institution': 'University of Sheffield'},\n",
       "  {'preprint_doi': '10.1101/023408',\n",
       "   'published_doi': '10.1186/s13059-015-0849-0',\n",
       "   'published_journal': 'Genome Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Circlator: automated circularization of genome assemblies using long sequencing reads',\n",
       "   'preprint_authors': 'Martin Hunt;Nishadi De Silva;Thomas D Otto;Julian Parkhill;Jacqueline A Keane;Simon R Harris;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-07-28',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'The assembly of DNA sequence data into finished genomes is undergoing a renais-sance thanks to emerging technologies producing reads of tens of kilobases. Assembling complete bacterial and small eukaryotic genomes is now possible, but the final step of circularizing sequences remains unsolved. Here we present Circlator, the first tool to automate assembly circularization and produce accurate linear rep-resentations of circular sequences. Using Pacific Biosciences and Oxford Nanopore data, Circlator correctly circularized 26 of 27 circularizable sequences, comprising 11 chromosomes and 12 plasmids from bacteria, the apicoplast and mitochondrion of Plasmodium falciparum and a human mitochondrion. Circlator is available at http://sanger-pathogens.github.io/circlator/.',\n",
       "   'preprint_author_corresponding': 'Martin  Hunt',\n",
       "   'preprint_author_corresponding_institution': 'Wellcome Trust Sanger Institute'},\n",
       "  {'preprint_doi': '10.1101/023531',\n",
       "   'published_doi': '10.1371/journal.pcbi.1004684',\n",
       "   'published_journal': 'PLOS Computational Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Formation and maintenance of robust long-term information storage in the presence of synaptic turnover',\n",
       "   'preprint_authors': 'Michael Fauth;Florentin Wrgtter;Christian Tetzlaff;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-07-31',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'A long-standing problem is how memories can be stored for very long times despite the volatility of the underlying neural substrate, most notably the high turnover of dendritic spines and synapses. To address this problem, here we are using a generic and simple probabilistic model for the creation and removal of synapses. We show that information can be stored for several months when utilizing the intrinsic dynamics of multi-synapse connections. In such systems, single synapses can still show high turnover, which enables fast learning of new information, but this will not perturb prior stored information (slow forgetting), which is represented by the compound state of the connections. The model matches the time course of recent experimental spine data during learning and memory in mice supporting the assumption of multi-synapse connections as the basis for long-term storage.\\\\n\\\\nAuthor SummaryIt is widely believed that information is stored in the connectivity, i.e. the synapses of neural networks. Yet, the morphological correlates of excitatory synapses, the dendritic spines, have been found to undergo a remarkable turnover on daily basis. This poses the question, how information can be retained on such a variable substrate.\\\\n\\\\nIn this study, using connections with multiple synapses, we show that connections which follow the experimentally measured bimodal distribution in the number of synapses can store information orders of magnitude longer than the lifetime of a single synapse. This is a consequence of the underlying bistable collective dynamic of multiple synapses: Single synapses can appear and disappear without disturbing the memory as a whole.\\\\n\\\\nFurthermore, increasing or decreasing neural activity changes the distribution of the number of synapses of multi-synaptic connections such that only one of the peaks remains. This leads to a desirable property: information about these altered activities can be stored much faster than it is forgotten. Remarkably, the resulting model dynamics match recent experimental data investigating the long-term effect of learning on the dynamics of dendritic spines.',\n",
       "   'preprint_author_corresponding': 'Michael  Fauth',\n",
       "   'preprint_author_corresponding_institution': 'University Gttingen, Bernstein Center for Computational Neuroscience'},\n",
       "  {'preprint_doi': '10.1101/009308',\n",
       "   'published_doi': '10.1016/j.neuroimage.2015.12.022',\n",
       "   'published_journal': 'NeuroImage',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Pinwheel-dipole configuration in cat early visual cortex',\n",
       "   'preprint_authors': 'Jrme Ribot;Alberto Romagnoni;Chantal Milleret;Daniel Bennequin;Jonathan D. Touboul;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2014-09-18',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'In the early visual cortex, information is processed within functional maps whose layouts are thought to underlie visual perception. However, the precise organization of these functional maps as well as their interrelationships remains largely unknown. Here, we show that spatial frequency representation in cat early visual cortex exhibits singularities around which the map organizes like an electric dipole potential. These singularities are precisely co-located with singularities of the orientation map: the pinwheel centers. To show this, we used high resolution intrinsic optical imaging in cat areas 17 and 18. First, we show that a majority of pinwheel centers exhibit in their neighborhood both semi-global maximum and minimum in the spatial frequency map, contradicting pioneering studies suggesting that pinwheel centers are placed at the locus of a single spatial frequency extremum. Based on an analogy with electromagnetism, we proposed a mathematical model for a dipolar structure, accurately fitting optical imaging data. We conclude that a majority of orientation pinwheel centers form spatial frequency dipoles in cat early visual cortex. Given the functional specificities of neurons at singularities in the visual cortex, it is argued that the dipolar organization of spatial frequency around pinwheel centers could be fundamental for visual processing.\\\\n\\\\nABBREVIATIONS',\n",
       "   'preprint_author_corresponding': 'Jrme  Ribot',\n",
       "   'preprint_author_corresponding_institution': 'Collge de France'},\n",
       "  {'preprint_doi': '10.1101/016105',\n",
       "   'published_doi': '10.1186/s12879-015-1318-9',\n",
       "   'published_journal': 'BMC Infectious Diseases',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Detecting Signals of Seasonal Influenza Severity through Age Dynamics',\n",
       "   'preprint_authors': 'Elizabeth C Lee;Ccile Viboud;Lone Simonsen;Farid Khan;Shweta Bansal;',\n",
       "   'preprint_category': 'Ecology ',\n",
       "   'preprint_date': '2015-03-06',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Background: Measures of population-level influenza severity are important for public health planning, but estimates are often based on case-fatality and case-hospitalization risks, which require multiple data sources, are prone to surveillance biases, and are typically unavailable in the early stages of an outbreak. In this study, we develop a severity index based on influenza age dynamics estimated from routine surveillance data that can be used in retrospective and early warning contexts.\\\\n\\\\nMethods and Findings: Our method relies on the observation that age-specific attack rates vary between seasons, so that key features of the age distribution of cases may be used as a marker of severity early in an epidemic. We illustrate our method using weekly outpatient medical claims of influenza-like illness (ILI) in the United States from the 2001 to 2009 and develop a novel population-level influenza severity index based on the relative risk of ILI among working-age adults to that among school-aged children. We validate our ILI index against a benchmark that comprises traditional influenza severity indicators such as viral activity, hospitalizations and deaths using publicly available surveillance data. We find that severe influenza seasons have higher relative rates of ILI among adults than mild seasons. In reference to the benchmark, the ILI index is a robust indicator of severity during the period of peak epidemic growth (87.5% accuracy in retrospective classification), and may have predictive power during the period between Thanksgiving and the winter holidays (57.1% accuracy in early warning). We further apply our approach at the state-level to characterize regional severity patterns across seasons. We hypothesize that our index is a proxy for severity because working-age adults have both pre-existing immunity to influenza and a high number of contacts, infecting them preferentially in severe seasons associated with antigenic changes in circulating influenza viruses. Our analysis is limited by its application to seasonal influenza epidemics and a relatively short study period.\\\\n\\\\nConclusions: Our severity index and research on the link between age dynamics and seasonal influenza severity will enable decision makers to better target public health strategies in severe seasons and improve our knowledge of influenza epidemiology and population impact. These findings demonstrate that routine surveillance data can be translated into operational information for policymakers. Our study also highlights the need for further research on the putative age-related mechanisms of severity in seasonal and pandemic influenza seasons.',\n",
       "   'preprint_author_corresponding': 'Shweta  Bansal',\n",
       "   'preprint_author_corresponding_institution': 'Georgetown University'},\n",
       "  {'preprint_doi': '10.1101/030247',\n",
       "   'published_doi': '10.1534/genetics.115.177816',\n",
       "   'published_journal': 'Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Sporadic, global linkage disequilibrium between unlinked segregating sites',\n",
       "   'preprint_authors': 'Daniel A Skelly;Paul M Magwene;Eric A Stone;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-10-30',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Demographic, genetic, or stochastic factors can lead to perfect linkage disequilibrium (LD) between alleles at two loci without respect to the extent of their physical distance, a phenomenon that Lawrence et al. (2005a) refer to as \\\\\"genetic indistinguishability\\\\\". This phenomenon can complicate genotype-phenotype association testing by hindering the ability to localize causal alleles, but has not been thoroughly explored from a theoretical perspective or using large, dense whole-genome polymorphism datasets. We derive a simple theoretical model of the prevalence of genetic indistinguishability between unlinked loci, and verify its accuracy via simulation. We show that sample size and minor allele frequency are the major determinants of the prevalence of perfect LD between unlinked loci but that demographic factors, such as deviations from random mating, can produce significant effects as well. Finally, we quantify this phenomenon in three model organisms and find thousands of pairs of moderate-frequency (> 5%) genetically indistinguishable variants in relatively large datasets. These results clarify a previously underexplored population genetic phenomenon with important implications for association studies, and define conditions under which it is likely to manifest.',\n",
       "   'preprint_author_corresponding': 'Eric A Stone',\n",
       "   'preprint_author_corresponding_institution': 'North Carolina State University'},\n",
       "  {'preprint_doi': '10.1101/010181',\n",
       "   'published_doi': '10.1016/j.baae.2015.07.004',\n",
       "   'published_journal': 'Basic and Applied Ecology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Pollinators, pests and soil properties interactively shape oilseed rape yield',\n",
       "   'preprint_authors': 'Ignasi Bartomeus;Vesna Gagic;Riccardo Bommarco;',\n",
       "   'preprint_category': '',\n",
       "   'preprint_date': '2014-10-09',\n",
       "   'published_date': '2015-07-31',\n",
       "   'preprint_abstract': 'O_LIPollination, pest control, and soil properties are well known to affect agricultural production. These factors might interactively shape crop yield, but most studies focus on only one of these factors at a time.\\\\nC_LIO_LIWe used 15 winter oilseed rape (Brassica napus L.) fields in Sweden to study how variation among fields in pollinator visitation rates, pollen beetle pest attack rates and soil properties (soil texture, pH and organic carbon) interactively determined crop yield. The fields were embedded in a landscape gradient with contrasting proportions arable and semi-natural land.\\\\nC_LIO_LIPollinator, pest and soil property variables formed bundles across the sites. In general, pollinator visitation and pest levels were negatively correlated and varied independently of soil properties. Because above- and below-ground processes reacted at contrasting spatial scales, it was difficult to predict bundle composition based on the surrounding landscape structure.\\\\nC_LIO_LIThe above-ground biotic interactions and below-ground abiotic factors interactively affected crop yield. Pollinator visitation was the strongest predictor positively associated with yield. High soil pH also benefited yield, but only at lower pest loads. Surprisingly, high pest loads increased the pollinator benefits for yield.\\\\nC_LIO_LISynthesis and applications Implementing management plans at different spatial scales can create synergies among bundles of above- and below-ground ecosystem processes, but both scales are needed given that different processes react to different spatial scales.\\\\nC_LI',\n",
       "   'preprint_author_corresponding': 'Ignasi  Bartomeus',\n",
       "   'preprint_author_corresponding_institution': 'Estacin Biolgica de Doana (EBD-CSIC)'},\n",
       "  {'preprint_doi': '10.1101/028001',\n",
       "   'published_doi': '10.1093/humrep/dev318',\n",
       "   'published_journal': 'Human Reproduction',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Genetic evidence that lower circulating FSH levels lengthen menstrual cycle, increase age at menopause, and impact reproductive health: a UK Biobank study',\n",
       "   'preprint_authors': 'Katherine S Ruth;Robin N Beaumont;Jessica Tyrrell;Samuel E Jones;Marcus A Tuke;Hanieh Yaghootkar;Andrew R Wood;Rachel M Freathy;Michael N Weedon;Timothy M Frayling;Anna Murray;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-10-01',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': 'Study question: How does a genetic variant altering follicle stimulating hormone (FSH) levels, which we identified as associated with length of menstrual cycle, more widely impact reproductive health?\\\\n\\\\nSummary answer: The T allele of the FSHB promoter polymorphism (rs10835638) results in longer menstrual cycles and later menopause and, while having detrimental effects on fertility, is protective against endometriosis.\\\\n\\\\nWhat is known already: The FSHB promoter polymorphism (rs10835638) affects levels of FSHB transcription and, as a result, levels of FSH. FSH is required for normal fertility and genetic variants at the FSHB locus are associated with age at menopause and polycystic ovary syndrome (PCOS).\\\\n\\\\nStudy design, size, duration: We conducted a genetic association study using cross-sectional data from the UK Biobank.\\\\n\\\\nParticipants/materials, setting, methods: We included white British individuals aged 40-69 years in 2006-2010, included in the May 2015 release of genetic data from UK Biobank. We conducted a genome-wide association study (GWAS) in 9,534 individuals to identify genetic variants associated with length of menstrual cycle. We tested the FSH lowering T allele of the FSHB promoter polymorphism (rs10835638) for associations with 29 reproductive phenotypes in up to 63,350 individuals.\\\\n\\\\nMain results and the role of chance: In the GWAS for menstrual cycle length, only variants near the FSHB gene reached genome-wide significance (P<5x10-8). The FSH-lowering T allele of the FSHB promoter polymorphism (rs10835638G>T; MAF 0.16) was associated with longer menstrual cycles (0.16 s.d. (approx. 1 day) per minor allele; 95% CI 0.12-0.20; P=6x10-16), later age at menopause (0.13 years per minor allele; 95% CI 0.04-0.22; P=5.7x10-3), greater female nulliparity (OR=1.06; 95% CI 1.02-1.11; P=4.8x10-3) and lower risk of endometriosis (OR=0.79; 95% CI 0.69-0.90; P=4.1x10-4). The FSH-lowering T allele was not associated more generally with other reproductive illnesses or conditions and we did not replicate associations with male infertility or PCOS.\\\\n\\\\nLimitations, reasons for caution: The data included might be affected by recall bias. Women with a cycle length recorded were aged over 40 and were approaching menopause, however we did not find evidence that this affected the results. Many of the illnesses had relatively small sample sizes and so we may have been under-powered to detect an effect.\\\\n\\\\nWider implications of the findings: We found a strong novel association between a genetic variant that lowers FSH levels and longer menstrual cycles, at a locus previously robustly associated with age at menopause. The variant was also associated with nulliparity and endometriosis risk. We conclude that lifetime differences in circulating levels of FSH between individuals can influence menstrual cycle length and a range of reproductive outcomes, including menopause timing, infertility, endometriosis and PCOS.',\n",
       "   'preprint_author_corresponding': 'Anna  Murray',\n",
       "   'preprint_author_corresponding_institution': 'University of Exeter Medical School'},\n",
       "  {'preprint_doi': '10.1101/024794',\n",
       "   'published_doi': '10.3389/fphar.2015.00307',\n",
       "   'published_journal': 'Frontiers in Pharmacology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The human experience with intravenous levodopa',\n",
       "   'preprint_authors': 'Shan H Siddiqi;Natalia K Abraham;Christopher L Geiger;Morvarid Karimi;Joel S Perlmutter;Kevin J Black;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-08-18',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': 'ObjectiveTo compile a comprehensive summary of published human experience with levodopa given intravenously, with a focus on information required by regulatory agencies.\\\\n\\\\nBackgroundWhile safe intravenous use of levodopa has been documented for over 50 years, regulatory supervision for pharmaceuticals given by a route other than that approved by the U.S. Food and Drug Administration (FDA) has become increasingly cautious. If delivering a drug by an alternate route raises the risk of adverse events, an investigational new drug (IND) application is required, including a comprehensive review of toxicity data.\\\\n\\\\nMethodsOver 200 articles referring to intravenous levodopa (IVLD) were examined for details of administration, pharmacokinetics, benefit and side effects.\\\\n\\\\nResultsWe identified 142 original reports describing IVLD use in humans, beginning with psychiatric research in 1959-1960 before the development of peripheral decarboxylase inhibitors. At least 2781 subjects have received IVLD, and reported outcomes include parkinsonian signs, sleep variables, hormones, hemodynamics, CSF amino acid composition, regional cerebral blood flow, cognition, perception and complex behavior. Mean pharmacokinetic variables were summarized for 49 healthy subjects and 190 with Parkinsons disease. Side effects were those expected from clinical experience with oral levodopa and dopamine agonists. No articles reported deaths or induction of psychosis.\\\\n\\\\nConclusionAt least 2781 patients have received i.v. levodopa with a safety profile comparable to that seen with oral administration.',\n",
       "   'preprint_author_corresponding': 'Kevin J Black',\n",
       "   'preprint_author_corresponding_institution': 'Washington University School of Medicine'},\n",
       "  {'preprint_doi': '10.1101/034009',\n",
       "   'published_doi': '10.1007/s00300-015-1871-0',\n",
       "   'published_journal': 'Polar Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Evidence of adoption, monozygotic twinning, and low inbreeding rates in a large genetic pedigree of polar bears',\n",
       "   'preprint_authors': 'Ren M. Malenfant;David W. Coltman;Evan S. Richardson;Nicholas J. Lunn;Ian Stirling;Elizabeth Adamowicz;Corey S. Davis;',\n",
       "   'preprint_category': 'Zoology ',\n",
       "   'preprint_date': '2015-12-15',\n",
       "   'published_date': '2015-12-26',\n",
       "   'preprint_abstract': 'Multigenerational pedigrees have been developed for free-ranging populations of many species, are frequently used to describe mating systems, and are used in studies of quantitative genetics. Here, we document the development of a 4449-individual pedigree for the Western Hudson Bay subpopulation of polar bears (Ursus maritimus), created from relationships inferred from field and genetic data collected over six generations of bears sampled between 1966 and 2011. Microsatellite genotypes for 22-25 loci were obtained for 2945 individuals, and parentage analysis was performed using the program FRANO_SCPLOWZC_SCPLOW, including additional offspring-dam associations known only from capture data. Parentage assignments for a subset of 859 individuals were confirmed using an independent high-density set of single nucleotide polymorphisms. To account for unsampled males in our population, we performed half-sib-full-sib analysis to reconstruct males using the program COLONY, resulting in a final pedigree containing 2957 assigned maternities and 1861 assigned paternities with only one observed case of inbreeding between close relatives. During genotyping, we identified two independently captured two-year-old males with identical genotypes at all 25 loci, showing--for the first time--a case of monozygotic twinning among polar bears. In addition, we documented six new cases of cub adoption, which we attribute to cub misidentification or misdirected maternal care by a female bereaved of her young. Importantly, none of these adoptions could be attributed to reduced female vigilance caused by immobilization to facilitate scientific handling, as has previously been suggested.',\n",
       "   'preprint_author_corresponding': 'Ren M. Malenfant',\n",
       "   'preprint_author_corresponding_institution': 'University of Alberta'},\n",
       "  {'preprint_doi': '10.1101/022699',\n",
       "   'published_doi': '10.1016/j.ajhg.2015.11.015',\n",
       "   'published_journal': 'American Journal Of Human Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Adaptive variation in human toll-like receptors is contributed by introgression from both Neandertals and Denisovans',\n",
       "   'preprint_authors': 'Michael Dannemann;Aida M. Andrs;Janet Kelso;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-07-16',\n",
       "   'published_date': '2016-01-07',\n",
       "   'preprint_abstract': 'Pathogens and the diseases they cause have been among the most important selective forces experienced by humans during their evolutionary history. Although adaptive alleles generally arise by mutation, introgression can also be a valuable source of beneficial alleles. Archaic humans, who lived in Europe and Western Asia for over 200,000 years, were likely well-adapted to the environment and its local pathogens, and it is therefore conceivable that modern humans entering Europe and Western Asia who admixed with them obtained a substantial immune advantage from the introgression of archaic alleles.\\\\n\\\\nHere we document a cluster of three toll-like receptors (TLR6-TLR1-TLR10) in modern humans that carries three distinct archaic haplotypes, indicating repeated introgression from archaic humans. Two of these haplotypes are most similar to Neandertal genome, while the third haplotype is most similar to the Denisovan genome. The toll-like receptors are key components of innate immunity and provide an important first line of immune defense against bacteria, fungi and parasites. The unusually high allele frequencies and unexpected levels of population differentiation indicate that there has been local positive selection on multiple haplotypes at this locus. We show that the introgressed alleles have clear functional effects in modern humans; archaic-like alleles underlie differences in the expression of the TLR genes and are associated with reduced microbial resistance and increased allergic disease in large cohorts. This provides strong evidence for recurrent adaptive introgression at the TLR6-TLR1-TLR10 locus, resulting in differences in disease phenotypes in modern humans.',\n",
       "   'preprint_author_corresponding': 'Janet  Kelso',\n",
       "   'preprint_author_corresponding_institution': 'Max Planck Institute for Evolutionary Anthropology'},\n",
       "  {'preprint_doi': '10.1101/019968',\n",
       "   'published_doi': '10.1093/beheco/arv184',\n",
       "   'published_journal': 'Behavioral Ecology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Environmental complexity favors the evolution of learning',\n",
       "   'preprint_authors': 'Slimane Dridi;Laurent Lehmann;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-05-29',\n",
       "   'published_date': '2015-12-29',\n",
       "   'preprint_abstract': 'Learning is a fundamental biological adaptation that is widespread throughout the animal kingdom. According to previous research, two conditions are necessary for learning to be adaptive: between-generation environmental variability and within-generation environmental predictability. In this paper, we show that between-generation variability is not necessary, and that instrumental learning can provide a selective advantage in complex environments, where an individual is exposed to a large number of different challenges during its lifespan. We construct an evolutionary model where individuals have a memory with limited storage capacity, and an evolving trait determines the fraction of that memory that should be allocated to innate responses to the environment versus learning these responses. The evolutionarily stable level of learning depends critically on the environmental process, but generally increases with environmental complexity. Overall, our work sheds light on the importance of global structural properties of the environment in shaping the evolution of learning.',\n",
       "   'preprint_author_corresponding': 'Slimane  Dridi',\n",
       "   'preprint_author_corresponding_institution': 'University of Pennsylvania, Department of Biology'},\n",
       "  {'preprint_doi': '10.1101/026823',\n",
       "   'published_doi': '10.1186/s12864-015-2239-0',\n",
       "   'published_journal': 'BMC Genomics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The neurotranscriptome of the Aedes aegypti mosquito',\n",
       "   'preprint_authors': 'Benjamin J Matthews;Carolyn S McBride;Matthew DeGennaro;Orion Despo;Leslie B Vosshall;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-09-15',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': 'BackgroundA complete genome sequence and the advent of genome editing open up non-traditional model organisms to mechanistic genetic studies. The mosquito Aedes aegypti is an important vector of infectious diseases such as dengue, chikungunya, and yellow fever, and has a large and complex genome, which has slowed annotation efforts. We used comprehensive transcriptomic analysis of adult gene expression to improve the genome annotation and to provide a detailed tissue-specific catalogue of neural gene expression at different adult behavioral states.\\\\n\\\\nResultsWe carried out deep RNA sequencing across all major peripheral male and female sensory tissues, the brain, and (female) ovary. Furthermore, we examined gene expression across three important phases of the female reproductive cycle, a remarkable example of behavioral switching in which a female mosquito alternates between obtaining blood-meals from humans and laying eggs. Using genome-guided alignments and de novo transcriptome assembly, our re-annotation includes 572 new putative protein-coding genes and updates to 13.5% and 50.3% of existing transcripts within coding sequences and untranslated regions, respectively. Using this updated annotation, we detail gene expression in each tissue, identifying large numbers of transcripts regulated by blood-feeding and sexually dimorphic transcripts that may provide clues to the biology of male- and female-specific behaviors, such as mating and blood-feeding, which are areas of intensive study for those interested in vector control.\\\\n\\\\nConclusionsThis neurotranscriptome forms a strong foundation for the study of genes in the mosquito nervous system and investigation of sensory-driven behaviors and their regulation. Furthermore, understanding the molecular genetic basis of mosquito chemosensory behavior has important implications for vector control.',\n",
       "   'preprint_author_corresponding': 'Leslie B Vosshall',\n",
       "   'preprint_author_corresponding_institution': 'The Rockefeller University'},\n",
       "  {'preprint_doi': '10.1101/029090',\n",
       "   'published_doi': '10.1109/EMBC.2015.7319726',\n",
       "   'published_journal': 'IEEE Engineering in Medicine and Biology Society',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Rapid three dimensional two photon neural population scanning',\n",
       "   'preprint_authors': 'Renaud Schuck;Peter Quicke;Caroline Copeland;Stefania Garasto;Luca A Annecchino;June Kyu Hwang;Simon Schultz;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-10-14',\n",
       "   'published_date': '2016-08-29',\n",
       "   'preprint_abstract': 'Recording the activity of neural populations at high sampling rates is a fundamental requirement for understanding computation in neural circuits. Two photon microscopy provides one promising approach towards this. However, neural circuits are three dimensional, and functional imaging in two dimensions fails to capture the 3D nature of neural dynamics. Electrically tunable lenses (ETLs) provide a simple and cheap method to extend laser scanning microscopy into the relatively unexploited third dimension. We have therefore incorporated them into our Adaptive Spiral Scanning (SSA) algorithm, which calculates kinematically efficient scanning strategies using radially modulated spiral paths. We characterised the response of the ETL, incorporated its dynamics using MATLAB models of the SSA algorithm and tested the models on populations of Izhikevich neurons of varying size and density. From this, we show that our algorithms can theoretically at least achieve sampling rates of 36.2Hz compared to 21.6Hz previously reported for 3D scanning techniques.',\n",
       "   'preprint_author_corresponding': 'Simon  Schultz',\n",
       "   'preprint_author_corresponding_institution': 'Imperial College London'},\n",
       "  {'preprint_doi': '10.1101/029108',\n",
       "   'published_doi': '10.1109/EMBC.2015.7319214',\n",
       "   'published_journal': 'IEEE Engineering in Medicine and Biology Society',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Multi-Unit Activity contains information about spatial stimulus structure in mouse primary visual cortex',\n",
       "   'preprint_authors': 'Marie Tolkiehn;Simon Schultz;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-10-14',\n",
       "   'published_date': '2016-08-29',\n",
       "   'preprint_abstract': 'This study investigates the spatial and directional tuning of Multi-Unit Activity (MUA) in mouse primary visual cortex and how MUA can reflect spatiotemporal structures contained in moving gratings. Analysis of multi-shank laminar electrophysiological recordings from mouse primary visual cortex indicates a directional preference for moving gratings around 180{degrees}, while preferred spatial frequency peaks around 0.02 cycles per degree, which is similar as reported in single-unit studies. Using only features from MUA, we further achieved a significant performance in decoding spatial frequency or direction of moving gratings, with average decoding performances of up to 58.54% for 8 directions, and 44% correctly identified spatial frequencies against chance level of 16.7%.',\n",
       "   'preprint_author_corresponding': 'Simon  Schultz',\n",
       "   'preprint_author_corresponding_institution': 'Imperial College London'},\n",
       "  {'preprint_doi': '10.1101/029116',\n",
       "   'published_doi': '10.1039/C5IB00230C',\n",
       "   'published_journal': 'Integrative Biology : Quantitative Biosciences From Nano To Macro',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A synthetic gene circuit for measuring autoregulatory feedback control.',\n",
       "   'preprint_authors': 'Miquel Angel Schikora-Tamarit;Carlos Toscano-Ochoa;Julia Domingo Espinos;Lorena Espinar;Lucas Carey;',\n",
       "   'preprint_category': 'Synthetic Biology ',\n",
       "   'preprint_date': '2015-10-14',\n",
       "   'published_date': '2016-01-05',\n",
       "   'preprint_abstract': 'Auto regulatory feedback loops occur in the regulation of molecules ranging from ATP to MAP kinases to zinc. Negative feedback loops can increase a systems robustness, while positive feedback loops can mediate transitions between cell states. Recent genome-wide experimental and computational studies predict hundreds of novel feedback loops. However, not all physical interactions are regulatory, and many experimental methods cannot detect self-interactions. Our understanding of regulatory feedback loops is therefore hampered by the lack of high-throughput methods to experimentally quantify the presence, strength, and temporal dynamics of auto regulatory feedback loops. Here we present a mathematical and experimental framework for high-throughput quantification of feedback regulation, and apply it to RNA binding proteins (RBPs) in yeast. Our method is able to determine the existence of both direct and indirect positive and negative feedback loops, and to quantify the strength of these loops. We experimentally validate our model using two RBPs which lack native feedback loops, and by the introduction of synthetic feedback loops. We find that the the RBP Puf3 does not natively participate in any direct or indirect feedback regulation, but that replacing the native 3UTR with that of COX17 generates an auto-regulatory negative feedback loop which reduces gene expression noise. Likewise, the RBP Pub1 does not natively participate in any feedback loops, but a synthetic positive feedback loop involving Pub1 results in increased expression noise. Our results demonstrate a synthetic experimental system for quantifying the existence and strength of feedback loops using a combination of high-throughput experiments and mathematical modeling. This system will be of great use in measuring auto-regulatory feedback by RNA binding proteins, a regulatory motif that is difficult to quantify using existing high-throughput methods.',\n",
       "   'preprint_author_corresponding': 'Lucas  Carey',\n",
       "   'preprint_author_corresponding_institution': 'Universitat Pompeu Fabra'},\n",
       "  {'preprint_doi': '10.1101/021618',\n",
       "   'published_doi': '10.18632/oncotarget.6811',\n",
       "   'published_journal': 'Oncotarget',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Comprehensive nucleosome mapping of the human genome in cancer progression',\n",
       "   'preprint_authors': 'Brooke Druliner;Daniel Vera;Ruth Johnson;Xiaoyang Ruan;Lynne Apone;Eileen Dimalanta;Fiona Stewart;Lisa Boardman;Jonathan H. Dennis;',\n",
       "   'preprint_category': 'Cancer Biology ',\n",
       "   'preprint_date': '2015-06-27',\n",
       "   'published_date': '2015-12-31',\n",
       "   'preprint_abstract': 'Altered chromatin structure is a hallmark of cancer, and inappropriate regulation of chromatin structure may represent the origin of transformation. Important studies have mapped human nucleosome distributions genome wide, but the role of chromatin structure in cancer progression has not been addressed. We developed a MNase-Transcription Start Site Sequence Capture method (mTSS-seq) to map the nucleosome distribution at human transcription start sites genome-wide in primary human lung and colon adenocarcinoma tissue. Here, we confirm that nucleosome redistribution is an early, widespread event in lung (LAC) and colon (CRC) adenocarcinoma. These altered nucleosome architectures are consistent between LAC and CRC patient samples indicating that they may serve as important early adenocarcinoma markers. We demonstrate that the nucleosome alterations are driven by the underlying DNA sequence and potentiate transcription factor binding. We conclude that DNA-directed nucleosome redistributions are widespread early in cancer progression. We have proposed an entirely new hierarchical model for chromatin-mediated genome regulation.\\\\n\\\\nSignificanceThis is the first report of human nucleosome distribution in cancer progression using sequence capture and HiSeq. We show in lung and colorectal adenocarcinoma patients that nucleosome distribution is a widespread, early response driven by genetically-encoded signals and potentiate regulatory factor binding. We present a model for chromatin-based hierarchical regulation.',\n",
       "   'preprint_author_corresponding': 'Jonathan H. Dennis',\n",
       "   'preprint_author_corresponding_institution': 'Florida State University'},\n",
       "  {'preprint_doi': '10.1101/022384',\n",
       "   'published_doi': '10.1038/srep18936',\n",
       "   'published_journal': 'Scientific Reports',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Maize pan-transcriptome provides novel insights into genome complexity and quantitative trait variation',\n",
       "   'preprint_authors': 'Minliang Jin;Haijun Liu;Cheng He;Junjie Fu;Yingjie Xiao;Yuebin Wang;Weibo Xie;Guoying Wang;Jianbing Yan;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-07-12',\n",
       "   'published_date': '2016-01-05',\n",
       "   'preprint_abstract': 'Variation in gene expression contributes to the diversity of phenotype. The construction of the pan-transcriptome is especially necessary for species with complex genomes, such as maize. However, knowledge of the regulation mechanisms and functional consequences of the pan-transcriptome is limited. In this study, we identified 13,382 nuclear expression presence and absence variation candidates (ePAVs, expressed in 5%~95% lines; based on the reference genome) by re-analyzing the RNA sequencing data from the kernels (15 days after pollination) of 368 maize diverse inbreds. It was estimated that only ~1% of the ePAVs are explained by DNA sequence presence and absence variations (PAV). The ePAV genes tend to be regulated by distant eQTLs when compared with non-ePAV genes (called here core expression genes, expressed in more than 95% lines). When the expression presence/absence status was used as the \\\\\" genotype\\\\\" to perform genome-wide association study, 56 (0.42%) ePAVs were significantly associated with 15 agronomic traits and 1,967 (14.74%) with 526 metabolic traits, measured from the mature kernels. While the above was majorly based on the reference genome, by using a modified  assemble-then-align strategy, 2,355 high confidence novel sequences with a total length of 1.9Mb were found absent in the current B73 reference genome (v2). Ten randomly selected novel sequences were validated with genomic PCR. A simulation analysis suggested that the pan-transcriptome of the maize whole kernel is approaching a maximum value of 63,000 genes. Two novel validated sequences annotated as NBS_LRR like genes were found to associate with flavonoid content and their homologs in rice were also found to affect flavonoids and disease-resistance. Novel sequences absent in the present reference genome might be functionally important and deserve more attentions. This study provides novel perspectives and resources to discover maize quantitative trait variations and help us to better understand the kernel regulation networks, thus enhancing maize breeding.',\n",
       "   'preprint_author_corresponding': 'Jianbing  Yan',\n",
       "   'preprint_author_corresponding_institution': 'National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University'},\n",
       "  {'preprint_doi': '10.1101/022871',\n",
       "   'published_doi': '10.1371/journal.pone.0146060',\n",
       "   'published_journal': 'PLOS ONE',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Automatic Synthesis of Anthropomorphic Pulmonary CT Phantoms',\n",
       "   'preprint_authors': 'Daniel Jimenez-Carretero;Raul San Jose Estepar;Mario Diaz Cacio;Maria J Ledesma-Carbayo;',\n",
       "   'preprint_category': 'Bioengineering ',\n",
       "   'preprint_date': '2015-07-21',\n",
       "   'published_date': '2016-01-05',\n",
       "   'preprint_abstract': 'The great density and structural complexity of pulmonary vessels and airways impose limitations on the generation of accurate reference standards, which are critical in training and in the validation of image processing methods for features such as pulmonary vessel segmentation or artery-vein (AV) separations. The design of synthetic computed tomography (CT) images of the lung could overcome these difficulties by providing a database of pseudorealistic cases in a constrained and controlled scenario where each part of the image is differentiated unequivocally. This work demonstrates a complete framework to generate computational anthropomorphic CT phantoms of the human lung automatically. Starting from biological and image-based knowledge about the topology and relationships between structures, the system is able to generate synthetic pulmonary arteries, veins, and airways using iterative growth methods that can be merged into a final simulated lung with realistic features. Visual examination and quantitative measurements of intensity distributions, dispersion of structures and relationships between pulmonary air and blood flow systems show good correspondence between real and synthetic lungs.',\n",
       "   'preprint_author_corresponding': 'Daniel  Jimenez-Carretero',\n",
       "   'preprint_author_corresponding_institution': 'Biomedical Image Technologies. Universidad Politecnica de Madrid, Madrid, Spain & CIBER-BBN'},\n",
       "  {'preprint_doi': '10.1101/024349',\n",
       "   'published_doi': '10.1371/journal.ppat.1005379',\n",
       "   'published_journal': 'PLOS Pathogens',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A conserved interaction between a C-terminal motif in Norovirus VPg and the HEAT-1 domain of eIF4G is essential for translation initiation',\n",
       "   'preprint_authors': 'Eoin N Leen;Frdric Sorgeloos;Samantha Correia;Yasmin Chaudhry;Fabien Cannac;Chiara pastore;Yingqi Xu;Stephen C Graham;Stephen J Matthews;Ian G Goodfellow;Stephen Curry;',\n",
       "   'preprint_category': 'Molecular Biology ',\n",
       "   'preprint_date': '2015-08-10',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': \"Translation initiation is a critical early step in the replication cycle of the positive-sense, single-stranded RNA genome of noroviruses, a major cause of gastroenteritis in humans. Norovirus RNA, which has neither a 5' m7G cap nor an internal ribosome entry site (IRES), adopts an unusual mechanism to initiate protein synthesis that relies on interactions between the VPg protein covalently attached to the 5'-end of the viral RNA and eukaryotic initiation factors (eIFs) in the host cell.\\\\n\\\\nFor murine norovirus (MNV) we previously showed that VPg binds to the middle fragment of eIF4G (4GM; residues 652-1132). Here we have used pull-down assays, fluorescence anisotropy, and isothermal titration calorimetry (ITC) to demonstrate that a stretch of [~]20 amino acids at the C terminus of MNV VPg mediates direct and specific binding to the HEAT-1 domain within the 4GM fragment of eIF4G. Our analysis further reveals that the MNV C-terminus binds to eIF4G HEAT-1 via a motif that is conserved in all known noroviruses. Fine mutagenic mapping suggests that the MNV VPg C terminus may interact with eIF4G in a helical conformation. NMR spectroscopy was used to define the VPg binding site on eIF4G HEAT-1, which was confirmed by mutagenesis and binding assays. We have found that this site is non-overlapping with the binding site for eIF4A on eIF4G HEAT-1 by demonstrating that norovirus VPg can form ternary VPg-eIF4G-eIF4A complexes. The functional significance of the VPg-eIF4G interaction was shown by the ability of fusion proteins containing the C-terminal peptide of MNV VPg to inhibit translation of norovirus RNA but not cap- or IRES-dependent translation. These observations define important structural details of a functional interaction between norovirus VPg and eIF4G and reveal a binding interface that might be exploited as a target for antiviral therapy.\",\n",
       "   'preprint_author_corresponding': 'Stephen  Curry',\n",
       "   'preprint_author_corresponding_institution': 'Imperial College'},\n",
       "  {'preprint_doi': '10.1101/026344',\n",
       "   'published_doi': '10.1534/genetics.115.183152',\n",
       "   'published_journal': 'Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Natural selection and recombination rate variation shape nucleotide polymorphism across the genomes of three related Populus species.',\n",
       "   'preprint_authors': 'Jing Wang;Nathaniel R Street;Douglas G Scofield;Pr K Ingvarsson;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-09-08',\n",
       "   'published_date': '2015-12-31',\n",
       "   'preprint_abstract': 'A central aim of evolutionary genomics is to identify the relative roles that various evolutionary forces have played in generating and shaping genetic variation within and among species. Here we use whole-genome re-sequencing data to characterize and compare genome-wide patterns of nucleotide polymorphism, site frequency spectrum and population-scaled recombination rates in three species of Populus: P. tremula, P. tremuloides and P. trichocarpa. We find that P. tremuloides has the highest level of genome-wide variation, skewed allele frequencies and population-scaled recombination rates, whereas P. trichocarpa harbors the lowest. Our findings highlight multiple lines of evidence suggesting that natural selection, both due to purifying and positive selection, has widely shaped patterns of nucleotide polymorphism at linked neutral sites in all three species. Differences in effective population sizes and rates of recombination are largely explaining the disparate magnitudes and signatures of linked selection we observe among species. The present work provides the first phylogenetic comparative study at genome-wide scale in forest trees. This information will also improve our ability to understand how various evolutionary forces have interacted to influence genome evolution among related species.',\n",
       "   'preprint_author_corresponding': 'Pr K Ingvarsson',\n",
       "   'preprint_author_corresponding_institution': 'Ume Universitet'},\n",
       "  {'preprint_doi': '10.1101/035188',\n",
       "   'published_doi': '10.7554/eLife.10921',\n",
       "   'published_journal': 'eLife',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Tunable protein synthesis by transcript isoforms in human cells',\n",
       "   'preprint_authors': 'Stephen N Floor;Jennifer A Doudna;',\n",
       "   'preprint_category': 'Systems Biology ',\n",
       "   'preprint_date': '2015-12-23',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': \"Eukaryotic genes generate multiple mRNA transcript isoforms though alternative transcription, splicing, and polyadenylation. However, the relationship between human transcript diversity and protein production is complex and not well understood at a global level. We fractionated a polysome profile and reconstructed transcript isoforms from each fraction, which we term Transcript Isoforms in Polysomes sequencing (TrIP-seq). Analysis of these data revealed regulatory features that control ribosome occupancy and the resulting translational output of each transcript isoform. We extracted a panel of 5' and 3' untranslated regions that control protein production from an unrelated gene in cells over a 100-fold range. Select 5' untranslated regions exert robust translational control between cell lines, while 3' untranslated regions can confer cell-type-specific expression. These results expose the large dynamic range of transcript-isoform-specific translational control, identify isoform-specific sequences that control protein output in human cells, and demonstrate that transcript isoform diversity must be considered when relating RNA and protein levels.\",\n",
       "   'preprint_author_corresponding': 'Jennifer A Doudna',\n",
       "   'preprint_author_corresponding_institution': 'University of California, Berkeley'},\n",
       "  {'preprint_doi': '10.1101/012096',\n",
       "   'published_doi': '10.1093/sysbio/syw001',\n",
       "   'published_journal': 'Systematic Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Detecting the anomaly zone in species trees and evidence for a misleading signal in higher-level skink phylogeny (Squamata: Scincidae).',\n",
       "   'preprint_authors': 'Charles W Linkem;Vladimir N. Minin;Adam D Leache;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2014-12-03',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': 'The anomaly zone presents a major challenge to the accurate resolution of many parts of the Tree of Life. The anomaly zone is defined by the presence of a gene tree topology that is more probable than the true species tree. This discrepancy can result from consecutive rapid speciation events in the species tree. Similar to the problem of long-branch attraction, including more data (loci) will only reinforce the support for the incorrect species tree. Empirical phylogenetic studies often implement coalescent based species tree methods to avoid the anomaly zone, but to this point these studies have not had a method for providing any direct evidence that the species tree is actually in the anomaly zone. In this study, we use 16 species of lizards in the family Scincidae to investigate whether nodes that are difficult to resolve are located within the anomaly zone. We analyze new phylogenomic data (429 loci), using both concatenation and coalescent based species tree estimation, to locate conflicting topological signal. We then use the unifying principle of the anomaly zone, together with estimates of ancestral population sizes and species persistence times, to determine whether the observed phylogenetic conflict is a result of the anomaly zone. We identify at least three regions of the Scindidae phylogeny that provide demographic signatures consistent with the anomaly zone, and this new information helps reconcile the phylogenetic conflict in previously published studies on these lizards. The anomaly zone presents a real problem in phylogenetics, and our new framework for identifying anomalous relationships will help empiricists leverage their resources appropriately for overcoming this challenge.',\n",
       "   'preprint_author_corresponding': 'Charles W Linkem',\n",
       "   'preprint_author_corresponding_institution': 'University of Washington, Seattle'},\n",
       "  {'preprint_doi': '10.1101/024331',\n",
       "   'published_doi': '10.1371/journal.pone.0146511',\n",
       "   'published_journal': 'PLOS ONE',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Reduced Glucose Sensation Can Increase the Fitness of Saccharomyces cerevisiae Lacking Mitochondrial DNA',\n",
       "   'preprint_authors': 'Emel Akdoan;Mehmet Tardu;Grkem Garipler;Glkz Baytek;brahim Halil Kavakl;Cory D. Dunn;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-08-10',\n",
       "   'published_date': '2016-01-11',\n",
       "   'preprint_abstract': 'Damage to the mitochondrial genome (mtDNA) can lead to diseases for which there are no clearly effective treatments. Since mitochondrial function and biogenesis are controlled by the nutrient environment of the cell, it is possible that perturbation of conserved, nutrient-sensing pathways may successfully treat mitochondrial disease. We found that restricting glucose or otherwise reducing the activity of the protein kinase A (PKA) pathway can lead to improved proliferation of Saccharomyces cerevisiae cells lacking mtDNA and that the transcriptional response to mtDNA loss is reduced in cells with diminished PKA activity. We have excluded many pathways and proteins from being individually responsible for the benefits provided to cells lacking mtDNA by PKA inhibition, and we found that robust import of mitochondrial polytopic membrane proteins may be required in order for cells without mtDNA to receive the full benefits of PKA reduction. Finally, we have discovered that the transcription of genes involved in arginine biosynthesis and aromatic amino acid catabolism is altered after mtDNA damage. Our results highlight the potential importance of nutrient detection and availability on the outcome of mitochondrial dysfunction.',\n",
       "   'preprint_author_corresponding': 'Cory D. Dunn',\n",
       "   'preprint_author_corresponding_institution': 'Ko University'},\n",
       "  {'preprint_doi': '10.1101/017855',\n",
       "   'published_doi': '10.1371/journal.pgen.1005616',\n",
       "   'published_journal': 'PLOS Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Genome-wide analysis reveals novel regulators of growth in Drosophila melanogaster',\n",
       "   'preprint_authors': 'Sibylle Chantal Vonesch;David Lamparter;Trudy FC Mackay;Sven Bergmann;Ernst Hafen;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-04-10',\n",
       "   'published_date': '2016-01-11',\n",
       "   'preprint_abstract': 'Organismal size depends on the interplay between genetic and environmental factors. Genome-wide association (GWA) analyses in humans have implied many genes in the control of height but suffer from the inability to control the environment. Genetic analyses in Drosophila have identified conserved signaling pathways controlling size; however, how these pathways control phenotypic diversity is unclear. We performed GWA of size traits using the Drosophila Genetic Reference Panel of inbred, sequenced lines. We find that the top associated variants differ between traits and sexes; do not map to canonical growth pathway genes, but can be linked to these by epistasis analysis; and are enriched for genes and putative enhancers. Performing GWA on well-studied developmental traits under controlled conditions expands our understanding of developmental processes underlying phenotypic diversity.\\\\n\\\\nAUTHOR SUMMARYGenetic studies in Drosophila have elucidated conserved signaling pathways and environmental factors that together control organismal size. In humans, hundreds of genes are associated with height variation, but these associations have not been performed in a controlled environment. As a result we are still lacking an understanding of the mechanisms creating size variability within a species. Here, under carefully controlled environmental conditions, we identify naturally occurring genetic variants that are associated with size diversity in Drosophila. We identify a cluster of associations close to the kek1 locus, a well-characterized growth regulator, but otherwise find that most variants are located in or close to genes that do not belong to the conserved pathways but may interact with these in a biological network. Many of these genes have a conserved role in humans. We validate 33 novel growth regulatory genes that participate in diverse cellular processes, most notably cellular metabolism and cell polarity. This study is the first genome-wide association analysis of natural variants underlying size in Drosophila and our results complement the knowledge we have accumulated on this trait from mutational studies of single genes.',\n",
       "   'preprint_author_corresponding': 'Ernst  Hafen',\n",
       "   'preprint_author_corresponding_institution': 'ETH Zurich'},\n",
       "  {'preprint_doi': '10.1101/024570',\n",
       "   'published_doi': '10.1534/g3.115.024380',\n",
       "   'published_journal': 'G3: Genes|Genomes|Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A computational approach to estimating nondisjunction frequency in Saccharomyces cerevisiae',\n",
       "   'preprint_authors': 'Daniel Chu;Sean M Burgess;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-08-13',\n",
       "   'published_date': '2016-01-08',\n",
       "   'preprint_abstract': 'Errors segregating homologous chromosomes during meiosis result in the formation of aneuploid gametes and are the largest contributing factor to birth defects and spontaneous abortions in humans. Saccharomyces cerevisiae has long served as a model organism for studying the gene network supporting normal chromosome segregation. Current methods of measuring homolog nondisjunction frequencies are laborious and involve dissecting thousands of tetrads to detect missegregation of individually marked chromosomes. Here we describe a holistic computational approach to determine the relative contributions of meiosis I nondisjunction and random spore death in mutants with reduced spore viability. These values are based on best-fit distributions of 4, 3, 2, 1, and 0 viable-spore tetrads to observed distributions in mutant and wild-type strains. We show proof-of-principle using published data sets that the calculated average meiosis I nondisjunction frequency closely matches empirically determined values. This analysis also points to meiosis I nondisjunction as an intrinsic component of spore inviability in wild-type strains. We uncover two classes of mutants that show distinct relationships between nondisjunction death and random spore death. Class I mutants, including those with known defects in establishing and maintaining the physical engagement of homologous chromosomes display a 4-fold greater ratio of nondisjunction death to random spore death compared to Class II mutants, which include those with defects in sister chromatid cohesion. Low numbers of required tetrads facilitates epistasis analysis to probe genetic interactions. Finally the application of the R-Scripts does not require any special strain construction and can be applied to previously observed tetrad distributions.',\n",
       "   'preprint_author_corresponding': 'Sean M Burgess',\n",
       "   'preprint_author_corresponding_institution': 'University of California, Davis'},\n",
       "  {'preprint_doi': '10.1101/025437',\n",
       "   'published_doi': '10.1093/bioinformatics/btw004',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'OEFinder: A user interface to identify and visualize ordering effects in single-cell RNA-seq data',\n",
       "   'preprint_authors': 'Ning Leng;Jeea Choi;Li-Fang Chu;James Thomson;Christina Kendziorski;Ron Stewart;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-08-25',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': 'A recent paper identified an artifact in multiple single-cell RNA-seq (scRNA-seq) data sets generated by the Fluidigm C1 platform. Specifically, Leng* et al. showed significantly increased gene expression in cells captured from sites with small or large plate output IDs. We refer to this artifact as an ordering effect (OE). Including OE genes in downstream analyses could lead to biased results. To address this problem, we developed a statistical method and software called OEFinder to identify a sorted list of OE genes. OEFinder is available as an R package along with user-friendly graphical interface implementations that allows users to check for potential artifacts in scRNA-seq data generated by the Fluidigm C1 platform.\\\\n\\\\nAvailability and ImplementationOEFinder is freely available at https://github.com/lengning/OEFinder\\\\n\\\\nContactrstewart@morgridge.org',\n",
       "   'preprint_author_corresponding': 'Ron  Stewart',\n",
       "   'preprint_author_corresponding_institution': 'Morgridge Institute for Research'},\n",
       "  {'preprint_doi': '10.1101/025510',\n",
       "   'published_doi': '10.1371/journal.pone.0146947',\n",
       "   'published_journal': 'PLOS ONE',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Systems Biology Approach to Model the Life Cycle of Trypanosoma cruzi',\n",
       "   'preprint_authors': 'Alejandra Carrea;Luis Diambra;',\n",
       "   'preprint_category': 'Systems Biology ',\n",
       "   'preprint_date': '2015-08-25',\n",
       "   'published_date': '2016-01-11',\n",
       "   'preprint_abstract': 'Due to recent advances in reprogramming cell phenotypes, many efforts have been dedicated to developing reverse engineering procedures for the identification of gene regulatory networks that emulate dynamical properties associated with the cell fates of a given biological system. In this work, we propose a systems biology approach for the reconstruction of the gene regulatory network underlying the dynamics of the Trypanosoma cruzis life cycle. By means of an optimisation procedure, we embedded the steady state maintenance, and the known phenotypic transitions between these steady states in response to environmental cues, into the dynamics of a gene network model. In the resulting network architecture we identified a small subnetwork, formed by seven interconnected nodes, that controls the parasites life cycle. The present approach could be useful for better understanding other single cell organisms with multiple developmental stages.\\\\n\\\\nAbbreviations',\n",
       "   'preprint_author_corresponding': 'Luis  Diambra',\n",
       "   'preprint_author_corresponding_institution': 'Universidad Nacional de La Plata'},\n",
       "  {'preprint_doi': '10.1101/025700',\n",
       "   'published_doi': '10.1186/s13062-015-0104-3',\n",
       "   'published_journal': 'Biology Direct',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The mysterious orphans of Mycoplasmataceae',\n",
       "   'preprint_authors': 'Tatiana Tatarinova;Inna Lysnyansky;Yuri Nikolsky;Alexander Bolshoy;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-08-29',\n",
       "   'published_date': '2016-01-08',\n",
       "   'preprint_abstract': 'BackgroundThe length of a protein sequence is largely determined by its function, i.e. each functional group is associated with an optimal size. However, comparative genomics revealed that proteins length may be affected by additional factors. In 2002 it was shown that in bacterium Escherichia coli and the archaeon Archaeoglobus fulgidus, protein sequences with no homologs are, on average, shorter than those with homologs [1]. Most experts now agree that the length distributions are distinctly different between protein sequences with and without homologs in bacterial and archaeal genomes. In this study, we examine this postulate by a comprehensive analysis of all annotated prokaryotic genomes and focusing on certain exceptions.\\\\n\\\\nResultsWe compared lengths distributions of \\\\\"having homologs proteins\\\\\" (HHPs) and \\\\\"non-having homologs proteins\\\\\" (orphans or ORFans) in all currently annotated completely sequenced prokaryotic genomes. As expected, the HHPs and ORFans have strikingly different length distributions in almost all genomes. As previously established, the HHPs, indeed, are, on average, longer than the ORFans, and the length distributions for the ORFans have a relatively narrow peak, in contrast to the HHPs, whose lengths spread over a wider range of values. However, about thirty genomes do not obey these rules. Practically all genomes of Mycoplasma and Ureaplasma have atypical ORFans distributions, with the mean lengths of ORFan larger than the mean lengths of HHPs. These genera constitute over 80% of atypical genomes.\\\\n\\\\nConclusionsWe confirmed on a ubiquitous set of genomes the previous observation that HHPs and ORFans have different gene length distributions. We also showed that Mycoplasmataceae genomes have very distinctive distributions of ORFans lengths. We offer several possible biological explanations of this phenomenon.',\n",
       "   'preprint_author_corresponding': 'Tatiana  Tatarinova',\n",
       "   'preprint_author_corresponding_institution': 'USC'},\n",
       "  {'preprint_doi': '10.1101/033043',\n",
       "   'published_doi': '10.1371/journal.pgen.1005774',\n",
       "   'published_journal': 'PLOS Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Adaptive Protein Evolution in Animals and the Effective Population Size Hypothesis.',\n",
       "   'preprint_authors': 'Nicolas Galtier;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-11-29',\n",
       "   'published_date': '2016-01-11',\n",
       "   'preprint_abstract': 'The rate at which genomes adapt to environmental changes and the prevalence of adaptive processes in molecular evolution are two controversial issues in current evolutionary genetics. Previous attempts to quantify the genome-wide rate of adaptation through amino-acid substitution have revealed a surprising diversity of patterns, with some species (e.g. Drosophila) experiencing a very high adaptive rate, while other (e.g. humans) are dominated by nearly-neutral processes. It has been suggested that this discrepancy reflects between-species differences in effective population size. Published studies, however, were mainly focused on model organisms, and relied on disparate data sets and methodologies, so that an overview of the prevalence of adaptive protein evolution in nature is currently lacking. Here we extend existing estimators of the amino-acid adaptive rate by explicitly modelling the effect of favourable mutations on non-synonymous polymorphism patterns, and we apply these methods to a newly-built, homogeneous data set of 44 non-model animal species pairs. Data analysis uncovers a major contribution of adaptive evolution to the amino-acid substitution process across all major metazoan phyla - with the notable exception of humans and primates. The proportion of adaptive amino-acid substitution is found to be positively correlated to species effective population size. This relationship, however, appears to be primarily driven by a decreased rate of nearly-neutral amino-acid substitution due to more efficient purifying selection in large populations. Our results reveal that adaptive processes dominate the evolution of proteins in most animal species, but do not corroborate the hypothesis that adaptive substitutions accumulate at a faster rate in large populations. Implications regarding the factors influencing the rate of adaptive evolution and positive selection detection in humans vs. other organisms are discussed.\\\\n\\\\nAuthor summaryThe rate at which species adapt to environmental changes is a controversial topic. The theory predicts that adaptation is easier in large than in small populations, and the genomic studies of model organisms have revealed a much higher adaptive rate in large population-sized flies than in small population-sized humans and apes. Here we build and analyse a large data set of protein-coding sequences made of thousands of genes in 44 pairs of species from various groups of animals including insects, molluscs, annelids, echinoderms, reptiles, birds, and mammals. Extending and improving existing data analysis methods, we show that adaptation is a major process in protein evolution across all phyla of animals: the proportion of amino-acid substitutions that occurred adaptively is above 50% in a majority of species, and reaches up to 90%. Our analysis does not confirm that population size, here approached through species genetic diversity and ecological traits, does influence the rate of adaptive molecular evolution, but points to human and apes as a special case, compared to other animals, in terms of adaptive genomic processes.',\n",
       "   'preprint_author_corresponding': 'Nicolas  Galtier',\n",
       "   'preprint_author_corresponding_institution': 'University Montpellier - CNRS'},\n",
       "  {'preprint_doi': '10.1101/027201',\n",
       "   'published_doi': '10.1093/bioinformatics/btw012',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'MTG2: An efficient algorithm for multivariate linear mixed model analysis based on genomic information',\n",
       "   'preprint_authors': 'Sang Hong Lee;Julius van der Werf;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-09-18',\n",
       "   'published_date': '2016-01-10',\n",
       "   'preprint_abstract': 'We have developed an algorithm for genetic analysis of complex traits using genome-wide SNPs in a linear mixed model framework. Compared to current standard REML software based on the mixed model equation, our method could be more than 1000 times faster. The advantage is largest when there is only a single genetic covariance structure. The method is particularly useful for multivariate analysis, including multitrait models and random regression models for studying reaction norms. We applied our proposed method to publicly available mice and human data and discuss advantages and limitations.\\\\n\\\\nAvailability: MTG2 is available in https://sites.google.com/site/honglee0707/mtg2.\\\\n\\\\nContact: hong.lee@une.edu.au\\\\n\\\\nSupplementary information: Supplementary data are available.',\n",
       "   'preprint_author_corresponding': 'Sang Hong  Lee',\n",
       "   'preprint_author_corresponding_institution': 'University of New England'},\n",
       "  {'preprint_doi': '10.1101/029538',\n",
       "   'published_doi': '10.1016/j.ygeno.2015.12.006',\n",
       "   'published_journal': 'Genomics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Robust and Stable Gene Selection via Maximum-Minimum Correntropy Criterion',\n",
       "   'preprint_authors': 'Majid Mohammadi;Hossein Sharifi Noghabi;Ghosheh Abed Hodtani;Habib Rajabi Mashhadi;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-10-21',\n",
       "   'published_date': '2016-01-04',\n",
       "   'preprint_abstract': 'One of the central challenges in cancer research is identifying significant genes among thousands of others on a microarray. Since preventing outbreak and progression of cancer is the ultimate goal in bioinformatics and computational biology, detection of genes that are most involved is vital and crucial. In this article, we propose a Maximum-Minimum Correntropy Criterion (MMCC) approach for selection of biologically meaningful genes from microarray data sets which is stable, fast and robust against diverse noise and outliers and competitively accurate in comparison with other algorithms. Moreover, via an evolutionary optimization process, the optimal number of features for each data set is determined. Through broad experimental evaluation, MMCC is proved to be significantly better compared to other well-known gene selection algorithms for 25 commonly used microarray data sets. Surprisingly, high accuracy in classification by Support Vector Machine (SVM) is achieved by less than10 genes selected by MMCC in all of the cases.',\n",
       "   'preprint_author_corresponding': 'Ghosheh Abed  Hodtani',\n",
       "   'preprint_author_corresponding_institution': 'Department of Electrical Engineering, Faculty of Engineering, Ferdowsi University of Mashhad, Mashh'},\n",
       "  {'preprint_doi': '10.1101/030296',\n",
       "   'published_doi': '10.1105/tpc.15.00637',\n",
       "   'published_journal': 'The Plant Cell',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Reconstitution of CO2 regulation of SLAC1 anion channel and function of CO2-permeable PIP2;1 aquaporin as carbonic anhydrase 4 interactor',\n",
       "   'preprint_authors': 'Cun Wang;Honghong Hu;Xue Qin;Brian Zeise;Danyun Xu;Wouter-Jan Rappel;Walter F Boron;Julian I Schroeder;',\n",
       "   'preprint_category': 'Plant Biology ',\n",
       "   'preprint_date': '2015-11-10',\n",
       "   'published_date': '2016-01-13',\n",
       "   'preprint_abstract': 'Daily dark periods cause an increase in the leaf CO2 concentration (Ci) and the continuing atmospheric [CO2] rise also increases Ci. Elevated Ci causes closing of stomatal pores thus regulating gas exchange of plants. The molecular signaling mechanisms leading to CO2-induced stomatal closure are only partially understood. Here we demonstrate that high intracellular [Formula] enhances currents mediated by the guard cell S-type anion channel SLAC1 when co-expressing either of the protein kinases OST1, CPK6 or CPK23 in Xenopus oocytes. Split-ubiquitin screening identified the PIP2;1 aquaporin as an interactor of the {beta}CA4 carbonic anhydrase, which was confirmed in split luciferase, bimolecular fluorescence complementation and co-immunoprecipitation experiments. PIP2;1 exhibited CO2 permeability. Co-expression of {beta}CA4 and PIP2;1 with OST1-SLAC1 or CPK6/23-SLAC1 enabled extracellular CO2 enhancement of SLAC1 anion channel activity. An inactive PIP2;1 point mutation was identified which abrogated water and CO2 permeability and extracellular CO2 regulation of SLAC1 activity in Xenopus oocytes. These findings identify the CO2-permeable PIP2;1 aquaporin as key interactor of carbonic anhydrases, show functional reconstitution of extracellular CO2 signaling to ion channel regulation and implicate SLAC1 as a bicarbonate-responsive protein in CO2 regulation of S-type anion channels.',\n",
       "   'preprint_author_corresponding': 'Julian I  Schroeder',\n",
       "   'preprint_author_corresponding_institution': 'University of California, San Diego'},\n",
       "  {'preprint_doi': '10.1101/019570',\n",
       "   'published_doi': '10.1093/nar/gkv1542',\n",
       "   'published_journal': 'Nucleic Acids Research',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'In Vivo Blunt-End Cloning Through CRISPR/Cas9-Facilitated Non-Homologous End-Joining',\n",
       "   'preprint_authors': 'Jonathan M Geisinger;Sren Turan;Sophia Hernandez;Laura P Spector;Michele P Calos;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-05-21',\n",
       "   'published_date': '2016-01-13',\n",
       "   'preprint_abstract': 'The ability to precisely modify the genome in a site-specific manner is extremely useful. The CRISPR/Cas9 system facilitates precise modifications by generating RNA-guided double-strand breaks. We demonstrate that guide RNA pairs generate deletions that are repaired with a high level of precision by non-homologous end-joining in mammalian cells. We present a method called knock-in blunt ligation for exploiting this excision and repair to insert exogenous sequences in a homology-independent manner without loss of additional nucleotides. We successfully utilize this method in a human immortalized cell line and induced pluripotent stem cells to insert fluorescent protein cassettes into various loci, with efficiencies up to 35.8% in HEK293 cells. We also present a version of Cas9 fused to the FKBP12-L106P destabilization domain for investigating repair dynamics of Cas9-induced double-strand breaks. Our in vivo blunt-end cloning method and destabilization-domain-fused Cas9 variant increase the repertoire of precision genome engineering approaches.',\n",
       "   'preprint_author_corresponding': 'Michele P Calos',\n",
       "   'preprint_author_corresponding_institution': 'Stanford University School of Medicine'},\n",
       "  {'preprint_doi': '10.1101/020305',\n",
       "   'published_doi': '10.1186/s12864-016-2364-4',\n",
       "   'published_journal': 'BMC Genomics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Pangenome-wide and molecular evolution analyses of the Pseudomonas aeruginosa species',\n",
       "   'preprint_authors': 'Jeanneth Mosquera-Rendon;Ana M. Rada-Bravo;Sonia Cardenas-Brito;Mauricio Corredor;Eliana Restrepo-Pineda;Alfonso Benitez-Paez;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-06-02',\n",
       "   'published_date': '2016-01-12',\n",
       "   'preprint_abstract': 'BackgroundDrug treatments and vaccine designs against the opportunistic human pathogen Pseudomonas aeruginosa have multiple issues, all associated with the diverse genetic traits present in this pathogen, ranging from multi-drug resistant genes to the molecular machinery for the biosynthesis of biofilms. Several candidate vaccines against P. aeruginosa have been developed, which target the outer membrane proteins; however, major issues arise when attempting to establish complete protection against this pathogen due to its presumably genotypic variation at the strain level. To shed light on this concern, we proposed this study to assess the P. aeruginosa pangenome and its molecular evolution across multiple strains.\\\\n\\\\nResultsThe P. aeruginosa pangenome was estimated to contain more than 16,000 non-redundant genes, and approximately 15% of these constituted the core genome. Functional analyses of the accessory genome indicated a wide presence of genetic elements directly associated with pathogenicity. An in-depth molecular evolution analysis revealed the full landscape of selection forces acting on the P. aeruginosa pangenome, in which purifying selection drives evolution in the genome of this human pathogen. We also detected distinctive positive selection in a wide variety of outer membrane proteins, with the data supporting the concept of substantial genetic variation in proteins probably recognized as antigens. Approaching the evolutionary information of genes under extremely positive selection, we designed a new Multi-Locus Sequencing Typing assay for an informative, rapid, and cost-effective genotyping of P. aeruginosa clinical isolates.\\\\n\\\\nConclusionsWe report the unprecedented pangenome characterization of P. aeruginosa on a large scale, which included almost 200 bacterial genomes from one single species and a molecular evolutionary analysis at the pangenome scale. Evolutionary information presented here provides a clear explanation of the issues associated with the use of protein conjugates from pili, flagella, or secretion systems as antigens for vaccine design, which exhibit high genetic variation in terms of non-synonymous substitutions in P. aeruginosa strains.',\n",
       "   'preprint_author_corresponding': 'Alfonso  Benitez-Paez',\n",
       "   'preprint_author_corresponding_institution': 'Bioinformatics Analysis Group, Centro de Investigacion y Desarrollo en Biotecnologia'},\n",
       "  {'preprint_doi': '10.1101/017210',\n",
       "   'published_doi': '10.1016/j.cell.2015.12.015',\n",
       "   'published_journal': 'Cell',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Visual cues determine hippocampal directional selectivity',\n",
       "   'preprint_authors': 'Lavanya Acharya;Zahra M. Aghajan;Cliff Vuong;Jason Moore;Mayank Mehta;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-03-28',\n",
       "   'published_date': '2015-12-17',\n",
       "   'preprint_abstract': 'Both spatial and directional information are necessary for navigation. Rodent hippocampal neurons show spatial selectivity in all environments1, but directional tuning only on linear paths2-8.The sensory mechanisms underlying directionality are unknown, though vestibular and visual cues are thought to be crucial. However, hippocampal neurons are thought to show no angular modulation during two-dimensional random foraging despite the presence of vestibular and visual cues6,7. Additionally, specific aspects of visual cues have not been directly linked to hippocampal responses in rodents. To resolve these issues we manipulated vestibular and visual cues in a series of experiments. We first measured hippocampal activity during random foraging in real world (RW) where we found that neurons firing exhibited significant modulation by head-direction. In fact, the fraction of modulated neurons was comparable to that in the head-direction system9. These findings are contrary to commonly held beliefs about hippocampal directionality6,7. To isolate the contribution of visual cues we measured neural responses in a visually similar virtual reality (VR) where the range of vestibular inputs is minimized5,10,11. Significant directional modulation was not only found in VR, but it was comparable to that in RW. Several additional experiments revealed that changes in the angular information contained in the visual cues induced corresponding changes in hippocampal head-directional modulation. Remarkably, for head-directionally modulated neurons, the ensemble activity was biased towards the sole visual cue. These results demonstrate that robust vestibular cues are not required for hippocampal directional selectivity, while visual cues are not only sufficient but also play a causal role in driving hippocampal responses.',\n",
       "   'preprint_author_corresponding': 'Mayank  Mehta',\n",
       "   'preprint_author_corresponding_institution': 'UCLA'},\n",
       "  {'preprint_doi': '10.1101/030650',\n",
       "   'published_doi': '10.1128/mSystems.00025-15',\n",
       "   'published_journal': 'mSystems',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'ADAGE analysis of publicly available gene expression data collections illuminates Pseudomonas aeruginosa-host interactions',\n",
       "   'preprint_authors': 'Jie Tan;John H Hammond;Deborah A Hogan;Casey S Greene;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-11-05',\n",
       "   'published_date': '2016-01-19',\n",
       "   'preprint_abstract': 'The growth in genome-scale assays of gene expression for different species in publicly available databases presents new opportunities for computational methods that aid in hypothesis generation and biological interpretation of these data. Here, we present an unsupervised machine-learning approach, ADAGE (Analysis using Denoising Autoencoders of Gene Expression) and apply it to the interpretation of all of the publicly available gene expression data for Pseudomonas aeruginosa, an important opportunistic bacterial pathogen. In post-hoc positive control analyses using curated knowledge, the P. aeruginosa ADAGE model found that co-operonic genes often participated in similar processes and accurately predicted which genes had similar functions. By analyzing newly generated data and previously published microarray and RNA-seq data, the ADAGE model identified gene expression differences between strains, modeled the cellular response to low oxygen, and predicted the involvement of biological processes despite low level expression differences in directly involved genes. Comparison of ADAGE with PCA and ICA revealed that ADAGE extracts distinct signals. We provide the ADAGE model with analysis of all publicly available P. aeruginosa GeneChip experiments, and we provide open source code for use in other species and settings.',\n",
       "   'preprint_author_corresponding': 'Casey S Greene',\n",
       "   'preprint_author_corresponding_institution': 'University of Pennsylvania'},\n",
       "  {'preprint_doi': '10.1101/025148',\n",
       "   'published_doi': '10.7717/peerj.1441',\n",
       "   'published_journal': 'PeerJ',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Snake venom gland cDNA sequencing using the Oxford Nanopore MinION portable DNA sequencer',\n",
       "   'preprint_authors': 'John F Mulley;Adam D Hargreaves;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-08-22',\n",
       "   'published_date': '2015-11-24',\n",
       "   'preprint_abstract': \"Portable DNA sequencers such as the Oxford Nanopore MinION device have the potential to be truly disruptive technologies, facilitating new approaches and analyses and, in some cases, taking sequencing out of the lab and into the field. However, the capabilities of these technologies are still being revealed. Here we show that single-molecule cDNA sequencing using the MinION accurately characterises venom toxin-encoding genes in the painted saw-scaled viper, Echis coloratus. We find the raw sequencing error rate to be around 12%, improved to 0-2% with hybrid error correction and 3% with de novo error correction. Our corrected data provides full coding sequences and 5' and 3' UTRs for 29 of 33 candidate venom toxins detected, far superior to Illumina data (13/40 complete) and Sanger-based ESTs (15/29). We suggest that, should the current pace of improvement continue, the MinION will become the default approach for cDNA sequencing in a variety of species.\",\n",
       "   'preprint_author_corresponding': 'John F Mulley',\n",
       "   'preprint_author_corresponding_institution': 'Bangor University'},\n",
       "  {'preprint_doi': '10.1101/026484',\n",
       "   'published_doi': '10.1016/j.bpj.2015.11.013',\n",
       "   'published_journal': 'Biophysical Journal',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Photon-HDF5: an open file format for single-molecule fluorescence experiments using photon-counting detectors',\n",
       "   'preprint_authors': 'Antonino Ingargiola;Ted Laurence;Robert Boutelle;Shimon Weiss;Xavier Michalet;',\n",
       "   'preprint_category': 'Biophysics ',\n",
       "   'preprint_date': '2015-09-09',\n",
       "   'published_date': '2016-01-05',\n",
       "   'preprint_abstract': 'We introduce Photon-HDF5, an open and efficient file format to simplify exchange and long term accessibility of data from single-molecule fluorescence experiments based on photon-counting detectors such as single-photon avalanche diode (SPAD), photomultiplier tube (PMT) or arrays of such detectors. The format is based on HDF5, a widely used platform- and language-independent hierarchical file format for which user-friendly viewers are available. Photon-HDF5 can store raw photon data (timestamp, channel number, etc...) from any acquisition hardware, but also setup and sample description, information on provenance, authorship and other metadata, and is flexible enough to include any kind of custom data.\\\\n\\\\nThe format specifications are hosted on a public website, which is open to contributions by the biophysics community. As an initial resource, the website provides code examples to read Photon-HDF5 files in several programming languages and a reference Python library (phconvert), to create new Photon-HDF5 files and convert several existing file formats into Photon-HDF5. To encourage adoption by the academic and commercial communities, all software is released under the MIT open source license.',\n",
       "   'preprint_author_corresponding': 'Antonino  Ingargiola',\n",
       "   'preprint_author_corresponding_institution': 'University of California Los Angeles'},\n",
       "  {'preprint_doi': '10.1101/005405',\n",
       "   'published_doi': '10.1371/journal.pone.0143489',\n",
       "   'published_journal': 'PLOS ONE',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Powerful tests for multi-marker association analysis using ensemble learning',\n",
       "   'preprint_authors': 'Badri Padhukasahasram;Chandan K Reddy;L. Keoki Williams;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2014-05-23',\n",
       "   'published_date': '2015-11-30',\n",
       "   'preprint_abstract': 'Multi-marker approaches are currently gaining a lot of interest in genome wide association studies and can enhance power to detect new associations under certain conditions. Gene and pathway based association tests are increasingly being viewed as useful complements to the more widely used single marker association analysis which have successfully uncovered numerous disease variants. A major drawback of single-marker based methods is that they do not consider pairwise and higher-order interactions between genetic variants. Here, we describe novel tests for multi-marker association analyses that are based on phenotype predictions obtained from machine learning algorithms. Instead of utilizing only a linear or logistic regression model, we propose the use of ensembles of diverse machine learning algorithms for constructing such association tests. As the true mathematical relationship between a phenotype and any group of genetic and clinical variables is unknown in advance and may be complex, such a strategy gives us a general and flexible framework to approximate this relationship across different sets of SNPs. We show how phenotype prediction obtained from ensemble learning algorithms can be used for constructing tests for the joint association of multiple variants. We first apply our method to simulated datasets to demonstrate its power and correctness. Then, we apply our method to previously studied asthma-related genes in two independent asthma cohorts to conduct association tests.',\n",
       "   'preprint_author_corresponding': 'Badri  Padhukasahasram',\n",
       "   'preprint_author_corresponding_institution': 'Henry Ford Health System'},\n",
       "  {'preprint_doi': '10.1101/002014',\n",
       "   'published_doi': '10.1093/nar/gkv1198',\n",
       "   'published_journal': 'Nucleic Acids Research',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Estimate of Within Population Incremental Selection Through Branch Imbalance in Lineage Trees',\n",
       "   'preprint_authors': 'Gilad Liberman;Jennifer Benichou;Lea Tsaban;yaakov maman;Jacob Glanville;yoram louzoun;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2014-01-23',\n",
       "   'published_date': '2015-11-19',\n",
       "   'preprint_abstract': 'Incremental selection within a population, defined as a limited fitness change following a mutation, is an important aspect of many evolutionary processes and can significantly affect a large number of mutations through the genome. Strongly advantageous or deleterious mutations are detected through the fixation of mutations in the population, using the synonymous to non-synonymous mutations ratio in sequences. There are currently to precise methods to estimate incremental selection occurring over limited periods. We here provide for the first time such a detailed method and show its precision and its applicability to the genomic analysis of selection. A special case of evolution is rapid, short term micro-evolution, where organism are under constant adaptation, occurring for example in viruses infecting a new host, B cells mutating during a germinal center reactions or mitochondria evolving within a given host. The proposed method is a novel mixed lineage tree/sequence based method to detect within population selection as defined by the effect of mutations on the average number of offspring. Specifically, we pro-pose to measure the log of the ratio between the number of leaves in lineage trees branches following synonymous and non-synonymous mutations. This method does not suffer from the need of a baseline model and is practically not affected by sampling biases. In order to show the wide applicability of this method, we apply it to multiple cases of micro-evolution, and show that it can detect genes and inter-genic regions using the selection rate and detect selection pressures in viral proteins and in the immune response to pathogens.',\n",
       "   'preprint_author_corresponding': 'Jennifer  Benichou',\n",
       "   'preprint_author_corresponding_institution': 'Bar Ilan University'},\n",
       "  {'preprint_doi': '10.1101/001230',\n",
       "   'published_doi': '10.1093/bioinformatics/btv363',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'libRoadRunner: A High Performance SBML Compliant Simulator',\n",
       "   'preprint_authors': 'Herbert M Sauro;Totte T Karlsson;Maciej Swat;Michal Galdzicki;Andy Somogyi;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2013-12-12',\n",
       "   'published_date': '2015-10-15',\n",
       "   'preprint_abstract': 'SummaryWe describe libRoadRunner, a cross-platform, open-source, high performance C++ library for running and analyzing SBML-compliant models. libRoadRunner was created primarily to achieve high performance, ease of use, portability and an extensible architecture. libRoadRunner includes a comprehensive API, Plugin support, Python scripting and additional functionality such as stoichio-metric and metabolic control analysis.\\\\n\\\\nAccessibility and ImplementationTo maximize collaboration, we made libRoadRunner open source and released it under the Apache License, Version 2.0. To facilitate reuse, we have developed comprehensive Python bindings using SWIG (swig.org) and a C API. Li-bRoadRunner uses a number of statically linked third party libraries including: LLVM [4], libSBML [1], CVODE, NLEQ2, LAPACK and Poco. LibRoadRunner is supported on Windows, Mac OS X and Linux.\\\\n\\\\nSupplementary informationOnline documentation, build instructions and git source repository are available at http://www.libroadrunner.org',\n",
       "   'preprint_author_corresponding': 'Herbert M Sauro',\n",
       "   'preprint_author_corresponding_institution': 'University of Washington'},\n",
       "  {'preprint_doi': '10.1101/005231',\n",
       "   'published_doi': '10.1111/gtc.12190',\n",
       "   'published_journal': 'Genes to Cells',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'SraTailor: GUI software for visualizing high-throughput sequence read archives',\n",
       "   'preprint_authors': 'Shinya Oki;Kazumitsu Maehara;Yasuyuki Ohkawa;Chikara Meno;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2014-05-16',\n",
       "   'published_date': '2014-10-07',\n",
       "   'preprint_abstract': 'Raw high-throughput sequence data are deposited in public databases as SRAs (Sequence Read Archives) and are publically available to every researcher. However, in order to graphically visualize the sequence data of interest, the corresponding SRAs must be downloaded and converted into BigWig format through complicated command-line processing. This task requires users to possess skill with script languages and sequence data processing, a requirement that prevents a wide range of biologists from exploiting SRAs. To address these challenges, we developed SraTailor, a GUI (Graphical User Interface) software package that automatically converts an SRA into a BigWig-formatted file. Simplicity of use is one of the most notable features of SraTailor: entering an accession number of an SRA and clicking the mouse are the only steps required in order to obtain BigWig-formatted files and to graphically visualize the extents of reads at given loci. SraTailor is also able to make peak calls and files of other formats, and the software also accepts various command-line-like options. Therefore, this software makes SRAs fully exploitable by a wide range of biologists. SraTailor is freely available at http://www.dev.med.kyushu-u.ac.jp/sra_tailor/.',\n",
       "   'preprint_author_corresponding': 'Chikara  Meno',\n",
       "   'preprint_author_corresponding_institution': 'Kyushu university'},\n",
       "  {'preprint_doi': '10.1101/013920',\n",
       "   'published_doi': '10.1093/bioinformatics/btv222',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'MultiMeta: an R package for meta-analysing multi-phenotype genome-wide association studies',\n",
       "   'preprint_authors': 'Dragana Vuckovic;Paolo Gasparini;Nicole Soranzo;Valentina Iotchkova;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-01-16',\n",
       "   'published_date': '2015-08-15',\n",
       "   'preprint_abstract': 'SummaryAs new methods for multivariate analysis of Genome Wide Association Studies (GWAS) become available, it is important to be able to combine results from different cohorts in a meta-analysis. The R package MultiMeta provides an implementation of the inverse-variance based method for meta-analysis, generalized to an n-dimensional setting.\\\\n\\\\nAvailabilityThe R package MultiMeta can be downloaded from CRAN Contact: dragana.vuckovic@burlo.trieste.it',\n",
       "   'preprint_author_corresponding': 'Dragana  Vuckovic',\n",
       "   'preprint_author_corresponding_institution': 'University of Trieste'},\n",
       "  {'preprint_doi': '10.1101/016345',\n",
       "   'published_doi': '10.1186/s12864-015-1860-2',\n",
       "   'published_journal': 'BMC Genomics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'ISMapper: Identifying insertion sequences in bacterial genomes from short read sequence data',\n",
       "   'preprint_authors': 'Jane Hawkey;Mohammad Hamidian;Ryan R Wick;David J Edwards;Helen Billman-Jacobe;Ruth M Hall;Kathryn E Holt;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-03-10',\n",
       "   'published_date': '2015-09-03',\n",
       "   'preprint_abstract': 'BackgroundInsertion sequences (IS) are small transposable elements, commonly found in bacterial genomes. Identifying the location of IS in bacterial genomes can be useful for a variety of purposes including epidemiological tracking and predicting antibiotic resistance. However IS are commonly present in multiple copies in a single genome, which complicates genome assembly and the identification of IS insertion sites. Here we present ISMapper, a mapping-based tool for identification of the site and orientation of IS insertions in bacterial genomes, direct from paired-end short read data.\\\\n\\\\nResultsISMapper was validated using three types of short read data: (i) simulated reads from a variety of species, (ii) Illumina reads from 5 isolates for which finished genome sequences were available for comparison, and (iii) Illumina reads from 7 Acinetobacter baumannii isolates for which predicted IS locations were tested using PCR. A total of 20 genomes, including 13 species and 32 distinct IS, were used for validation. ISMapper correctly identified 96% of known IS insertions in the analysis of simulated reads, and 98% in real Illumina reads. Subsampling of real Illumina reads to lower depths indicated ISMapper was reliable for average genome-wide read depths >20x. All ISAba1 insertions identified by ISMapper in the A. baumannii genomes were confirmed by PCR. In each A. baumannii genome, ISMapper successfully identified an IS insertion upstream of the ampC beta-lactamase that could explain phenotypic resistance to third-generation cephalosporins. The utility of ISMapper was further demonstrated by profiling genome-wide IS6110 insertions in 138 publicly available Mycobacterium tuberculosis genomes, revealing lineage-speific inserction and multi inserction hotspot.\\\\n\\\\nConclusionsISMapper provides a rapid and robust method for identifying IS insertion sites direct from short read data, with a high degree of accuracy demonstrated across a wide range of bacteria.',\n",
       "   'preprint_author_corresponding': 'Jane  Hawkey',\n",
       "   'preprint_author_corresponding_institution': 'The University of Melbourne'},\n",
       "  {'preprint_doi': '10.1101/019372',\n",
       "   'published_doi': '10.1186/s12864-016-2366-2',\n",
       "   'published_journal': 'BMC Genomics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'svclassify: a method to establish benchmark structural variant calls',\n",
       "   'preprint_authors': 'Hemang Parikh;Marghoob Mohiyuddin;Hugo Y. K. Lam;Hariharan Iyer;Desu Chen;Mark Pratt;Gabor Bartha;Noah Spies;Wolfgang Losert;Justin M. Zook;Marc L. Salit;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-05-16',\n",
       "   'published_date': '2016-01-16',\n",
       "   'preprint_abstract': 'Background: The human genome contains variants ranging in size from small single nucleotide polymorphisms (SNPs) to large structural variants (SVs). High-quality benchmark small variant calls for the pilot National Institute of Standards and Technology (NIST) Reference Material (NA12878) have been developed by the Genome in a Bottle Consortium, but no similar high-quality benchmark SV calls exist for this genome. Since SV callers output highly discordant results, we developed methods to combine multiple forms of evidence from multiple sequencing technologies to classify candidate SVs into likely true or false positives. Our method (svclassify) calculates annotations from one or more aligned bam files from many high-throughput sequencing technologies, and then builds a one-class model using these annotations to classify candidate SVs as likely true or false positives.\\\\n\\\\nResults: We first used pedigree analysis to develop a set of high-confidence breakpoint-resolved large deletions. We then used svclassify to cluster and classify these deletions as well as a set of high-confidence deletions from the 1000 Genomes Project and a set of breakpoint-resolved complex insertions from Spiral Genetics. We find that likely SVs cluster separately from likely non-SVs based on our annotations, and that the SVs cluster into different types of deletions. We then developed a supervised one-class classification method that uses a training set of random non-SV regions to determine whether candidate SVs have abnormal annotations different from most of the genome. To test this classification method, we use our pedigree-based breakpoint-resolved SVs, SVs validated by the 1000 Genomes Project, and assembly-based breakpoint-resolved insertions, along with semi-automated visualization using svviz.\\\\n\\\\nConclusions: We find that candidate SVs with high scores from multiple technologies have high concordance with PCR validation and an orthogonal consensus method MetaSV (99.7% concordant), and candidate SVs with low scores are questionable. We distribute a set of 2676 high-confidence deletions and 68 high-confidence insertions with high svclassify scores from these call sets for benchmarking SV callers. We expect these methods to be particularly useful for establishing high-confidence SV calls for benchmark samples that have been characterized by multiple technologies.',\n",
       "   'preprint_author_corresponding': 'Justin M. Zook',\n",
       "   'preprint_author_corresponding_institution': 'National Institute of Standards and Technology'},\n",
       "  {'preprint_doi': '10.1101/021659',\n",
       "   'published_doi': '10.1021/acs.jctc.5b00784',\n",
       "   'published_journal': 'Journal Of Chemical Theory And Computation',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'A simple method for automated equilibration detection in molecular simulations',\n",
       "   'preprint_authors': 'John D Chodera;',\n",
       "   'preprint_category': 'Biophysics ',\n",
       "   'preprint_date': '2015-06-29',\n",
       "   'published_date': '2016-01-15',\n",
       "   'preprint_abstract': 'Molecular simulations intended to compute equilibrium properties are often initiated from configurations that are highly atypical of equilibrium samples, a practice which can generate a distinct initial transient in mechanical observables computed from the simulation trajectory. Traditional practice in simulation data analysis recommends this initial portion be discarded to equilibration, but no simple, general, and automated procedure for this process exists. Here, we suggest a conceptually simple automated procedure that does not make strict assumptions about the distribution of the observable of interest, in which the equilibration time is chosen to maximize the number of effectively uncorrelated samples in the production timespan used to compute equilibrium averages. We present a simple Python reference implementation of this procedure, and demonstrate its utility on typical molecular simulation data.',\n",
       "   'preprint_author_corresponding': 'John D Chodera',\n",
       "   'preprint_author_corresponding_institution': 'Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center'},\n",
       "  {'preprint_doi': '10.1101/032987',\n",
       "   'published_doi': '10.1016/j.jmb.2015.09.002',\n",
       "   'published_journal': 'Journal of Molecular Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'The limitations of in vitro experimentation in understanding biofilms and chronic infection',\n",
       "   'preprint_authors': 'Aled E L Roberts;Kasper N Kragh;Thomas Bjarnsholt;Stephen P Diggle;',\n",
       "   'preprint_category': 'Microbiology ',\n",
       "   'preprint_date': '2015-11-26',\n",
       "   'published_date': '2015-11-20',\n",
       "   'preprint_abstract': 'We have become increasingly aware that during infection, pathogenic bacteria often grow in multicellular biofilms which are often highly resistant to antibacterial strategies. In order to understand how biofilms form and contribute to infection, in vitro biofilm systems such as microtitre plate assays and flow cells, have been heavily used by many research groups around the world. Whilst these methods have greatly increased our understanding of the biology of biofilms, it is becoming increasingly apparent that many of our in vitro methods do not accurately represent in vivo conditions. Here we present a systematic review of the most widely used in vitro biofilm systems, and we discuss why they are not always representative of the in vivo biofilms found in chronic infections. We present examples of methods that will help us to bridge the gap between in vitro and in vivo biofilm work, so that our bench-side data can ultimately be used to improve bedside treatment.',\n",
       "   'preprint_author_corresponding': 'Stephen P Diggle',\n",
       "   'preprint_author_corresponding_institution': 'University of Nottingham'},\n",
       "  {'preprint_doi': '10.1101/033597',\n",
       "   'published_doi': '10.1016/j.tpb.2015.12.002',\n",
       "   'published_journal': 'Theoretical Population Biology',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Evolutionary dynamics of a quantitative trait in a finite asexual population.',\n",
       "   'preprint_authors': 'Florence Debarre;Sarah Otto;',\n",
       "   'preprint_category': 'Evolutionary Biology ',\n",
       "   'preprint_date': '2015-12-03',\n",
       "   'published_date': '2016-01-06',\n",
       "   'preprint_abstract': 'In finite populations, mutation limitation and genetic drift can hinder evolutionary diversification. We consider the evolution of a quantitative trait in an asexual population whose size can vary and depends explicitly on the trait. Previous work showed that evolutionary branching is certain (\\\\\"deterministic branching\\\\\") above a threshold population size, but uncertain (\\\\\"stochastic branching\\\\\") below it. Using the stationary distribution of the populations trait variance, we identify three qualitatively different sub-domains of \\\\\"stochastic branching\\\\\" and illustrate our results using a model of social evolution. We find that in very small populations, branching will almost never be observed; in intermediate populations, branching is intermittent, arising and disappearing over time; in larger populations, finally, branching is expected to occur and persist for substantial periods of time. Our study provides a clearer picture of the ecological conditions that facilitate the appearance and persistence of novel evolutionary lineages in the face of genetic drift.',\n",
       "   'preprint_author_corresponding': 'Florence  Debarre',\n",
       "   'preprint_author_corresponding_institution': 'CNRS'},\n",
       "  {'preprint_doi': '10.1101/017830',\n",
       "   'published_doi': '10.1038/srep19233',\n",
       "   'published_journal': 'Scientific Reports',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'An evaluation of the accuracy and speed of metagenome analysis tools',\n",
       "   'preprint_authors': 'Stinus Lindgreen;Karen L Adair;Paul Gardner;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-04-09',\n",
       "   'published_date': '2016-01-18',\n",
       "   'preprint_abstract': 'Metagenome studies are becoming increasingly widespread, yielding important insights into microbial communities covering diverse environments from terrestrial and aquatic ecosystems to human skin and gut. With the advent of high-throughput sequencing platforms, the use of large scale shotgun sequencing approaches is now commonplace. However, a thorough independent benchmark comparing state-of-the-art metagenome analysis tools is lacking. Here we present a benchmark where the most widely used tools are tested on complex, realistic data sets. Our results clearly show that the most widely used tools are not necessarily the most accurate, that the most accurate tool is not necessarily the most time consuming, and that there is a high degree of variability between available tools. These findings are important as the conclusions of any metagenomics study are affected by errors in the predicted community composition and functional capacity. Data sets and results are freely available from http://www.ucbioinformatics.org/metabenchmark.html',\n",
       "   'preprint_author_corresponding': 'Stinus  Lindgreen',\n",
       "   'preprint_author_corresponding_institution': 'University of Copenhagen'},\n",
       "  {'preprint_doi': '10.1101/026682',\n",
       "   'published_doi': '10.1371/journal.pone.0146500',\n",
       "   'published_journal': 'PLOS ONE',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Model-free Estimation of Tuning Curves and Their Attentional Modulation, Based on Sparse and Noisy Data',\n",
       "   'preprint_authors': 'Markus Helmer;Vladislav Kozyrev;Valeska Stephan;Stefan Treue;Theo Geisel;Demian Battaglia;',\n",
       "   'preprint_category': 'Neuroscience ',\n",
       "   'preprint_date': '2015-09-12',\n",
       "   'published_date': '2016-01-19',\n",
       "   'preprint_abstract': 'Tuning curves are the functions that relate the responses of sensory neurons to various values within one continuous stimulus dimension (such as the orientation of a bar in the visual domain or the frequency of a tone in the auditory domain). They are commonly determined by fitting a model e.g. a Gaussian or other bell-shaped curves to the measured responses to a small subset of discrete stimuli in the relevant dimension. However, as neuronal responses are irregular and experimental measurements noisy, it is often difficult to determine reliably the appropriate model from the data. We illustrate this general problem by fitting diverse models to representative recordings from area MT in rhesus monkey visual cortex during multiple attentional tasks involving complex composite stimuli. We find that all models can be well-fitted, that the best model generally varies between neurons and that statistical comparisons between neuronal responses across different experimental conditions are affected quantitatively and qualitatively by specific model choices. As a robust alternative to an often arbitrary model selection, we introduce a model-free approach, in which features of interest are extracted directly from the measured response data without the need of fitting any model. In our attentional datasets, we demonstrate that data-driven methods provide descriptions of tuning curve features such as preferred stimulus direction or attentional gain modulations which are in agreement with fit-based approaches when a good fit exists. Furthermore, these methods naturally extend to the frequent cases of uncertain model selection. We show that model-free approaches can identify attentional modulation patterns, such as general alterations of the irregular shape of tuning curves, which cannot be captured by fitting stereotyped conventional models. Finally, by comparing datasets across different conditions, we demonstrate effects of attention that are cell-and even stimulus-specific. Based on these proofs-of-concept, we conclude that our data-driven methods can reliably extract relevant tuning information from neuronal recordings, including cells whose seemingly haphazard response curves defy conventional fitting approaches.',\n",
       "   'preprint_author_corresponding': 'Markus  Helmer',\n",
       "   'preprint_author_corresponding_institution': 'Max Planck Institute for Dynamics and Self-organization'},\n",
       "  {'preprint_doi': '10.1101/026948',\n",
       "   'published_doi': '10.1038/nmeth.3734',\n",
       "   'published_journal': 'Nature Methods',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis',\n",
       "   'preprint_authors': 'Jean Fan;Neeraj Salathia;Rui Liu;Gwen Kaeser;Yun Yung;Joseph L Herman;Fiona Kaper;Jian-Bing Fan;Kun Zhang;Jerold Chun;Peter Kharchenko;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-09-16',\n",
       "   'published_date': '2016-01-18',\n",
       "   'preprint_abstract': 'Single-cell transcriptome measurements are being applied at rapidly increasing scales to study cellular repertoires underpinning functions of complex tissues and organs, including mammalian brains. The transcriptional state of each cell, however, reflects a variety of biological factors, including persistent cell-type specific regulatory configurations, transient processes such as cell cycle, local metabolic demands, and extracellular signals. Depending on the biological setting, all such aspects of transcriptional heterogeneity can be of potential interest, but detecting complex heterogeneity structure from inherently uncertain single-cell data presents analytical challenges. We developed PAGODA to resolve multiple, potentially overlapping aspects of transcriptional heterogeneity by identifying known pathways or novel gene sets that show significant excess of coordinated variability among the measured cells. We demonstrate that PAGODA effectively recovers the subpopulations and their corresponding functional characteristics in a variety of single-cell samples, and use it to characterize transcriptional diversity of neuronal progenitors in the developing mouse cortex.',\n",
       "   'preprint_author_corresponding': 'Peter  Kharchenko',\n",
       "   'preprint_author_corresponding_institution': 'Harvard Medical School'},\n",
       "  {'preprint_doi': '10.1101/027342',\n",
       "   'published_doi': '10.1093/bioinformatics/btw018',\n",
       "   'published_journal': 'Bioinformatics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'FINEMAP: Efficient variable selection using summary data from genome-wide association studies',\n",
       "   'preprint_authors': 'Christian Benner;Chris C.A. Spencer;Samuli Ripatti;Matti Pirinen;',\n",
       "   'preprint_category': 'Genetics ',\n",
       "   'preprint_date': '2015-09-22',\n",
       "   'published_date': '2016-01-14',\n",
       "   'preprint_abstract': 'MotivationThe goal of fine-mapping in genomic regions associated with complex diseases and traits is to identify causal variants that point to molecular mechanisms behind the associations. Recent fine-mapping methods using summary data from genome-wide association studies rely on exhaustive search through all possible causal configurations, which is computationally expensive.\\\\n\\\\nResultsWe introduce FINEMAP, a software package to efficiently explore a set of the most important causal configurations of the region via a shotgun stochastic search algorithm. We show that FINEMAP produces accurate results in a fraction of processing time of existing approaches and is therefore a promising tool for analyzing growing amounts of data produced in genome-wide association studies.\\\\n\\\\nAvailabilityFINEMAP v1.0 is freely available for Mac OS X and Linux at http://www.christianbenner.com.\\\\n\\\\nContact: christian.benner@helsinki.fi, matti.pirinen@helsinki.fi',\n",
       "   'preprint_author_corresponding': 'Christian  Benner',\n",
       "   'preprint_author_corresponding_institution': 'Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland'},\n",
       "  {'preprint_doi': '10.1101/028449',\n",
       "   'published_doi': '10.1142/9789814749411_0013',\n",
       "   'published_journal': 'Pacific Symposium on Biocomputing',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Investigating the importance of anatomical homology for cross-species phenotype comparisons using semantic similarity.',\n",
       "   'preprint_authors': 'Prashanti Manda;Christopher Mungall;James Balhoff;Hilmar Lapp;Todd Vision;',\n",
       "   'preprint_category': 'Bioinformatics ',\n",
       "   'preprint_date': '2015-10-05',\n",
       "   'published_date': '2016-01-10',\n",
       "   'preprint_abstract': 'There is growing use of ontologies for the measurement of cross-species phenotype similarity. Such similarity measurements contribute to diverse applications, such as identifying genetic models for human diseases, transferring knowledge among model organisms, and studying the genetic basis of evolutionary innovations. Two organismal features, whether genes, anatomical parts, or any other inherited feature, are considered to be homologous when they are evolutionarily derived from a single feature in a common ancestor. A classic example is the homology between the paired fins of fishes and vertebrate limbs. Anatomical ontologies that model the structural relations among parts may fail to include some known anatomical homologies unless they are deliberately added as separate axioms. The consequences of neglecting known homologies for applications that rely on such ontologies has not been well studied. Here, we examine how semantic similarity is affected when external homology knowledge is included. We measure phenotypic similarity between orthologous and non-orthologous gene pairs between humans and either mouse or zebrafish, and compare the inclusion of real with faux homology axioms. Semantic similarity was preferentially increased for orthologs when using real homology axioms, but only in the more divergent of the two species comparisons (human to zebrafish, not human to mouse), and the relative increase was less than 1% to non-orthologs. By contrast, inclusion of both real and faux random homology axioms preferentially increased similarities between genes that were initially more dissimilar in the other comparisons. Biologically meaningful increases in semantic similarity were seen for a select subset of gene pairs. Overall, the effect of including homology axioms on cross-species semantic similarity was modest at the levels of divergence examined here, but our results hint that it may be greater for more distant species comparisons.',\n",
       "   'preprint_author_corresponding': 'Prashanti  Manda',\n",
       "   'preprint_author_corresponding_institution': 'University of North Carolina, Chapel Hill'},\n",
       "  {'preprint_doi': '10.1101/029199',\n",
       "   'published_doi': '10.1534/g3.115.023655',\n",
       "   'published_journal': 'G3: Genes|Genomes|Genetics',\n",
       "   'preprint_platform': 'bioRxiv',\n",
       "   'preprint_title': 'Major improvements to the Heliconius melpomene genome assembly used to confirm 10 chromosome fusion events in 6 million years of butterfly evolution',\n",
       "   'preprint_authors': 'John Davey;Mathieu Chouteau;Sarah L. Barker;Luana Maroja;Simon W. Baxter;Fraser Simpson;Mathieu Joron;James Mallet;Kanchon K. Dasmahapatra;Chris D. Jiggins;',\n",
       "   'preprint_category': 'Genomics ',\n",
       "   'preprint_date': '2015-10-15',\n",
       "   'published_date': '2016-01-15',\n",
       "   'preprint_abstract': 'The Heliconius butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridise in the wild. Here, we present a substantially improved assembly of the Heliconius melpomene genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. Firstly, we whole genome sequenced a pedigree to produce a linkage map incorporating 99% of the genome. Secondly, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Thirdly, we incorporated ~20x coverage of Pacific Biosciences sequencing and scaffolded the haploid genome using an assembly of this long read sequence. These improvements result in a genome of 795 scaffolds, 275 Mb in length, with an L50 of 2.1 Mb, an N50 of 34 and with 99% of the genome placed and 84% anchored on chromosomes. We use the new genome assembly to confirm that the Heliconius genome underwent 10 chromosome fusions since the split with its sister genus Eueides, over a period of about 6 million years.',\n",
       "   'preprint_author_corresponding': 'John  Davey',\n",
       "   'preprint_author_corresponding_institution': 'University of Cambridge'}]}"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
